# Conclusion and outlook {#conclusion .unnumbered}

## Overall conclusion{.unnumbered}

In this Doctoral thesis, first, the work is presented on the development of pipelines for multi-omics high-throughput sequencing data, [Methods](#methods), and data analysis pipelines have been used to analyze muli-omics data from the lab and literature, [Appendix A](#aa). Next as summarized in [Chapter 1](#chapter1), we compare the open chromatin landscape of developing and adult spermatogonial cells to explore the development of spermatogonial cells in postnatal development. We discovered and characterized genomic regions with differential genomic activity in spermatogonial cells by combining the data generated by the host lab with the data available from the literature. After that, as summarized in [Chapter 2](#chapter2), we studied the impact of early postnatal stress on the transcriptome of epididymal extracellular vesicles. Our findings show that chronic stress in early postnatal life affects miRNAs in adult male reproductive tract extracellular vesicles, with consequences for mature sperm and zygotes. Finally, in [Chapter 3](#chapter3) we created an `R` package, `shortRNA`, for analyzing short RNA sequencing data that is platform-independent, provides a good overview of the quality, performs quality control, aligns the data to a custom-made genome, performs read annotation and assignment with adaptable assignment rules, organizes reads and features as a phylogenetic tree, performs differential analysis, and provides interactive plots for exploratory data analysis. However, there are a number of changes or further investigations that might be needed, which we will explore in this section.


## MSUS-associated differences {.unnumbered}

The MSUS model is appropriate for examining epigenetic inheritance owing to environmental perturbations, but low effect sizes make it difficult to discover candidates that passed multiple testing corrections. The costs associated with sample creation and sequencing, makes it challenging to have a higher number of replicates in the experimental designs. After multiple testing correction, obtaining significant p-values requires either appropriate data distribution assumptions or a high number of replicates. Clipper, a new statistical framework for reducing false discovery rates without relying on p-values or specific data distributions, was published recently (Ge et al. 2021). `Clipper` has been shown to outperform conventional methods for limiting false discoveries, thus pipelines can be updated to implement it.


## RNA sequencing data analysis pipeline {.unnumbered}

Data analysis pipelines for RNA sequencing could be adapted. `Salmon`, a tool for obtaining quantifications from RNA-seq data, is used for pseudo-alignment while accounting for GC biases at the fragment level, sequence-specific biases, and position-specific fragment starts distribution. `Salmon Anomaly Detection` (`SAD`) tool can be incorporated into the RNA sequencing data analysis pipeline in addition to `Salmon` quantifications. `SAD` identifies probable misquantifications in `Salmon`'s RNA-seq transcript expression estimation. However, due to several external dependencies, implementing this tool may be difficult (https://github.com/Kingsford-Group/sad/issues), and this will almost certainly necessitate containerization from the authors, either via `conda` or `Docker`. Also, while evaluating and quantifying lowly-expressed genes and small RNAs with alignment-free pipelines, such as `Salmon`, potential pitfalls have been recognized, particularly when the small RNAs contain biological variability. As a result, when the analysis isn't limited to simply protein-coding genes, an alignment-based technique like `Rsubread` or `STAR` can be used (Wu et al. 2018). Further, for differential transcript usage and isoform switching, we employed `IsoformSwitchAnalyzeR`. Under the hood, `IsoformSwitchAnalyzeR` employs `DEXseq` for differential transcript usage. However, `DEXseq` has the drawback of not being able to account for blocking variables. `BANDITS` could also be used instead of `DEXseq` in the `IsoformSwitchAnalyzeR` pipeline because it outperforms `DEXseq` (Tiberi and Robinson 2020).


## ATAC sequencing data analysis pipeline {.unnumbered}

In recent years, ATAC-seq has grown in popularity as a tool for investigating chromatin accessibility. Despite improvements in protocols, bioinformatics analysis tools are still in their development, with no comprehensive analytical pipeline defined. The interpretation of ATAC-seq results is currently hampered by this. The majority of the tools used to analyze ATAC-seq data are from the ChIP-seq or DNA-seq fields. For example, `MACS2`, the peak caller we used for ATAC-seq, was designed originally for ChIP-seq data. `HMMRATAC` is a peak calling method for ATAC-seq data that combines the nucleosome-free and nucleosome-enriched signals from a single ATAC-seq dataset to find open chromatin regions (Tarbell and Liu 2019). Our strategy was instead to use the nucleosome-free fragments (which are indicative of TF binding) for peak calling. Because we only looked into nucleosome-free fragments, `HMMRATAC` was not used. Also, `HMMRATAC`'s authors advise against using it on ATAC-seq datasets that have undergone physical or computational size selection. But, `HMMRATC` could be employed for investigating outside the nucleosome-free fragments.

The ATAC-seq pipeline can also be extended to include tools for nucleosome positioning and transcription factor footprinting analysis. However, nucleosome detection is problematic due to an inherent shortcoming of ATAC-seq data, which typically has inadequate read coverage beyond peaks (Yan et al. 2020). Based on empirical and computational estimates, the recommended minimum number of mapped reads for open chromatin detection and differential analysis in mammalian species is 50 million, and 200 million for TF footprinting (Yan et al. 2020). Due to the reduced read coverage outside the open chromatin regions, nucleosome detection in ATAC-seq data was shown to be more difficult than in MNase-seq data (Yan et al. 2020). Because we did not sequence this deep, we did not include TF footprinting in the pipeline, which would necessitate a thorough investigation of tools. These drawbacks necessitate a detailed study of the tools employed in ATAC-seq for normalization, peak calling on NFFs, and TF footprinting.


## Updating manuscript from [Chapter 1](#chapter1) in light of latest RNA-seq data {.unnumbered}

The datasets used for the study of spermatogonial cell development were not initially designed for that purpose, but to investigate the changes at different developmental stages because of the MSUS paradigm. As a result, the datasets were not ideal for this new purpose: the study originally employed polyA RNA sequencing data from the PNW21 stage, while RNA sequencing data from the stages PND8 and PND15 came from Total RNA sequencing. We, therefore, used data from the literature but determined to re-generate the data from the PNW21 stage in the lab in order to have more impactful research and accurate findings. Because of these changes, the entire downstream analysis, including functional analysis, motif analysis, data integration, and re-creation of the figures, will have to be repeated. Some of these analyses have been completed and are included in [Chapter 1](#chapter1) as an update. In general, however, given the high consistency of the results across the two analyses, we are confident that the main results will remain unchanged.


## Low correlations of quantification between `shortRNA` and qPCR {.unnumbered}

In terms of relative miRNA expression across samples, we found a low correlation between sequencing-based quantifications (with `shortRNA` as well as with other tools) and qPCR. We covered a variety of possible explanations for this mismatch in [Chapter 3](#chapter3), including normalizing the data with U6 RNA expression and the lack of UMIs. We don't have an exact explanation for the poor correlations, but we want to look into them further using a variety of techniques, including testing the shortRNA tool with UMI-based data with corresponding qPCR quantifications and creating a more extended simulation dataset.
