---
title: 'Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance'
author: 'Deepak Kumar Tanwar'
# date: 'December 2021'
institution: 'ETH Zurich'
# division: 'Mathematics and Natural Sciences'
# advisor: 'Advisor F. Name'
# If you have more two advisors, un-silence line 7
#altadvisor: 'Your Other Advisor'
# department: 'Mathematics'
# degree: 'Bachelor of Arts'
knit: bookdown::render_book
site: bookdown::bookdown_site

# fontfamily: accanthis
mainfont: Arial

# The next two lines allow you to change the spacing in your thesis. You can 
# switch out \onehalfspacing with \singlespacing or \doublespacing, if desired.
header-includes:
    - \usepackage{setspace}\onehalfspacing
    - \usepackage{acronym}
    - \usepackage{glossaries}
    - \usepackage{biblatex}
    - \usepackage{fancyhdr}
    - \usepackage{graphics}
    - \usepackage{float}
    # - \usepackage{placeins}
    - \usepackage{lmodern}
    - \usepackage{pdflscape}
    - \usepackage{caption}
    - \captionsetup{font=footnotesize}
    - \usepackage[labelfont=bf]{caption}
    - \usepackage{wrapfig}
    - \usepackage{pdfpages}
    - \usepackage{afterpage}
    - \usepackage[section]{placeins}

# This will automatically install the {remotes} package and {thesisdown}
# Change this to FALSE if you'd like to install them manually on your own.
params:
  'Install needed packages for {thesisdown}': True
  
# Remove the hashtag to specify which version of output you would like.
# Can only choose one at a time.
output:
  thesisdown::thesis_pdf:
    toc_depth: 5
    extra_dependencies: ["flafter", "float", "subfig", "subfloat"]
 # thesisdown::thesis_gitbook: default
 # thesisdown::thesis_word: default
 # thesisdown::thesis_epub: default
# # 
acknowledgements: |
  This doctoral thesis has been an accomplishment that would not have been possible, or anywhere near as enjoyable, without the help and support of a number of incredible individuals. Thank you for all that you have contributed to this thesis. \newline\newline I am eternally grateful to Prof. Dr. Isabelle Mansuy for allowing me to do Doctoral research in her lab. It is a pleasure to be a member of the Neuroepigenetics lab. Thank you to all the members of the Neuroepigenetics lab, past and present. The Neuroepigenetics group was always fun to work with and I learned a lot about experiments. Thank you so much! A special thanks to the present and past members of Y38 for motivating, and various scientific and non-scientific discussions. \newline\newline My sincere thanks to Prof. Dr. Mark Robinson and Prof. Dr. Tuncay Baubec for accepting to be my co-examiners and supporting me throughout my doctoral research. Thank you so much Mark for always being available and providing all the help in writing proposals, helping in supervising students, and discussing and providing solutions to the problems I was facing. \newline\newline I am indebted to Dr. Pierre-Luc Germain for his expert supervision and all the support he has provided. I am grateful for all discussions and suggestions on both scientific and non-scientific issues. Thank you for being so nice and for all the support which was extremely helpful to reach this point. I have learnt a lot from you. \newline\newline I would also like to thank the Directors and members of the Brain Research Institute for creating such a stimulating environment for research. \newline\newline I am extremely grateful for my doctoral funding, the Swiss Government Excellence Scholarship. Thank you so much Sandra Zweifel for always being there. \newline\newline Looking back into the past, this thesis would not have happened without my bachelor’s thesis supervisors Rainer König and Sandro Lindig, who got me interested in the field of bioinformatics data science. \newline\newline Finally, I am very grateful to my family for their continuous support and belief in me. Visiting you always helped me to relax and to not worry about work so much. Last of all, I want to thank my wife, Palni Kundra, for always cheering me up when I was down and for helping me with stupid issues.

#   # https://tex.stackexchange.com/questions/25798/how-can-i-add-abstract-and-acknowledgement-pages-into-the-table-of-contents/25799
#   
# dedication: |
#   To Palni
  
# Specify the location of the bibliography below
bibliography: [bib/anaraEV.bib, rmd/references.bib,  bib/thesis.bib]
link-citations: true
biblatexoptions: [refsegment=chapter]
# Download your specific csl file and refer to it in the line below.
csl: csl/apa.csl
lot: true
lof: true
---

```{=html}
<!--
Above is the YAML (YAML Ain't Markup Language) header that includes a lot of 
metadata used to produce the document.  Be careful with spacing in this header!

If you'd prefer to not include a Dedication, for example, simply delete the section entirely, or silence them (add # before each line). 

If you have other LaTeX packages you would like to include, delete the # before header-includes and list the packages after hyphens on new lines.

If you'd like to include a comment that won't be produced in your resulting file enclose it in a block like this.

If you receive a duplicate label error after knitting, make sure to delete the index.Rmd file and then knit again.
-->
```
```{r include_packages, include=FALSE}
# This chunk ensures that the thesisdown package is
# installed and loaded. This thesisdown package includes
# the template files for the thesis.
if (!require(remotes)) {
  if (params$`Install needed packages for {thesisdown}`) {
    install.packages("remotes", repos = "https://cran.rstudio.com")
  } else {
    stop(
      paste('You need to run install.packages("remotes")",
            "first in the Console.')
    )
  }
}
if (!require(thesisdown)) {
  if (params$`Install needed packages for {thesisdown}`) {
    remotes::install_github("ismayc/thesisdown")
  } else {
    stop(
      paste(
        "You need to run",
        'remotes::install_github("ismayc/thesisdown")',
        "first in the Console."
      )
    )
  }
}
library(thesisdown)
library(png)
library(grid)
library(jpeg)
library(dplyr)
library(kableExtra)
# Set how wide the R output will go
# options(width = 70)
knitr::opts_chunk$set(fig.pos = "htbp")
# defOut <- knitr::knit_hooks$get("plot")  # save the default plot hook 
# knitr::knit_hooks$set(plot = function(x, options) {  # set new plot hook ...
#   x <- defOut(x, options)  # first apply the default hook
#   if(!is.null(options$wrapfigure)) {  # then, if option wrapfigure is given ...
#     # create the new opening string for the wrapfigure environment ...
#     wf <- sprintf("\\begin{wrapfigure}{%s}{%g\\textwidth}", options$wrapfigure[[1]], options$wrapfigure[[2]])
#     x  <- gsub("\\begin{figure}", wf, x, fixed = T)  # and replace the default one with it.
#     x  <- gsub("{figure}", "{wrapfigure}", x, fixed = T)  # also replace the environment ending
#   }
#   return(x)
# })
```

```{=html}
<!--
The acknowledgments, preface, dedication, and abstract are added into the PDF
version automatically by inputting them in the YAML at the top of this file.
Alternatively, you can put that content in files like 00--prelim.Rmd and
00-abstract.Rmd like done below.
-->
```
<!-- ```{r eval=knitr::is_latex_output(), child=here::here("prelims", "00--prelim.Rmd")} -->

<!-- ``` -->

<!-- ```{r eval=!knitr::is_latex_output(), child=here::here("prelims", "00-abstract.Rmd")} -->

<!-- ``` -->



<!--chapter:end:index.Rmd-->

# List of Abbreviations {.unnumbered}

```{r, echo=FALSE, message=FALSE, warning=FALSE}
library(readxl)
library(dplyr)
library(knitr)
df <- read_xlsx("figure/abbr.xlsx")
colnames(df) <- c("Abbreviation", "Term")
df %>%  arrange(Term) %>% # i.e. alphabetical order by Term
  kable(booktab = T) # booktab = T gives us a pretty APA-ish table
```

<!--chapter:end:rmd/00-abbr.Rmd-->

# Summary {- #summary}

The environment has a strong influence on an organism's development, especially in the early stages. Environmental influences and life experiences can alter the phenotypic features of exposed individuals and their offspring in many animals. In mammals, traumatic stress is a type of environmental event that can change their behaviour, cognition, and physiological functions. Some of the impacts of severe stress can be passed down through generations, even if those generations have never been exposed to a traumatic stressor. Environmentally generated effects that involve non-genetic germline alterations can be an important route of trait transmission over generations. Even if the subsequent generations have never been exposed to similar stressful exposures, they may be impaired in similar functions. The host lab's unpredictable maternal separation combined with unpredictable maternal stress (MSUS) mouse model is perfect for learning more about the mechanisms behind epigenetic inheritance induced by environmental perturbations. 

In this thesis, to study the mechanisms of epigenetic inheritance, multi-omics data analysis pipelines were built, [Methods](#methods), and a number of high-throughput sequencing datasets generated using the MSUS model were analyzed using these pipelines, [Appendix A](#aa) and [Appendix B](#ab). Also, the datasets generated from spermatogonial cells were used to study the developmental trajectory across postnatal and adult stages. Further, the effect of early life trauma on epididymal extracellular vesicles transcriptome was investigated using short RNA sequencing data. At last, a tool was built to analyze short RNA sequencing data to address issues in the current tools and pipelines.

Spermatogonial cells are the postnatal initiators of spermatogenesis, and they undertake critical proliferative and differentiation tasks throughout one's life. They are the only type of stem cell capable of transmitting genetic information to an embryo. Open chromatin in spermatogonial cells undergoes significant remodelling throughout testes development. Throughout postnatal testes development, they have significantly different transcriptional pathways. We investigated the dynamics of chromatin accessibility and gene expression during the developmental stages of postnatal day 8 (PND8), PND15, and postnatal week 21 (PNW21) using multi-omics data collected from spermatogonial cells. By comparing chromatin accessibility changes between early postnatal and steady-state adult cells in the mouse testes, the age-dependent molecular phenotype of spermatogonial cells was identified. The differentially accessible regions were discovered to be relevant to developmental and metabolic processes and enriched for important transcription factors. Also, the accessibility of chromatin at transposable elements in spermatogonia has been examined. Specific transposable elements subtypes become less accessible as the adult testis matures, whereas specific LINE L1 subtypes become more accessible. The genome's olfactory receptor gene regions were discovered to be highly connected to several of the more accessible subtypes. In conclusion, these findings reveal that chromatin accessibility in spermatogonial cells changes from postnatal to adult stage, as well as interesting biological patterns when combined with expression and histone modifications data. Furthermore, we show how bioinformatics approaches may be used to integrate multi-omics datasets. These results are available in [Chapter 1](#chapter1).

If environmentally-triggered marks are not inherited from SSCs, another possible way in which sperm may carry them is through the uptake of extracellular vesicles, in particular those produced by epididymal cells. By using the MSUS model, we studied the impact of early postnatal stress on the transcriptome of epididymal extracellular vesicles. The short RNA profile of epididymal EVs, particularly miRNAs, were found to be altered in adult males exposed to postnatal stress. In certain cases, these miRNA changes have been related to differences in the expression of their target genes in sperm and zygotes produced from those sperm. To summarize, chronic stress in early postnatal life changes miRNAs in mature extracellular vesicles of the male reproductive tract, with implications for mature sperm and zygotes. These results are summarized in [Chapter 2](#chapter2). 

While working on [Chapter 1](#chapter1) and [Chapter 2](#chapter2), we discovered a lot of issues with tools available for analyzing short RNA sequencing data. Short RNAs are significant molecules that play an important role in the control of the genome. Short RNAs have been classified into several categories, including miRNA, tRNA and tRNA fragments, and piRNAs, all of which have a complex biosynthesis. As a result, an analysis framework is necessary that is both sufficiently specialized to reflect the specificities of different classes of short RNA and their biogenesis, as well as sufficiently generic and thorough to conduct global analyses. We created a user-friendly, highly customizable, and comprehensive `R` package, `shortRNA` to do a full end-to-end analysis of short RNA sequencing data. Our package has been specifically designed to account for changes in short RNA types and their biogenesis, and it may readily be modified to include further annotations. We also use a customised genome annotation with pseudo chromosomes to account for post-transcriptional modifications, as well as a flexible rule-based approach to allocate reads along a tree of hierarchically ordered attributes. This enables systematic querying, exploration, and differential expression analysis of short RNAs at several levels of granularity, ranging from single sequences to entire RNA classes. The shortRNA R package, which is platform-agnostic, was designed to carry out all of the analysis steps from within R. These results are available in [Chapter 3](#chapter3).

In summary, the work presented in this thesis investigated various mechanisms of epigenetic inheritance and developed several pipelines for data analysis from the MSUS model, investigated changes in spermatogonial cells across development, studied the impact of early life stress on the transcriptome of epididymal extracellular vesicles, and developed an `R` package for short RNA sequencing data analysis.

<!--chapter:end:rmd/01-summary.Rmd-->

# Zusammenfassung {- #summary-de}

Die Umwelt hat einen starken Einfluss auf die Entwicklung eines Organismus, insbesondere in den frühen Stadien. Umwelteinflüsse und Lebenserfahrungen können bei vielen Tieren die phänotypischen Merkmale exponierter Individuen und ihrer Nachkommen verändern. Bei Säugetieren ist traumatischer Stress eine Art Umweltereignis, das ihr Verhalten, ihre Wahrnehmung und ihre physiologischen Funktionen verändern kann. Einige der Auswirkungen von starkem Stress können über Generationen weitergegeben werden, auch wenn diese Generationen noch nie einem traumatischen Stressor ausgesetzt waren. Umweltbedingte Effekte, die nicht-genetische Veränderungen der Keimbahn beinhalten, können ein wichtiger Übertragungsweg von Merkmalen über Generationen hinweg sein. Auch wenn die nachfolgenden Generationen noch nie ähnlichen Belastungen ausgesetzt waren, können sie in ähnlichen Funktionen beeinträchtigt sein. Die unvorhersehbare mütterliche Trennung des Wirtslabors in Kombination mit dem Mausmodell für unvorhersehbaren mütterlichen Stress (MSUS) ist perfekt, um mehr über die Mechanismen hinter der epigenetischen Vererbung durch Umweltstörungen zu erfahren.

Um die Mechanismen der epigenetischen Vererbung zu untersuchen, wurden in dieser Dissertation Multi-Omics-Datenanalysepipelines erstellt, [Methoden](#methods) und eine Reihe von Hochdurchsatz-Sequenzierungsdatensätzen, die mit dem MSUS-Modell generiert wurden, unter Verwendung dieser Pipelines analysiert, [Anhang A](#aa) und [Anhang B](#ab). Auch , wurden die aus Spermatogonien generierten Datensätze verwendet, um den Entwicklungsverlauf über postnatale und adulte Stadien hinweg zu untersuchen. Darüber hinaus wurde die Wirkung eines frühen Lebenstraumas auf das Transkriptom der extrazellulären Vesikel der Nebenhoden unter Verwendung von kurzen RNA-Sequenzierungsdaten untersucht. Schließlich wurde ein Tool entwickelt, um kurze RNA-Sequenzierungsdaten zu analysieren, um Probleme in den aktuellen Tools und Pipelines zu beheben.

Spermatogoniale Zellen sind die postnatalen Initiatoren der Spermatogenese und übernehmen lebenslang kritische Proliferations- und Differenzierungsaufgaben. Sie sind die einzige Art von Stammzellen, die genetische Informationen an einen Embryo übertragen können. Offenes Chromatin in Spermatogonialzellen wird während der Hodenentwicklung signifikant umgebaut. Während der postnatalen Hodenentwicklung weisen sie signifikant unterschiedliche Transkriptionswege auf. Wir untersuchten die Dynamik der Zugänglichkeit von Chromatin und der Genexpression während der Entwicklungsstadien des 8. postnatalen Tages (PND8), PND15 und postnatalen Woche 21 (PNW21) unter Verwendung von Multi-Omics-Daten aus Spermatogonien. Durch den VergleichChromatins der Veränderungen der Zugänglichkeit deszwischen frühen postnatalen und stationären adulten Zellen in den Hoden von Mäusen wurde der altersabhängige molekulare Phänotyp von Spermatogonien identifiziert. Die unterschiedlich zugänglichen Regionen wurden als relevant für Entwicklungs- und Stoffwechselprozesse entdeckt und um wichtige Transkriptionsfaktoren angereichert. Auch die Zugänglichkeit von Chromatin an transponierbaren Elementen in Spermatogonien wurde untersucht. Spezifische Subtypen der transponierbaren Elemente werden mit zunehmender Reife des adulten Hodens weniger zugänglich, während spezifische LINE L1-Subtypen leichter zugänglich werden. Es wurde entdeckt, dass die Genregionen des olfaktorischen Rezeptors des Genoms stark mit mehreren der leichter zugänglichen Subtypen verbunden sind. Zusammenfassend zeigen diese Ergebnisse, dass sich die Zugänglichkeit von Chromatin in Spermatogonien vom postnatalen bis zum adulten Stadium ändert, sowie interessante biologische Muster, wenn sie mit Expressions- und Histonmodifikationsdaten kombiniert werden. Darüber hinaus zeigen wir, wie bioinformatische Ansätze verwendet werden können, um Multi-Omics-Datensätze zu integrieren. Diese Ergebnisse sind in [Kapitel 1](#chapter1) verfügbar. 

Wenn umweltbedingte Markierungen nicht von SSCs vererbt werden, können Spermien sie auch durch die Aufnahme extrazellulärer Vesikel, insbesondere solcher, die von Nebenhodenzellen produziert werden, übertragen. Mit dem MSUS-Modell untersuchten wir den Einfluss von frühem postnatalem Stress auf das Transkriptom extrazellulärer Nebenhodenvesikel. Es wurde festgestellt, dass das kurze RNA-Profil von Nebenhoden-EVs, insbesondere miRNAs, bei erwachsenen Männern, die postnatalem Stress ausgesetzt waren, verändert war. In bestimmten Fällen wurden diese miRNA-Veränderungen mit Unterschieden in der Expression ihrer Zielgene in Spermien und Zygoten, die aus diesen Spermien produziert wurden, in Verbindung gebracht. Zusammenfassend lässt sich sagen, dass chronischer Stress im frühen postnatalen Leben miRNAs in reifen extrazellulären Vesikel des männlichen Fortpflanzungstrakts verändert, mit Auswirkungen auf reife Spermien und Zygoten. Diese Ergebnisse sind in [Kapitel 2](#chapter2) zusammengefasst.

Während der Arbeit an [Kapitel 1](#chapter1) und [Kapitel 2](#chapter2) entdeckten wir viele Probleme mit verfügbaren Werkzeugen zur Analyse von kurzen RNA-Sequenzierungsdaten. Kurze RNAs sind bedeutende Moleküle, die eine wichtige Rolle bei der Kontrolle des Genoms spielen. Kurze RNAs wurden in mehrere Kategorien eingeteilt, darunter miRNA, tRNA und tRNA-Fragmente sowie piRNAs, die alle eine komplexe Biosynthese aufweisen. Daher ist ein Analyserahmen erforderlich, der sowohl ausreichend spezialisiert ist, um die Spezifitäten verschiedener Klassen kurzer RNA und deren Biogenese widerzuspiegeln, als auch ausreichend generisch und gründlich ist, um globale Analysen durchzuführen. Wir haben ein benutzerfreundliches, hochgradig anpassbares und umfassendes `R`-Paket, `shortRNA`, erstellt, um eine vollständige End-to-End-Analyse von kurzen RNA-Sequenzierungsdaten durchzuführen. Unser Paket wurde speziell entwickelt, um Veränderungen in kurzen RNA-Typen und deren Biogenese zu berücksichtigen, und es kann leicht modifiziert werden, um weitere Anmerkungen aufzunehmen. Wir verwenden auch eine angepasste Genom-Annotation mit Pseudochromosomen, um posttranskriptionelle Modifikationen zu berücksichtigen, sowie einen flexiblen regelbasierten Ansatz, um Reads entlang eines Baums hierarchisch geordneter Attribute zuzuordnen. Dies ermöglicht die systematische Abfrage, Exploration und differentielle Expressionsanalyse kurzer RNAs auf mehreren Granularitätsebenen, die von einzelnen Sequenzen bis hin zu ganzen RNA-Klassen reichen. Das plattformunabhängige Paket shortRNA R wurde entwickelt, um alle Analyseschritte innerhalb von R durchzuführen. Diese Ergebnisse sind in [Kapitel 3](#chapter3) verfügbar. 

Zusammenfassend wurden in dieser Arbeit verschiedene Mechanismen der epigenetischen Vererbung untersucht und entwickelt mehrere Pipelines für die Datenanalyse aus dem MSUS-Modell, untersuchten Veränderungen in Spermatogonien während der Entwicklung, untersuchten die Auswirkungen von Stress im frühen Leben auf das Transkriptom extrazellulärer Vesikel der Nebenhoden und entwickelten ein `R`-Paket für die Analyse von kurzen RNA-Sequenzierungsdaten.

<!--chapter:end:rmd/02-summaryDE.Rmd-->

# Thesis overview {- #overview}
This thesis is divided into three parts: introduction and methods, scientific contributions, and concluding remarks. First, I introduce the biological context for my thesis, [Introduction](#intro), followed by the method section with the flowcharts and description of various data analysis pipelines, [Methods](#methods). Next, in scientific contributions, there are three chapters: [Chapter 1](#chapter1) is the latest version of the manuscript submitted to BioRxiv (Lazar-Contes et al. 2020), [Chapter 2](#chapter2) is published (Alshanbayeva et al. 2021), and [Chapter 3](#chapter3) is included as a descriptive chapter on shortRNA and a manuscript is in preparation. My contribution to each paper is summarized at the beginning of each chapter. In the end, the thesis concludes with a summary and outlook, [Conclusion and outlook](#conclusion).

<!--chapter:end:rmd/03-overview.Rmd-->

---
bibliography: rmd/references.bib
---
<!-- \FloatBarrier -->
# Introduction {#intro .unnumbered}

## Epigenetics {.unnumbered}

Conrad Hal Waddington, a British developmental biologist, coined the
term "epigenetics" in 1942. He was also a pioneer in the fields of
epigenetics, systems biology, and evolutionary developmental biology.
What came to be known as "Waddington's landscape" describes how genes
interact with the environment to produce, from the same genotype, the
variety of cellular phenotypes of a multicellular organism
[@waddington_1942]. While epigenetics has been given various meanings
throughout history (Figure \ref{fig:if1}, taken from [@pisco2016], it
can be broadly understood as the study of how our genes, which we
inherit from our forefathers and mothers, are controlled and interact
with the environment: how our genes form who we are. Epigenetics is the
study of how additional factors communicate with genes to guide the
processes that make our cells and bodies function. "Epi" means "on top
of" or "in addition to."

Epigenetic mechanisms regulate the transcriptional programs in response
to developmental and environmental cues and encompass a variety of
pre-transcriptional to post-translational events, including DNA
methylation (DNAme), post-translational histone modifications (e.g. acetylation
and methylation), histone variations, non-coding RNA (ncRNA) and their
modifications, and nucleosome remodelling [@susiarjo2016]. Epigenetic
mechanisms in the somatic cell regulate vital cellular function,
including differentiation or adaptation of cellular function in response
to environmental indications. However, epigenetic control in germ cells
can trigger changes in phenotype over many successive generations, known
as transgenerational epigenetic inheritance, section [Transgenerational
epigenetic inheritance](#tei), and contribute to changes in
susceptibility to health and disease.

(ref:ifc1) \textbf{History of the term ‘epigenetics’.} The discovery of
DNA methylation launched the associated modern (molecular) notion of
'epigenetics (1975) and is reflected in the surge of the number of
publications after that. Note the stagnation in the two decades after
that DNA recombinant technology and genetic determinism dominated
mindset in biology (grey horizontal bar). We are now seeing an enormous
increase in the use of the term, as the post-genome era findings
challenge the textbook gene-centred mindset. Textboxes display quotes,
of selected authors (vertical text). The red curve indicates the number
of Medline entries containing the wildcarded search term 'epigen\*' per
year, normalized for a number of publications that contain the generic
keywords 'cell' and 'function'. Diagonal text boxes highlight key
discoveries or findings in biology influencing the history of
epigenetics but not directly related to it. Taken from [@susiarjo2016].

```{r if1, echo=FALSE, fig.align='center', fig.cap="(ref:ifc1)", fig.scap="History of the term epigenetics", fig.width=6.42, fig.height=4.45}
img <- readPNG("figure/intro/1.png")
grid.raster(img)
```

## Germline epigenetics {.unnumbered}

### Spermatogonial Stem Cells {.unnumbered}

The germ stem cells of the seminiferous epithelium in the testis are
known as spermatogonial stem cells (SSCs). They are the building blocks
of spermatogenesis, the mechanism of producing mature male germ cells.
The SSCs are responsible for the continuous creation of the massive
number of sperm cells required for a male's reproductive lifespan. They
are present in the testis from birth and for the rest of the male's
life. SSCs have a dual biological function: they replicate themselves to
maintain the stem cell pool while also generating progenitors that will
differentiate into spermatozoa. They begin life in the testicular
seminiferous tubules and continue to serve this dual purpose throughout
adulthood. SSCs are the male adult stem cells of the seminiferous
epithelium in the testis that are capable of transferring a species'
genome from generation to generation while also possessing the ability
to differentiate into pluripotent stem cells.

In mice, the primordial germ cells (PGCs), the precursors of
prospermatogonia, appear at 5.5 days post-coitum. Prospermatogonia
establishes SSCs shortly after birth (gonocytes). PGCs undergo global
DNA demethylation (which involves the full erasure of parental imprints)
through both active and passive mechanisms as they migrate toward the
gonads. DNA methylation (DNAme) is gradually restored after that
(between embryonic day 13.5 [E13.5] and E16.5), and both maternal and
paternal-specific imprints are believed to be completely formed by birth
in male prospermatogonia [@hammoud2015].

The prospermatogonia are mitotically arrested at birth, but they begin
cycling on postnatal days 1--2 (PND1 -- PND2). A population of
spermatogonial cells continues to proliferate throughout the first week
after birth, giving rise to the undifferentiated A\textsubscript{single}, A\textsubscript{paired}, and A\textsubscript{aligned} spermatogonial pools [@kolasa2012]. Cycling
spermatogonia proliferate and populate the seminiferous tubule over the
next week (PND3--PND10); here, a subset gives rise to self-renewing
SSCs, while the rest differentiates without self-renewal, resulting in
the first wave of spermatogenesis or commits apoptosis. As a result,
this stage of germ cell growth is critical for initiating and
maintaining male fertility during adulthood. Following puberty, after
35-37 days, SSCs go through many phases of proliferation, eventually
forming sister stem cells that will maintain a stable pool or more
differentiated intermediates that will initiate meiosis I and II and
form mature sperm cells. Please see Figure \ref{fig:if2} for an overview. These transitions are timed to correlate with
global epigenome shifts, which represent the spermatogonial cell's
developmental path, current transcriptional program, and future
commitment to differentiation. Since the transcriptome and epigenetic
landscape of spermatogonial stem cells is so complex during
proliferation and differentiation, drastic changes in the environment
may not go unnoticed.

(ref:ifc2) \textbf{Graphical summary of the biology of germline stem cell specification and transitions.} Adapted from [@hammoud2015]

```{r if2, echo=FALSE, fig.align='center', fig.cap="(ref:ifc2)", fig.scap="Graphical summary of the biology of germline stem cell specification and transitions", fig.width=6.42, fig.height=4.45}
img <- readPNG("figure/intro/2.png")
grid.raster(img)
```

It is difficult to distinguish SSCs from progenitor spermatogonia that
don't have stem cell activity and are committed to
differentiation. Specific cell surface markers like Thy1 and TSPAN8 can
be used to enrich SSCs. Further, cells extracted from Id4-eGFP
transgenic mice that express a high amount of GFP signal, and
combinations of reporter gene and cell surface marker signal can be used
to further enrich the population [@mutoji2016; @chan2014].

### Reprogramming {.unnumbered}

Reprogramming refers to the erasure and remodelling of epigenetic
markers such as DNAme throughout mammalian development [@reik2001].
Diverse cells and tissues acquire different gene expression patterns
during the evolution of multicellular organisms. Some cells undergo
extensive epigenetic reprogramming in normal developmental or
pathological conditions, which involves the removal of epigenetic marks
in the nucleus and the installation of a new set of marks. Genome-wide
epigenetic reprogramming occurs at stages when the developmental potency
of cells changes. There is passive DNA demethylation and further
reorganization of histone modifications. The epigenetic reprogramming is
likely to be needed for totipotency, correct initiation of embryonic
gene expression, and early lineage development in the embryo. Extensive
epigenetic reprogramming of DNA and histone marks occur in mammalian
development in totipotent early embryos and pluripotent germ cells
[@canovas2016].

The germ cell lineage is unique in carrying genetic and epigenetic
information from one generation to the next. In the germline, epigenetic
reprogramming resets genomic potential and erases epigenetic memory.
Epigenetic reprogramming involves genome-wide demethylation of 5mC,
which is important for genome imprinting, X inactivation, transposon
silencing, centromeric/telomeric structural stability, and gene
expression [@surani2007]. After a sperm fertilizes an ovum, nearly all
epigenetic marks are reprogrammed in two short intervals early in
development. By eliminating random changes in epigenetic marks (i.e.,
epimutations) that have occurred in the germ cells, reprogramming restores the ability
of the fertilized egg cell (i.e., zygote) to develop into all the
different cells types and tissues [@reik2001]. During embryo
development, there are two waves of epigenetic reprogramming, Figure \ref{fig:if3}.
After fertilization in the preimplantation embryo (i.e., the
blastocyst), the first phase begins; preimplantation refers to the time
between the embryo implanting and forming a placenta. The embryo in mice
undergoes genome-wide demethylation after fertilization, which is
finished by E5 [@reik2001; @smallwood2012]. The
paternal genome undergoes rapid, active demethylation, whereas the
maternal genome experiences passive demethylation. The embryonic DNA
begins to be re-methylated on day E5 and is finished before birth. The
second wave of epigenetic reprogramming takes place in the germ cells of
the developing embryo, which will eventually give rise to gametes with
sex-specific epigenetic markers (gametogenesis). The methylation
signatures of the parental genomes are found in the PGCs, which are progenitors to both male and female germ cells
in the developing embryo. PGCs are epiblast cells that first appear in
the posterior primitive streak at E7.5, after which they begin to
migrate from E8.5 to the genital ridge, arriving by E11.5. PGCs undergo
rapid demethylation beginning around E7--8, which is completed by
E15--16. In PGCs, parental imprints are erased and totipotency is
restored, followed by the restoration of sex-specific methylation.
Reprogramming is complete in the male germline at birth, but not in the
female germline until adolescence.

(ref:ifc3) \textbf{Methylation reprogramming in the germline.} \newline \textbf{A:} Primordial germ cells (PGCs) in the mouse become demethylated early in development. Remethylation begins in prospermatogonia on E16 in male germ cells, and after birth in growing oocytes. Some stages of germ cell development are shown. \newline \textbf{B:} Methylation reprogramming in preimplantation embryos. The paternal genome (blue) is demethylated by an active mechanism immediately after fertilization. The maternal genome (red) is demethylated by a passive mechanism that depends on DNA replication. Both are remethylated around the time of implantation to different extents in embryonic (EM) and extraembryonic (EX) lineages. Methylated imprinted genes and some repeat sequences (dashed line) do not become demethylated. Unmethylated imprinted genes (dashed line) do not become methylated. Taken from [@reik2001].

```{r if3, echo=FALSE, fig.align='center', fig.cap="(ref:ifc3)", fig.scap="Graphical summary of the biology of germline stem cell specification and transitions", fig.width=3.5, fig.height=4.5}
img <- readJPEG("figure/intro/3.jpeg")
grid.raster(img)
```

Embryonic development involves epigenetic reprogramming to replenish
totipotency from a germline state in each generation. To begin with,
because there is no hereditary germplasm and germ cell precursors are
formed from a post-implantation epiblast that has acquired epigenetic
markers throughout early development, a germline reprogramming phase may
be required to ensure that this acquired information is erased. Then, in
the post-implantation epiblast, the parent-of-origin markers of mature
gametes fuse in the zygote and are passed down to nascent PGCs from
their predecessors. Another goal could be to reverse the effects of
accumulating epigenetic changes that are influenced by both genetic and
environmental factors. However, epigenetic information can be inherited
through the germline (or after fertilization) on occasion, which is most
likely due to insufficient germline erasure. Finally, demethylation in
the germline may have the added benefit of reducing the frequency of
mutations caused by 5-methylcytosine (5mC) deamination.

## Epigenetic Inheritance {.unnumbered}

Inheritance is the process of passing traits or information from one
generation of individuals or cells to the next. Charles Darwin proposed
natural selection as the primary mechanism for species evolution in his
1859 book: "On the origin of species". According to Darwin, natural
selection is a key mechanism of evolution, characterized by changes in
the heritable traits of a population over generations. Jean-Baptiste
Lamark was an early proponent of the idea that biological evolution
occurred and proceeded in accordance with natural laws. He is well known
for the principle of "inheritance of acquired characteristics", an idea
known as Lamarckism. According to Lamarckian theories of inheritance,
physical traits inherited by a generation during its lifetime (e.g.
strength and body size) are passed down to offspring. In 1868, Darwin
proposed pangenesis theory as a supplement to his 1859 theory of
evolution by natural selection in his book "The Variation of Animals and
Plants Under Domestication." The term pangenesis was coined by Darwin,
with "pan-" referring to the whole and "genesis" referring to the
origin. Using the term pangenesis, Darwin proposed that all parts of the
parents could contribute to the evolution and development of the
offspring. Darwin's theory of pangenesis was based on the idea that
somatic cells would release 'gemmules' or 'pangenes' in response to
environmental stimulation (use and disuse), which travelled around the
body but not necessarily in the bloodstream. These pangenes were
microscopic particles that supposedly contained information about the
characteristics of their parent cell, and Darwin believed that they
eventually accumulated in germ cells, where they could pass on the newly
acquired characteristics of the parents to the next generation
[@darwin1871]. As a result, Darwin advocated Lamarckian use and disuse
inheritance, as well as Lamarckian soft inheritance. In the 1880s,
August Weismann challenged this concept. Weismann proposed the germ
plasm theory, which states that germ cells in the gonads contain
information that is passed down from generation to generation unaffected
by experience and independent of somatic cells [@weismann1893].
Hereditary information is only transferred from germline cells to
somatic cells. This is referred to as the Weismann barrier.


However, recent epigenetic discoveries have revealed that acquired
traits can be passed down to future generations. Our recent work showed
that serum from adult male mice exposed to postnatal stress, injected
chronically to naïve males, can induce metabolic symptoms associated with
stress in the offspring of the injected males [@vansteenwyk2020]. Also,
we identified circulating extracellular vesicles as vectors of
communication with germ cells that can reproduce symptoms of exposure in
the offspring when injected into fathers in vivo [@alshanbayeva2021].
These results highlight the importance of circulating factors in the
mechanisms of epigenetic inheritance, [Chapter 2](#chapter2).

### Transgenerational Epigenetic Inheritance {#tei .unnumbered}

The transmission of epigenetic information from one generation to the
next is referred to as intergenerational epigenetic inheritance, while
that across multiple generations is referred to as transgenerational
epigenetic inheritance. Both intergenerational and transgenerational
imply the parents' exposure to the environmental factor in question, as
well as the gametes they already have. Transgenerational exposure refers
to people who have never been specifically exposed to the environmental
factor during their lifetime (either as individuals or as future
gametes). This may be the third generation after in utero exposure or
the second generation after preconception exposure. It is assumed that
fetal or in utero exposure to an environmental factor is not heritable
because the offspring are specifically exposed. However, the mechanisms
of adverse development following in utero exposure to drugs, alcohol,
diet, and other factors also influence offspring through epigenetic
mechanisms.

### Reprogramming in epigenetic inheritance {.unnumbered}

Epigenetic reprogramming refers to the process of removing and resetting
relevant epigenetic information in germ cells, which is primarily
accomplished through DNA demethylation. Epigenetic reprogramming
prevents the transmission of epigenetic information from mature gametes,
which is in line with the germ-plasm theory of Weissman [@weismann1893].
The extent of epigenetic reprogramming, however, is unknown, leaving the
question of how much epigenetic information from adult gametes is
preserved in the embryo unanswered. In mammals, DNAme and histone marks
are effectively reset during reprogramming both in the germline, and in
the zygote immediately after conception, leaving epigenetic changes with
minimal possibility of passing down. Evidence for the survival of
histone modifications across reprogramming has been identified in sperm,
where histones are largely replaced by protamines. Surprisingly,
H3K27me3 enrichment has been discovered in reprogramming resistant sperm
nucleosomes [@casas2020], suggesting that this mark may be a hereditary
signal. Further, findings in [@kremsky2020] imply that transcription
factors can operate as epigenetic information carriers during germ cell
and pre-implantation development by maintaining the CpG methylation
state.

## Models of Epigenetic Inheritance {.unnumbered}

Given limitations such as the inability to conduct controlled
experiments in humans and the availability of data for successive
generations, rodent models have become critical for correlating human
findings [@jawaid2021] and to investigate the penetrance of effects
across generations to provide mechanistic insight. There are several
experimental paradigms for studying epigenetic inheritance. In mammals,
dietary changes, stress, medications, and environmental toxins have all
been linked to altered phenotypes in offspring [@gapp2018]. Epigenetic
inheritance appears to be initiated or modulated by mutations in the
epigenetic machinery, such as DNA/RNA methylation enzymes or histone
modification complexes, such as overexpression of _Kdm1a_, an H3K4me3
demethylase in mice [@siklenka2015].

Alternatively, environmental perturbations may initiate epigenetic
inheritance, with various insults leading to offspring phenotypes in a
variety of model systems [@jawaid2019]. Environmental factors like
vinclozolin (an antiandrogenic compound), methoxychlor (an estrogenic
compound), stress responses, high-fat diet, high-sugar diet, low-protein
diet, air pollutant exposure in pregnant mice, in utero Bisphenol-A (BPA),
di(2- Ethylhexyl) phthalate (DEHP) and dibutyl phthalate (DBP) exposure
in rats, in utero exposure to jet fuel (JP-8) in mice, in utero valproic
acid exposure in mice, olfactory fear conditioning in Filial 0 (F0) mice, and
Female mice fed a high-fat/ high-sugar diet have been linked to
epigenetic inheritance [@rando2016; @bohacek2015; @hanson2016; @mao2017;
@nagy2015; @heard2014; @schaefer2015]. These models shed new light on
the mechanisms of epigenetic inheritance. However, distinguishing
evidence of epigenetic mark transmission from the evidence of exposure
transmission is difficult.

### MSUS mouse model {.unnumbered}

**A unique transgenerational model to assess the effects of trauma:**
unpredictable maternal separation combined with unpredictable maternal
stress (MSUS) is a firmly established transgenerational model in the
Mansuy lab, that does not involve any chemical or mutagenic agent but,
depends only on psychological and emotional trauma, Figure \ref{fig:if4}. It is an
ideal model to study the possible transgenerational effects of life
experiences on the genome and the epigenome. 

(ref:ifc4) \textbf{MSUS paradigm.} MSUS consists of \textbf{(A)} separating mouse pups (F1) from their mother (F0, naïve primiparous control females mated with naïve males) daily for 3 hours per day at an unpredictable time during the 12 hours active cycle, starting 1 day after birth (postnatal day 1, PND1) until PND14. \textbf{(B)} During separation, dams are exposed to an additional unpredictable stressor by being subjected to either, a forced swim in 18°C water for 5 minutes or a 20-minute physical restraint in a tube, anytime (unpredictably) during the 3 hours. From PND15, mice are left undisturbed with their mother until PND21 (no further MSUS), are then weaned at PND21 and raised normally until adulthood \textbf{(C)}. Control litters are raised normally (left). Males used to generate the pups are removed from the breeding cage shortly after mating thus, fathers never encounter their offspring and do not contribute to their rearing. When adult (3–8 months of age), F1 males are paired with naïve primiparous control females to sire the F2 generation, then F2 and F3 males are bred with naïve primiparous control females to generate an F3 and F4 offspring, respectively. Males from each generation are tested on the elevated plus-maze, forced swim test, weight measurements and glucose response after physical restraint. MSUS is applied only to F1 mice, mice from F2, F3 and F4 generations are not exposed to any manipulation. Phenotypes transmitted from father to offspring are intergenerational, phenotypes that persist from father to offspring then grand-offspring or great grand-offspring are transgenerational. Taken from [@vansteenwyk2018].

```{r if4, echo=FALSE, fig.align='center', fig.cap="(ref:ifc4)", fig.scap="MSUS paradigm", fig.width=5.3, fig.height=3.3}
img <- readPNG("figure/intro/4.png")
grid.raster(img)
```


MSUS consists in separating
mouse pups from their mother unpredictably (any time during the day)
each day for 3 hours from postnatal day 1 (PND1) to PND14 and exposing the mothers to stressful situations such as restraint stress or
forced swim in cold water (18ºC), unpredictably during separation. F1 generation males are mated with naïve control females, and F2
males are mated with control females to obtain F3. MSUS induces several
behavioural symptoms including depression, increased risk-taking,
antisocial behaviours and memory deficits, and metabolic alterations
across several generations [@franklin2010; @vansteenwyk2018]. Further,
metabolic phenotypes like decreased bodyweight, insulin hypersensitivity
and glucose intolerance are also observed. For studying the molecular
mechanisms of epigenetic inheritance, a number of high-throughput sequencing (HTS) datasets have been
collected and sequenced, Figure \ref{fig:if5}.



(ref:ifc5) \textbf{Dataset collected from the MSUS model.}

```{r if5, echo=FALSE, fig.align='center', fig.cap="(ref:ifc5)", fig.scap="Dataset collected from the MSUS model", fig.width=6.42, fig.height=4.45}
img <- readPNG("figure/intro/5.png")
grid.raster(img)
```


## Vectors of epigenetic inheritance {.unnumbered}

Epigenetic marks, such as DNAme, histone modifications, and non-coding
RNAs, have been proposed as information-carrying vectors passed down
through generations. These three candidates have been investigated in a
variety of epigenetic inheritance paradigms. It is hypothesized that stable changes in
epigenetic marks caused by an environmental insult (epimutations) may
affect gene expression or chromatin stability, increasing the risk of a
phenotype or disease in offspring (Miska and Ferguson-Smith, 2016).
DNAme, histone modifications, RNA, and ncRNAs are all interconnected and
interdependent within individual cells. They are likely to act in a
coordinated and collaborative manner to cause disease phenotypes in the
context of epigenetic inheritance.

### DNA methylation {.unnumbered}

The most common type of DNAme in mammals is cytosine residue
methylation. Cytosine residues are methylated on the 5' carbon and are
referred to as 5-methylcytosines (5mC). Methylation of a cytosine
residue is a binary process -- either methylation or non-methylation. In
mammals, DNA methylation is typically found in the context of CpG
dinucleotides [@bird2002]. Cytosine methylation can be copied from one
DNA strand to the next in this manner during replication and meiosis.
DNAme patterns, on the other hand, are dynamic. The patterns of DNAme
vary over time and space, changing during development and differing
between cell types [@luo2018]. DNA methyltransferase (DNMT) enzymes
methylate cytosine residues. DNMT1 is a methyltransferase that reads
hemimethylated DNA after cell division and adds methylation to the newly
synthesized strand [@chen2011]. The de novo methyltransferases DNMT3a
and 3b can add methylation to previously unmethylated residues
[@smith2013]. Although DNAme in gene control regions is generally
associated with gene repression, its precise role in gene regulation is
likely locus dependent [@deaton2011]. Methylated DNA is a true
epigenetic mark because there are "readers" that preferentially
recognize 5mC and can interpret the methylation mark's meaning
[@zhu2016a]. Many properties of DNAme make it an appealing mechanistic
candidate for epigenetic inheritance, including the following: it can be
environmentally modulated; machinery exists to replicate methylation
patterns onto newly synthesised DNA, making it mitotically heritable;
and methylated loci resistant to epigenetic reprogramming have been
identified in the zygote and germline. Disruption of the DNAme machinery
may also play a role in the initiation of EI.

### Histone modifications {.unnumbered}

In somatic cells, DNA is wrapped around an octameric core of histone
proteins (H2A, H2B, H3, and H4) to form a nucleosome [@marmorstein2009].
Histones' N-terminal tails and globular domains undergo
post-translational modifications (PTMs). Histone modifications are
dynamic during development, vary across tissues, are regulated by
specific enzymes, play a significant role in gene expression, and
interact with other epigenetic control systems such as DNAme
[@barth2010]. On the histone tails, there are over 60 different amino
acid residues where modifications have been detected [@iwasaki2013].
Histones can undergo various modifications, including acetylation,
methylation, phosphorylation, ubiquitination and sumoylation among
others [@kouzarides2007]. Histone methyltransferase (HMT) enzymes
methylate histones on Lysine (K) and Arginine (R) amino acids. Histone
lysine residues can be mono-, di-, or tri-methylated, and these
modifications can function as either active or repressive marks.
Methylation of H3K4, H3K36, and H3K79 is associated with transcriptional
activation, whereas methylation of H3K9, H3K27, and H4K20 is associated
with transcriptional repression [@vakoc2006; @sims2006]. These
modifications can influence chromatin packaging and nucleosome
positioning. Histone modifications, like DNAme, are spatially and
temporally dynamic. Certain histone modifications, such as
trimethylation of lysine 27 on histone 3 (H3K27me3), are generally
associated with gene repression, whereas acetylation of the same residue
(H3K27ac) is generally associated with gene activation [@lawrence2016].
Histone modifications regulate gene expression by recruiting protein
complexes and controlling the accessibility of DNA to the transcription
machinery [@lawrence2016]. Histone modifications are a plausible
mechanistic candidate in epigenetic inheritance because they can be modulated by the
environment. During epigenetic reprogramming, some histone modifications
in the germline and zygote are retained. Histones are spread onto newly
assembled chromatin by sophisticated machinery [@marmorstein2009].
However, it is still unclear how histone codes are maintained during
mitosis and meiosis [@erkek2013]. Furthermore, during spermatogenesis,
most histones are removed and replaced with protamines [@casas2014].
Sperm contains protamine modifications that are similar to histone
modifications [@brunner2014], despite this, their functional
significance remains unknown, particularly during epigenetic
inheritance. Disruption of the machinery that regulates histone
modification could also be a key initiator of epigenetic inheritance [@siklenka2015]. H3K9
and H3K27 methylation appear to be the most likely to be epigenetic
among the various histone lysine methylation states, not only because
they are key regulators of classic epigenetic phenomena [@huang2013].
According to one model of epigenetic inheritance, marked parental
histones inherited via DNA replication recruit histone modifiers to
deposit similar marks on new adjacent nucleosomes, reestablishing the
previous local landscape of histone modification [@saxton2019]. These
findings suggest that histone modifications can aid in epigenetic
inheritance, though such a mechanism is normally obscured by H3K9
demethylation activity [@saxton2019].

### RNA {.unnumbered}

One haploid copy of the genome exists in the sperm nucleus, which is
transcriptionally silent and not replicated due to the packaging of
sperm chromatins during spermiogenesis. Also, the sources of sperm RNAs
are unknown. Some RNAs are probably expressed endogenously within germ
cells, but considering that short RNAs have been demonstrated to impact
gene expression outside of their tissue of origin, sperm RNA could come
from elsewhere. Because RNAs are easily degraded, they are most likely
transported with protein chaperones and, more likely, enclosed in
extracellular vesicles. For example, in epididymosomes, which are
extracellular vesicles that are released from the epididymal epithelium
contain proteins, ncRNAs and a distinct set of lipids that are
transferred to spermatozoa while they pass through the different
epididymal regions [@sullivan2016]. Aside from epididymosomes, RNAs have
been found in exosomes and high-density lipoproteins (HDLs) in the
bloodstream, as well as vesicles secreted by prostate cells
(protostomes) and Sertoli cells, which come into direct contact with
developing sperm. These interactions have not been studied in the
context of transgenerational inheritance. Microinjections of RNA
mixtures were used to recapitulate mutant phenotypes in functional tests
of RNAs contribution to inheritance patterns. These findings imply that
RNA plays a significant role in non-genetic inheritance. It is unclear
how these transient RNA molecules can mediate epigenetic inheritance
across cell divisions and generations on their own. Nonetheless, a
growing body of evidence suggests that RNA may aid in the stable
propagation of paternal effects from the soma to the germline, where
they will be inherited by the next generation [@vansteenwyk2020;
@andersson2014; @gapp2020].

### Non-coding RNA {.unnumbered}

ncRNAs are broadly classified into two types: 1) long-ncRNA, greater
than 200 nucleotides in length (lncRNAs) [@li2012] and 2) small-ncRNA,
less than 200 nucleotides (sncRNA) [@bouckenheimer2018]. Small
regulatory RNAs (siRNAs, miRNAs, tRNAs, and piRNAs) are non-coding RNAs
that are expressed in the majority of eukaryotic cells. These small RNAs
are bound by the Argonaute (AGO) or PIWI proteins and function as
specificity factors (guide RNAs) to regulate other homologous RNAs at
the transcription, stability, and translation levels. Small RNAs are
found in the nuclei of most eukaryotes, where they regulate genome
stability, heterochromatin formation, and transcription. ncRNAs are
uniquely suited to transmit information across generations. A variety of
ncRNA species can be found in both sperm and oocytes [@rando2016;
@veselovska2015]. There is evidence that sperm can acquire ncRNAs from
surrounding somatic tissues [@robles2019]. Thus, RNAs can circumvent and
penetrate the theoretically impenetrable wall between somatic cells and
the germline (the so-called Weismann barrier). This has significant
implications for our understanding of epigenetic inheritance as a result
of adult-life environmental insults. However, it is currently unknown
how epigenetic messages transmitted by RNA are maintained across
multiple cell divisions, let alone multiple generations.

### Transcription factors{.unnumbered}

Transcription factors (TFs) may also act as epigenetic information
carriers. TFs are proteins that play a role in the
process of transcribing DNA to RNA. TFs are found on the genomes of
mature gametes, and there is evidence that TF binding can influence
DNAme at the binding sites [@smith2013; @jones2001]. CpG sites bound by
TFs during germ cell re-methylation are protected from methylation
throughout development [@kremsky2020]. During PGC and preimplantation
development, DNAme can only occur at CpGs that are resistant to
demethylation. This is due to the fact that TF binding during the
re-methylation phase maintains the methylation status between
generations [@kremsky2020]. This suggests that transcription factors can
act as carriers of epigenetic information during germ cell and
pre-implantation development by maintaining CpG methylation status, and
serve as the foundation for a mechanistic description of the
transgenerational transmission of such information in mammals
[@kremsky2020]. Mammalian gametes contain complex patterns of 3D
interactions that can also be transmitted to the zygote after
fertilization [@jung2019]. TFs may remain bound to chromatin after
fertilization and influence gene expression in the developing embryo.
According to a recent publication, BPA, either directly or indirectly,
induces the binding of TFs at different regions of the genome in F1
generation sperm, altering the 3D organization of chromatin in mature F1
generation sperm, many of which are conserved in subsequent generations
[@jung2020]. Thus, TFs are a plausible candidate for transmitting
epigenetic information.

### Extracellular vesicles {.unnumbered}

All cells can secrete various types of membrane vesicles known as
extracellular vesicles (EVs). EVs vary greatly in size (30 nm to several
micrometres in diameter), chemical composition, secretion mechanism and
carrying molecules, all of the properties that lead to their different
functions. EVs mediate cell-to-cell communication by activating
receptors on the surface of acceptor cells [@bonsergent2021] and play
roles in a variety of physiological and pathological processes,
including blood coagulation, inflammation, stem cell expansion, neuronal
communication, and tumorigenesis [@locicero2015]. There are two major EV
biogenesis pathways. The first biogenesis pathway buds EVs directly from
the plasma membrane, resulting in what is known as microvesicles. The
second biogenesis pathway involves intraluminal vesicle release caused
by multivesicular endosome fusion with the plasma membrane, resulting in
the formation of exosomes. Exosomes and microvesicles contain cargoes
such as nucleic acids, proteins and lipids, RNAs (extracellular RNAs,
exRNAs) and DNA [@obrien2020]. The majority of known ncRNA biotypes,
including small nuclear RNAs, small nucleolar RNAs (snoRNAs), rRNAs,
lncRNAs, piRNAs, tRNAs,
mitochondrial RNAs, Y RNAs, and vault RNAs (vtRNAs), have been
discovered in EVs in more comprehensive studies [@obrien2020;
@abels2016]. ExRNA extraction, purification, and sequencing methods are
much more susceptible to contamination and artefacts than cellular RNA
preparations, due to the low amount of RNA available from EVs. As a
result, analyzing exRNA samples can be difficult, and a specific tool
for EV ncRNA data analysis has been developed [@rozowsky2019]. A number
of studies have shown the role of extracellular vesicles in epigenetic
inheritance. Also, we found that extracellular vesicles are vectors of
communication with the germline that play a role in epigenetic
inheritance [@alshanbayeva2021], [Chapter 2](#chapter2).

## Challenges in data analysis {.unnumbered}

### Multi-omics data integration {.unnumbered}

A single omics analysis may not provide enough information to provide a
deep understanding of a biological system, but combining multiple omics
experiments and data analyses can provide a more holistic view of a
system. Performing different experiments on the same or different
samples from the same biological conditions, such as RNA sequencing (RNA-seq), Assay for Transposase-Accessible Chromatin sequencing (ATAC-seq),
chromatin immunoprecipitation with sequencing (ChIP-seq), and Bisulfite sequencing (BS-seq), results in multi-dimensional omics datasets, which
enable the study of relationships between different biological
processes, such as gene expression, chromatin accessibility, histone
modifications, transcription factor binding, and DNAme, as well as the
leveraging of multiple data types to draw inferences about these
processes. Data integration is the process of combining disparate
sources of information in order to gain a better understanding of
complex systems. The researchers execute two forms of data integration:
custom or ad-hoc integration and more systematic integration.

#### Ad-hoc approaches {.unnumbered}

Integrating BS-seq data with RNA-seq is one example of an ad-hoc
technique. The expressed genes are known to be demethylated. As a
result, the methylation values of the promoters can be linked to the
genes that are expressed. Integrating ATAC-seq data with RNA-seq data
can be used to analyze TF activity. Regions with
increased accessibility and expression would show a TF's role as an
activator, while regions with increased accessibility and decreased
expression would represent a TF's role as a repressor. For this, a tool
exists, diffTF [@berest2019], which estimates differential TF activity
based on expression and chromatin data. The ad-hoc methods are
scientifically motivated and produce biologically relevant outcomes.
However, these ad-hoc approaches have several drawbacks, such as the
need for extensive exploratory investigation and the inability to be
applied consistently to all projects. Furthermore, these methods are
frequently applied to a maximum of two omics datasets.

#### Systematic approaches{.unnumbered}

Systematic methods developed for data integration in biological research
can be classified into three types: supervised, unsupervised, and
semi-supervised. Unsupervised data integration refers to methods that
draw conclusions from input datasets without the use of labelled
response variables (e.g., factorization, Bayesian, clustering,
dimensionality reduction, coexpression and correlation). On the other
hand, supervised data integration methods take into account phenotype
labels (e.g., control or treatment) and use, for example, machine
learning approaches to evaluate models (network-based and multi-kernel);
semi-supervised data integration methods fall somewhere between
supervised and unsupervised methods and use both labelled and unlabeled
samples to develop a learning algorithm to infer information from
unlabeled samples [@huang2017]. A few methods for integrating data from
multivariate multi-omics experiments have been developed: Data
Integration Analysis Using Latent Components for Biomarker Discovery
(DIABLO) [@rohart2017] - an extension to sparse generalized canonical
correlation analysis (sGCCA) [@tenenhaus2014], a supervised approach
which seeks for common information across different data types through
the selection of a subset of molecular features; Multi-Omics Factor
Analysis (MOFA) - an unsupervised method for decomposing the sources of
heterogeneity in multi-omics data sets [@argelaguet2018;
@argelaguet2020] and; mixOmics [@rohart2017] - a package that hosts
several supervised methods under its umbrella (DIABLO; sparse Partial
Least Square---Discriminant Analysis, PLS-DA [@lêcao2011] and
Multivariate INTegrative method, MINT [@rohart2017]).

Despite the fact that these tools have addressed critical challenges for
multi-omics applications, they are not without limitations. One of the
major limitations of most of these tools is that they can only
incorporate data from the same omics experiments. This restricts users
to utilize the same samples for all experiments, which is never the case
in biological research due to the limited quantity of components of
interest (DNA or RNA) and the high demand for these components for
library preparation and sequencing. Another issue is that these methods
assume that users have already normalized the data and corrected it for
batches or confounding factors such as cage effect, sample composition,
and time of day [@bruning2016]. Based on the data types, methods should
be able to optimally normalize the data and correct it for confounding
factors using supervised learning, which could improve uniformity,
robustness, and precision. Another disadvantage is that some of the
existing methods necessitate exploratory analysis prior to data
integration. Exploratory data analysis is a critical process in many
data science projects that requires advanced coding skills and takes a
significant amount of time. The method should be able to perform
automated data cleaning functions, such as outlier clipping, missing
value imputation, and zero-variance variables, depending on the data
types. It should also aid in exploratory analysis by providing summary
statistics and plots for visual examination. Finally, most existing
methods are designed to work with linear relationships and thus cannot
be applied to more complex datasets that are related in a non-linear
manner.

To summarize, integrating diverse and huge omics data is both a
conceptual and practical problem in data analysis. Data integration has
been used in omics research since the beginning of the era, and it is
both a challenge and an opportunity. It is surely growing more popular.
We devised and implemented a more straightforward and scientifically
motivated approach to integrate multi-omics datasets, involving more
than two omics datasets, please see [Methods](#methods) and [Chapter 1](#chapter1).

### Short RNA data analysis {.unnumbered}

Short RNAs are a type of non-coding RNA that plays an important role in
nearly all aspects of organismal development in both health and disease.
Despite their importance in a wide range of biological processes,
analyzing short RNA sequencing data remains a difficult task.
Specifically, a number of computational challenges arise when analyzing
short RNA sequencing:

-   Even after size selection, we expect to sequence other RNA biotypes
    in addition to short RNAs.

-   Short RNAs have a length of less than 200bp, often much shorter
    (e.g. miRNAs at 21-22bp), and can therefore often map to multiple
    positions. In this case, standard pipelines either map the reads
    randomly to one position, making it difficult to determine their
    true origin, or they are discarded, resulting in a large proportion
    of reads being lost.

-   Many short RNAs are post-transcriptionally modified. For example,
    tRNAs receive a "CCA" sequence at the 3' end [@hou2010],
    additionally, Histidine tRNAs receive a "G" sequence at the 5' end
    [@cooley1982], resulting in an unalignment problem.

-   Because of their biogenesis, short RNAs often overlap multiple
    features. For example, mature miRNAs overlap with precursors, which
    can result in misassignment.

-   rRNAs are typically masked in the genome, and the Ensembl
    [@howe2021] or GENCODE [@harrow2012] databases' annotations are
    unavailable, which make it difficult to quantify for rRNAs
    fragments.

All the aforementioned issues could lead to misalignment and low
mappability of the sequenced data. Several tools have been developed to
analyze short RNA-seq data, however, they typically fall prey to some of
these issues. In [Chapter 3](#chapter3), we, therefore, developed a novel approach to
address them.

## Goals of thesis {.unnumbered}

We defined the following goals for my thesis:

1.  Collaboration with all the members of the Mansuy lab to study
    epigenetic inheritance in the MSUS mouse model. Please see [Chapter 2](#chapter2), [Appendix A](#aa), and [Appendix B](#ab) for this part. I was responsible for:

    -   Version controlled and reproducible data analysis for data
        generated from RNA-seq, short RNA-seq (sRNA-seq), ATAC-seq, whole-genome bisulfite sequencing (WGBS), and reduced representation bisulfite sequencing (RRBS)

    -   Determine the qualitative and quantitative effects of early life
        experiences on the epigenome, particularly in germ cells. This
        includes identifying the epigenetic changes, including DNAme,
        RNA expression, ncRNAs, and chromatin
        accessibility, induced in germ cells due to early life
        experiences like traumatic stress in mice relating these changes
        to genome activity. This also includes determining whether and
        to what extent changes in the epigenome persist across life, and
        if they can be passed to the offspring and underlie epigenetic
        inheritance, and further, whether they can result in changes in
        DNA sequence across generations

    -   Supervising and collaborating with colleagues who were
        performing data analyses themselves

    -   Analyses of datasets using different tools

    -   Multi-omics analysis of data from same cells/ tissues

    -   Figure generation and interpretation for the manuscripts


2.  Development of data analysis pipelines. Please see the [Methods](#methods), for
    this part. I was responsible for:

    -   Benchmarking methods for including in the pipeline

    -   Testing different parameters of the tools for including in the
        pipeline

3.  Collaboration with Dr. Irina Lazar for studying changes in
    spermatogonial stem cells across development. Please see [Chapter 1](#chapter1)
    for this part. I was responsible for:

    -   Computational design of the project

    -   Version controlled and reproducible analysis of the datasets

    -   Analyzing spermatogonial stem cells dataset generated from
        RNA-seq, sRNA-seq, RRBS, and ATAC-seq from the data generated in
        the lab

    -   Analyzing spermatogonial stem cells dataset generated from
        RNA-seq, WGBS, and ChIP-seq from the publicly available datasets

    -   Comparison of published datasets with the lab's

    -   Analysis of ID4 ATAC-seq data from the literature and comparison
        with lab's dataset

    -   Multi-omics data integration

    -   Figures outline, figures generation, and interpretation for the
        manuscript

    -   Writing methods, figure legends, and updating/ revising
        manuscript


4.  Collaboration with Dr. Pierre-Luc Germain for the development of
    `shortRNA`, a tool for comprehensive analysis of sRNA-seq data
    in R. Please see [Chapter 3](#chapter3) for this part. I was responsible for:

    -   Development and testing of the tool

    -   Data analysis using different methods for qualitative and
        quantitative comparison with shortRNA

<!--chapter:end:rmd/04-intro.Rmd-->

---
bibliography: [rmd/references.bib, bib/thesis.bib]
link-citations: true
---

<!-- \FloatBarrier -->

```{r setup, include=FALSE}
# knitr::opts_chunk$set(dpi=300)
```

# Bioinformatics methods {#methods .unnumbered}

## Version controlled data analysis using git {#m1 .unnumbered}

Version control (also known as source control or revision control) is a
software engineering process that involves managing changes to a
computer program or document using a version control system (VCS). In
graph theory, revisions form a directed acyclic graph that represents
the growth line. The revisions to a computer program occur over time and
can be tracked using revision numbers, timestamps, or descriptive text.
While file copies have been used in the past to provide some form of
version control, they are error-prone and time-consuming. The file is
typically copied to another location, and unless the files are cleverly
time-stamped, this simple approach may be error-prone.

(ref:mfc1) \textbf{Overview of version control system (VCS).} Adapted
from [@chacon2021].

```{r mf1, echo=FALSE, fig.align='center', fig.cap="(ref:mfc1)", fig.scap="Overview of version control system", fig.width = 4.96, fig.height=5.6}
img <- readPNG("figure/methods/in/1.png")
grid.raster(img)
```

Programmers developed a local VCS with a database that records all
modifications to the files under revision control to keep track of the
versions. A local VCS is an example of a revision control system (RCS).
Since it is difficult to collaborate with a local VCS, a centralized VCS
was developed, such as Apache Subversion (SVN). All changes to files are
monitored under the centralized server using Centralized VCS. A
centralized VCS, on the other hand, has significant drawbacks. No one
can connect or save versioned changes to something they are working on
if the server goes down for an hour. This leads to the development of
the distributed VCS, such as git. In a distributed VCS, the repository
can be completely mirrored, including the entire version history. A
schematic of distributed VCS is shown in Figure \ref{fig:mf1}, adapted
from [@chacon2021].

GitHub ([github.com](https://github.com)) a distributed version control
system, was used to manage data analysis projects and software
development projects throughout this doctoral thesis work. One downside
of using GitHub is that files greater than 50 megabytes cannot be
hosted, which means that large files such as sequencing data, FASTQ
files, and BAM files are not version managed. To solve this issue,
git-fat was used, a version-controlled tool that offloads files to a
local directory or another server [@brown2018].

## Data analysis directory organization {#m2 .unnumbered}

ScienceCloud service of the University of Zurich was used for computing.
On ScienceCloud, a data volume is mounted on my instance (virtual
computer). In the data volume, two directories were created, namely
"Data" and "Project". The "Data" repository stores raw sequencing files,
FASTQ files, and "gitfat" objects, while the Project directory stores
all analysis and code and is version controlled with git. A schematic
representation of this workflow can be seen in Figure \ref{fig:mf2}.
This is done for each dataset analysed.

(ref:mfc2) \textbf{Organization of data analysis projects.}

```{r mf2, echo=FALSE, fig.align='center', fig.cap="(ref:mfc2)", fig.scap="Organization of data analysis projects", fig.width = 5.25, fig.height=2.98}
img <- readPNG("figure/methods/in/2.png")
grid.raster(img)
```

This organization of the data analysis projects ensures open and
reproducible research. As all the analyses are version controlled, one
can look at the analysis conducted back in time. Also, each analysis
script has a log file, which stores the information of the tools version
and any warnings or errors encountered during the analyses. Further, at
the end of the project, the GitHub repository can be made public and all
the code for intermediate data analysis are available to the research
community.

## Pipelines for data analysis {#m3 .unnumbered}

I worked on multi-omics datasets for my doctoral research, which led to
the development of data analysis pipelines for RNA sequencing (RNA-seq),
Assay of Transposase Accessible Chromatin sequencing (ATAC-seq),
Whole-genome bisulfite sequencing (WGBS), Reduced representation
bisulfite sequencing (RRBS), Chromatin immunoprecipitation followed by
sequencing (ChIP-seq), and short RNA sequencing (sRNA-seq).

### Quality check and trimming of sequencing datasets {#m3.1 .unnumbered}

The goal of data quality check and trimming is to find and clean any
data quality issues in the dataset and possibly account for them. The
sequencer determines the nucleotide bases in a DNA or RNA library during
sequencing. A small sequence, known as a read, is created for each
fragment in the library, which is a sequence of nucleotides. In a single
experiment, sequencing technologies, such as Illumina, can generate a
large number of sequence reads. The first step after receiving the
sequenced data is to assess the data quality to determine the
reliability of the sequenced reads. The FASTQ files are examined using
`FastQC` [@andrews2012]. `FastQC` provides a comprehensive view of the
data, including the number of sequenced reads, base sequence quality,
"N" content (if a sequencer cannot call a base, due to the technical
limitations, it enters "N"; of each sequencing platform.), sequence
duplication levels, and adapter information. A Hypertext Markup Language
(HTML) report generated by the software, for each sample, helps to
understand the pre-processing steps required. The quality assessment is
done using the quality scores (Q-scores) in the FASTQ files, which are
translated to quality statistics in `FastQC` reports and plotted as
boxplots. Q score is defined as the base-calling error probabilities and
is calculated by the formula: `**Q = -log10 (P)**`. For instance, if a
nucleotide base is assigned a Q score of 30, this is equivalent to the
probability of an incorrect base call of 1/1000 times, which means that
the accuracy of base calling is 99.9%. A Q score of 30 represents
perfect base calling with no errors and ambiguities and is considered a
benchmark for quality in high-throughput sequencing (HTS) [@ewing1998].

(ref:mfc3)
\textbf{Workflow for quality check and quality control for sequencing data.}
QC: Quality Control.

```{r mf3, echo=FALSE, fig.align='center', fig.cap="(ref:mfc3)", fig.scap="Workflow for quality check and quality control for sequencing data", fig.width = 4, fig.height=3.9}
img <- readPNG("figure/methods/in/3.png")
grid.raster(img)
```

Adaptors are artificial DNA oligonucleotides and are required for
sequencing by platforms like Illumina. During the process of Illumina
library preparation, adaptors are ligated to the short DNA sequences.
Because of the adapters (attached at the end of reads), the sequencing Q
score is usually low, towards the end of the reads. Therefore, these are
required to be removed before downstream processing of the data.
Furthermore, the sequenced DNA reads could have trailing and leading 'N'
(if a base caller fails to call a base at a genomic location, it will
put 'N' instead of 'A', 'T', 'G' or 'C'). Also, there is a chance that
although long reads were sequenced, small reads appeared after
sequencing, or they became shorter after removal of trailing or leading
'N'. It is always better to get rid of small reads below a certain
length, as they would be mapped to multiple locations and will lead to
misinterpretation. Trimming is the process of modifying the ends of
reads. Trimming can help to increase the number of reads that the
aligner or assembler can successfully use, lowering the amount of
unmapped or unassembled reads. `TrimGalore` is used to trim adapters and
inferior ends of reads with a Phred score of less than 30 (`-q 30`), to
remove Ns from both sides of reads (`--trim-n`), and to discard trimmed
reads that are too short to be informative (e.g. `--length 30`, or `15`
for small RNA-seq data). After this, all results are used to generate an
overview report with `multiQC` [@ewels2016], which is a reporting tool
that parses summary statistics from results of other bioinformatics data
analysis tools. Finally, it is critical to recognize, identify, and rule
out issues that may affect downstream analysis interpretation.

The trimmed data is then aligned to the reference genome, indexed with
the specific tool needed for alignment. For example, for aligning WGBS
data with the reference genome, the reference genome is first indexed
with `Bismark`. This workflow is shown in Figure \ref{fig:mf3}.

Some library preparation kits use the unique molecular identifier (UMI),
which are complex indices and are added to the sequencing libraries
before the polymerase chain reaction (PCR) amplification step. UMIs were
first implemented in the iCLIP protocol [@könig2010] but has been
implemented for RNA-seq, small RNA-seq, single-cell sequencing,
ChIP-seq, and whole-genome sequencing (WGS).

### RNA sequencing {#m3.2 .unnumbered}

RNA sequencing (RNA-seq) is a technique that uses high-throughput
sequencing (HTS) to investigate the quantity and sequences of RNA. It
examines the transcriptome to determine, which of our DNA-encoded genes
are turned on or off, and to what degree. RNA-seq pipeline schematic is
shown in Figure \ref{fig:mf4}.

(ref:mfc4) \textbf{Workflow for RNA sequencing data analysis.}

```{r mf4, echo=FALSE, fig.align='center', fig.cap="(ref:mfc4)", fig.scap="Workflow for RNA sequencing data analysis", fig.width = 6.5, fig.height=3.5}
img <- readPNG("figure/methods/in/4.png")
grid.raster(img)
```

#### Pseudo-alignment and alignment {#m3.2.1 .unnumbered}

Sequence alignment is used to determine where the sequences are similar
to the reference genome and how similar they are. We can estimate where
a read came from by aligning or "mapping" it to a reference genome or
transcriptome. There are two families of methods for mapping reads in
RNA-seq data: alignment and pseudo-alignment. For alignment, aligners
such as `STAR` or `Rsubread` are used. They align reads to a genome or
transcriptome and perform spliced alignment. They return the reads'
base-level alignments. Transcript quantification tools such as `Salmon`
[@patro2017] and `Kallisto` [@bray2016] instead perform
pseudo-alignment. These tools use statistical inference to determine
transcript abundances by mapping reads to the transcriptome. Alignments
from the aligners can then be used for a variety of purposes, such as
for feeding transcript assembly tools, for variant calling pipelines, or
for transcript quantification tools.

Quality-controlled reads were pseudo-aligned using `Salmon` with
automatic detection of the library type (`-l A`), correcting for
sequence-specific bias (`--seqBias`), and correcting for fragment GC
bias correction (`--gcBias`) on a `transcript index` prepared from
GENCODE, with additional piRNA precursors and transposable elements
(concatenated by family) from Repeat Masker as in [@gapp2020].
Downstream analyses including differential analysis; differential
transcript usage and differential isoform switching; and functional
analyses, are performed on the pseudo-alignment from `Salmon`.

Additionally, alignment is performed using the `Rsubread` package
[@liao2019] using the `subjunc()` function, specific for RNA-seq data.
Analysis of duplicates, differential exon usage, and differential 3'
usage are performed on alignment obtained from `Rsubread`.

A direct comparison of genome alignment with transcriptome
pseudo-alignment was performed in [@yi2018] and the authors found that
both approaches produce similar quantifications. However, [@wu2018]
suggested that it is not optimal to use alignment-free methods to
analyze and quantify lowly expressed genes and small RNAs.

#### Duplicated reads analysis {#m3.2.2 .unnumbered}

Duplicated reads are different copies of the exact same sequence. Unless
a genomic region is highly expressed, most mRNA fragments are expected
to be unique. A low degree of repetition may indicate that the target
sequence is well-covered, whereas a high level of duplication is more
likely to signal bias. Duplicate reads might be caused by PCR
duplication or legitimately overrepresented sequences. Because PCR
amplification is more efficient for some sequences than others, it can
misrepresent the true proportion of sequences in the input, whereas
truly overrepresented sequences are the result of very abundant
transcripts in an RNA-Seq library and are an expected case and not of
concern because they accurately represent the input.

Throughout the research, duplicated reads were marked in the aligned
data using `sambamba` [@tarasov2015], and further, the quality issues
with PCR duplicates are analyzed using `dupRadar` [@sayols2016].
`dupRadar` aids in separating the fraction of readings produced by
artefacts from the fraction resulting from high expression.

#### Differential expression analysis {#m3.2.3 .unnumbered}

Differential expression analysis (at the transcript or gene level) is
the statistical examination of normalised read count data to find
quantitative differences in expression levels between experimental
groups.

Read counts at the transcript level are obtained from pseudo-aligned
data and are aggregated at the gene level. Low counts of candidate
genes/transcripts across samples give minimal evidence for differential
expression. When evaluating false discovery rates, they also add to the
multiple testing burden, lowering the ability to find differentially
expressed genes. A pre-filtering of counts is therefore performed using
the `filterByExpr()` function from `edgeR` [@robinson2010] with a design
matrix (matrix containing data about multiple characteristics of several
samples, such as biological group, age, and batch for library
preparation) and requiring at least 20 counts (min.counts = 20)
[@germain2020]. Next, the counts are normalized for different sequencing
depths between samples and to eliminate composition biases between
samples, for example, if there are a few highly expressed genes
dominating in some samples, leading to fewer reads from other genes.
Normalization factors are obtained using the `TMM` normalization method
[@robinson2010] from the `edgeR` package, using the calcNormFactors
function. `TMM` stands for Trimmed Mean of M values, in which the counts
for the samples are scaled using a weighted trimmed mean of the log
expression ratios. Differential expression analysis at gene and
transcript level is performed using limma-voom [@law2014] pipeline from
limma [@ritchie2015] package.

#### Exploratory data analysis {#m3.2.4 .unnumbered}

Exploratory data analysis of counts (using principal component analysis,
PCA), differential analysis results (volcano plot, MA plot, and
heatmap), and assessment of library composition is performed using
`plgINS`, a versatile `R` package,
([github.com/ETHZ-INS/plgINS](https://github.com/ETHZ-INS/plgINS)),
under development by Dr. Pierre-Luc Germain. The package also comes with
a shiny application.

#### Differential transcript usage and differential isoform switching {#m3.2.5 .unnumbered}

Differential transcript usage (DTU) analysis looks for proportional
differences in the makeup of a gene's transcripts between conditions.

DTU is performed using `IsoformSwitchAnalyzeR` [@vitting-seerup2017;
@vitting-seerup2019]. Isoforms are annotated via integration of a wide
range of (predicted) annotations: Pfam was used for prediction of
protein domains, CPC2 (version 2.0) for calculation of the coding
potential, SignalP (version 5.0) for prediction of Signal Peptides, and
IUPred2A was used to predict Intrinsically Disordered Region (IDRs) and
Intrinsically Disordered Binding Regions (IDBRs). The results from Pfam,
CPC2, IUPred2A, and SignalP are used with `IsoformSwitchAnalyzeR` for
annotation and prediction of functional consequences for the identified
isoform changes.

#### Differential exon usage and differential 3' untranslated regions usage {#m3.2.6 .unnumbered}

Differential exon usage (DEU) analysis looks for variations in exon
usage between experimental conditions. mRNAs' 3' untranslated regions
(3' UTRs) are best recognized for regulating mRNA-based functions such
as mRNA localization, stability, and translation [@mayr2019].
Differential UTR usage looks for variations in 3' UTR usage between
experimental conditions.

DEU and differential 3' UTR usage are performed using the diffUTR R
package [@gerber2021].

#### Functional analysis {#m3.2.7 .unnumbered}

The output of RNA-seq differential expression analysis is a list of
significant differentially expressed genes (DEGs). The goal of
functional analysis is to provide biological insight of DEGs. Functional
analysis on Kyoto Encyclopedia of Genes and Genomes (KEGG)
[@kanehisa2021], Reactome [@jassal2020], Gene Ontology (GO)
[@consortium2021] and curated pathway list from the Gene Set Enrichment
Analysis (GSEA) [@subramanian2005] databases is performed using
multiGSEA
([github.com/lianos/multiGSEA](https://github.com/lianos/multiGSEA)),
which facilitates the analysis with GOseq [@young2010], fGSEA
[@korotkevich2016], Correlation Adjusted MEan RAnk gene set test
(CAMERA) and camera pre-ranked (CAMERA-PR) [@wu2012], ROtation testing
using MEan Ranks (ROMER) and fry [@ritchie2015], and ROtAtion gene Set
Testing (ROAST) [@wu2010]. An advantage of using multiGSEA is that it
has standardized input and output and provides a Shiny app for
exploratory data analysis
([github.com/lianos/multiGSEA.shiny](https://github.com/lianos/multiGSEA.shiny)).
With multiGSEA, it is easy to specify to use testing relative to a
threshold (TREAT) method [@mccarthy2009], which takes a user-specified
log fold change cut-off and recalculates the moderated t-statistics and
p-values. Using TREAT only affects enrichment tests that first threshold
the genes in the experiment as "significant" or not, like GOseq and not
tests like the camera.

### Assay of Transposase Accessible Chromatin sequencing {#m3.3 .unnumbered}

The Assay of Transposase Accessible Chromatin sequencing (ATAC-seq) is
widely used in studying chromatin biology to study chromatin
accessibility and to identify open chromatin regions (OCRs). The
nucleosome is the most basic component of chromatin. Each nucleosome is
made up of about two turns of DNA wrapped around a set of eight proteins
called histones known as histone octamer. With the help of nucleosomes,
the genome is securely packed and structured, known as chromatin. The
organization and accessibility of DNA are influenced by a number of
factors, including chromatin structure, nucleosome location,
transcription factors, and histone modifications. As a result, these
factors have a role in the activation and inactivation of genes. The
genome is treated with a hyperactive version of the Tn5 transposase to
insert sequencing adapters into OCRs [@buenrostro2015]. Because it is
easier, faster, and uses fewer cells than competing approaches like
FAIRE-Seq and DNase-Seq, ATAC-Seq has become popular for discovering
accessible regions of the genome. The schematic of the ATAC-seq pipeline
is shown in Figure \ref{fig:mf5}.

(ref:mfc5) \textbf{Workflow for ATAC sequencing data analysis.} QC:
Quality Control, NFF: Nucleosome Free Fragments.

```{r mf5, echo=FALSE, fig.align='center', fig.cap="(ref:mfc5)", fig.scap = "Workflow for ATAC sequencing data analysis", fig.width = 3.55, fig.height=7}
img <- readPNG("figure/methods/in/5.png")
grid.raster(img)
```

#### Alignment, post-alignment quality control, and peak calling {#m3.3.1 .unnumbered}

Alignment of the ATAC-seq data is performed using `Bowtie2`
[@langmead2012] with the following parameters: fragments up to 2 kb were
allowed to align (-X 2000), entire read alignment (--end-to-end),
suppressing unpaired alignments for paired reads (--no-mixed),
suppressing discordant alignments for paired reads (--no-discordant) and
minimum acceptable alignment score with respect to the read length
(--score-min L,-0.4,-0.4). Tn5 inserts adapters separated by 9bp when it
cuts an accessible chromatin region [@kia2017]. This indicates that
reads on the positive strand should be adjusted 4 bp to the right and
reads on the negative strands should be shifted 5 bp to the left in
order for the read start site to reflect the centre of where Tn5 binds
as done in [@buenrostro2013], which is important for the footprint
analysis. Using alignmentSieve from deepTools [@ramirez2016], aligned
data (BAM files) are adjusted for the read start sites to represent the
centre of the transposon cutting event (--ATACshift), and filtered for
reads with a high mapping quality (--minMappingQuality 30). In addition,
because the mitochondrial genome is devoid of chromatin packaging
[@bogenhagen2012] a large number of mitochondrial reads can be a concern
in ATAC-seq. Some ATAC-Seq samples have been found to include 80%
mitochondrial reads, prompting the development of wet-lab approaches to
address the problem [@corces2017]. Also, because the open chromatin
regions of interest are typically found in the nuclear genome,
mitochondrial reads are typically excluded from the analysis. Hence,
reads mapping to the mitochondrial chromosome and ENCODE blacklisted
regions (regions where genome assembly results in erroneous signal) were
filtered using alignmentSieve. Because ATAC-seq does not require
rigorous size selection during library preparation, it can identify
nucleosome positions using fragments representing nucleosome-free
fragments (NFFs) nucleosome monomers, and multimers. There is
approximately 147 bp of DNA wrapped around a nucleosome, and in order to
obtain NFFs, which are indicative of transcription factor bindings, the
fragments less than 147 bp must be extracted. To extract NFFs, all
aligned files are merged within experimental groups, reads are sorted by
left-most coordinates and indexed using SAMtools [@li2009], and NFFs
were obtained by selecting alignments with a template length between 40
and 140 inclusively [@jung2017; @goodnight2019]. Peak calling is a
statistical approach that leverages data coverage properties to identify
locations that are enriched as a result of protein/ transcription factor
binding. To find regions corresponding to potential OCRs in the
nucleosome-free regions, we want to identify regions where reads have
piled up greater than the background read coverage. Peak calling on the
NFFs is performed using MACS2 [@zhang2008] with genome size in base-pair
(-g genomeSize) and specifying the paired-end BAM file format (-f
BAMPE).

#### Peaks annotation and differential accessibility analysis {#m3.3.2 .unnumbered}

The goal of peak annotation is to map peaks to the regulatory elements.
The peaks are annotated with transcript, and the distance to the nearest
transcription start site, based on overlap with gene transfer format
(GTF) file obtained from the GENCODE. The number of extended reads
overlapping in the peak regions is calculated using the csaw package
[@lun2015]. Peak regions that did not have at least 15 reads in at least
40% of the samples were filtered out. Normalization factors were
obtained on the filtered peak regions using the `TMM` normalization
method [@robinson2010] and differential accessibility analysis is
performed using the Genewise Negative Binomial Generalized Linear Models
with Quasi-likelihood (glmQLFit) test from the `edgeR` package. `edgeR`
has recently been benchmarked and recommended for differential chromatin
accessibility analysis [@gontarz2020].

#### Functional analysis {#m3.3.3 .unnumbered}

GO analysis is performed on differentially accessible regions (DARs)
with the rGREAT package
([github.com/jokergoo/rGREAT](https://github.com/jokergoo/rGREAT)),
which is a wrapper around the Genomic Regions Enrichment of Annotations
Tool (GREAT) [@mclean2010]. Transcription factor motif enrichment
analysis is performed using the marge package [@amezquita2018], which is
a wrapper around the Homer tool [@heinz2010].

#### Differential accessibility analysis at transposable elements {#m3.3.4 .unnumbered}

Transposable elements (TEs) GTF file is obtained from
[http://labshare.cshl.edu/shares/mhammelllab/www-data/TEtranscripts/TE_GTF/mm10_rmsk_TE.gtf.gz](http://labshare.cshl.edu/shares/mhammelllab/www-data/TEtranscripts/TE_GTF/mm10_rmsk_TE.gtf.gz%20on%2003.02.2020).
The GTF file provides hierarchical information about TEs: **Class**
(level 1, eg. LTR), **Family** (level 2, eg. LTR L1), **Subtype** (level
3, eg. LTR L1 L1_Rod), and **Locus** (level 4, eg. LTR L1 L1_Rod
L1_Rod_dup1). TE loci are annotated based on overlap with GENCODE as
described above for ATAC-seq peaks. Filtered BAM files (without reads
mapping to blacklisted or mitochondrial regions) are used for analyzing
TEs. Mapped reads are assigned to TEs using featureCounts from the R
package `Rsubread` [@liao2019] and were summarized to Subtypes (level
3), allowing for multi-overlap with fractional counts, while ignoring
duplicates. The number of extended reads overlapping at the TE loci is
obtained using the csaw package [@lun2015]. Subtypes that did not have
at least 15 reads, and loci that did not have at least 5 reads in at
least 40% of the samples, were filtered out. Normalization and
differential accessibility analysis is performed as described above for
the peaks. GO and motif enrichment analysis is performed as described
above for the peak regions.

### Chromatin immunoprecipitation followed by sequencing {#m3.4 .unnumbered}

A substantial part of gene expression regulation is controlled by
protein-DNA interactions. Proteins like transcription factors and
histones influence how much and where genes are expressed. ChIP,
followed by sequencing, can be used to investigate the interaction
between a DNA sequence and a protein. There are two types of ChIP:
cross-linked ChIP (X-ChIP) and native ChIP (N-ChIP). Formaldehyde is
utilized to crosslink histones to DNA, and ultrasound sonication is used
to fragment the DNA in X-ChIP, whereas the native covalent interaction
between protein and DNA is exploited in N-ChIP [@park2009]. The
fundamental purpose of ChIP is to map global binding sites for any
protein of interest, such as transcription factors (TF) and histones, in
order to investigate specific modifications. In vivo, the approach
generates a library of DNA locations bound to the protein of interest.
Following ChIP on samples, library preparation, PCR amplification, and
ultimately deep sequencing are performed. The schematic of the ChIP-seq
pipeline is shown in Figure \ref{fig:mf6}.

(ref:mfc6) \textbf{Workflow for ChIP sequencing data analysis.} QC:
Quality Control.

```{r mf6, echo=FALSE, fig.align='center', fig.cap="(ref:mfc6)", fig.scap="Workflow for ChIP sequencing data analysis", fig.width = 4, fig.height=6}
img <- readPNG("figure/methods/in/6.png")
grid.raster(img)
```

#### Alignment and peak calling {#m3.4.1 .unnumbered}

Alignment is performed using `Bowtie2`. Reads with more than 3
mismatches are removed from the aligned data, as suggested in
[@royo2016], and reads with low mapping quality (--minMappingQuality 30)
or mapping to the mitochondrial chromosome or aforementioned blacklisted
regions are also filtered out. Peak calling is performed using MACS2
with mouse genome size in base-pair (-g genomeSize) and the BAM file
format (-f BAM/PEBAM), together with the ChIP input sample.

#### Peaks annotation, differential analysis, and functional analysis {#m3.4.2 .unnumbered}

These analyses are performed as described above for the ATAC-seq peaks.

### Bisulfite sequencing {#m3.5 .unnumbered}

DNA methylation (DNAme) involves the addition of methyl groups to the
DNA molecule. It most normally occurs at the C5 position of cytosines
inside CpG dinucleotides, where it is stable during mitosis and meiosis.
Methylation can change the activity of DNA without changing its
sequence. DNAme is a reversible, cell-type-specific DNA modification
that is usually persistent through cell division. It is not equally
distributed across the genome, but rather is linked to CpG density.
High-throughput bisulfite sequencing is one of the most reliable methods
for measuring DNA methylation. In bisulfite conversion, unmethylated
cytosine residues are converted to uracil, but methylated cytosines
remain intact [@frommer1992], allowing methylation and unmethylated
cytosine bases to be distinguished. Following PCR amplification, these
uracils are converted to thymines. This is followed by the WGBS. WGBS is
considered a gold-standard technology for DNA methylation detection as
it provides a qualitative, quantitative, and efficient technique to
identify 5-methylcytosine at single base-pair resolution. The cost of
sequencing an entire genome is typically quite high. As a result,
despite its use on huge genomes like the human genome, significant
numbers of individual samples are rarely sequenced. For this reason,
RRBS was developed, in which the bisulfite process occurs but only
around 1% of the genome is sequenced. This makes it possible to sequence
the genomes in greater numbers. The idea behind RRBS is to enrich the
sequencing library with CpG-dense sections of the genome to boost the
sequencing depth of cytosine-rich areas [@meissner2005]. This is done by
using CpG-specific restriction enzymes like MspI, which identifies the
sequence 5'-CCGG-3'. MspI cleaves the complete genome in RRBS, leaving
at least two cytosines per cleaved DNA fragment (one cytosine at either
end of the MspI-MspI fragment). Sequencing libraries are made from
size-selected MspI DNA fragments, followed by bisulfite conversion and
sequencing of the converted library. The schematic of the BS-seq
pipeline is shown in Figure \ref{fig:mf6}.

(ref:mfc7) \textbf{Workflow for Bisulfite sequencing data analysis.} QC:
Quality Control, DML: differentially methylated loci, DMR:
differentially methylated regions, CNV: copy-number variation.

```{r mf7, echo=FALSE, fig.align='center', fig.cap = "(ref:mfc7)", fig.scap="Workflow for Bisulfite sequencing data analysis", fig.width = 6.47, fig.height=2.83}
img <- readPNG("figure/methods/in/7.png")
grid.raster(img)
```

#### Alignment and methylation calling {#m3.5.1 .unnumbered}

Mismatches between the reads and the reference genome are introduced
during bisulfite conversion, resulting in slow and imprecise mapping.
Further, the DNA code's complexity is reduced. Hence, specialized tools
are required for aligning the bisulfite sequencing data. Alignment of
the bisulfite sequencing data is performed using `Bismark`
[@krueger2011] on a genome index built using
`Bismark`\_genome_preparation. Methylation information for individual
cytosines (methylation calls) is extracted using the
`Bismark`\_methylation_extractor tool from the `Bismark` package.

#### Differential methylation analysis {#m3.5.2 .unnumbered}

Differential analysis can be performed at the loci level (differentially
methylated loci, DML) or at the region's level (differentially
methylated regions, DMRs). For the identification of DML and DMRs, the
data is first filtered by the coverage among samples. DML are identified
using `edgeR`. Identification of DMRs is not so straightforward, as
defining regions boundaries is not easy the tool has to take into
account correlation across loci of a region. Hence, during a course, we
benchmarked different tools for detecting DMRs
[dktanwar.github.io/sta426-project-dmr-comparison](https://dktanwar.github.io/sta426-project-dmr-comparison/report/project_report.html)
and found dmrseq to outperform all other methods that we compared. DMRs
from RRBS data are identified using the parameters recommended by Dr.
Keegan Korthauer**,** author of dmrseq
([github.com/kdkorthauer/dmrseq/issues/14](https://github.com/kdkorthauer/dmrseq/issues/14)).

#### Copy number variation analysis {#m3.5.3 .unnumbered}

The phenomenon of copy number variation (CNV) occurs when regions of the
genome are repeated and the amount of repeats differs between
individuals. CNV is a type of structural variation that occurs when a
duplication or deletion event occurs that impacts a large number of base
pairs. To infer and visualize CNV from WGBS data, we use the CNVkit
[@talevich2016].

#### DML and DMR annotation, and functional analysis {#m3.5.4 .unnumbered}

These analyses are performed as described above for the ATAC-seq data.

### Short RNA-seq {#m3.6 .unnumbered}

The short RNA-seq pipeline is not described in this section, as a method
for analyzing small RNA-seq data has been developed and is discussed in
detail in Chapter 3.

### Multi-omics analysis {#m3.7 .unnumbered}

Using different experiments, followed by sequencing, on the same samples
or cells/tissues, such as RNA-seq, ATAC-seq, ChIP-seq, and BS-seq,
results in multi-dimensional omics datasets that can be used to
investigate relationships between different biological processes, such
as gene expression, chromatin accessibility, histone modifications, and
DNA methylation. Integrating multi-omics datasets is most challenging
and is not straightforward to perform, as there are no standard recipes.

We present a straightforward and biologically useful approach for
performing multi-omics analysis on RNA-seq, ATAC-seq, ChIP-seq, and
BS-seq datasets. First, we overlap the DARs with ChIP-seq peaks
enrichment, RNA-seq expression of overlapping genes with DARs, and
methylation values of DARs overlapping loci from BS-seq data into one
matrix, overlapMatrix. The DARs are initially divided into proximal
(less than +/- 2.5 kb from a TSS) and distal (more than +/- 2.5 kb from
a TSS) regions, as per the ENCODE guidelines [@thurman2012;
@harrow2012]; the proximal regions are further divided into active and
inactive (proximal active and proximal inactive) groups based on nearby
gene expression (if nearby gene expression is detected) (**step 1**).
Next, these three groups are separated based on whether the biological
groups are more or less accessible (**step 2**). This classification
would reveal six separate clusters of DARs, each with its own biological
profile. Further, proximally active genes are categorised based on their
up/ down expression (**step 3**). Moreover, we categorised DARs
depending on whether they had H3K4me3, which is a critical marker for
the active transcription and developmentally stable genes [@fu2019];
H3K27ac, which is associated with the higher activation of transcription
and therefore defined as an active enhancer mark [@zhang2020]; or dual
H3K4me3/H3K27ac enrichment [@beacon2021]; as well as H3K27me3
enrichment, which is involved in transcriptional silencing
[@bogliotti2012] (**step 4**). The schematic is shown in Figure
\ref{fig:mf8}. Furthermore, gene ontology and TF motif enrichment
analysis can be done on these categorised DARs. For a more in-depth
understanding, all of the regions can be further classified based on
their methylation state (high or low). Please see [Chapter 2](#chapter2)
for an application of this multi-omics analysis.

```{=tex}
\newpage
\begin{landscape}

(ref:mfc8) \textbf{Workflow for multi-omics data integration.}

```{r mf8, echo=FALSE, fig.align='center', fig.cap="(ref:mfc8)", fig.scap="Workflow for multi-omics data integration", fig.width = 9, fig.height=7}
img <- readPNG("figure/methods/in/8.png")
grid.raster(img)
```
\end{landscape}
```
Visualization of these regions is an essential step. We used
EnrichedHeatmap [@gu2018] and ComplexHeatmap [@gu2016] packages for the
visualization of categorized DARs.

**ATAC-seq:** ATAC-seq data is the starting point and we first find the
middle of each DARs. After that, we obtain the coverage for 1kb upstream
and downstream from the middle of DARs. We use the `normalizeToMatrix`
function from the EnrichedHeatmap package and normalized the signal
using the weighted mean between the intersected parts and un-intersected
parts(mean_mode = "w0") in a window of 50bp. In the normalized matrix,
each row corresponds to DAR and each column corresponds to a window
either on upstream or downstream of middle of the peak.

$$
v_{w 0}=\frac{\sum_{i=1}^{n} x_{i} w_{i}}{W+W^{\prime}}
$$

${W}$ is the sum of width of the intersected parts
$\left(\sum_{i}^{n} w_{i}\right)$ and $W^{\prime}$ is the sum of width
for the non-intersected parts.

For visualization, DARs are ordered by the enriched scores. For each DAR
in the matrix, values are denoted as ${x}$, indices as
$1, \ldots, n_{1}$ for upstream windows, indices as $n_{1}+1, \ldots, n$
for downstream windows, and $n_{2}=n-n_{1}$ are the enriched score that
are calculated as the sum of ${x}$ weighted by distance to middle of the
peak. DAR has a higher enriched score when there is more signal focused
on it.

$$
\sum_{i=1}^{n_{1}} x_{i} \cdot i / n_{1}+\sum_{i=n_{1}+1}^{n} x_{i} \cdot(n-i+1) / n_{2}
$$

Moreover, we also plot the log\textsubscript[2]fold-changes of DARs using
ComplexHeatmap.

**ChIP-seq:** ChIP-seq data coverage for 1kb upstream and downstream
from the mid of DARs is obtained. We use the `normalizeToMatrix` function
from the EnrichedHeatmap package and normalized the signal using the
weighted mean between the intersected parts and un-intersected parts
(mean_mode = "w0") in a window of 50bp.

**BS-seq:** BS-seq data methylation values for 1kb upstream and
downstream from the mid of DARs is obtained. We use the
`normalizeToMatrix` function from the EnrichedHeatmap package and
normalized the signal using the mean of all the signals (mean_mode =
"absolute") in a window of 50bp.

$$
v_{a}=\frac{\sum_{i=1}^{n} x_{i}}{n}
$$

**RNA-seq:** For RNA-seq data, we used ComplexHeatmap to plot the
log\textsubscript[2]-fold-changes of expression values between the groups, and also the
log\textsubscript[2]-fold-changes obtained after the differential analysis.

<!--chapter:end:rmd/05-methods.Rmd-->

---
output: html_document
editor_options: 
  chunk_output_type: console
bibliography: [rmd/references.bib,bib/thesis.bib]
---

# Dynamic chromatin accessibility in spermatogonial cells for transcriptional programmings from early postnatal to adult stages {#chapter1}

**Deepak K Tanwar**\textsuperscript{1,2,3,$\dagger$}, Irina
Lazar-Contes\textsuperscript{1,2,3,$\dagger$}, Pierre-Luc
Germain\textsuperscript{1,2,3,4}, Isabelle M
Mansuy\textsuperscript{1,2,3,\#}\newline \newline
\textsuperscript{1}Laboratory of Neuroepigenetics, Brain Research
Institute at the Medical Faculty of the University of Zurich. \newline
\textsuperscript{2}Institute for Neuroscience of the Department of
Health Sciences and Technology, ETH Zurich, Zurich, Switzerland.\newline
\textsuperscript{3}Zurich Neuroscience Center, ETH and University of
Zurich, Zurich, Switzerland. \newline \textsuperscript{4}Statistical
Bioinformatics Group, Swiss Institute of Bioinformatics, Zurich,
Switzerland. \newline \newline \textsuperscript{$\dagger$}Equal
contributions \newline\newline \textsuperscript{\#}Corresponding
author \newline\newline 
__Contributions:__ *I performed data analysis, generated figures, wrote computational methods and figure captions, revised manuscript.*

\newpage

## Update on the new data analysis

This project has undergone two major changes as a result of the
replacement of one of the spermatogonial cells (SCs) RNA-seq datasets by
a better one.

### Code availibility

-   BioRxiv version:
    [bioRxiv_v1](https://github.com/mansuylab/SC_postnatal_adult/tree/bioRxiv_v1)

-   This thesis version:
    [thesis_manuscript](https://github.com/mansuylab/SC_postnatal_adult/tree/thesis_manuscript)

-   New analysis after inclusion of new RNA-seq data:
    [deepak](https://github.com/mansuylab/SC_postnatal_adult/tree/deepak)

### Inclusion of RNA-seq data from the literature

Adult SCs RNA-seq data from the lab was excluded since it was obtained
from polyA RNA-seq, as opposed to pups' RNA-seq, which was Total
RNA-seq. In the preprint version of the manuscript [@lazarcontes_2020],
we used RNA-seq data generated in the lab from SCs at the developmental
stages postnatal day 8 (PND8), PND15, and postnatal week 21 (PNW21; also
termed as adults in the manuscript). The RNA-seq datasets from PND8 and
PND15 were generated using total RNA libraries, whereas the datasets
from PNW21 were generated using polyA RNA libraries. Because polyA and
total RNA-seq datasets are not directly comparable [@zhao_2018], we
obtained RNA-seq data from the PND7, PND14, and PNW8 stages from the
literature [@hammoud2014; @hammoud2015] and compared it to the lab's
RNA-seq data.

(ref:dnc1) \textbf{Log\textsubscript{2} fold-change comparison of RNA-seq data from the literature and lab (old and new).}
(ref:dnsc1) Log\textsubscript{2} fold-change comparison of RNA-seq data
from the literature and lab (old and new)

```{r dn1, echo=FALSE, fig.align='center', fig.cap="(ref:dnc1)", fig.scap="(ref:dnsc1)", fig.width=5, fig.height=4}
img <- readPNG("figure/dev/new/01.png")
grid.raster(img)
```

First, we compared the gene expression of lab PND8 samples to PND7 from
literature and lab PND15 samples to PND14 from literature and discovered
a strong correlation; however, data from PNW8 and PNW21 were not well
correlated (result not shown). Following that, we performed differential
analysis on data from the lab: PND15 vs PND8 and PNW21 vs PND15, as well
as data from the literature: PND14 and PND7, and PNW8 and PND14. We
compared the log2 fold-changes from PND14 vs PND8 with PND15 vs PND8,
and PNW8 vs PND14 with PNW21 vs PND15 after differential analysis. We
discovered that the correlation between PND14 vs PND8 and PND15 vs PND8
was 52% (R = 0.52, Pearson method) and that most of the genes were
changing in the same direction, whereas the correlation between PNW8 vs
PND14 and PNW21 vs PND15 was negative 16% (R = -0.16; Pearson method)
and that around 50% of the genes were changing in the opposite
direction, as shown in Figure \ref{fig:dn1}A. As a result, we incorporated the PND14
and PNW8 datasets into our project to better compare developmental
stages. We used differential analysis between PNW8 and PND14 rather than
PNW8 and PND15 because the PND14 and PND15 samples were sequenced
separately and could have unaccountable batch effects. These analyses
were then updated in the manuscript and appear in [Chapter
1](\#chapter1).

### New RNA-seq data from lab

Because of these limitations, we decided to generate new SCs data in the
lab. Previously, the "Takara total RNA pico input mammalian kit v2" was
used to generate libraries, but this kit is no longer available. The new
RNA-seq dataset was obtained from the total RNA libraries prepared using
the "Takara total RNA pico input mammalian kit v3." Furthermore, PND8
and PND15 libraries were previously sequenced on the Illumina HiSeq 4000
platform, which is no longer available for sequencing at the Functional
Genomics Center Zurich (FGCZ). To avoid any technical/ batch effects, we
therefore re-sequenced the total RNA libraries from PND8 and PND15 on
the Illumina NovaSeq platform using the "Takara total RNA pico input
mammalian kit v3" along with the PNW21 total RNA libraries.

First, we compared the newly sequenced data to data generated previously
in the lab and from the literature. We discovered that gene expression
from newly generated PND8 data was highly correlated with data generated
in the lab previously and data from the literature. The data from the
PNW21 stage is also well correlated with the literature (result not
shown).

(ref:dnc2) \textbf{Volcano plot of differentially expressed genes (adjusted \textit{P} $\leq$ 0.05 and absolute Log\textsubscript{2}FC $\geq$ 1)}.\newline
\textbf{A:} PND15 and PND8. \newline \textbf{B:} PNW21 and PND15.

```{r dn2, echo=FALSE, fig.align='center', fig.cap="(ref:dnc2)", fig.scap="Volcano plot of differentially expressed genes", fig.width=6.4, fig.height=2.6}
img <- readPNG("figure/dev/new/02.png")
grid.raster(img)
```

We processed the RNA-seq data using the methods described in [Chapter
1](\#chapter1)
and [Methods](#methods). Following that, we conducted a differential analysis
between PND15 and PND8, as well as PNW21 and PND15. Figure \ref{fig:dn2} shows that
we found 663 genes that were differentially expressed between PND15 and
PND8, and 2,493 genes that were differentially expressed between PNW21
and PND15.

We compared log2 fold-changes from new data PND15 vs PND8 to literature
data PND14 vs PND8, and new data PNW21 vs PND15 to literature data PNW8
vs PND14. We discovered that the correlation between PND14 vs PND8 and
PND15 vs PND8 was 43% (R = 0.43, Pearson method) and that most of the
genes were changing in the same direction, and that the correlation
between PNW8 vs PND14 and PNW21 vs PND15 was 70% (R = 0.7; Pearson
method) and that most of the genes were changing in the same direction,
as shown in Figure \ref{fig:dn1}C, \ref{fig:dn1}D. Furthermore, we compared the lab's old data
with the new data and discovered an 87% correlation (R = 0.87; Pearson
method) and nearly all the genes changing in the same direction for
PND15 vs PND8, whereas the correlation for PNW21 vs PND15 was 6% (R =
0.06; Pearson method) and nearly 50% genes changing in the opposite
direction, Figure \ref{fig:dn1}E, \ref{fig:dn1}F. This is due to the fact that the previous
PNW21 data were obtained from ployA libraries, whereas the new PNW21
data were obtained from total RNA libraries.

#### Identification of clusters of genes with distinct expression profiles

We discovered five clusters of genes that were differentially expressed.
We have used the union of differentially expressed genes between PND15
and PND8, as well as PNW21 and PND15, and ran k-means clustering (k = 2
to 10) on the log2 fold-changes of normalized expression values, using
PND8 samples as the baseline. Based on visual inspection, we select 7
clusters (k = 7) and merge the clusters with visually observed similar
expression profiles to obtain 5 clusters, Figure \ref{fig:dn3}.

(ref:dnc3) \textbf{K-means clustering (k = 5) of genes with differential expression in postnatal versus adult spermatogonial cells.}
Each row depicts a gene and each column depicts a sample. Gene
expression fold-change is calculated by subtracting mean
log\textsubscript{2} counts per million (CPM) of PND8 from PND15 and
PNW21. Samples are clustered using Ward's method and genes are ordered
by the "MDS_angle method" using `seriation` (R package).

```{r dn3, echo=FALSE, fig.align='center', fig.cap="(ref:dnc3)", fig.scap="K-means clustering of differentiallly expressed genes", fig.width=4.5, fig.height=4}
img <- readPNG("figure/dev/new/03.png")
grid.raster(img)
```

#### Multi-omics data integration

After combining data from ATAC-seq, RNA-seq, and ChIP-seq, we were able
to generate clusters of distinct genome activity. We combined these
datasets, with ATAC-seq as the starting point, [Methods](#methods). We classified
the differentially accessible regions (DARs) based on ATAC-seq and
RNA-seq data into distal (\> 2.5 kb of the transcription start site,
TSS) and proximal (2.5kb of TSS). Following that, distal DARs were
divided into two groups based on increased or decreased accessibility in
PNW21, Figure \ref{fig:dn4},\ref{fig:dn6}, while proximal DARs were divided into six groups
based on increased or decreased accessibility in PNW21 and up or
down-regulation of nearby gene expression in PNW21 Figure \ref{fig:dn5}, \ref{fig:dn7}.
Category 1: increased chromatin accessibility and upregulation of
neighbouring genes; Category 2: increased chromatin accessibility and
downregulation of neighbouring genes Category 3: decreased chromatin
accessibility and downregulation of neighbouring genes, Category 4:
decreased chromatin accessibility and upregulation of neighbouring
genes, Category 5: increased chromatin accessibility with no detectable
gene expression, and Category 6: decreased chromatin accessibility with
no detectable gene expression. All such clusters had also been
identified in the previous analysis (see [Chapter
1](\#chapter1) and
[@lazarcontes_2020], however one of the original categories (Category 5
from Figure \ref{fig:dn5}) disappeared in the new analysis. This category is related
to TSS-proximal DARs which, against expectations, increase in expression
despite a decrease in accessibility. Since expression implies promoter
accessibility, it is safe to assume that this category was attributable
to lack of comparability of the two RNAseq protocols.

To conclude, the results obtained from the newly generated RNA-seq data
from the lab are quite similar to the results generated previously We
have obtained similar results in terms of differential analysis and
multi-omics data integrations. However, a number of analyses are still
to be performed, for instance, gene-ontology analysis on differentially
expressed genes, identified clusters of gene expression, and clusters
obtained from multi-omics data integration; transcription factor motif
analysis of clusters obtained from multi-omics data integration; and
revision of figures and manuscript.

Figures \ref{fig:dn4}, \ref{fig:dn5}, \ref{fig:dn6}, \ref{fig:dn7} are enriched heatmaps that show different clusters for
distal and proximal regions. Each row represents a peak region, which is
ordered by the ATAC signal. The mid-x-axis corresponds to the centre of
a peak region and is extended to +/1 kbp. The ATAC heatmap's colour key
represents the signal. For RNA-seq, the fold-change in gene expression
is calculated by subtracting the mean log\textsubscript{2} CPM of PND8
from all samples.

(ref:dnc4) \textbf{Enriched heatmap of distal differentially accessible regions.}

```{r dn4, echo=FALSE, fig.align='center', fig.cap="(ref:dnc4)", fig.scap="Enriched heatmap of distal DARs", fig.width=6.5, fig.height=8}
img <- readPNG("figure/dev/new/04.png")
grid.raster(img)
```

(ref:dnc5) \textbf{Enriched heatmap of proximal differentially accessible regions.}

```{r dn5, echo=FALSE, fig.align='center', fig.cap="(ref:dnc5)", fig.scap="Enriched heatmap of proximal DARs", fig.width=6.5, fig.height=8}
img <- readPNG("figure/dev/new/05.png")
grid.raster(img)
```

(ref:dnc6) \textbf{Enriched heatmap of distal differentially accessible regions with ChIP-seq and BS-seq data.}

```{r dn6, echo=FALSE, fig.align='center', fig.cap="(ref:dnc6)", fig.scap="Enriched heatmap of distal DARs with ChIP-seq and BS-seq data", fig.width=6.5, fig.height=8}
img <- readPNG("figure/dev/new/06.png")
grid.raster(img)
```

(ref:dnc7) \textbf{Enriched heatmap of proximal differentially accessible regions with ChIP-seq and BS-seq data.}

```{r dn7, echo=FALSE, fig.align='center', fig.cap="(ref:dnc7)", fig.scap="Enriched heatmap of proximal DARs with ChIP-seq and BS-seq data", fig.width=6.5, fig.height=8}
img <- readPNG("figure/dev/new/07.png")
grid.raster(img)
```

<!--chapter:end:rmd/06-dev/1.Rmd-->

---
output: html_document
editor_options: 
  chunk_output_type: console
bibliography: rmd/references.bib
---

## Graphical Abstract

```{r dga, echo=FALSE, fig.align='center', fig.cap="Graphical abstract.", fig.scap="Graphical abstract"}
img <- readPNG("figure/dev/ga.png")
grid.raster(img)
```

\newpage

## Abstract

Stem cells have the capacity to self-renew and differentiate to give
rise to tissues. In mammals, spermatogonial stem cells are unique
undifferentiated cells in the male germline that are present throughout
life and produce functional sperm. They have a remarkable dynamic
transcriptome between postnatal life and adulthood, thought to allow
adaptation to the maturing gonadal environment. To assess if chromatin
remodelling is associated with this dynamic, we characterized chromatin
accessibility in postnatal and adult spermatogonial cells in mice using
ATAC-seq. Extensive changes in chromatin accessibility were observed in
spermatogonial cells across postnatal development, that correlate with
distinct gene expression profiles and transcription factor motif
enrichment. We identify genomic regions with differential chromatin
accessibility in adult spermatogonial cells, that are marked by distinct
histone modifications and are situated in proximity to the transcription
start site of genes important for cell maintenance and proliferation.
Some of these regions correspond to transposable element subtypes
enriched in multiple transcription factor motifs and with increased
transcription. Together, our results reveal profiles of chromatin
organization, histone modifications and gene expression in
spermatogonial cells, and underscore the dynamic nature of the germline
genome.

\newpage

## Introduction

Spermatogonial cells are cells in male gonads that actively self-renew
and differentiate into spermatogenic cells to produce adult gametes. In
mice, they become active one to two days after birth, when they exit
mitotic arrest and start dividing, to populate the basement membrane of
seminiferous tubules. During the first week of postnatal life, a
population of spermatogonial cells continues to proliferate and gives
rise to undifferentiated Asingle (As), Apaired (Apr) and Aaligned (Aal)
cells. The remaining spermatogonia differentiate to form chains of
daughter cells that become primary and secondary spermatocytes around
postnatal day (PND) 10 to 12. Spermatocytes undergo meiosis and give
rise to haploid spermatids that develop into spermatozoa. Spermatozoa
are then released in the lumen of the seminiferous tubules and continue
to mature in the epididymis until becoming capable of fertilization by
PND 42-48 [@thebiol2017; @rooij2017; @kubota2018].

Recent work using population or single-cell RNA sequencing (RNA-seq)
showed that distinct transcriptional profiles characterize
spermatogonial cells in postnatal and adult life [@hammoud2014;
@hammoud2015; @hermann2018; @law2019; @green2018]. During the first week
after birth, spermatogonia have unique features necessary for their
rapid establishment and expansion along the basement membrane. This
includes high expression of genes involved in cell cycle regulation,
stem cell proliferation, transcription and RNA [@grive2019]. In
comparison, in the adult testis, genes are involved in the maintenance
of a steady spermatogonial cells population, with a balance between
proliferation and differentiation to ensure sperm formation
predominates. This includes pathways related to paracrine signalling and
niche communication, as well as mitochondrial function and oxidative
phosphorylation [@hermann2018; @grive2019]. Epigenetic changes such as
histone tail posttranslational modifications and DNA methylation
accompany transcriptional differences in spermatogonial cells across
postnatal stages [@hammoud2014; @hammoud2015]. However, little is known
about the accessible chromatin landscape, during the transition from
postnatal to the adult stage.

We examined chromatin reorganization in the transition from postnatal to
adult spermatogonia and its functional relevance by profiling chromatin
accessibility by ATAC-seq (assay for transposase-accessible chromatin
with high-throughput sequencing) using an Omni-ATAC protocol
[@corces2017] and integrating data with transcriptomic and epigenetic
profiles. The results show that chromatin is extensively remodelled in
spermatogonial cells during development, and its accessibility is
increased at several genomic regions particularly transposable elements
(TEs), that correlate with transcriptional and epigenetic signatures.

## Results

### Enrichment of spermatogonial cells from postnatal and adult mouse testis

We collected testes from mouse pups at postnatal days (PND) 8 and 15 and
from males at postnatal week (PNW) 20, and prepared cell suspensions by
enzymatic digestion. Spermatogonial cells were enriched by
fluorescence-activated cell sorting (FACS) using surface markers (Supp. Figure \ref{fig:ds1a}A) [@kubota2004a]. Immunocytochemistry using PLZF, a well-established
marker of undifferentiated spermatogonia [@costoya2004], confirmed that
our preparation was enriched in spermatogonial cells, with 85-95% PLZF+
cells after FACS compared to 3-6% PLZF+ cells before (Supp. Figure \ref{fig:ds1a}B).
Transcriptomic analyses by RNA-seq validated the spermatogonial identity
of the cells, showing high expression of stem cell and undifferentiated
spermatogonial markers, and low expression of somatic cells (Leydig and
Sertoli cells) markers both at PND 8 and 15 (Supp. Figure \ref{fig:ds1b}C).

### Chromatin is remodelled in spermatogonial cells during the development

We profiled chromatin accessibility in postnatal and adult
spermatogonial cells using an Omni-ATAC protocol [@corces2017].
Omni-ATAC has a higher signal-to-noise ratio than classical ATAC-seq and
can be used with low input material (a few hundred cells). Accessible
regions in the genome were identified by peak-calling on merged
nucleosome-free fragments (NFF). Following the removal of lowly enriched
regions, 158,978 regions were selected for downstream analyses (see
Methods section for details). Most Tn5-accessible regions are intergenic
(38%) and located in gene bodies (33%) or in the proximity of a
transcription start site (TSS) (28% +/-1 kb from TSS) (Supp. Figure \ref{fig:ds2a}A). 3212
differentially accessible regions were identified between PND15 and
adult spermatogonia with the majority showing a gain in accessibility in
adult cells (Figure \ref{fig:df1a}A and [Table S1](#st)). Regions of differential
accessibility are predominantly intergenic (45%) and intronic (34%), and
15% of all differentially accessible regions are +/- 1kb from a TSS
(Figure \ref{fig:df1a}B). Gene ontology (GO) analysis, showed that regions with
increased accessibility in adult spermatogonia are associated with cell
fate and stem cell population maintenance, protein metabolism and RNA
metabolic processes (Figure \ref{fig:df1b}C and [Table S1](#st)). Separate GO analyses
depending on genomic location showed that regions located in gene bodies
(mainly introns) are enriched for terms related to reproduction and
protein metabolism, whilst regions close to or overlapping with a TSS
relate to cell fate specification and tissue morphogenesis (Supp. Figure \ref{fig:ds2b}B and
[Table S1](#st)). Regions with decreased accessibility in adult spermatogonia
are predominantly located in intergenic regions and associated with
multiple terms related to embryonic development (Figure \ref{fig:df1b}C and [Table S1](#st)).

### Differentially accessible chromatin regions associated with distinct gene expression dynamics

To assess the relevance of changes in chromatin accessibility, we first
integrated our ATAC-Seq data with transcriptomic datasets and examined
the correlation between chromatin states and transcriptome. We conducted
RNA-seq analyses on postnatal spermatogonial cells and used published
data from Thy1+ adult spermatogonial cells. We found that 719 genes are
differentially expressed in spermatogonial cells between PND8 and PND15,
with 515 being upregulated and 212 downregulated (Figure \ref{fig:df2a}A and [Table S2](#st)).
Fast Gene Set Enrichment Analysis (FGSEA) showed that pathways related
to RNA processing and splicing, cell cycle, redox homeostasis and
protein catabolism are downregulated between PND8 and PND15 while
pathways associated with cellular transport, exocytosis and signal
transduction are upregulated (Figure \ref{fig:df2a}B and [Table S2](#st)). Comparable analyses
using published RNA-seq data from PND14 and adult spermatogonia cells
[@hammoud2014; @hammoud2015] similarly identified transcriptional
changes between postnatal and adult stages (Fig. S?A and [Table S2](#st)) and
downregulation of pathways related to RNA processing, ribosome
biogenesis and cell cycle in adults compared to postnatal cells.
Pathways related to developmental programs and mitochondrial functions
were also downregulated and those related to spermatogenesis or
involving cytokine signalling were upregulated (Fig. S? and [Table S2](#st)).

To integrate chromatin accessibility and transcriptomic datasets, we
first divided differentially-accessible regions into proximal (situated
less than +/- 2.5 kb from a TSS) and distal (situated more than +/- 2.5
kb from a TSS) following ENCODE practice [@harrow2012; @ausers2011;
@thurman2012]. For proximal regions, we further defined 6 different
categories based on the change in expression of the nearest gene. The
first two include proximal regions with increased chromatin
accessibility and upregulated (Category 1) or downregulated (Category 2)
expression of the nearest gene. Category 3 and 4 include regions of less
chromatin accessibility with the nearest gene either downregulated or
upregulated, respectively. Category 5 and 6 include regions with
increased or decreased chromatin accessibility respectively, whose
nearest gene is not expressed in spermatogonial cells (Figure \ref{fig:df3a}A and [Table S3](#st))).

### Differentially accessible chromatin regions associated with distinct epigenetic profiles

Next, we examined the relationship between chromatin accessibility and
epigenetic marks using published ChIP-seq and bisulfite sequencing (BS)
datasets from Thy1+ spermatogonia [@hammoud2014; @hammoud2015]. Profiles
of histone marks including H3K4me3, H3K27ac and H3K27me3 in adult
spermatogonial cells, and DNA methylation in PND7, PND14 and adult
spermatogonial cells were used. .....analyses showed that for a subset
of regions in Category 1, there is an overlap with active H3K4me3,
H3K27ac or dual H3K4me3/K27ac modifications, and an overall lack of
H3K27me3 (Figure \ref{fig:df3a}B and [Table S3](#st)). Notably, several of the genes in
Category 1 with chromatin opening marked by histone modification(s) are
known regulators of stem cell potency. Pdpk1 promoter region was marked
by dual H3K4me3/K27ac, while Pdpk1 mRNA was upregulated in adult
spermatogonia (Supp. Figure \ref{fig:ds3}A). Pdpk1 (phosphoinositide-dependent protein
kinase 1) is a glycolysis factor important for stem cell self-renewal
[@kanatsu-shinohara2016; @chen2020]. In contrast, the Gata2 promoter
region was marked by the bivalent H3K4me3/K27me3 mark, while Gata2
expression showed an upregulation across testis maturation (Supp. Figure \ref{fig:ds3}).
Gata2 (GATA-binding factor 2) is a known target of NANOS2, an essential
regulator of spermatogonial stem cell potential [@barrios2010;
@sada2009]. Other exemplary genes in Category 1 include pyruvate
cellular carriers Slc25a18, Slc23a1 and Slc2a5, suggesting differences
in glycolysis regulation in adult spermatogonial cells ([Table S3](#st)).
Notably, we found increased chromatin accessibility at the TSS of GDNF
receptor Gfra2 (Figure \ref{fig:df3b}C). At the mRNA level, Gfra2displayed a marked
upregulation in adult spermatogonial cells, indicating an increased
utilization of GFRA2 receptors in adult spermatogonial cells compared to
early postnatal stages, in which GFRA1-mediated signalling is dominant
(Figure \ref{fig:df3b}C) [@hammoud2015; @grive2019]. We also observed increased
chromatin accessibility and upregulation of DNA damage responsive gene
Fus, and the multifunctional redox gene Prdx4 ([Table S3](#st)).

Interestingly, the highest number of differentially accessible chromatin
regions were in Category 2, and included proximal regions with increased
chromatin accessibility and decreased expression of nearby genes in
adult spermatogonia, indicative of active repression taking place (Fig.
3A and [Table S3](#st)). A subset of Category 2 regions was marked by H3K27me3
together with H3K4me3 and mainly associated with developmental genes
such as Satb1 and Hmx1, (Supp. Figure \ref{fig:ds3}B). Other developmental genes such as
Tbx4 also displayed a decreased accessibility and an increase in mRNA
expression (Figure \ref{fig:df3b}C). GO enrichment analysis revealed that regions in
Category 2 are associated with regulation of cell cycle, RNA processing,
DNA repair and cell division ([Table S4](#st)). Such an example is is Fgf8,
important for Fgf8-Fgfr1 mediated maintenance of undifferentiated
spermatogonia [@hasegawa2014]. Fgf8 showed increased chromatin
accessibility close to its TSS and a downregulated expression in adult
spermatogonia, in agreement with recent findings from scRNA-seq data
suggesting downregulation of Fgfr1-mediated signalling with age(Fig.
S3B) [@grive2019; @hasegawa2014].

Regions in Category 3 displayed decreased chromatin accessibility and
downregulation of nearby genes in adult spermatogonia and were mostly
depleted of any of the 3 histone marks investigated (Figure \ref{fig:df3a}A and \ref{fig:df3a}B). GO
enrichment on the nearby genes revealed an association with
developmental processes and WNT signalling ([Table S4](#st)). A notable example
we identified in this category is Pdgfra, a gene involved in the hepatic
stellate cell activation pathway, which was recently identified by
scRNA-seq to be upregulated in spermatogonial stem cells in the immature
testis compared to the adult stage [@hermann2018]. Pdgfra displayed a
marked downregulation in adult spermatogonia, and a decrease in
chromatin accessibility overlapping its TSS (Figure \ref{fig:df3b}C). Another example
of a gene important for early postnatal spermatogonial cell
establishment is Dap2ip, which we also found to carry a marked decrease
in chromatin accessibility at its TSS and lower expression in adult
spermatogonial cells (Supp. Figure \ref{fig:ds3}C). Surprisingly, we also identified a
category of proximal regions (Category 4) with decreased accessibility
(\<20 regions) at genes that were upregulated in adult spermatogonial
cells and with a previously uncharacterized role in spermatogonial cells
(Figure \ref{fig:df3a}A). Notably, DNAme profiles across postnatal stages did not show
any significant change across any of the 6 categories of proximal
regions, suggesting a relatively stable DNAme profile in the transition
from early postnatal to adult stage (Supp. Figure \ref{fig:ds3}D). Aside from proximal
regions, we also identified numerous distal regions with differential
chromatin accessibility between PND15 and adult spermatogonial cells
(Supp. Figure \ref{fig:ds4}A). Similar to proximal regions, accessibility in distal regions
mainly increased in adult spermatogonial cells compared to the early
postnatal stage. When integrating the literature ChIP-seq data, we
observed enrichment for H3K4me3, H3K27ac and H3K27me3 at a small number
of the differentially accessible distal regions, indicative of potential
regulatory roles (Supp. Figure \ref{fig:ds4}B and [Table S3](#st)). Similar to proximal regions,
DNAme levels did not display major changes in the transition from early
postnatal to adult stage (Supp. Figure \ref{fig:ds4}B). Taken together, our data
integration reveals novel associations between chromatin regions of
differential accessibility, histone marks and gene transcription in
spermatogonial cells from postnatal to adult stages of development. They
suggest that chromatin accessibility is a mechanism of control of
transcriptional programs and of certain signalling pathways during
development.

### Accessibility changes at open chromatin regions carry binding sites for distinct families of transcription factors

Transcription factors (TFs) are essential for establishing and
maintaining transcriptional programs across developmental stages in
cells [@shavlakadze2019 @fushan2015]. To examine if regions of different
chromatin accessibility between PND15 and adult spermatogonia are
enriched in regulatory elements such as TF binding motifs, we performed
motif enrichment analysis using the Hypergeometric Optimization of Motif
EnRichment (HOMER) tool [@heinz2010]. In regions with increased
chromatin accessibility, we identified 41 enriched TF motifs (q-value
$\leq$ 0.05) (Figure \ref{fig:df4a}A). Top candidates by significance (q-value $\leq$
0.0001) are members of the Fos/Jun family (FOS, FOSB, FOSL1 and FOSL2,
JUN, JUNB and JUND) (Figure \ref{fig:df4a}B). Notably, at the mRNA level, some of the
TFs displayed age-specific differences (log\textsubscript{2}CPM $\geq$ 1 and abs log\textsubscript{2}
fold change $\geq$ 1): Fos,Junb and Jund were downregulated in adult
spermatogonial cells (Figure \ref{fig:df4b}C). JUN, FOS and CREB are all part of the
AP-1 (activating protein-1) superfamily, and play an important role in
regulating cell proliferation and death, by mediating the
senescence-associated chromatin and transcriptional landscape
[@martínez-zamudio2020; @shaulian2002]. JUND and c-FOS specifically
promote the proliferative potential of spermatogonial stem cells
[@he2008; @wang2018]. USF1 and POU3F1, 2 factors important in the
maintenance of the spermatogonial stem cell pool also displayed enriched
binding motifs in the more accessible regions. However, their mRNA
levels were downregulated in adult spermatogonial cells (Figure \ref{fig:df4b}C)
[[@faisal2019; @faisal2019]. POU3F1 is a GDNF-regulated TF, which has
been shown to play an important role in promoting spermatogonial cell
self-renewal capacity [@wu2010a; @niu2011]. Notably, Pou3f1
downregulation in adult spermatogonial cells coincides with marked
downregulation of Gfra1 and upregulation of Gfra2 (Figure \ref{fig:df3b}C and [Table S2](#st)).TF motif analysis using HOMER also revealed enriched binding sites
for retinoic acid receptors such as RXRa and RARa
(Figure \ref{fig:df4a}A). Recently, expression of RA receptors Rxra and
Rara was reported in the stem cell-containing population of
spermatogonial cells in both pup and adult testis, together with
evidence that their utilization in spermatogonial cells is vastly
dependent on the niche microenvironment [@lord2018].

To check if some TF binding motifs are preferentially enriched in
certain genomic locations, we performed motif enrichment analysis for
more accessible chromatin regions situated in gene bodies, intergenic
regions and in regions +/- 1kb from TSS. We identified several TF motifs
specifically enriched in intergenic regions, specifically members of the
ubiquitously expressed NF-Y complex, NF-YA, NF-YB and NF-YC (Figure \ref{fig:df4b}D).
In mESCs, NF-Y TF family members located in distal regions facilitate a
permissive chromatin conformation and play an important role in the
expression of core ESC pluripotency genes [@oldfield2014]. Furthermore,
NF-YA/B motif enrichment has also been found in regions of open
chromatin in human spermatogonial cells [@guo2017].

Although regions of more accessible chromatin encompass the majority of
the differentially accessible regions in adult spermatogonia compared to
PND15, less accessible chromatin also displayed a high number of
enriched TF binding motifs (Figure \ref{fig:df4a}A). Notably, almost all of these TF
motifs were uniquely enriched in the regions of decreased chromatin
accessibility and predominantly associated with developmental factors.
Top hits included members of the FOX family (FOXO1, FOXO3, FOXP2, FOXK1,
FOXA2) and members of the ETS and ETS-related families (ETS1, GABPA,
ETV4, ELF1, ELF3) (Figure \ref{fig:df4a}B). The gene expression levels of most of these
TFs were decreased in adult spermatogonial cells (Figure \ref{fig:df4a}A). FOXO1 is a
pivotal regulator of the self-renewal and differentiation of
spermatogonial stem cells in both pup and adult testis, via the PI3K-Akt
signalling pathway [@goertz2011; @chan2014]. The roles of the various
ETS-related TFs in spermatogonial cells have not been clarified, however
recently published data found ETV4 in the stem-cell enriched fraction of
the spermatogonial population, particularly during the spermatogonial
stem cell pool establishment immediately after birth [@law2019;
@cheng2020]. Motif enrichment analysis on the regions with decreased
chromatin accessibility situated in the gene body and intergenic regions
revealed that TFs are important in numerous developmental processes
(FOXC1, FOXJ2, FOXM1, LHX6) were specifically enriched in intergenic
regions of decreased chromatin accessibility (Figure \ref{fig:df4b}D). This is
consistent with the association of intergenic regions with embryonic
development-related GO terms that we previously detected (Figure \ref{fig:df1b}C). Our
findings provide a detailed characterization of the enriched TF motifs
present at the regions of dynamic accessibility between PND15 and adult
spermatogonial populations and point towards novel candidate regulators
of the differential transcriptome of the pup and adult mouse
spermatogonia.

### Chromatin accessibility at transposable elements undergoes significant remodelling in the transition from postnatal to adult spermatogonia

Transposable elements (TEs) are under tight control in the germline,
through coordinated epigenetic mechanisms involving DNA methylation,
chromatin silencing and PIWI proteins -- piRNA pathway [@thompson2016;
@deniz2019]. Recent evidence suggests an important role for long
terminal repeat (LTR) - type elements, specifically for ERVKs, the
youngest class of endogenous retroviruses (ERVs), in the transcriptional
regulation of mRNAs and long non-coding RNAs (lncRNAs) during
mitosis-to-meiosis transition [@sakashita2020; @davis2017]. Furthermore,
accessibility analysis at LTRs in mitotic and meiotic germ cells,
revealed a unique chromatin accessibility landscape in spermatogonial
cells, compared to the rest of the germ cells in the testis
[@sakashita2020]. To explore potential differences in TEs regulation
driven by postnatal age, we compared the accessibility of TEs in PND15
and adult spermatogonia. For this purpose, we quantified the ATAC-seq
reads overlapping TEs defined by UCSC RepeatMasker and performed
differential accessibility analysis at the subtype level (see Methods
section). Our results revealed that the transition from PND15 to the
adult stage is accompanied by significant chromatin accessibility
differences at 135 TE subtypes (Figure \ref{fig:df5a}A and 5B and [Table S5](#st)). Although
most of the differentially accessible TE subtypes displayed a decrease
in chromatin accessibility between PND15 and adult stages (68,9%,
93/135) (Figure \ref{fig:df5a}A), we also observed 42 TE subtypes which increased in
accessibility in adult spermatogonia (Figure \ref{fig:df5a}B). Of note, more accessible
subtypes also displayed an increased expression in adult spermatogonia
compared to the early postnatal stage (Figure \ref{fig:df5a}B). TE loci within the
subtypes harbouring changes in chromatin accessibility were situated in
intergenic and intronic regions (68% intergenic and 25% intronic), and
around 6% were located in the proximity of a gene (+/- 1kb from a TSS)
(Figure \ref{fig:df5a}C). LTRs were the most abundant TEs to display changes in
chromatin accessibility, specifically ERVK and ERV1 subtypes (Figure \ref{fig:df5a}A
and 5B). Exemplary ERVK subtypes harbouring less accessible chromatin
included RLTR17, RLTR9A3, RLTR12B and RMER17B ([Table S5](#st)). Enrichment of
RLTR17 and RLTR9 repeats have been reported previously in mESCs,
specifically at TFs important for pluripotency maintenance such as Oct4
and Nanog [@fort2014]. Interestingly, we identified the promoter region
of the lncRNA Lncenc1, an important regulator of pluripotency in mESCs
[@fort2014; @sun2018], harboring several LTR loci with decreased
accessibility in our adult spermatogonia, with the RLTR17 locus falling
within the TSS of Lncenc1. This decrease in accessibility correlated
with a marked decrease in expression of Lncenc1 in adult spermatogonia
(Figure \ref{fig:df5a}D). Lncenc1 (also known as Platr18) is part of the
pluripotency-associated transcript (Platr) family of lncRNAs which were
recently identified as potential regulators of the
pluripotency-associated genes Oct4, Nanog and Zfp42 in mESCs [@wu2018;
@dann2008; @bergmann2015]. We were also able to identify several other
Platr genes, such as Platr27 and Platr14, for which the TSS overlapped
LTRs with reduced accessibility, RLTR17 and RLTR16B_MM, respectively
(Figure \ref{fig:df5a}D and [Table S5](#st)). These 2 pluripotency-associated transcripts also
showed a decrease in mRNA expression in adult spermatogonia, while their
expression was unchanged between PND8 and PND15 (Figure \ref{fig:df5a}D and [Table S5](#st)).
The remaining LTR subtypes with decreased accessibility in adult
spermatogonia belonged to the ERV1, ERVL and MaLR families (Figure \ref{fig:df5a}A).
Only very few other non-LTR TEs showed a decrease in chromatin
accessibility, with 7 DNA element subtypes, 2 Satellite subtypes and 1
LINE subtype, respectively (Figure \ref{fig:df5a}A and [Table S5](#st)). Emerging evidence
suggests an important contribution of TEs in providing tissue-specific
substrates for TF binding [@fort2014; @sundaram2020; @sundaram2014]. To
investigate the regulatory potential of the less accessible LTR
subtypes, we assessed the enrichment of TF motifs in these regions using
HOMER. To do so, we focused on the family level and grouped together all
LTR subtypes coming from one family (EVK, ERV1, ERVL and ERVL-MaLR
families). Among the less accessible LTR families, ERVKs showed the
highest number of enriched TF motifs in adult spermatogonial cells. Top
hits included TFs with known regulatory roles in cell proliferation and
differentiation such as FOXL1 and FOXQ1, stem cell maintenance factors
ELF1, EBF1 and THAP11 and TFs important in spermatogenesis PBX3, ZNF143
and NFYA/B (Figure \ref{fig:df5b}E and S6A). ERVLs displayed motif enrichment for very
few TFs, among which the previously undescribed ETV2, newly reported
spermatogonial stem cell factor ZBTB7A and the testis-specific CTCF
paralog CTCFL (Figure \ref{fig:df5b}E) [@green2018].

Among the TE subtypes which increased in chromatin accessibility,
members of the ERVK, ERVL and ERV1 families were predominant (57,1%,
24/42) (Figure \ref{fig:df5a}B). Interestingly, we also found a considerable number of
LINE L1 subtypes with increased chromatin accessibility in adult
spermatogonial cells (Figure \ref{fig:df5a}B). When parsing the data for more
accessible loci within the L1 subtypes, we found several L1 loci
situated less than +/- 5 kb from the TSS of numerous olfactory (Olfr)
genes.Most of them were located in Olfr gene clusters on chromosomes 2,
7 and 11 ([Table S5](#st)). Furthermore, the increase in accessibility of the
L1 loci correlated with an increase in mRNA expression of the nearby
Olfr gene in adult spermatogonial cells (Figure \ref{fig:df6}A). Representative
examples were Olfr362 and Olfr1307, both situated in the Olfr gene
cluster on Chr2 (Figure \ref{fig:df6}B). Interestingly, when visualizing the data in
IGV, we also observed that the Olfr gene cluster on chromosome 2
exhibited a higher density of L1 loci compared to neighbouring regions
(Supp. Figure \ref{fig:ds6}B). Similar to before, we performed TF motif enrichment analysis
at the family level by grouping together all differentially accessible
TE subtypes coming from one family. More accessible LINE L1s were highly
enriched in TF motifs, particularly in multiple members of the ETS, E2F
and FOX families (Figure \ref{fig:df6}C). The most significant motifs belonged to
spermatogonial stem cell maintenance and stem cell potential regulators
FOXO1 and ZEB1, as well as TFs which have been recently associated with
active enhancers of the stem cell-enriched population of spermatogonia
such as ZBTB17 and KLF5 (Figure \ref{fig:df6}C and S6C) [@cheng2020]. More accessible
ERV1s also displayed enrichment of several TF binding sites, including
spermatogenesis-related TFs (PBX3, PRDM1, NFYA/B), hypoxia-inducible
HIF1A and cytokine regulators STAT5A/B, suggestive of different
spermatogonial cell metabolic demands between early postnatal and adult
stage (Figure \ref{fig:df6}C and S6C). Overall, we provide an extensive
characterization of the chromatin accessibility landscape of TEs in
PND15 and adult spermatogonia, reveal differences in accessibility and
TF motif landscape at distinct subtypes of TEs between these 2-time
points, and suggest potential gene programs that may be regulated by
these changes.


## Discussion

As initiators of the spermatogenic cascade, spermatogonial cells are
essential in germ cell proliferation and differentiation throughout
postnatal life. Although recent studies employing bulk and scRNA-seq
have revealed distinct transcriptional signatures of spermatogonial
cells across postnatal life, very few have focused on describing the
underlying landscape of open chromatin, and the extent to which it can
contribute to the gene expression dynamics [@hammoud2015; @hermann2018;
@grive2019].

Our ATAC-seq revealed a reorganization of open chromatin in adult
spermatogonia compared to the PND15 cell population. This striking
reorganization of accessible chromatin, which we detected mainly in
intergenic and intronic regions, may be indicative of novel regulatory
regions governing at least in part the vast transcriptome changes that
spermatogonial cells undergo in their transition from early postnatal to
adult stage [@hammoud2015; @grive2019]. Indeed, we found that the
regions of differential accessibility were associated with distinct gene
pathways, with morphogenesis and developmental pathways associated to
the regions of decreased chromatin accessibility, while regions of
increased chromatin openness were enriched for DNA repair pathways, stem
cell maintenance, RNA processing and protein metabolic processes.
Furthermore, we identified numerous enriched TF motifs in the regions of
differential accessibility, indicative of their potential regulatory
role. AP-1 TFs with previously described roles in spermatogonial cell
proliferation such as JUND and c-FOS, and pluripotency factors such as
POU3F1, displayed enriched binding sites in the regions of increased
chromatin accessibility. In contrast, FOX and ETS TF motifs, known
regulators of developmental pathways, mainly mapped to regions that
decreased in accessibility in adult spermatogonial cells. For some of
the enriched TF motifs, a preference for certain genomic locations was
evident: NF-YA and B binding sites exhibited enrichment specifically in
intergenic regions of more accessible chromatin which interestingly,
were also associated with spermatogenesis-related pathways. NF-YA/B is
also localized in intergenic regions of open chromatin in humans
spermatogonial cells [@guo2017], prompting an additional investigation
of their roles in regulating spermatogonial cell programs, with
potential consequences for sperm formation.

Our comparison of the gene expression changes from PND8 to PND15
spermatogonial cells confirmed the dynamic transcriptome associated with
developing spermatogonial cell states [@hammoud2015; @grive2019].
Notably, PND15 corresponds to the maturation of the Sertoli cell niche
in the mouse testis [@shinohara2001; @flickinger1967]. Upregulation of
pathways associated with signal transduction and cellular transport in
PND15 spermatogonia suggests an increased cellular communication with
the somatic niche, compared to the first week of postnatal development.
We complemented these findings with literature RNA-seq data from PND14
and adult spermatogonia and confirmed further transcriptome changes in
signalling pathways related to mitochondria, developmental processes and
cell-to-cell signalling. Furthermore, we revealed numerous pathways
related to cytokine signalling upregulated in adult spermatogonia, which
point to the recently suggested role of testis resident macrophages in
maintaining and regulating spermatogonial proliferation [@defalco2015;
@garbuzov2018]. These findings suggest that, as the testis matures and
the somatic niche develops, spermatogonial cells rely more on paracrine
signalling and undergo vast changes in gene expression programs.

To obtain a comprehensive profile of the chromatin and the transcriptome
differences between early postnatal and adult spermatogonial cells, we
have integrated the chromatin accessibility and gene expression, with
known histone H3 modifications and global DNAme patterns of THY1+
spermatogonial cells from [@hammoud2014; @hammoud2015]. This allowed us
to identify 4 distinct categories of differentially accessible chromatin
regions for which the nearest gene was dynamically expressed between
early postnatal and adult stages. Interestingly, we found a similar
number of genes with increased chromatin accessibility around their TSS
which exhibited either an up- or a downregulated expression between pup
and adult spermatogonia. In the category of upregulated genes with
increased nearby chromatin accessibility, we identified several factors
associated with redox processes, mitochondria function and cell
proliferation. In contrast, genes marked by an increase in chromatin
accessibility and downregulated expression in adult spermatogonia
comprised factors important for cell cycle, RNA processing and
developmental genes, suggesting that active repression is taking place
at these genes [@starks2019]. For some of the developmental genes, more
accessible chromatin was also marked by a bivalent H3K4me3/H3K27me3,
indicative of a poised state. Notably, previous findings in THY1+ adult
spermatogonial cells and in sperm also revealed a poised state at
promoters of developmental genes [@hammoud2014; @erkek2013; @jung2017].
Therefore, our findings suggest that open chromatin reorganization may
contribute to the poised status that is already established at the
spermatogonial cell stage for certain developmental genes. We also
identified a category of regions for which the decrease in chromatin
accessibility correlated with a decreased expression, the category which
also included developmental factors. The stable methylation patterns we
detected at the differentially accessible chromatin regions, in both
distal and proximal genomic regions, suggest a minimal impact for DNAme
in regulating gene expression dynamics of spermatogonial cells across
postnatal age.

Lastly, by investigating chromatin accessibility specifically at TEs, we
revealed that distinct TE subtypes undergo changes in chromatin
accessibility between PND15 and adult spermatogonial populations. ERVK
and ERV1 subtypes were the most abundant categories of TEs to become
less accessible in adult spermatogonia, whilst LINE L1 subtypes gained
in accessibility. Although the majority of these TEs resided in
intergenic and intronic regions, we were able to detect specific loci
belonging to the differentially accessible ERVK and LINE L1 subtypes,
which localized nearby TSS of distinct gene families. RLTR17, one of the
LTR subtypes with decreased chromatin accessibility in adult
spermatogonial cells, overlapped the TSS of several downregulated
long-non coding RNAs from the Platr family. Platr genes, including the
ones identified in our study, Lncenc1 and Platr14, are LTR-associated
long non-coding RNAs important for embryonic stem cell gene expression
programs [@bergmann2015]. Interestingly, RLTR17 has also been previously
linked to pluripotency maintenance. In mouse embryonic stem cells,
RLTR17 is highly expressed and enriched in open chromatin regions and
has been shown to provide binding sites for pluripotency factors Oct4
and Nanog [@fort2014]. On the basis of these findings, we suggest that
RLTR17 chromatin organization may play a significant role in regulating
pluripotency programs between early postnatal and adult spermatogonial
cells. In contrast to the decreased accessibility of LTRs, LINE L1
subtypes displayed an increase in chromatin accessibility in adult
spermatogonial cells. Some of these L1 loci were situated in the
vicinity of olfactory receptor genes with upregulated mRNA expression in
adult spermatogonia. Recent findings in mouse and human embryonic stem
cells have suggested a non-random genomic localization for L1 elements,
specifically at genes that encode proteins with specialized functions
[@lu2020]. Among these, the Olfr gene family was the most enriched in L1
elements [@lu2020]. Although their role in spermatogonial cells is
currently not established, Olfr proteins have been implicated in the
swimming behaviour of sperm [@fukuda2005; @vanderhaeghen1997]. Given
their dynamic regulation across age in spermatogonial cells, we
speculate that Olfr genes could play additional roles in
spermatogenesis, other than in sperm physiology. This data together with
the high number of enriched TF motifs identified at the differentially
accessible ERVKs and LINE L1 elements underscores previously undescribed
regulatory roles for chromatin organization of TEs in spermatogonial
cells during the transition from developing to adult stages
[@sundaram2020; @sundaram2014].

One limitation of our study is the incomplete purification achieved
using FACS, which doesn't fully remove other testis cell types from our
cell preparations. Therefore, we cannot entirely exclude the influence
of contaminating cells on some of the transcriptome and chromatin
accessibility data interpretation. Secondly, differences can also stem
from the literature datasets which involve similar but not identically
enriched populations of spermatogonial cells. Nevertheless, by comparing
open chromatin landscape between developing and adult spermatogonial
cells, our results reveal for the first time that there is an
age-dependent dynamic reorganization of chromatin accessibility in
spermatogonial cells. By integrating this newly generated data with gene
expression profiles and known histone modifications, we provide novel
insight into the chromatin - transcriptome dynamics of mouse
spermatogonial cells between developing and adult stages and compile an
information-rich resource for further germline studies.

\newpage

## Methods

### Mouse husbandry

Male C57Bl/6J mice were purchased from Janvier Laboratories (France) and
bred in-house to generate male mice used for experiments. All animals
were kept on a reversed 12-h light/12-h dark cycle in a temperature- and
humidity-controlled facility, with food (M/R Haltung Extrudat, Provimi
Kliba SA, Switzerland) and water provided ad libitum. Cages were changed
once weekly. Animals from 2 independent breedings were used for the
experiments.

### Germ cells isolation

Germ cells were isolated from male mice at postnatal day (PND) 8 or 15
for RNA-seq and ATAC-seq experiments, and adults at 20 weeks of age
(PNW20) for ATAC-seq. Testicular single-cell suspensions were prepared
as previously described with slight modifications [@kubota2004a;
@kubota2004]. For preparations using PND8 and PND15 pups, testes from 2
animals were pooled for each sample. Pup testes were collected in
sterile HBSS on ice. Tunica albuginea was gently removed from each
testis, making sure to keep the seminiferous tubules as intact as
possible. Tubules were enzymatically digested in 0.25% trypsin-EDTA
(ThermoFisher Scientific) and 7mg/ml DNase I (Sigma-Aldrich) solution
for 5 min at 37oC. The suspension was vigorously pipetted up and down 10
times and incubated again for 3 min at 37oC. The digestion was stopped
by adding 10% fetal bovine serum (ThermoFisher Scientific) and the cells
were passed through a 20\textmu m-pore-size cell strainer (Miltenyi
Biotec) and pelleted by centrifugation at 600g for 7 min at 4oC. Cells
were resuspended in PBS-S (PBS with 1% PBS, 10 mM HEPES, 1 mM pyruvate,
1mg/ml glucose, 50 units/ml penicillin and 50 \textmu g/ml streptomycin)
and used for sorting. For preparations from adult testis, one adult male
was used for each sample. The tunica was removed and seminiferous
tubules were digested in 2 steps. The first consisted of incubation in
1mg/ml collagenase type IV (Sigma-Aldrich) for 5 min at 37oC and
vigorous swirling until the tubules were completely separated. Then
tubules were placed on ice for 5 min to sediment, the supernatant was
removed and washed with HBSS. Washing/sedimentation steps were repeated
3 times and were necessary to remove interstitial cells. After the last
washing step, sedimented tubule fragments were digested again with 0.25%
trypsin-EDTA and 7mg/ml DNase I solution, and the digestion was stopped
by adding 10% FBS. The resulting single-cell suspension was filtered
through a 20\textmu m strainer (Corning Life Sciences) and washed with
HBSS. After centrifugation at 600g for 7 min at 4oC, the cells were
resuspended in PBS-S, layered on a 30% Percoll solution (Sigma-Aldrich)
and centrifuged at 600g for 8 min at 4oC without braking. The top 2
layers (HBSS and Percoll) were removed and the cell pellets were
resuspended in PBS-S and used for sorting.

### Spermatogonial cells enrichment by FACS

For pup testis, dissociated cells were stained with BV421-conjugated
anti-b2M, biotin-conjugated anti-THY1 (53-2.1), and
PE-conjugated anti-av-integrin (RMV-7) antibodies. THY1 was
detected by staining with Alexa Fluor 488-Sav. For adult testes, cells
were stained with anti-a6-integrin (CD49f; GoH3),
BV421-conjugated anti-b2 microglobulin (b2M; S19.8), and
R-phycoerythrin (PE)-conjugated anti-THY1 (CD90.2; 30H-12) antibodies.
a6-Integrin was detected by Alexa Fluor 488-SAv after staining
with biotin-conjugated rat anti-mouse IgG1/2a (G28-5) antibody. Prior to
FACS, 1 \textmu g/ml propidium iodide (Sigma) was added to the cell
suspensions to discriminate dead cells. All antibody incubations were
performed in PBS-S for at least 30 min at 4oC followed by washing in
PBS-S. Antibodies were obtained from BD Biosciences (San Jose, United
States) unless otherwise stated. Cell sorting was performed at 4oC on a
FACS Aria III 5L using an 85\textmu m nozzle at the Cytometry Facility
of the University of Zurich. For RNA-seq on PND8 and PND15
spermatogonia, cells were collected in 1.5 ml Eppendorf tubes in 500
\textmu L PBS-S, immediately pelleted by centrifugation and snap-frozen
in liquid N2. Cell pellets were stored at -80oC until RNA extraction.
For OmniATAC on PND15 spermatogonia, 25'000 cells were collected in a
separate tube, pelleted by centrifugation and immediately processed
using the OmniATAC library preparation protocol 10. For OmniATAC on
adult spermatogonia, 5000 cells from each animal were collected in a
separate tube and further processed using the same protocol.

### Immunocytochemistry

The protocol used for assessing spermatogonial cell enrichment after
sorting was kindly provided by the Oatley Lab at Washington State
University, Pullman, USA [@yang2013]. Briefly, 30,000-50,000 cells
adhered to poly-L-Lysine coated coverslips (Corning Life Sciences) in
24-well plates for 1 h. Cells were fixed in freshly prepared 4% PFA for
10 min at room temperature then washed in PBS with 0.1% Triton X-100
(PBS-T). Non-specific antibody binding was blocked by incubation with
10% normal goat serum for 1 h at room temperature. Cells were incubated
overnight at 4oC with mouse anti-PLZF (0.2 \textmu g/ml, Active Motif,
clone 2A9) primary antibody. Alexa488 goat anti-mouse IgG (1 \textmu
g/mL, ThermoFisher Scientific) was used for secondary labelling at 4oC
for 1 h. Coverslips were washed 3x and mounted onto glass slides with
VectaShield mounting medium containing DAPI (Vector Laboratories) and
examined by fluorescence microscopy. Stem cell enrichment was determined
by counting PLZF+ cells in 10 random fields of view from each coverslip
and dividing by the total number of cells present in the field of view
(DAPI-stained nuclei).

### RNA extraction and library preparation for RNA-seq

For RNA-seq on PND8 and PND15 spermatogonial cells, total RNA was
extracted from sorted cells using AllPrep RNA/DNA Micro kit (Qiagen).
RNA quality was assessed using a Bioanalyzer 2100 (Agilent
Technologies). Samples were quantified using Qubit RNA HS Assay
(ThermoFisher Scientific). 10 ng of total RNA from each sample were used
to prepare total long RNA sequencing libraries using SMARTer® Stranded
Total RNA-Seq Kit v2 - Pico Input Mammalian (Takara Bio USA, Inc.) at
the Functional Genomics Center Zurich (FGCZ) according to the
manufacturer's instructions.

### Library preparation for Omni-ATAC

Chromatin accessibility was profiled in PND15 and adult spermatogonial
cells. Libraries were prepared starting from 25 000 PND15 and 5000
adults sorted spermatogonia, respectively 10. Briefly, sorted cells were
lysed in cold lysis buffer (10 mM Tris-HCl pH 7.4, 10 mM NaCl, 3 mM
MgCl2, 0.1% NP40, 0.1% Tween-20, and 0.01% digitonin) and nuclei were
pelleted and transposed using Nextera Tn5 (Illumina) for 30 min at 37oC
in a thermomixer with shaking at 1000 rpm. Transposed fragments were
purified using the MinElute Reaction Cleanup Kit (Qiagen). Following
purification, libraries were generated by PCR amplification using the
NEBNext High-Fidelity 2X PCR Master Mix (New England Biolabs), and
purified using Agencourt AMPure XP magnetic beads (Beckman Coulter) to
remove primer dimers (78bp) and fragments \>1000bp. Library quality was
assessed on an Agilent High Sensitivity DNA chip using the Bioanalyzer
2100 (Agilent Technologies).

### RNA sequencing

Quality control and alignment:Single-end (SE) sequencing was performed
using an Illumina HiSeq4000 at the FGCZ. PND8 raw data (FASTQ files) was
merged from two individual runs. For analysis of published RNA-seq data
(PND14 and PNW8 spermatogonia), FASTQ files were obtained using
fastq-dump (version 2.10.8), quality-controlled using FastQC
[@andrews2012] (version 0.11.8). TrimGalore [@krueger2015] (version
0.6.2) was used to trim adapters and low-quality ends from reads with
Phred score less than 30 (-q 30), and for discarding trimmed reads
shorter than 30 bp (--length 30). Trimmed reads were pseudo-aligned
using Salmon [@patro2017] (version 0.9.1) with automatic detection of
the library type (-l A), correcting for sequence-specific bias
(--seqBias) and correcting for fragment GC bias correction (--gcBias) on
a transcript index prepared for the Mouse genome (GRCm38) from GENCODE
(version M18) [@harrow2012], with additional piRNA precursors and
transposable elements (concatenated by family) from Repeat Masker as in
[@gapp2020].

Downstream analysis:Analysis was conducted with R[@rcoreteam2019]
(version 3.6.2) using packages from The Comprehensive R Archive Network
(CRAN)
([[cran.r-project.org](https://cran.r-project.org/)]([cran.r-project.org](https://cran.r-project.org/)))
and Bioconductor [@huber2015]. Pre-filtering of genes was performed
using the filterByExpr function from edgeR [@robinson2010] (version
3.28.1) with a design matrix and requiring at least 15 counts
(min.counts = 15). Normalization factors were obtained using TMM
normalization [@robinson2010a] from edgeR package and differential gene
expression (DGE) analysis was performed using limma-voom [@law2014]
pipeline from limma [@ritchie2015] (version 3.42.2). log\textsubscript{2} fold change
between samples was calculated by subtracting log\textsubscript{2} normalized expression
values. Gene ontology (GO) analysis was performed on expressed genes
with fGSEA (version 1.15.2) using fGSEAMultilevel function on sets with
10 to 1000 annotated genes (minSize = 10, maxSize = 1000), and p-values
boundary of 1E-100 (eps = 1e-100) [@korotkevich2016]. For PND8 versus
PND15 comparison, genes were pre-ranked using t-statistic; for PND14
versus PNW8 comparison, log\textsubscript{2} fold change was used due to the lack of
multiple replicates for each time point. REVIGO was used to summarize GO
terms obtained following fGSEA [@supek2011].

### Omni-ATAC

Quality control, alignment, and peak calling:Paired-end (PE) sequencing
was performed on PND15 and adult spermatogonial cells samples on an
Illumina HiSeq2500 platform (FGCZ). FASTQ files were assessed for
quality using FastQC [@andrews2012] (version 0.11.8). Quality control
(QC) was performed using TrimGalore [@krueger2015] (version 0.6.2) in PE
mode (--paired), trimming adapters, low-quality ends (-q 30) and
discarding reads \< 30 bp after trimming (--length 30). Alignment on the
GRCm38 genome was performed using Bowtie2 [@langmead2012] (version
2.3.5) with the following parameters: fragments up to 2 kb were allowed
to align (-X 2000), entire read alignment (--end-to-end), suppressing
unpaired alignments for paired reads (--no-mixed), suppressing
discordant alignments for paired reads (--no-discordant) and minimum
acceptable alignment score with respect to the read length (--score-min
L,-0.4,-0.4). Using alignmentSieve (version 3.3.1) from deepTools
[@ramirez2016] (version 3.4.3), aligned data (BAM files) were adjusted
for the read start sites to represent the centre of the transposon
cutting event (--ATACshift), and filtered for reads with a high mapping
quality (--minMappingQuality 30). Reads mapping to the mitochondrial
chromosome and ENCODE blacklisted regions were filtered out. To call
nucleosome-free regions, all aligned files were merged within groups
(PND15 and adult), sorted, and indexed using SAMtools [@li2009] (version
0.1.19), and nucleosome-free fragments (NFFs) were obtained by selecting
alignments with a template length between 40 and 140 inclusively. Peak
calling (identifying areas in a genome that have been enriched for
transcription factors) on the NFFs was performed using MACS2
[@zhang2008] (version 2.2.7.1) with mouse genome size (-g 2744254612)
and PE BAM file format (-f BAMPE).

Differential accessibility analysis:The downstream analysis was
performed in R(version 3.6.2), using packages from CRAN and Bioconductor
[@huber2015]. The peaks were annotated based on overlap with GENCODE
(version M18) [@harrow2012] transcript, and/or the distance to the
nearest transcription start site (available at the following link:
[https://github.com/mansuylab/SC_postnatal_adult/bin/annoPeaks.R)](https://github.com/mansuylab/SC_postnatal_adult/bin/annoPeaks.R).The)
number of extended reads overlapping in the peak regions was calculated
using the csaw package [@lun2015] (version 1.20.0). Peak regions that
did not have at least 15 reads in at least 40% of the samples were
filtered out. Normalization factors were obtained on the filtered peak
regions using the TMM normalization method [@robinson2010a] and
differential analysis on the peaks (adults vs PND15) was performed using
the Genewise Negative Binomial Generalized Linear Models with
Quasi-likelihood (glmQLFit) Tests from the edgeR package [@robinson2010]
(version 3.28.1). Peak regions that had an absolute log\textsubscript{2} fold change
$\geq$ 1 and an FDR $\leq$ 0.05 were categorized as differentially
accessible regions. GO analysis was performed on DARs with the rGREAT
package [@gu2021] (version 1.18.0), which is a wrapper around the GREAT
tool [@mclean2010] (version 4.0). Transcription factor motif enrichment
analysis was performed using the marge package [@amezquita2018] (version
0.0.4.9999), which is a wrapper around the Homer tool [@heinz2010]
(version 4.11.1).

Differential accessibility analysis at transposable elements: TE gene
transfer format (GTF) file was obtained from
[[labshare.cshl.edu/shares/mhammelllab/www-data/TEtranscripts/TE_GTF/mm10_rmsk_TE.gtf.gz%20on%2003.02.2020](http://labshare.cshl.edu/shares/mhammelllab/www-data/TEtranscripts/TE_GTF/mm10_rmsk_TE.gtf.gz)](http://labshare.cshl.edu/shares/mhammelllab/www-data/TEtranscripts/TE_GTF/mm10_rmsk_TE.gtf.gz%20on%2003.02.2020)
on 03.02.2020. The GTF file provides hierarchical information about TEs:
**Class** (level 1, eg. LTR), **Family** (level 2, eg. LTR L1),
**Subtype** (level 3, eg. LTR L1 L1_Rod), and **Locus** (level 4, eg.
LTR L1 L1_Rod L1_Rod_dup1). TE loci were annotated based on overlap with
GENCODE (version M18) as described above for ATAC-seq peaks. Filtered
BAM files (without reads mapping to blacklisted or mitochondrial
regions) were used for analyzing TEs. Mapped reads were assigned to TEs
using featureCounts from the R package Rsubread [@liao2019] (version
2.0.1) and were summarized to Subtypes (level 3), allowing for
multi-overlap with fractional counts, while ignoring duplicates. The
number of extended reads overlapping at the TE loci were obtained using
the csaw package [@lun2015] (version 1.20.0). Subtypes which did not
have at least 15 reads, and loci that did not have at least 5 reads in
at least 40% of the samples, were filtered out. Normalization and
differential accessibility analysis were performed as described above.
Subtypes that had an absolute log\textsubscript{2}fold change $\geq$ 0.5 and an FDR
$\leq$ 0.05 were categorized as differentially accessible subtypes and
the loci with an absolute log\textsubscript{2} fold change $\geq$ 1 and an FDR $\leq$
0.05 were categorized as differentially accessible loci. For further
downstream data analysis, only the differentially accessible loci of
differentially accessible subtypes were considered. GO and motif
enrichment analysis were performed as described above.

### Chromatin immunoprecipitation sequencing

Quality control, alignment, and peak calling:ChIP-Seq SE data for PNDW8
(adults) were obtained from GEO accession GSE49621 [@hammoud2014]. FASTQ
files were obtained using fastq-dump (version 2.10.8), and different
runs were merged. The FASTQ files were assessed for quality using FastQC
[@andrews2012] (version 0.11.8). Quality control (QC) was performed
using TrimGalore [@krueger2015] (version 0.6.0), trimming adapters,
low-quality ends (-q 30) and discarding trimmed reads shorter than 30 bp
(--length 30). Alignment to the GRCm38 genome was performed using
Bowtie2 [@langmead2012] (version 2.3.5). Reads reads with low mapping
quality (--minMappingQuality 30) or mapping to the mitochondrial
chromosome or aforementioned blacklisted regions were filtered out. Peak
calling was performed using MACS2 [@zhang2008] (version 2.2.7.1) with
mouse genome size (-g 2744254612) and SE BAM file format (-f BAM).

### Bisulfite sequencing (BS)

Quality control and alignment:BS paired-end data for PND7, PND14, and
PNW8 (adults) were obtained from GEO accession GSE49623 [@hammoud2015].
FASTQ files were obtained using fastq-dump (version 2.10.8), and
different runs were merged. FASTQ files were assessed for quality using
FastQC [@andrews2012] (version 0.11.8). QC was performed using
TrimGalore [@krueger2015] (version 0.6.4_dev) in PE mode (--paired),
trimming adapters, low-quality ends (-q 30) and discarding trimmed reads
shorter than 30 bp (--length 30). Alignment of the QC data was performed
using Bismark [@krueger2011] (version 0.22.3) on a GRCm38 index built
using bismark_genome_preparation (version 0.17.0). Methylation
information for individual cytosines was extracted using the
bismark_methylation_extractor tool from the Bismark package (version
0.22.3).

### High-throughput sequencing data analysis

Data availability:the datasets used in this study are available from the
following GEO accessions: **GSE___**, GSE49621, GSE49622,
GSE62355, and GSE49623. An overview of the datasets included in the
study is shown in Table \ref{tab:devtab1}

```{r, devtab1, echo = FALSE}

df <- data.frame(Source = c("GSE", "GSM1525703", "GSM1415671", "GSE",
                            "GSM1202705", "GSM1202708", "GSM1202713", 
                            "GSM1202723", "GSE49623"),
           Sequencing = c("RNA-seq", "RNA-seq", "RNA-seq",
                          "ATAC-seq", "ChIP-seq (H3K4me3)",
                          "ChIP-seq (H3K27me3)", "ChIP-seq (H3K27ac)",
                          "ChIP-seq (Input)", "BS-seq"),
           'Stages (n)' = c("PND8 (8), PND15 (9)", "PND14 (1)",
                            "PNW8 (1)", "PND15 (6), PNW20 (5)",
                            "PNW8 (1)", "PNW8 (1)", "PNW8 (1)", "PNW8 (1)",
                            "PND7 (1), PND14 (1), PNW8 (7)"), check.names = F)


kable(df, 
    caption = "Datasets used in this manuscript", 
    caption.short = "Datasets used in this manuscript",
    longtable = TRUE,
    booktabs = TRUE, format = "latex", escape = FALSE)
```

### Figures

All figures in this study were generated using ggplot2 [@wickham2016],
EnrichedHeatmap [@gu2018], and ComplexHeatmap [@gu2016] packages, and
using base plotting functions in R. Genomic tracks were generated in IGV
and colour coded in Inkscape.

## Authors contribution

ILC and IMM conceived and designed the study. ILC prepared samples,
performed all RNA-seq, ICC and ATAC-seq experiments. DKT analyzed
RNA-seq, ATAC-seq, ChIP-seq and BS data, with significant support from
PLG. ILC and DKT prepared figures. ILC interpreted the data with
significant input from DKT, PLG and IMM. ILC wrote the manuscript with
significant help from DKT, PLG and IMM. All authors read and accepted
the final version of the manuscript.

## Acknowledgements

We thank Francesca Manuella and Martin Roszkowski for taking care of
animal breeding, Yvonne Zipfel for animal care, Andrew McDonald for
assistance with basic lab needs, Silvia Schelbert and Alberto Corcoba
for taking care of the animal licenses and lab organization.We thank
Rodrigo Arzate for conceptual support and critical reading of the
manuscript. We thank Catherine Aquino and Emilio Yángüez from the
Functional Genomics Center Zurich for support and advice with library
preparation and sequencing. We are very grateful to Jon Oatley, Melissa
Oatley, Tessa Lord and Nathan Law for conceptual advice, hands-on
training, and for providing detailed protocols for testis dissection and
preparation, and immunocytochemistry of spermatogonial cells. We thank
Zuguang Gu for support with heatmaps. We thank Service and Support for
Science IT ([www.s3it.uzh.ch](http://www.s3it.uzh.ch/)) for
computational infrastructure.

## Competing interest

The authors declare that they have no competing interests.

## Funding

The lab is funded by the University Zurich, ETH Zurich, the Swiss
National Science Foundation Grant No. 31003A_175742 / 1, ETH fellowship
ETH-25 19-2. DKT is supported by a Swiss Government Excellence
Scholarship.

## Data and materials availability

Repository accession numbers will be available at publication or by
request to the corresponding author. The code employed for the data
analysis is available from
<https://github.com/mansuylab/SC_postnatal_adult>.

\newpage

<!--chapter:end:rmd/06-dev/2.Rmd-->

---
output: html_document
editor_options: 
  chunk_output_type: console
bibliography: rmd/references.bib
# header-includes:
  # \usepackage{float} \floatplacement{figure}{H}
  # \newcommand{\beginfig}{\setcounter{table}{0}  \renewcommand{\thetable}{\arabic{table}} \setcounter{figure}{0} \renewcommand{\thefigure}{\arabic{figure}}}
---

## Figures

\begin{subfigures}
(ref:dfc1a) \textbf{Regions of differential chromatin accessibility in spermatogonial cells between postnatal and adult stages associate with distinct biological processes.}\newline \textbf{A:} Volcano plot of differentially accessible regions (adjusted P $\leq$ 0.05 and absolute Log\textsubscript{2} FC $\geq$ 1) between PND15 (n = 6) and adult (n = 5) spermatogonial cells identified by ATAC-seq.\newline \textbf{B:} Bar plot illustrating the genomic distribution of differentially accessible chromatin regions in PND15 and adult spermatogonial cells. Genomic regions are categorized in exonic, intronic, intergenic and +/- 1kb from the TSS of a gene.

```{r df1a, echo=FALSE, fig.align='center', fig.cap="(ref:dfc1a)", fig.scap="Regions of differential chromatin accessibility in spermatogonial cells between postnatal and adult stages"}
img <- readPNG("figure/dev/1a.png")
grid.raster(img)
```

(ref:dfc1b) \textbf{Regions of differential chromatin accessibility in spermatogonial cells between postnatal and adult stages associate with distinct biological processes.}\newline \textbf{C:} Dot plots of top enriched GO biological processes for regions with increased or decreased chromatin accessibility in adult spermatogonia. Dots size indicates the number of genes in the term, and their colour corresponds to the adjusted P-value of the term’s enrichment.

```{r df1b, echo=FALSE, fig.align='center', fig.cap="(ref:dfc1b)", fig.scap="Regions of differential chromatin accessibility in spermatogonial cells between postnatal and adult stages"}
img <- readPNG("figure/dev/1b.png")
grid.raster(img)
```

\end{subfigures}

\begin{subfigures}

(ref:dfc2a) \textbf{Transcriptomic profile in postnatal and adult spermatogonial cells.}\newline \textbf{A:} Heatmap of differentially expressed genes (coding and non-coding) (adjusted P $\leq$ 0.05 and absolute Log\textsubscript{2} FC $\geq$ 1) between PND8 (n = 9) and PND15 (n = 8) spermatogonial cells. Shown are Log\textsubscript{2} FC with respect to the average of PND8 data. Samples are clustered using Ward’s method. Genes are ordered by principal component analysis (PCA) method using seriation (R package).\newline \textbf{B:} Dot plots of top 20 enriched GO biological processes  (adjusted P $\leq$ 0.05) from GSEA analysis of PND8 versus PND15. GO terms are summarized by REVIGO and ordered by their normalized enrichment scores (NES). The size of the dot indicates the number of expressed genes annotated in the GO term, and the colour corresponds to the adjusted P-value.

```{r df2a, echo=FALSE, fig.align='center', fig.cap="(ref:dfc2a)", fig.scap="Transcriptomic profile in postnatal and adult spermatogonial cells"}
img <- readPNG("figure/dev/2a.png")
grid.raster(img)
```

(ref:dfc2b) \textbf{Transcriptomic profile in postnatal and adult spermatogonial cells.} \newline \textbf{C,D:} Dot plots of top 10 enriched GO BP terms (adjusted P $\leq$ 0.05) from GSEA analysis of PND15 vs PND8 and PNW8 vs PND14 comparison, respectively. GO terms are summarized by REVIGO and ordered by their normalized enrichment scores (NES). The size of the dot indicates the number of expressed genes annotated in the GO term, and the colour corresponds to the adjusted P value.

```{r df2b, echo=FALSE, fig.align='center', fig.cap="(ref:dfc2b)", fig.scap="Transcriptomic profile in postnatal and adult spermatogonial cells"}
img <- readPNG("figure/dev/2b.png")
grid.raster(img)
```
\end{subfigures}


\begin{subfigures}

(ref:dfc3a) \textbf{Chromatin accessibility and histone modifications at proximal regions of genes dynamically expressed between PND15 and adult spermatogonial cells.} \newline \textbf{A:} Heatmap of mean coverage across conditions showing proximal regions of increased chromatin accessibility and increased (Category 1, n = 171) or decreased (Category 2, n = 233) gene expression, decreased chromatin accessibility and decreased (Category 3, n = 32) or increased (Category 4, n = 14) gene expression when comparing PND15 with adult spermatogonia. Proximal inactive regions were defined as regions with increased (Category 5, n = 291) or decreased (Category 6, n = 88) accessibility for which no expression of the nearest gene could be detected by RNA-seq. \newline \textbf{B:} Heatmap showing the overlap between Category 1-4 regions and ChIP-seq data in PNW8 spermatogonia for H3K4me3, H3K27ac and H3K27me3. For each of the Category 1-4, the following sub-categorization was applied: regions enriched for H3K4me3 (with or w/o H3K27ac and/or H3K27me3), regions enriched for H3K27ac (and lack both H3K4me3 and H3K27me3) and regions enriched for H3K27me3 (and lack both H3K4me3 and H3K27ac). For (\textbf{A}, \textbf{B}), each line represents a peak region and regions are ordered within a category by the ATAC-seq signal. Mid-x-axis corresponds to the middle of a peak region and is extended to +/- 1 kb. The colour-key of the ATAC-seq heatmap represents the ATAC-seq signal in Log\textsubscript{2} Reads Per Kilobase per Million (RPKM) reads sequenced. For RNA-seq, Log\textsubscript{2} FC is shown from PND8 versus PND15 and PND14 versus PNW8 comparisons. Rows are ordered by the enriched scores of ATAC-seq data.

```{r df3a, echo=FALSE, fig.align='center', fig.cap="(ref:dfc3a)", fig.scap="Chromatin accessibility and histone modifications at proximal regions of genes dynamically expressed between PND15 and adult spermatogonial cells"}
img <- readPNG("figure/dev/3a.png")
grid.raster(img)
```

(ref:dfc3b) \textbf{Chromatin accessibility and histone modifications at proximal regions of genes dynamically expressed between PND15 and adult spermatogonial cells.} \newline \textbf{C:} Genomic snapshots from the Integrative Genomics Viewer (IGV, Broad Institute) of exemplary genes from Category 1 (Gfra2), category 2 (Hmx1) and category 3 (Pdgfra) showing the relative abundance of transcripts from RNA-seq and chromatin accessibility from ATAC-seq. RNA-seq data correspond to literature PND14 and PNW8 spermatogonial cells and ATAC-seq data to PND15 and adult spermatogonial cells. Genomic snapshots from the Integrative Genomics Viewer (IGV, Broad Institute) of exemplary genes from Category 1 (Gfra2), Category 2 (Hmx1) and Category 3 (Pdgfra) showing the relative abundance of transcripts from RNA-seq, chromatin accessibility from ATAC-seq and enrichment of 3 different histone marks (H3K27ac, H3K4me3 and H3K27me3) from ChIP-seq. Pup (PND15 ATAC-seq or PND14 RNA-seq samples) spermatogonia are represented in green and adult (PNW20 ATAC-seq samples or PNW8 RNA-seq and ChIP-seq samples) spermatogonia are represented in orange and overlayed in each of the IGV tracks.

```{r df3b, echo=FALSE, fig.align='center', fig.cap="(ref:dfc3b)", fig.scap="Chromatin accessibility and histone modifications at proximal regions of genes dynamically expressed between PND15 and adult spermatogonial cells"}
img <- readPNG("figure/dev/3b.png")
grid.raster(img)
```

\end{subfigures}

\begin{subfigures}
(ref:dfc4a) \textbf{Transcription factor dynamics at differentially accessible regions is predicted by motif enrichment in adult spermatogonial cells.} \newline \textbf{A:} Dot plot of all transcription factor motifs enriched in the regions of decreased and increased accessibility between adult and PND15 spermatogonia. Each dot corresponds to a motif. The differential gene expression of each transcription factor was extracted from the PND14 and PNW8 literature RNA-seq and is shown as colour-coded Log\textsubscript{2} FC. The size of the dot indicates the HOMER motif enrichment adjusted P of each motif. \newline \textbf{B:} Genomic snapshots from the Integrative Genomics Viewer (IGV, Broad Institute) of mRNA expression levels of representative enriched TFs in the regions of increased chromatin accessibility. RNA-seq data corresponds to literature PND14 and adult (PNW8) spermatogonial cells and  ATAC-seq data to PND15 and adult (PNW20) spermatogonial cells, respectively.

```{r df4a, echo=FALSE, fig.align='center', fig.cap="(ref:dfc4a)", fig.scap="Transcription factor dynamics at differentially accessible regions is predicted by motif enrichment in adult spermatogonial cells"}
img <- readPNG("figure/dev/4a.png")
grid.raster(img)
```

(ref:dfc4b) \textbf{Transcription factor dynamics at differentially accessible regions is predicted by motif enrichment in adult spermatogonial cells.} \newline \textbf{C:} HOMER extracted consensus sequences for each transcription factor motif. Representative examples from the most enriched transcription factor families are depicted. \newline \textbf{D:} Dot plots of top transcription factor motifs enriched in differentially accessible chromatin regions situated in gene bodies and in intergenic areas of the genome. Each dot corresponds to a motif. The differential gene expression of each transcription factor was extracted from the PND14 and PNW8 literature RNA-seq and is shown as colour-coded Log\textsubscript{2} FC. The size of the dot indicates the HOMER motif enrichment adjusted P of each motif.

```{r df4b, echo=FALSE, fig.align='center', fig.cap="(ref:dfc4b)", fig.scap="Transcription factor dynamics at differentially accessible regions is predicted by motif enrichment in adult spermatogonial cells"}
img <- readPNG("figure/dev/4b.png")
grid.raster(img)
```
\end{subfigures}

\begin{subfigures}

(ref:dfc5a) \textbf{Differential chromatin accessibility at transposable elements (TEs) in adult spermatogonial cells compared to PND15.} \newline \textbf{A:} Heatmap of the LTR and LINE subtypes with decreased accessibility between adult and PND15 spermatogonia (adjusted P $\leq$ 0.05 and Log\textsubscript{2} FC $\geq$ 0.5). \newline \textbf{B:} Heatmap of the LTR and LINE subtypes with increased accessibility between adult and PND15 spermatogonia (adjusted P $\leq$ 0.05 and Log\textsubscript{2} FC $\geq$ 0.5). Expression changes of these subtypes between PNW8 and PND14 spermatogonia literature RNA-seq is represented as Log\textsubscript{2} FC. \newline (\textbf{A}, \textbf{B}) Log\textsubscript{2} FC are shown with respect to the average of the PND15 samples. Samples are clustered using Ward’s method. Subtypes are ordered by principal component analysis (PCA) method using seriation (R package). \newline \textbf{C:} Bar plot illustrating the genomic distribution of differentially accessible TEs between adult and PND15 spermatogonial cells. \newline \textbf{D:} Genomic snapshots from the Integrative Genomics Viewer (IGV, Broad Institute) of Lncenc1 and Platr14 showing LTRs from RepeatMasker and the average normalized RNA-seq and ATAC-seq coverage (RPKM). LTR loci were extracted using RepeatMasker. RNA-seq data corresponds to literature PND14 and adult (PNW8) spermatogonial cells and  ATAC-seq data to PND15 and adult (PNW20) spermatogonial cells, respectively.

```{r df5a, echo=FALSE, fig.align='center', fig.cap="(ref:dfc5a)", fig.scap="Differential chromatin accessibility at transposable elements (TEs) in adult spermatogonial cells compared to PND15"}
img <- readPNG("figure/dev/5a.png")
grid.raster(img)
```


(ref:dfc5b)  \textbf{Differential chromatin accessibility at transposable elements (TEs) in adult spermatogonial cells compared to PND15.} \newline \textbf{E:} Dot plots of top transcription factor motifs enriched in the less accessible ERVKs and ERVL subtypes. Each dot corresponds to a motif. The differential gene expression of each transcription factor was extracted from the PNW8 vs PND14 comparison from literature RNA-seq and is shown as colour-coded Log\textsubscript{2} FC. The size of the dot indicates the HOMER motif enrichment adjusted P of each motif.

```{r df5b, echo=FALSE, fig.align='center', fig.cap="(ref:dfc5b)", fig.scap="Differential chromatin accessibility at transposable elements (TEs) in adult spermatogonial cells compared to PND15"}
img <- readPNG("figure/dev/5b.png")
grid.raster(img)
```

\end{subfigures}

(ref:dfc6) \textbf{Increased accessibility at LINE L1 subtypes located near Olfr gene clusters.} \newline \textbf{A:} Heatmap of the Olfr genes for which we identified an upregulated expression from PND14 to PNW8 timepoints and an increase in accessibility at a nearby L1 locus. RNA expression levels are expressed as Log\textsubscript{2} CPM at each timepoint. Accessibility changes at each of the corresponding L1 locus situated within +/- 5kbp from the gene are expressed as Log\textsubscript{2} FC calculated from the ATAC-seq analysis of the differentially accessible TEs between adult and PND15 spermatogonial cells. Locus are ordered by PCA method using seriation (R package). \newline \textbf{B:} Genomic snapshots from the Integrative Genomics Viewer (IGV, Broad Institute) of exemplary genes Olfr1497 and Olfr362 showing relative abundance of transcripts from RNA-seq and chromatin accessibility from ATAC-seq. LINE loci were extracted using Repeat Masker. RNA-seq data corresponds to literature PND14 and adult (PNW8) spermatogonial cells and  ATAC-seq data to PND15 and adult (PNW20) spermatogonial cells, respectively. \newline \textbf{C:} Dot plots of top transcription factor motifs enriched in the more accessible L1 and ERV1 subtypes. Each dot corresponds to a motif. The differential gene expression of each transcription factor was extracted from the PNW8 vs PND14 comparison from literature RNA-seq and is shown as colour-coded Log\textsubscript{2} FC. The size of the dot indicates the HOMER motif enrichment adjusted P of each motif.

```{r df6, echo=FALSE, fig.align='center', fig.cap="(ref:dfc6)", fig.scap="Increased accessibility at LINE L1 subtypes located near Olfr gene clusters"}
img <- readPNG("figure/dev/6.png")
grid.raster(img)
```

<!--chapter:end:rmd/06-dev/3.Rmd-->

---
output: html_document
editor_options: 
  chunk_output_type: console
bibliography: rmd/references.bib
# header-includes:
#   \usepackage{float} \floatplacement{figure}{H}
#   \newcommand{\beginsupplement}{\setcounter{table}{0}  \renewcommand{\thetable}{S\arabic{table}} \setcounter{figure}{0} \renewcommand{\thefigure}{S\arabic{figure}}}
---

## Supplementary Figures

\begin{subfigures}

(ref:dsc1a) \textbf{FACS leads to enrichment of PLZF+ cells in testicular preparations from mouse.} \newline \textbf{A:} Representative dot plots of the sorting strategy for spermatogonial cell enrichment. Gating based on side scatter/forward scatter (SSC-A/FSC-A) and forward scatter – height/ forward scatter – area (FSC-H/FSC-A) was performed to exclude cell debris and cell clumps. \newline \textbf{B:} PLZF+ cells are enriched following FACS, illustrated by immunocytochemistry on unsorted and sorted cell preparations. Immunocytochemistry of PND15 unsorted and the sorted testis cell suspension was performed by fixating cells on poly-L-lysine coated slides. Cells were stained with anti-PLZF antibody (S19 clone, Active Motif) and VECTASHIELD (with DAPI) antifade mounting medium was used for mounting. Cells were visualized with a fluorescence microscope and counted in 10 different fields of view/slide. The number of PLZF+ and PLZF- cells from 10 different fields of view were averaged.

```{r ds1a, echo=FALSE, fig.align='center', fig.cap="(ref:dsc1a)", fig.scap="FACS leads to enrichment of PLZF+ cells in testicular preparations from mouse"}
img <- readPNG("figure/dev/s1a.png")
grid.raster(img)
```

(ref:dsc1b) \textbf{FACS leads to enrichment of PLZF+ cells in testicular preparations from mouse.} \newline \textbf{C} Heatmap of the expression profile of selected markers for spermatogonial and different testicular somatic cells from our PND8 and PND15  RNA-seq data, and PND14 and PNW8 datasets from the literature. Key genes for stem cell potential, stem and progenitor spermatogonia, and Leydig and Sertoli cells were chosen to evaluate spermatogonial cells enrichment in the populations. Gene expression is represented in Log\textsubscript{2}CPM (counts per million).

```{r ds1b, echo=FALSE, fig.align='center', fig.cap="(ref:dsc1b)", fig.scap="FACS leads to enrichment of PLZF+ cells in testicular preparations from mouse"}
img <- readPNG("figure/dev/s1b.png")
grid.raster(img)
```

\end{subfigures}


\begin{subfigures}

(ref:dsc2a) \textbf{Omni-ATAC profiles of PND15 and adult spermatogonial cells and their genomic distribution.} \newline \textbf{A:} Genomic distribution of the 158, 978 identified Omni-ATAC regions in between exonic, intronic, intergenic and located +/- 1kb of a transcriptional start site (TSS).

```{r ds2a, echo=FALSE, fig.align='center', fig.cap="(ref:dsc2a)", fig.scap="Omni-ATAC profiles of PND15 and adult spermatogonial cells and their genomic distribution"}
img <- readPNG("figure/dev/s2a.png")
grid.raster(img)
```

(ref:dsc2b) \textbf{Omni-ATAC profiles of PND15 and adult spermatogonial cells and their genomic distribution.} \newline \textbf{B:} Dot plots of top enriched GO biological processes for regions with increased chromatin accessibility in adult spermatogonia, within gene bodies and around TSS of nearby genes (TSS +/- 1kb). The size of dots indicates the number of genes in the term and the colour of each dot corresponds to the adjusted P-value of the term’s enrichment.

```{r ds2b, echo=FALSE, fig.align='center', fig.cap="(ref:dsc2b)", fig.scap="Omni-ATAC profiles of PND15 and adult spermatogonial cells and their genomic distribution"}
img <- readPNG("figure/dev/s2b.png")
grid.raster(img)
```

\end{subfigures}


(ref:dsc3) \textbf{Representative examples from Categories 1-3 resulted from the overlap of chromatin accessibility, gene expression and histone profiling datasets.} Genomic snapshots from the Integrative Genomics Viewer (IGV, Broad Institute) of exemplary genes from \textbf{A:} Category 1 (Pdpk1 and Gata2), \textbf{B:} Category 2 (Hmx1 and Fgf8), and \textbf{C:} Category 3 (Dap2ip) showing the relative abundance of transcripts from RNA-seq, chromatin accessibility from ATAC-seq and enrichment of H3K27ac, H3K4me3 and H3K27me3 from ChIP-seq. RNA-seq data correspond to literature PND14 and PNW8 spermatogonial cells and  ATAC-seq data to PND15 and adult spermatogonial cells.

```{r ds3, echo=FALSE, fig.align='center', fig.cap="(ref:dsc3)", fig.scap="Representative examples from Categories 1-3 resulted from the overlap of chromatin accessibility, gene expression and histone profiling datasets"}
img <- readPNG("figure/dev/s3.png")
grid.raster(img)
```

(ref:dsc4) \textbf{DNAme profiles of spermatogonial cells do not show major changes over the period of testis postnatal maturation.} Enriched heatmaps showing the overlap between Category 1-4 regions, literature ChIP-seq data in PNW8 spermatogonia for H3K4me3, H3K27ac and H3K27me3, and DNAme data from BS in PND7, PND14 and PNW8 spermatogonia. For each of the Category 1-4 the following sub-categorization was applied: regions that are enriched for H3K4me3 (with or w/o H3K27ac and/or H3K27me3), regions that are enriched for H3K27ac (and lack both H3K4me3 and H3K27me3) and regions that are enriched for H3K27me3 (and lack both H3K4me3 and H3K27ac).  Each line represents a peak region and the regions are ordered by the ATAC-seq signal. Mid-x-axis corresponds to the middle of a peak region and is extended to +/- 1 kbp. The colour-key of the ATAC-seq, ChIP-seq and BS heatmaps represent ATAC-seq, ChIP-seq and BS signal, respectively. For RNA-seq, Log\textsubscript{2} FC is shown from PND8 vs PND15 and PND14 vs PNW8 comparisons. For BS, the level of DNAme is between 0 and 1, with 0 representing completely unmethylated loci and 1 fully methylated loci, respectively.

```{r ds4, echo=FALSE, fig.align='center', fig.cap="(ref:dsc4)", fig.scap="DNAme profiles of spermatogonial cells do not show major changes over the period of testis postnatal maturation"}
img <- readPNG("figure/dev/s4.png")
grid.raster(img)
```


(ref:dsc5) \textbf{Distinct chromatin profiles between PND15 and adult spermatogonia are present at distal regions across the genome.} \textbf{(A-B)} Enriched heatmaps showing the overlap between more accessible regions \textbf{(A)} and less accessible regions \textbf{(B)} situated in distal regions in spermatogonial cells, and literature ChIP-seq and DNAme data in PNW8 spermatogonia. The following sub-categorization was applied: regions that are enriched for H3K4me3 (with or w/o H3K27ac and/or H3K27me3), regions that are enriched for H3K27ac (and lack both H3K4me3 and H3K27me3) and regions that are enriched for H3K27me3 (and lack both H3K4me3 and H3K27ac).  Each line represents a peak region and the regions are ordered by the ATAC-seq signal. Mid-x-axis corresponds to the middle of a peak region and is extended to +/- 1 kbp. The colour-key of the ATAC-seq, ChIP-seq and BS heatmaps represent ATAC-seq, ChIP-seq and BS signal, respectively. For BS, the level of DNAme is between 0 and 1, with 0 representing completely unmethylated loci and 1 fully methylated loci, respectively.


```{r ds5, echo=FALSE, fig.align='center', fig.cap="(ref:dsc5)", fig.scap="Distinct chromatin profiles between PND15 and adult spermatogonia are present at distal regions across the genome"}
img <- readPNG("figure/dev/s5.png")
grid.raster(img)
```


(ref:dsc6) \textbf{Differentially accessible TEs exhibit enriched TF motifs and correspond to regions nearby non-random gene families.} \textbf{A:} HOMER extracted consensus sequences for TF motifs enriched in less accessible ERVK subtypes. Representative examples from the most enriched transcription factor families are depicted. \textbf{B:} Genomic snapshots from the Integrative Genomics Viewer (IGV, Broad Institute) of a cluster of Olfr genes on Chr2 overlapping an increased density of LINE elements relative to the neighbouring regions. The relative abundance of transcripts from RNA-seq and chromatin accessibility from ATAC-seq are shown. RNA-seq data corresponds to literature PND14 and adult (PNW8) spermatogonial cells and ATAC-seq data to PND15 and adult (PNW20) spermatogonial cells, respectively.


```{r ds6, echo=FALSE, fig.align='center', fig.cap="(ref:dsc6)", fig.scap="Differentially accessible TEs exhibit enriched TF motifs and correspond to regions nearby non-random gene families"}
img <- readPNG("figure/dev/s6.png")
grid.raster(img)
```


\newpage

## Supplementary tables{#st}

Supplementary tables are too big to be included in the thesis, hence they are provided as links

1. [Supplementary table 1](../../figures/dev/t1.xlsx)
2. [Supplementary table 1](../../figures/dev/t2.xlsx)
3. [Supplementary table 1](../../figures/dev/t3.xlsx)
4. [Supplementary table 1](../../figures/dev/t4.xlsx)
5. [Supplementary table 1](../../figures/dev/t5.xlsx)
6. [Supplementary table 1](../../figures/dev/t6.xlsx)

<!--chapter:end:rmd/06-dev/4.Rmd-->

---
output: html_document
editor_options: 
  chunk_output_type: console
# bibliography: []
---

\chaptermark{Chapter 2}

# Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse {#chapter2}

Anar Alshanbayeva\textsuperscript{1,2,3}, **Deepak K
Tanwar**\textsuperscript{1,2,3}, Martin
Roszkowski\textsuperscript{1,2,3}, Francesca
Manuella\textsuperscript{1,2,3}, Isabelle M
Mansuy\textsuperscript{1,2,3,\#} \newline \newline
\textsuperscript{1}Laboratory of Neuroepigenetics, Brain Research
Institute at the Medical Faculty of the University of Zurich. \newline
\textsuperscript{2}Institute for Neuroscience of the Department of
Health Sciences and Technology, ETH Zurich, Zurich, Switzerland.
\newline \textsuperscript{3}Zurich Neuroscience Center, ETH and
University of Zurich, Zurich, Switzerland. \newline \newline
\textsuperscript{\#}Corresponding author \newline \newline
**Journal:** *Biology of Reproduction* \newline \newline 
**DOI:** [10.1093/biolre/ioab156](https://doi.org/10.1093/biolre/ioab156)
\newline \newline
__Contributions:__ *I performed data analysis with Anar Alshanbayeva,
generated figures with Anar Alshanbayeva, assisted Anar Alshanbayeva in
writing the manuscript, and revised manuscript with Anar Alshanbayeva.*

\newpage

## Graphical Abstract

```{r cf1, echo=FALSE, fig.align='center', fig.cap="Graphical abstract.", fig.scap="Graphical abstract", fig.width=6.4, fig.height=4.45}
img <- readPNG("figure/anaraEV/ga.png")
grid.raster(img)
```


\newpage

## Abstract

Sperm RNA can be modified by environmental factors and has been
implicated in communicating signals about changes in a father's
environment to the offspring. The small RNA composition of sperm could
be changed during its final stage of maturation in the epididymis by
extracellular vesicles (EVs) released by epididymal cells. We studied
the effect of exposure to stress in early postnatal life on the
transcriptome of epididymal EVs using a mouse model of transgenerational
transmission. We found that the small RNA signature of epididymal EVs,
particularly miRNAs, is altered in adult males exposed to postnatal
stress. In some cases, these miRNA changes correlate with differences in
the expression of their target genes in sperm and zygotes generated from
that sperm. These results suggest that stressful experiences in early
life can have persistent biological effects on the male reproductive
tract that may in part be responsible for the transmission of the
effects of exposure to the offspring.

## Summary sentence

miRNA cargo of extracellular vesicles in cauda epididymis is altered by
paternal exposure to early life stress. This correlates with changes in
the expression of target genes in sperm and in zygotes generated from
that sperm.


## Key words

epigenetics, epididymis, epididymosomes, early life stress,
extracellular vesicles, miRNAs, sperm.

\newpage

## Introduction

Post-testicular maturation of spermatozoa in the epididymis is an
elaborate process that involves modifications of sperm RNA, protein, and
lipid content [@nixon_2015; @rejraji_2006; @sharma_2016; @skerget_2015;
@tamessar_2021]. The epididymis is segmented into different parts,
including the initial segment, caput, corpus, and cauda. Each segment
has a distinct gene expression profile, and different protein and lipid
composition. Some modifications in epididymal spermatozoa are conserved
across species [@sellem_2020]. For example, \~50% of miRNAs, a class of
small RNAs that are modified during caput to cauda epididymis transit,
is identical in mouse and bovine spermatozoa [@sellem_2020]. One
mechanism by which small RNA load in spermatozoa is modified along the
epididymis is by uptake of extracellular vesicles (EVs), also known as
epididymosomes, which are produced by epididymal epithelial cells
[@reilly_2016]. Studies have shown that epididymosomes can be taken up
by maturing sperm through proteins present on the sperm head such as
dynamin in mice and tetraspanins or syntenins in humans
[@caballero_2013; @reilly_2016; @thimon_2008; @zhou_2019]. Co-incubation
experiments provided evidence for epididymosome-mediated transfer of
miRNAs to spermatozoa [@reilly_2016]. Exogenous DNA and RNA can also be
directly taken up by spermatozoa via artificial liposomes
[@bachiller_1991].

However, it is still not clear if changes in small RNA composition of
spermatozoa occurring during epididymal transit are required for
embryonic development, and studies on the subject have been conflicting
[@conine_2018; @suganuma_2005]. Changes in sperm small RNA have
nevertheless been suggested to play a role in the transmission of
information about paternal experiences to the progeny and can influence
their developmental trajectory [@chan_2020; @gapp_2014; @sharma_2016].
Epididymosomal small RNA content can also be altered by exposure, for
instance, to dietary insult and stress [@chan_2020; @sharma_2016]. For
instance, epididymosomal miRNAs are changed by exposure to chronic
stress [@chan_2020] and low-protein diet [@sharma_2016] in mice.

Transmission of information about paternal exposure to the offspring
depends on the type of exposure, its duration and the developmental
window at which it is applied. To date, little is known about the
long-term effects of early life stress, particularly stress experienced
after birth, on epididymosomal small RNA composition in adulthood, and
whether any changes to this composition can influence gene expression in
sperm and in zygotes generated from that sperm. Using a
transgenerational mouse model of postnatal stress induced by
unpredictable maternal separation combined with unpredictable maternal
stress, unpredictable maternal separation combined with unpredictable
maternal stress (MSUS) [@franklin_2010], we show that the miRNA
signature of cauda epididymosomes in adult males is altered by exposure,
and that this alteration, correlates with changes in the expression of
their target genes in sperm and in zygotes.

## Results

### Isolation of cauda epididymosomes confirmed by several methods

To characterize the RNA composition of cauda epididymosomes,
epididymosomes were isolated by high-speed ultracentrifugation from
adult control males and males exposed to MSUS (Figure \ref{fig:ef1}A). MSUS
consists of exposing newborn pups to unpredictable maternal separation
for 3h daily and subjecting dams to unpredictable maternal stress during
separation [@franklin_2010]. Adult MSUS-exposed and control males were
euthanized and cauda epididymis was collected. Successful isolation of
cauda epididymosomes was confirmed by electron microscopy,
immunoblotting, and nanoparticle-tracking analyses (Figure \ref{fig:ef1}). The
presence and purity of epididymosomes was further validated by staining
with the EV-specific marker CD9 and confirmation of the absence of the
cellular marker Glyceraldehyde 3-phosphate dehydrogenase (GAPDH)
(Figure \ref{fig:ef1}B). Size analysis by nanoparticle-tracking indicated that the
collected particles are 50--300nm in diameter (Figure \ref{fig:ef1}D). Imaging by
transmission electron microscopy showed the typical cup-shaped
structures of epididymosomes (Figure \ref{fig:ef1}C, Supplementary Figure \ref{fig:es1}A)
[@choi_2017]. RNA profiling by high-resolution automated electrophoresis
showed enrichment for small RNAs of different length, similar to
previous studies on cauda epididymosomal RNA (Supplementary Figure \ref{fig:es1}B) [@sharma_2016; @conine_2018].


### The number and size of epididymosomes in adult males are not altered by postnatal stress

We next examined the number and size of cauda epididymosomes in adult
MSUS and control males by dynamic light scattering. No significant
difference could be detected between MSUS and control males (Figure \ref{fig:ef2}A
and B). Since most epididymosomal production occurs via apocrine
secretion from principal cells located in caput epididymis, we also
examined the expression of genes involved in EVs exocytosis. We chose
Ras-related protein Rab-5A (*Rab5*) and Ras-related protein Rab-7A
(*Rab7*), which are involved in vesicle trafficking, the soluble NSF
attachment protein (SNARE) family protein vesicle-associated membrane
protein 7 (*Vamp7*) and SNARE recognition molecule synaptobrevin homolog
YKT6 (*Ykt6*), involved in vesicle fusion. No significant change in the
expression of these genes could be detected in caput epididymis between
MSUS and control adult males (Figure \ref{fig:ef2}C). However, we observed a
consistent trend (not statistically significant) for decreased
expression of all genes involved in EVs secretion in caput epididymis of
MSUS mice (Figure \ref{fig:ef2}C).

### miRNAs are persistently altered by postnatal stress in cauda epididymosomes

Epididymosomal small RNAs are known to be affected by changing
environmental conditions in rodents. Small RNAs, like tRNA-derived
fragments (tRFs), miRNAs and more recently rRNA-derived small RNA
fragments are believed to act as messengers of a father's experiences
that can be transferred to the offspring [@sharma_2016; @chan_2020;
@wang_2021; @zhang_2018]. Our previous work showed that early postnatal
stress alters small and long RNA content in sperm of adult males
[@gapp_2014; @gapp_2020]. Since caudal sperm and epididymosomal small RNA
profiles are highly similar [@sharma_2016], we examined whether small RNA
content of cauda epididymosomes is also altered by postnatal stress. We
extracted RNA from cauda epididymosomes of adult MSUS and control males
and prepared small RNA-seq (sRNA-seq) libraries. RNAs of different size
were observed in cauda epididymosomes, with the majority of small RNA
reads mapping to tRNAs as previously observed (Figure \ref{fig:ef3}A)
[@sharma_2016]. When plotting the results of differential expression
analysis of small RNAs (*P*\<0.05) between MSUS and control sample, the
majority of nonsignificant differences in small RNAs appeared to be in
miRNAs, although some differences in tRNAs and piRNAs were also detected
(Figure \ref{fig:ef3}B). We next performed size-selection on the same libraries to
enrich for miRNAs and then, we re-sequenced the libraries
(Supplementary Figure \ref{fig:es2}A and C). As expected, size-selection did not
alter the abundance of miRNAs and uniformly enriched the miRNA fraction
in all samples (Supplementary Figure \ref{fig:es2}B--D). Differential expression
analysis of miRNAs biotypes from the combined sequencing datasets after
batch effect correction revealed changes in several miRNAs in MSUS cauda
epididymosomes (Supplementary Table S\ref{tab:est1}). These include upregulation of
miR-878-5p, miR34c-5p, miR-881-3p and downregulation of miR-31-5p and
miR-155-5p (adjusted *P* $\geq$ 0.1). Differential expression analysis
on all small RNA biotypes from the combined datasets showed that 70% of
all significantly altered small RNAs correspond to miRNAs, 15% to tRNAs,
15% to piRNAs and snoRNAs, while rRNAs are not changed (Supplementary
Table S\ref{tab:est4}). Pathway analysis of top candidate RNAs from miRNA-based
analysis after size-selection revealed that the most up- and
down-regulated miRNAs (*P*\<0.05) have target mRNAs that encode proteins
involved in fatty acid metabolism, steroid biosynthesis, lysine
degradation, and thyroid hormone signaling (Figure \ref{fig:ef3}C). Notably,
similar pathways are altered in plasma of MSUS males during postnatal
life and adulthood as shown by metabolomic analysis [@vansteenwyk_2020]. In
particular, metabolites implicated in polyunsaturated fatty acid
biogenesis were up-regulated, whereas steroidogenesis and the
steroidogenic ligand aldosterone were down-regulated [@vansteenwyk_2020].
Steroidogenesis was already altered at postnatal day 28 in MSUS males,
with total cholesterol significantly decreased in testis (Figure \ref{fig:ef3}D)
and HDL cholesterol significantly increased in liver (Figure \ref{fig:ef3}F).
Since the primary role of HDL cholesterol in blood is to transport
excess cholesterol from peripheral tissues to liver, an increase in HDL
in liver is consistent with a decrease in testis. However, cholesterol
was no longer altered in testis of adult MSUS males (Figure \ref{fig:ef3}E),
suggesting a transient alteration. The androgen receptor, which the
cholesterol derivatives, androgens bind to, was decreased in adult caput
epididymis (Figure \ref{fig:ef3}G), suggesting potential secondary effects of
lower cholesterol in testis when occurring in early postnatal life.

### mRNA targets of miRNAs from cauda epididymosomes are altered by postnatal stress in sperm and in zygotes

The relative abundance of miRNAs in cauda epididymosomes and mature
sperm significantly correlated (Figure \ref{fig:ef4}C), consistent with prior
findings [@sharma_2016; @chan_2020]. Since cauda epididymosomes carry
small RNA payloads matching those of mature sperm and are part of the
ejaculate [@frenette_2005; @belleanne_2013a], they may contribute to the
information delivered to the oocyte upon fertilization. Therefore, we
looked at the mRNA targets of miRNAs significantly changed in MSUS cauda
epididymosomes in two previously published analysis of genes identified
in MSUS sperm and in zygotes derived from MSUS males (*P* \< 0.05)
[@gapp_2020]. For this, we plotted the cumulative log fold-change
distribution of all genes from differential expression analysis of sperm
or zygotes versus the number of conserved binding sites for miRNAs
significantly changed by MSUS in cauda epididymosomes (Figure \ref{fig:ef4}A and
B, Supplementary Figure \ref{fig:es5}). Target genes with three binding sites
for miR-31-5p, a miRNA differentially expressed in MSUS cauda
epididymosomes, had increased expression in sperm and decreased
expression in zygotes from MSUS males (Figure \ref{fig:ef4}A and B, Supplementary
Table S\ref{tab:est2} and S\ref{tab:est3}). However, not all targets of miRNAs significantly
altered in MSUS cauda epididymosomes showed corresponding changes in
expression in sperm and zygotes (Supplementary Figure \ref{fig:es5}). We then
conducted miRNA-gene interaction analysis based on experimentally
validated data from Tarbase [@vlachos_2017]. This analysis showed that,
overall, the five miRNAs significantly changed in MSUS cauda
epididymosomes target genes that are part of pathways involved in
steroid biosynthesis, extracellular matrix (ECM)--receptor interaction,
and cell-adhesion molecules (Figure \ref{fig:ef4}D).

## Discussion

The effects of environmental factors on RNA in the male reproductive
tract, in particular, the epididymis have been examined in rodent
models. Until now, most models have used invasive exposure such as
dietary insult or injection of endocrine disruptors, applied prenatally
and sometimes before conception. Few studies have examined the effects
of non-invasive psychological/emotional exposure such as stress in early
life and the effects on epididymal RNA in adulthood [@chan_2020]. This
study examines if postnatal stress affects RNAs in EVs released from the
cauda epididymis and whether this has consequences for mature sperm and
zygotes generated from that sperm.

Using a transgenerational mouse model of early postnatal stress, we show
that several miRNAs, including miR-871-3p, miR-31-5p, miR-155-5p,
miR-878-5p, and miR-34c-5p are altered in cauda epididymosomes in adult
males exposed to postnatal stress, and that the targets of some of these
miRNAs are affected in mature sperm and zygotes. Particularly, miR-31-5p
is significantly decreased in cauda epididymosomes and its target genes
are up-regulated in sperm but down-regulated in zygotes generated from
that sperm, suggesting an over-compensation during early development.
This may also be due to the heterogeneity of epididymosomes which have
different size, biogenesis, and cellular targeting [@sullivan_2015],
leading to a dissociation between the RNA content of epididymosomes and
transcriptional changes in zygotes. It has been suggested that different
subsets of epididymosomes have different roles. While a subset
communicates with spermatozoa during sperm epididymal transit
[@reilly_2016; @sharma_2016], another subset serves in the communication
within epididymal epithelial cells [@belleanne_2013a], and a third one
is delivered as part of seminal fluid during fertilization
[@belleanne_2013; @frenette_2005]. Thus owing to their heterogeneity,
not all cauda epididymosomes or their cargo is delivered to the oocyte
upon fertilization, which may explain the differences in miRNAs targets
that are affected in sperm and zygotes.

Several of the differentially expressed miRNAs in MSUS cauda
epididymosomes play a role in metabolic processes and early development
[@reza_2019]. For instance, miR-31-5p is involved in glucose metabolism
and fatty acid oxidation [@reza_2019]. In humans, its target complement
C1q Tumor Necrosis Factor-Related Protein 9A (CTRP9) protein is
negatively correlated with the amount of visceral fat and positively
associated with a beneficial glucose and metabolic phenotype
[@shao_2017]. This is consistent with the observation that glucose and
insulin metabolism are also affected by MSUS [@franklin_2010;
@gapp_2014]. The level of other miRNAs is significantly increased or
decreased in MSUS epididymosomes, such as miR-155-5p, which facilitates
differentiation of mouse embryonic stem cells, or miR-34c-5p that
initiates the first embryonic cleavage in mice [@reza_2019].

The first days after birth are a sensitive period for the development
and the establishment of cellular niches in tissues. Epithelial cells in
the epididymis, which are the source of epididymosomes, undergo
differentiation and expansion postnatally until puberty [@na_2002]. Once
their expansion is completed, epididymal epithelial cells remain at a
nearly constant number in adulthood. If they can be modified by prior
exposure, they may therefore carry a memory of exposure into adulthood.
The postnatal development and differentiation of epididymal epithelial
cells primarily depend on testicular signals [@biliska_2006; @na_2002;
@robaire_2011; @zhu_2000]. Since chronic stress affects the coupling of
the hypothalamus--pituitary and hypothalamus--gonadal axes,
stress-related decrease in steroidogenesis can profoundly affect the
differentiation and expansion of epididymal epithelial cells in early
postnatal life. A number of studies have shown the importance of the
abundance of androgens during postnatal life for epididymal development
[@na_2002]. Thus, the availability of testicular cholesterol during
epididymal cells differentiation has implications for these cells.
Systemic alteration in cholesterol metabolism seen in young MSUS males
(decreased total cholesterol in testis and increased HDL cholesterol in
liver) may contribute to metabolic changes seen in adult animals, for
instance in plasma steroidogenesis and fatty acid pathways, and to
alterations in glucose and insulin metabolism in adult MSUS males.
Moreover, androgen-dependent miRNAs miR-878-5p and miR-871-3p are
significantly increased in cauda epididymosomes in MSUS.

In conclusion, our results provide evidence that chronic stress in early
postnatal life alters miRNAs in EVs of the male reproductive tract in
adulthood, with effects in mature sperm and zygotes. These persistent
and intergenerational effects in vivo point to the sensitivity of the
reproductive system to stress exposure and the detrimental consequences
for descendants. These consequences may differ depending on the time
window and severity of paternal stress exposure. Further studies are
necessary to more precisely define these effects and the source of
vesicles and their cargo miRNAs.


## Materials and methods

### Animals

Animal experiments were conducted according to the Swiss Law for Animal
Protection and were approved by the cantonal veterinary office in Zürich
under license number 83/2018. C57Bl/6 J mice were obtained from Janvier
(France) and bred in-house to generate animals for the experiments.
Animals were maintained under a reverse light--dark cycle in a
temperature and humidity-controlled environment with constant access to
food and water. Nine months old age-matched MSUS and control males were
used for small RNA-sequencing (sRNA-seq) of cauda epididymosomes, tissue
collection for RT-qPCR, nanoparticle-tracking analysis, and total
cholesterol measurements. HDL cholesterol and total cholesterol
measurements were performed on MSUS and control males at postnatal day
28. Datasets from previous publication [@gapp_2020]: caudal sperm
RNA-seq was performed on 5-months old males, and zygote RNA-seq was
performed on zygotes from 3-months old males.

### MSUS

To obtain MSUS mice, 3-months old C57Bl/6 J females and their litters
were subjected to daily 3 h separation unpredictably and females were
exposed to an unpredictable stressor during separation as previously
described [@franklin_2010]. Control dams and pups were left undisturbed.
After weaning at postnatal day 21, pups from different litters were
randomly assigned to cages of 4--5 mice, in corresponding treatment
groups to avoid litter effects.

### Tissue collection

After decapitation and blood collection, mice were pinned on a
dissection board and cleaned with alcohol. Epididymis and testis were
excised and separated from surrounding adipose tissue. The epididymis
was separated into caput, corpus, and cauda. Cauda was excised with
several incisions and sperm collected with a swim-up protocol. The
supernatant was collected to isolate epididymosomes. The whole testis
and caput epididymis were crushed with stainless steel beads in a tissue
crusher in cold PBS, centrifuged at 3000 rcf for 10 min to pellet the
tissue and cells and used for total cholesterol and HDL cholesterol
measurements.

### Electron microscopy images

Negative staining of cauda epididymosomes was performed with
methylcellulose. Briefly, the carrier grid was glow-discharged in plasma
for 10 min and washed with a drop of PBS, then incubated in 1%
glutaraldehyde (GA) in water for 5 min and washed with water five times
for 2 min each. Afterwards the grid was incubated in 1% UAc (uranyl
acetate) for 5 min and then kept on ice in methylcellulose/UAc (900 ul
methylcellulose 2% and 100 ul 3% UAc) solution. After incubation with
methylcellulose/UAc, the excess liquid was removed by dipping onto a
filter paper. The grid was air-dried on ice for 5 min. Imaging was
performed with a transmission electron microscope.

### Epididymosomes isolation by ultracentrifugation

After pelleting sperm following the sperm swim-up protocol in M2 medium
(Sigma, M7167), the supernatant was centrifuged at 2000 rcf for 10 min,
10 000 rcf for 30 min and then ultracentrifuged at 120 000 rcf at 4
$^{\circ}$C for 2 h (TH 64.1 rotor, Thermo Fisher Scientific). The
epididymosomal pellet was then washed in PBS at 4 $^{\circ}$C and
ultracentrifuged at 120 000 rcf at 4 $^{\circ}$C for 2 h. The resulting
pellet was resuspended in 60 \textmu l of PBS for all downstream
analysis.

### Immunoblotting

PBS-resuspended pellet containing epididymosomes was lysed in 10x RIPA
buffer for 5 min at 4 $^{\circ}$C. Equal amounts of protein were mixed
with 4x Laemmli Sample Buffer (Bio-Rad Laboratories, USA) and loaded on
4--20% Tris-Glycine polyacrylamide gels (Bio-Rad Laboratories, USA). The
membranes were blocked in 5% SureBlock (in Tris-buffer with 0.05%
Tween-20) for 1 h at room temperature and incubated with overnight at 4
$^{\circ}$C with primary anti-Cd9 ([1:3000; System Biosciences, USA] and
anti-Gapdh [1:5000; Cell Signaling, USA; 14C10]) antibodies.

### Nanoparticle tracking analysis

Particle number and size of epididymosomes were measured using a
Nanosight NS300 (Malvern, UK) at 20 $^{\circ}$C, according to the
manufacturer's instructions and lots were generated using a published
method [@dragovic_2011]. The following parameters were kept constant for
all samples: "Camera level" = 14 and "Detection threshold" = 7. For
measurements with Nanosight, the resuspended pellet from
ultracentrifugation was diluted to a 1:1000 concentration.

### RNA isolation and epididymosomes profiling

To lyse purified epididymosomes, 33 \textmu l of lysis buffer (6.4 M
guanidine HCl, 5% Tween 20, 5% Triton, 120 mM EDTA, and 120 mM Tris pH
8.0) per 60 \textmu l of PBS resuspended pellet was added to each
sample, together with \textmu l Proteinase K and 3.3 \textmu l water.
Samples were incubated at 60 $^{\circ}$C for 15 min with shaking. A
total of 40 \textmu l water were added and RNA was extracted using
Trizol LS protocol, according to the manufacturer's instructions.
Profiling of extracted RNA was done using high-resolution automated
electrophoresis on a 2100 Bioanalyzer (Agilent, G2939BA), according to
instructions for the RNA 6000 Pico Kit (Agilent, 5067-1513) reagent.

### Preparation and sequencing of sRNA-seq libraries from epididymosomes

sRNA-seq libraries were prepared using the NEB Next Small RNA-sequencing
kit (NEB \#E7300, New England BioLabs), according to the manufacturer's
instructions. About 80--90 ng of total RNA per sample was used to
prepare the libraries. The same libraries were sequenced before and
after size-selection (target peak 150 bp) with the BluePippin System.
200 million reads were obtained for 10 samples, with 125 bp
single-stranded read-length on a HiSeq2500 sequencer.

### RT-qPCR

For gene expression analysis in caput epididymis, RNA was extracted
using the phenol/chloroform extraction method (TRIzol; Thermo Fisher
Scientific). Reverse transcription was performed using miScript \RN{2}
RT reagents (Qiagen) - HiFlex buffer, and RT qPCR was performed with
QuantiTect SYBR (Qiagen) on the Light-Cycler \RN{2} 480 (Roche). All
samples were run in cycles as follows: 95 $^{\circ}$C for 15 min, 45
cycles of 15 s at 94 $^{\circ}$C, 30 s at 55 $^{\circ}$C and 30 s at 70
$^{\circ}$C, followed by gradual increase of temperature to $^{\circ}$C.
The endogenous control \textit{Gapdh} was used for normalization. The
expression level of genes was analyzed using two-tailed Student's
\textit{t}-test.

### Cholesterol measurements

Testicular and epididymal total cholesterol and HDL cholesterol levels
were measured using the CHOL reagent, in conjunction with SYNCHRON LX
System(s), UniCel DxC 600/800 System(s) and Synchron Systems Multi
Calibrator (Beckman Coulter), according to the manufacturer's
instructions at the Zurich Integrative Rodent Physiology (ZIRP) facility
of the University of Zurich.

### Bioinformatics data analysis

sRNA-sequencing FASTQ files were processed using the `ExceRpt` pipeline,
previously established for EV small RNA data analysis [@rozowsky_2019].
Briefly, `ExceRpt` first automatically identifies and removes known 3$'$
adapter sequences, then aligns against known spike-in sequences used for
library construction, filters low-quality reads and aligns them to
annotated sequences in the UniVec database. Reads not filtered out in
pre-processing steps are then aligned to the mouse genome and
transcriptome using `STAR` aligner [@dobin_2013]. The annotations were
performed in the following order: miRbase, tRNAscan, piRNA, GENCODE, and
circRNA. rRNA counts were obtained using `Oasis 2` tool. Reads mapped to
miRNAs were combined from sequencing obtained before and after
size-selection and corrected for batch effect using `RUVSeq`
[@leek_2014]. Normalization factors were calculated using `TMM`
[@robinson_2010] method and differential expression was performed using
`edgeR` package [@robinson_2010a] in `R`. For cumulative distribution
plots, miRNA targets (all and conserved) were downloaded from TargetScan
release 7.2 [@agarwal_2015]. When using context++ scores, targets were
split into three same-frequency groups according to their scores.
\textit{P}-values were calculated using a `Kolmogorov–Smirnov` test
between the first and last groups (i.e. strongest and weakest targets).
The miRNA pathway analysis was conducted using a web-server tool
`DIANA-miRPath` [@vlachos_2017], where targets were predicted-derived
from DIANA-TarBase v6.0, a database of experimentally validated miRNA
targets. The adjusted \textit{P} cutoff value of 0.05 was used for the
identification of expressed pathways. The miRNAs and their corresponding
target pathways information was extracted and plots were generated in
`R`. `ggplot2` [@wickham_2016] and `ComplexHeatmap` [@gu_2016] `R`
packages were used for generation of figures.

## Data availability

The datasets collected for this study are available as follows:

-   sRNA-seq dataset of cauda epididymosomes before and after
    sizeselection: NCBI GEO under accession number
    [GSE175976](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175976).

-   Codes for bioinformatics analysis of RNA-sequencing datasets and all
    corresponding differential expression analyses: Github repository
    [mansuylab/alshanbayeva_et_al_2021](https://github.com/mansuylab/alshanbayeva_et_al_2021).

-   Sperm and zygote sequencing datasets from previous publications can
    be found in ArrayExpress database at EMBL-EBI
    ([www.ebi.ac.uk/arrayexpress](www.ebi.ac.uk/arrayexpress)) with the
    accession number
    [E-MTAB-5834](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5834/)
    (sperm) and
    [E-MTAB-6589](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6589/)
    (zygotes).

## Authors' contributions

AA and IMM conceived and designed the study. FM and MR performed the
MSUS breeding and collected tissue samples. AA and DKT performed data
analysis and generated figures. AA wrote the manuscript with input from
DKT and IMM. AA performed all experiments for RNA sequencing and all
molecular analyses. IMM supervised the project and raised funds.

## Grant Support

The work was supported by Swiss National Science Foundation
(31003A-135715), ETH grants (ETH-10 15-2 and ETH-17 13-2), the Slack-Gyr
Foundation, the Escher Foundation. The Mansuy lab is funded by the
University Zürich, the Swiss Federal Institute of Technology, the Swiss
National Science Foundation (31003A-135715), ETH grants (ETH-10 15-2 and
ETH-17 13-2), the Slack-Gyr foundation, the Escher Foundation. Deepak K.
Tanwar is supported by the Swiss Government Excellence Scholarship.
Martin Roszkowski was funded by the ETH Zurich Fellowship (ETH-10 15-2).

## Acknowledgements

We thank Pierre-Luc Germain for advice on data analysis and for
generating cumulative distribution plots, Irina Lazar-Contes for help
with MSUS breeding, Silvia Schelbert for work on the animal license,
Emilio Yandez at Function Genomics Center Zurich (FGCZ) for advice on
the sRNA sequencing, Alekhya Mazumkhar for help with
nanoparticle-tracking analysis, Yvonne Zipfel for animal care, Zurich
Integrative Rodent Physiology facility for performing cholesterol
measurements. We also thank Eloise Kremer, Ali Jawaid, and Mea Holmes
for their initial contributions to the project.

**Conflict of interest:** The authors declare no conflict of interest.

\newpage

## Figures

(ref:efc1) \textbf{Isolation and characterization of cauda epididymosomes.} \newline \textbf{A:} Schematic representation of cauda epididymosomes preparation. \newline \textbf{B:} Immunoblot analysis was used to confirm the purity of isolation by staining with epididymosomal marker CD9 and absence of cellular marker GAPDH in the ultracentrifuged pellet. \newline \textbf{C:} Electron microscopy images of the preparations were used to assess the size and heterogeneity of the isolated populations. Arrows indicate cauda epididymosomes. \newline \textbf{D:} Nanoparticle-tracking analysis by dynamic light scattering showed particles of expected size of 50–300 nm.

```{r ef1, echo=FALSE, fig.align='center', fig.cap="(ref:efc1)", fig.scap="Isolation and characterization of cauda epididymosomes", fig.width=6.5, fig.height=3.54}
img <- readJPEG("figure/anaraEV/1.jpeg")
grid.raster(img)
```


(ref:efc2) \textbf{Comparison of epididymosomal number, size, and release machinery in adult MSUS and control males.} \newline \textbf{A:} Nanoparticle-tracking analysis showed no difference in the number of cauda epididymosomes between MSUS and control males. The plots were generated from average values across replicates (_N_ = 5 animals/group). Data are presented as mean ± standard error of the mean (SEM). _P_ < 0.05. \newline \textbf{B:} Quantification of nanoparticle-tracking analysis showed the mean size of cauda epididymosomes was not changed between the two groups (_N_ = 5 animals/group). Data are presented as mean ± SEM. _P_ < 0.05. \newline \textbf{C} Expression of genes involved in vesicular secretion in the caput epididymis from adult males measured by qRT-PCR. The experiments were performed in triplicates without pooling (_N_ = 8 animals/group). \newline Expression of GAPDH was used as endogenous control to normalize the expression level of the target genes. Data are presented as mean ± SEM. _P_ < 0.05.

```{r ef2, echo=FALSE, fig.align='center', fig.cap="(ref:efc2)", fig.width=5.85, fig.height=5.78, fig.scap="Comparison of epididymosomal number, size, and release machinery in adult MSUS and control males"}
img <- readJPEG("figure/anaraEV/2.jpeg")
grid.raster(img)
```


(ref:efc3) \textbf{Target pathways of up- and down-regulated miRNAs in adult MSUS cauda epididymosomes and alterations in steroidogenesis in postnatal and adult MSUS males.} \newline \textbf{A:} Representative distribution of RNA biotypes from cauda epididymosomal sequencing (_N_ = 10 animals, 5 animals/group). \newline \textbf{B:} Heatmap of the most abundant small RNAs (_n_ = 39). Expression fold-change (log\textsubscript{2} FC) was calculated by subtracting log\textsubscript{2} counts per million of MSUS from controls. Each row depicts a small RNA and each column depicts a sample. Samples and RNAs are ordered by “PCA” method using `seriation` (R package). \newline \textbf{C:} Dot plot of miRNAs and pathways. Color-scale of the dot represents -log\textsubscript{10} adjusted _P_ of miRNA in a pathway and size of the dot represents –log\textsubscript{10} adjusted _P_ of the pathway. \newline Total cholesterol measurements in whole testes of males at postnatal day 28 (_N_ = 4 males/group) \textbf{D} and adulthood (_N_ = 10 controls, _N_ = 7 MSUS) \textbf{E}. \textbf{F:} HDL cholesterol level in the liver of males at postnatal day 28 (_N_ = 6 males/group). \newline \textbf{G:} Relative expression level of the androgen receptor in adult caput epididymis measured by qRT-PCR. qRT-PCR experiments were performed in triplicates, without pooling (_N_ = 10 animals/group). \newline \textbf{(D–G)} Data are presented as mean ± SEM. _P_ < 0.05.

```{r ef3, echo=FALSE, fig.align='center', fig.cap="(ref:efc3)", fig.width=4, fig.height=6.3, fig.scap="Target pathways of up- and down-regulated miRNAs in adult MSUS cauda epididymosomes and alterations in steroidogenesis in postnatal and adult MSUS males"}
img <- readJPEG("figure/anaraEV/3.jpeg")
grid.raster(img)
```


(ref:efc4) \textbf{Targets of miRNAs from cauda epididymosomes are altered by MSUS in sperm and zygotes.} \newline Cumulative distribution plots of miR-31-5p targets from 5-months old MSUS and control caudal sperm RNA-seq \textbf{A} and from RNA-seq of zygotes originating from 3-months old MSUS and control males \textbf{B}. \newline \textbf{C:} miRNA abundance of sperm plotted against abundance in cauda epididymosomes. Coefficient of determination (_R_\textsuperscript{2}=0.74). \newline \textbf{D:} Dot plot of the top target pathways (adjusted _P_ < 0.05) of miRNAs differentially expressed (adjusted _P_ $\leq$ 0.1) in adult MSUS cauda epididymosomes. Color-scale of the dot represents –log\textsubscript{10} adjusted _P_ of miRNA in a pathway and size of the dot represents –log\textsubscript{10} adjusted _P_ of the pathway.

```{r ef4, echo=FALSE, fig.align='center', fig.cap="(ref:efc4)", fig.width=6.49, fig.height=5.59}
img <- readJPEG("figure/anaraEV/4.jpeg")
grid.raster(img)
```

## Supplementary figures

(ref:esc1) \textbf{Electron microscopy image and electrophoresis.} \newline \textbf{A:} Electron microscopy image of the preparations were used to access the size and heterogeneity of the isolated populations. \newline \textbf{B:} The RNA amount of epididymosomes preparations was assessed by high-resolution automated electrophoresis.

```{r es1, echo=FALSE, fig.align='center', fig.cap="(ref:esc1)", fig.width=5.15, fig.height=1.93, fig.scap="Electron microscopy image and electrophoresis"}
img <- readPNG("figure/anaraEV/s1.png")
grid.raster(img)
```


(ref:esc2) \textbf{Data overview.} \newline \textbf{A:} RNA read-length distributions of cauda epididymosomal small RNAs after size-selection. \newline \textbf{B:} Heatmap of top miRNAs of cauda epididymosomal small RNA-seq before and after size-selection. Each row depicts a miRNA and each column depicts a sample. (_N_ = 10 Controls, _N_ = 10 MSUS). \newline \textbf{C:} Representative distribution of RNA biotypes from cauda epididymosomal small RNA-seq after size-selection. (_N_ = 5 Controls, _N_ = 5 MSUS). \newline \textbf{D} miRNA abundance distributions plot of cauda epididymosomal small RNA-seq. (_N_ = 10 Controls, _N_ = 10 MSUS)
 
```{r es2, echo=FALSE, fig.align='center', fig.cap="(ref:esc2)", fig.width=5.66, fig.height=4.83, fig.scap="Data overview"}
img <- readPNG("figure/anaraEV/s2.png")
grid.raster(img)
```

(ref:esc3) \textbf{Scatterplot of log fold-change of miRNAs in sperm and cauda epididymosomes.} miRNAs down-regulated (blue) and up-regulated (red) in MSUS cauda epididymosomes.

```{r es3, echo=FALSE, fig.align='center', fig.cap="(ref:esc3)", fig.width=1.9, fig.height=1.9, fig.scap="Scatterplot of log fold-change of miRNAs in sperm and cauda epididymosomes"}
img <- readPNG("figure/anaraEV/s3.png")
grid.raster(img)
```

(ref:esc4) \textbf{Profiles of cauda epididymosomal small RNAs by high-resolution automated electrophoresis.} (_N_ = 10 Controls, _N_ = 10 MSUS).

```{r es4, echo=FALSE, fig.align='center', fig.cap="(ref:esc4)", fig.width=5.58, fig.height=3.99, fig.scap="Profiles of cauda epididymosomal small RNAs by high-resolution automated electrophoresis"}
img <- readPNG("figure/anaraEV/s4.png")
grid.raster(img)
```

(ref:esc5) \textbf{Cumulative distribution plots of miR-34c-5p, miR-155-5p targets in differentially expressed genes (P $\le$ 0.05) from MSUS sperm and zygote RNA-sequencing.}

```{r es5, echo=FALSE, fig.align='center', fig.cap="(ref:esc5)", fig.width=5.33, fig.height=4.06, fig.scap="Cumulative distribution plots of miRNA targets in differentially expressed genes"}
img <- readPNG("figure/anaraEV/s5.png")
grid.raster(img)
```

(ref:esc6) \textbf{Immunoblot images of cauda epididymosomes and liver cell lysate, stained with cellular marker GAPDH (37kDa) and extracellular vesicle marker CD9 (25kDa).} Same membrane was incubated and imaged separately.

```{r es6, echo=FALSE, fig.align='center', fig.cap="(ref:esc6)", fig.width=5.54, fig.height=2.19, fig.scap="Immunoblot images of cauda epididymosomes and liver cell lysate, stained with cellular marker GAPDH (37kDa) and extracellular vesicle marker CD9 (25kDa)"}
img <- readPNG("figure/anaraEV/s6.png")
grid.raster(img)
```




## Supplementary Tables

```{r est1, message=FALSE, warning=FALSE, echo=FALSE}
library(readxl)
df1 <- data.frame(read_xlsx("figure/anaraEV/t1.xlsx", sheet = 1), check.names = F)

library(kableExtra)

kbl(df1, booktabs = T, caption = "Supplementary table 1") %>%
kable_styling(latex_options = c("striped", "scale_down")) %>%
  kable_styling(latex_options = c("HOLD_position")) 
```


```{r est2, message=FALSE, warning=FALSE, echo=FALSE}
library(readxl)
df2 <- data.frame(read_xlsx("figure/anaraEV/t2.xlsx", sheet = 1), check.names = F)

library(kableExtra)

kbl(df2, booktabs = T, caption = "Supplementary table 2") %>%
kable_styling(latex_options = c("striped", "scale_down")) %>%
  kable_styling(latex_options = c("HOLD_position")) 
```

```{r est3, message=FALSE, warning=FALSE, echo=FALSE}
library(readxl)
df1 <- data.frame(read_xlsx("figure/anaraEV/t3.xlsx", sheet = 1), check.names = F)

library(kableExtra)

kbl(df1, booktabs = T, caption = "Supplementary table 3") %>%
kable_styling(latex_options = c("striped", "scale_down")) %>%
  kable_styling(latex_options = c("HOLD_position")) 
```

```{r est4, message=FALSE, warning=FALSE, echo=FALSE}
library(readxl)
df1 <- data.frame(read_xlsx("figure/anaraEV/t4.xlsx", sheet = 1), check.names = F)

library(kableExtra)

kbl(df1, booktabs = T, caption = "Supplementary table 4") %>%
kable_styling(latex_options = c("striped", "scale_down")) %>%
  kable_styling(latex_options = c("HOLD_position")) 
```

<!--chapter:end:rmd/07-ev.Rmd-->

---
bibliography: rmd/references.bib
editor_options: 
  chunk_output_type: console
---


# shortRNA {#chapter3}
\begingroup\LARGE
`shortRNA`: a flexible framework for the analysis of short RNA sequencing data
\begingroup\Large
\normalsize


__Contributions:__ *I worked towards the tool development, data analysis, results interpretation. This has been done together with Pierre-Luc Germain.*


## Abstract

Short RNA are important molecules that play key functional roles in the
regulation of the genome. Several classes of short RNAs, such as miRNA,
tRNA and tRNA fragments, and piRNAs have been characterized and have
complex biogenesis. Short RNA sequencing is becoming increasingly
relevant in the research of regulatory mechanisms in a wide range of
biological functions. From an analysis point of view, each type of RNA
has its own features, and hence, specialized methods have been developed
focused on particular types, which not only multiply the work needed for
a comprehensive analysis but potentially create misassignment problems.
In addition, methods are typically divided into genome-based methods
that do not deal adequately with post-transcriptional modifications, and
transcript-based methods that are blind to unannotated features.
Finally, there are several outstanding issues in the analysis of short
RNAs, which are critical in the analysis of some samples. There is,
therefore, a need for an analysis framework that is: sufficiently
tailored to consider the specificities of different classes of short RNA
and their biogenesis, and sufficiently general and exhaustive to perform
global analyses. We developed a user-friendly, highly flexible and
comprehensive `R` package for a thorough end-to-end analysis of short RNA
sequencing data (planned submission to Bioconductor). Our package is
appropriately adapted to take into account the differences between
different types of short RNA and their biogenesis and is seamlessly
expandable to include additional annotation. We use a customized genome
annotation with artificial chromosomes to account for
post-transcriptional modifications, and a flexible rule-based approach
to assign reads along a tree of hierarchically organized features. This
enables the systematic querying, exploration, and differential
expression analysis of short RNAs at various degrees of granularity,
from specific sequences to RNA classes. We also include various
normalization and experimental-bias correction methods.The `shortRNA` `R` package is developed to perform all the steps from within `R` and is
platform-independent.

## Introduction and background

Previous decades have revealed a number of RNAs that are distinct from
the messenger RNAs. Short-RNAs are non-coding RNA molecules that are
fewer than 200 nucleotides long and play an important role in genome
regulation. Several types of short RNAs, of different lengths, have been
discovered including microRNA (miRNA; 18-24 nt); small interfering RNA
(siRNA; 21-27 nt); small nucleolar RNA (snoRNA; 60-170 nt); small
nuclear RNA (snRNA/ U-RNA; 10-300 nt); Small temporal RNA (stRNA; 18-24
nt); tRNA-derived small RNA (tsRNA), including tRNA halves (tsRNA; 28-36
nt) and tRNA fragments (tRFs; 14-22 nt); small rDNA-derived RNA (srRNA;
18-30 nt); repeat-associated small interfering RNA (rasiRNA; 24-27 nt);
and Piwi-interacting RNA [piRNA; 26-31 nt, present in animals in the
germline cells and also in the brain [@zuo2016]. Some short RNAs, for
instance, miRNA and siRNA, are known to suppress gene expression by
sequence-specific interactions with regulatory regions during
transcription, RNA processing and translation [@zhu2016], and by forming
the core with RNA-induced silencing complex (RISC) [@pratt2009]. In
addition, short-RNAs have been related to cancer, Parkinson's disease,
Alzheimer's disease, and prion disease [@gong2005], and hence, are used
as biomarkers.

Several methods have been developed for the analyses of short RNA
sequencing data. The development of high-throughput sequencing (HTS)
techniques allows researchers to study short RNAs in diverse tissues or
cells. HTS not only allows for the quantification of known short RNAs
but also for the identification and quantification of novel short RNAs.
The small RNA sequencing (sRNA-Seq) bears challenges and biases that
researchers need to be informed of in order to properly analyze the
data. First, most of the methods concentrate on a specific type of short
RNA, such as `MINTmap` [@loher2017] focus on analyzing tRFs, `TAM 2.0`
[@li2018], `Prost!` [@desvignes2019], `Chimira` [@vitsios2015], and `mirTools`
2.0 [@wu2013] focus on miRNAs. Next, a few tools concentrate on most
types of short RNAs (not all), such as `ncPRO-seq` [@chen2012],
`sRNAtoolbox` [@rueda2015], `SPAR` [@kuksa2018], `Threshold-seq` [@magee2017],
`sRNAnalyzer` [@wu2017], `UEA sRNA Workbench` [@stocks2012], `Sports1`
[@shi2018], and `Oasis2` [@rahman2018]. Tools that concentrate on many
types of short RNAs, such as `Oasis2` and `Sports1`, perform sequential
mapping, for example, they perform sequential mapping by first assigning
the reads to miRNAs, then assigning the leftover to tRNAs, and then the
remaining ones to mRNAs, which can create a misassignment problem as
short reads can map to multiple locations. But this information is lost
because only unmapped reads are aligned against the next annotation. In
addition, althoughthe libraries for sequencing are tailored to short
RNAs, other long RNAs are also often detected in the sequencing data,
and also because short RNAs often overlap with other features,
prioritization between the biotypes is crucial. Further, there are quite
a high number of sequenced reads left that map to the reference genome
but are not assigned to the known features. Moreover, although most of
the tools do not consider assignment rules for reads assignment, few
tools such as Prost! have a set of defined rules for reads assignment.
But, the rules are only for the miRNAs. Furthermore, the multi-mapping
reads are either excluded or randomly assigned to the multi-mapping
positions in the genome. As well as, not all the tools deal with the
post-transcriptional modifications, such as the addition of "CCA"
towards the 3' end of tRNAs [@hou2010; @ibba2000; @barraud2019], which
is important for the recognition of tRNA by enzymes and translation
[@sprinzl1979; @green1997]; and the addition of "G" towards the 5' end
in histidine tRNAs [@cooley1982; @cozen2015], this is critical for
histidyl-tRNA synthetase (HisRS) recognition [@fromant2000], which is
responsible for the integration of histidine into proteins
[@freist1999]. On top of these limitations, most of the tools are
written in different programming languages, are platform-dependent and
depend on external tools, which can create a barrier for installation
for the user; and the tools are either web-based, which restrict the
user for additional downstream analysis or command-line based, which
creates a restriction for the non-computational researchers. Besides,
the available tools use feature-based counting (number of reads, counts,
associated with each feature), except for `seqpac` [@skog2021], a recently
published tool, which uses sequence-based counting (number of reads,
counts, associated with each unique sequence). Sequence-based counting
would prevent the same sequence from being annotated multiple times
within and across samples, hence increasing the efficiency for alignment
and reads assignment. Also, it directly enables looking at specific
variations in sequences or their boundaries. However, due to this
complexity, it can be challenging to explore data at the level of
individual sequences, which calls for methods that can afford different
levels of granularity. Hence, specialized tools are required with
specialized pipelines to analyze sRNA-seq data.

We developed `shortRNA`, an R tool that addresses all of the issues listed
above for processing sRNA-seq data. Our tool is cross-platform (it may
run on any operating system) and built on the Bioconductor framework, in
particular making use of efficient data structures (e.g., `DataFrame`,
`FactorList`, and `TreeSummarizedExperiment`) enabling interoperability with
other Bioconducto`R` packages. Users can use `shortRNA` to conduct a
thorough analysis of their data, from quality control (trimming, adapter
removal, UMI compressing) and alignment to quantification and downstream
analysis. `shortRNA` also has a set of customizable criteria for assigning
reads to various types of short RNAs and seamlessly enables querying
features at the level of individual fragments. We have tested our tool
using a published mouse dataset from sperm, simulated data from sperm,
and human data from peripheral blood mononuclear cells (PBMC;
unpublished).

## Methods

### Development and testing environment

The `shortRNA` package is developed and tested on the Linux (GNU/Linux
4.4.0-210-generic x86_64) operating system, Ubuntu (version 16.04.7 LTS)
with 16 processors and 124 GB of RAM, using R 3.6.3 and R 4.0.5. As the
`shortRNA` tool is developed as an `R` package, it is platform-independent
(can easily be installed and run on other operating systems).

### Backbone data structures of `shortRNA`

The `shortRNA` package is developed around four main data structures from
Bioconductor: phylo, `FactorList`, `DataFrame`, and
`TreeSummarizedExperiment`. These four data structures store the data and
results in the most efficient manner and enable fast computation.

#### phylo

The phylogenetic tree is a branching diagram used to depict evolutionary
relationships. In R, the phylo class stores the phylogenetic
relationship. In `shortRNA`, we save the relationship between RNA
biotypes, features, and reads as a phylo object, section [Features Tree](#ft).

#### `FactorList`

In the case of a long vector of repeated characters, R takes more memory
to store them and also the computation could be quite slow. These
repeated character vectors could be stored as factors, where levels are
provided to each unique character in the vector
(<https://datascience.stackexchange.com/questions/12018/when-to-choose-character-instead-of-factor-in-r>).
However, when factors are stored in a list, the levels are traditionally
defined for each element of the list, which can be very inefficient
memory-wise. The `FactorList`, a class from the IRanges Bioconductor
package, instead stores lists of factors as a single factor vector with
added list membership information. In addition, like other AtomicList
objects, it enables list operations without iteration. This is helpful
for efficiently saving the object in R memory and for fast computations.

#### `DataFrame`

Rectangular data can be stored as `data.frame` class object in R.
`DataFrame` functions from S4Vectors Bioconducto`R` package and behaves
similar to `data.frame` in terms of construction and subsetting. An
advantage of using `DataFrame` is that it can store any type of object in
a column, for example, a `FactorList`, or even another `DataFrame`, while
retaining all the methods and functionalities of traditional
`data.frame`s.

#### `TreeSummarizedExperiment`

`TreeSummarizedExperiment` is a Bioconducto`R` package extending the
classical `SummarizedExperiment` [@morgan2020] and with additional
hierarchy data and operations [@huang2020]. The `SummarizedExperiment`
class holds rectangular matrices of experiment data, accompanied by row
and column annotation data. There are two classes in the
SummarizedExperiment package: `SummarizedExperiment` and
RangedSummarizedExperiment. Instead of a `DataFrame` of features,
RangedSummarizedExperiment objects represent genomic ranges of interest.
Figure \ref{fig:3f1}A depicts the structure of the `SummarizedExperiment`. The
rectangular data matrices are stored as assays, rowData stores
annotation for corresponding rows in the assays, colData stores
annotation for corresponding columns in the assays, ranges are described
by a GRanges or a GRangesList object, which are stored as rowRanges, and
metadata can be used to store additional data-related information.

In extension to the `SummarizedExperiment`, `TreeSummarizedExperiment` has
rowTree, which stores the hierarchical structure of rows of assays;
colTree, which stores hierarchical structure of columns of the assays;
rowLinks, which stores the link information between the rows of the
assays and rowTree; colLinks, which stores the link information between
the columns of the assays; and referenceSeq, which stores the reference
sequence for the features, depicted in Figure \ref{fig:3f1}B.

(ref:3cf1) Schematic of A: `SummarizedExperiment` [@morgan2020], B: `TreeSummarizedExperiment` [@huang2020]

```{r 3f1, echo=FALSE, fig.align='center', fig.cap="(ref:3cf1)", fig.width = 6.44, fig.height=7.18}
img <- readPNG("figure/shortRNA/in/1.png")
grid.raster(img)
```

### Pipeline

The `shortRNA` pipeline includes three major data analysis steps:
preprocessing of data, alignment and reads assignment with the
customisable assignment rules, and statistical analysis and
visualization. Figure \ref{fig:3f2} shows the outline of the pipeline. First, the
data goes through the preprocessing steps that consist of filtering and
alignment of unique sequenced reads. The raw data undergo quality
assessment and then the quality control steps are performed. The data is
then aligned to a custom genome containing, in addition to the reference
genome, extra pseudo-chromosomes for handling post-transcriptional
modifications. Aligned reads are then overlapped with the features
annotation and reads are assigned to the features using customisable
assignment rules, which is an important part of this pipeline, after
that, a features tree is formed. After the reads assignment, the data
goes through the statistical analysis steps that consist of
normalization and differential analysis. The quality controlled data are
normalized and then the test for differential expression of short RNAs
is performed in the user-defined biological groups.

(ref:3cf2) Overview of `shortRNA` pipeline
```{r 3f2, echo=FALSE, fig.align='center', fig.cap="(ref:3cf2)", fig.width = 5, fig.height=5.5}
img <- readPNG("figure/shortRNA/in/2.png")
grid.raster(img)
```

#### Quality control and trimming

Quality control (QC) and trimming is a vital step in HTS data analysis.
To assess and control the quality of sRNA-Seq data, we adapted the
functionalities from two `R` packages: `Rfastp` [@wang2020] and seqTools
[@kaisers2020]. `Rfastp` is used for trimming the low-quality reads and
adapter trimming. Further, with the summary files from `Rfastp`, the
functions in our tool could be used for making a table of comparison for
before and after QC, plots for duplicated reads, reads quality plot, and
base-content plots. We benchmarked Biostrings [@pagès2020], qrqc
[@buffalo2020], ShortRead [@morgan2009], and seqTools for reading and
storing FASTQ files quality data in `R`, using an 863 MB FASTQ file.
seqTools outperformed the other three tools in terms of reading and
storing the FASTQ file quality data, as shown in Figure \ref{fig:3f3}. Hence, we
used it for reading the FASTQ files quality data, before and after QC,
to make reads length distribution plots. To summarize, the QC functions
provide information about sequencing depth, reads quality, possible
adapter sequences, duplicated reads, and sequence length distribution,
both before and after QC. The interactive plots provide users with
greater insight into their data. Exemplary plots and tables from the QC
report of a sample are shown in Figure \ref{fig:3f4}.

(ref:3cf3) Benchmark of four different Bioconductor tools for reading and storing FASTQ file quality data. Performance of four tools is measured in **A:** the amount of time they take for reading the file, **B:** the amount of memory they take to store the data.

```{r 3f3, echo=FALSE, fig.align='center', fig.cap="(ref:3cf3)", fig.width = 5.34, fig.height=2.45}
img <- readPNG("figure/shortRNA/in/3.png")
grid.raster(img)
```

(ref:3cf4) Exemplary QC report plots of a sperm sample from [@gapp2021]. **A:** Quality control table, **B:** Quality score lines plot at each nucleotide in the sequencing reads, **C:** Barplot of reads duplication level with the mean GC ratio represented as lines, **D:** Histogram of reads distribution, before and after quality control

```{r 3f4, echo=FALSE, fig.align='center', fig.cap="(ref:3cf4)", fig.width = 6.5, fig.height=6.72}
img <- readPNG("figure/shortRNA/in/4.png")
grid.raster(img)
```

Unique Molecular Identifiers (UMIs) are complex indices of 8-16
nucleotide lengths that are introduced to sequencing libraries before
PCR amplification steps. Researchers can use UMI to evaluate the
efficiency with which they collect input molecules, identify sampling
bias, and, most importantly, identify and compensate for the effects of
PCR amplification bias [@fu2018]. Preprocessing of data sequenced with
UMIs is required for reads deduplication and correction, as well as the
creation of consensus sequences from each UMI. In `shortRNA`, we have
adapted the `UMIc` tool for UMIs collapsing [@tsagiopoulou2021], which
takes into account the frequency and the Phred quality of nucleotides
and the distances between the UMIs and the actual sequences for
collapsing the sequences.

Schematic of quality checks and quality control is shown in Figure \ref{fig:3f5}.

(ref:3cf5) Schematic of quality check and quality control of FASTQ files, UMIs collapsing (if present), and collapsing FASTQ files to obtain unique sequences for alignment and making a sequence by samples counts matrix

```{r 3f5, echo=FALSE, fig.align='center', fig.cap="(ref:3cf5)", fig.width = 3.4, fig.height=6}
img <- readPNG("figure/shortRNA/in/5.png")
grid.raster(img)
```

#### Sequence count matrix

From the trimmed FASTQ files, we extract the unique reads sequences and
make a sample by sequence count matrix. This matrix represents the
counts of sequences in each sample, keeping only sequences occurring
more than once. This is done by the `fastq2SeqCountMatrix()` function from
`shortRNA`. Further, we export these sequences as a fasta file. An
overview of this step is shown in Figure \ref{fig:3f5}. The two main outputs from
this step are sequenceFasta, which would be used for alignment, and
countsMatrix.

#### Annotation preparation

Sequencing generates nucleotide sequences with unknown functions, while
sequence annotation provides descriptive information about sequenced DNA
sequences. Several databases are available for the annotation of the
sequences. In `shortRNA`, we used miRBase [@griffiths-jones2008] for
miRNAs; GtRNADb [@chan2016] for tRNAs; mitoRNADb [@jühling2009] for
mitochondrial tRNAs; for mouse, we use piRNA precursors from [@li2013],
rRNAs from SILVA [@quast2013], and mRNAs and other biotypes from Ensembl
[@howe2021]. It is possible for users to have additional databases of
their choice.

The miRBase database is a collection of miRNA sequences and annotations.
In miRBase, miRNAs are named as, for example, org-mir-20a. The first 3
letters signify the organism, "20" tell us that it was an
early-discovered family (20th family that was named), "20a" tell us that
there is possibly another related miRNA, for example, org-mir-20a. Here,
org-mir-20a is a precursor. The mature miRNA species may be derived from
both the 5' and 3' arms of the precursor duplex, and are called the
miRNA-5p and -3p species, respectively. In this example, it would be
org-mir20a-5p and org-mir20a-3p. All the miRNAs overlapping between
Ensembl and miRBase were removed from Ensembl and the remaining one from
Ensembl, not overlapping, were renamed to match the nomenclature of
miRBase. For example, Mir7679 is renamed to org-miR-7679. Further, if
the length of Ensembl miRNAs were more than 25bp, we labelled them as
precursors.

A miRNA cluster is a group of two or more miRNA hairpins that are
transcribed from miRNA genes that are physically adjacent, transcribed
in the same direction, and are not separated by a transcription unit or
a miRNA in the opposite orientation. There are mostly 2-3 mature miRNAs
in a miRNA cluster, but there is an existence of a bigger miRNA cluster,
which is found in humans on chromosome 13: miR-17 to miR-92, involved in
tumour formation, and development of heart lungs and immune systems
[@lai2013]. Researchers group the miRNAs to form a miRNA cluster by the
distance between them. For example, [@griffiths-jones2008;
@baumgart2017] call a group of miRNA as clusters if miRNAs are within
10kb and [@yuan2009] used a distance of 50kb. In `shortRNA`, we use a
distance of 10kb for clustering miRNAs as more than 40% of
experimentally validated human miRNA cluster genes have been identified
within 10kb [@griffiths-jones2008; @lai2013].


The genomic tRNA database (GtRNADb) is a database of tRNA gene
predictions created by tRNAscan-SE [@chan2021] on whole or almost
complete genomes. In GtRNADb, tRNAs symbols consist of 5 parts,
separated by a "-", for example, tRNA-Ala-AGC-9-2. In this example, tRNA
(prefix) represents tRNA genes that are high scoring and not predicted
as pseudogenes. If it was a pseudogene, it would have been represented
as "tRX". At the second position, we have three-letter amino acids
(isotype), which stand for tRNA isotype. At the third position, we have
anticodon detected in the gene sequence. At the fourth position, there
is a unique ID (transcript ID) of a tRNA transcript or "isodecoder" with
a particular isotype and anticodon. The fifth position represents the
gene locus ID and for tRNA genes with multiple identical copies, this
gene locus ID represents the particular gene copy in the genome. For the
mitochondrial tRNAs, we obtained the sequences from mitotRNAdb
[@jühling2009] using the tRNAdbImport [@gm2018] `R` package. Further, we
removed all the duplicated sequences and renamed the mitochondrial tRNAs
as per the naming convention of GtRNADb, adding "mt" as a prefix. Mature
tRNAs receive a post-transcriptional addition of "CCA" [@barraud2019],
hence, we added "CCA" to all the tRNA/ mt-tRNA sequences at the 3' end.
All histidine tRNAs of known sequence are one nucleotide longer at the
5' end than are other tRNA species[@cozen2015],hence we added "G" at the 5' end of the sequences of Histidine tRNAs/ mt-tRNAs to account for this post-translational modification, as also
done in [@shi2018; @cozen2015]. Because of these post-transcriptional
modifications, we add the sequences of tRNA into the genome FASTA file
as pseudo chromosomes. For example:

```git
>pseudoChr_tRNA-Thr-CGT-1-1
GGCGCGGTGGCCAAGTGGTAAGGCGTCGGTCTCGTAAACCGAAGATCACGGGTTCGAACCCCGTCCGTGCCTCCA

>pseudoChr_mt_tRNA-His-GTG-1-1
GGTGAATATAGTTTACAAAAAACATTAGACTGTGAATCTGACAACAGGAAATAAACCTCCTTGTTCACCCCA
```

All the tRNAs overlapping between Ensembl and GtRNADb were removed from
Ensembl and the remaining tRNA from Ensembl, not overlapping, were
labelled as pseudo tRNAs.

SILVA is an rRNA database, which provides extensive, quality-checked,
and regularly updated datasets of aligned short (16S/18S, SSU) and large
subunit (23S/28S, LSU) ribosomal RNA sequences. rRNAs are repeated
sequences and are masked in the genomes in the Ensembl database, and are
not listed in the features annotation. Hence, we add them as well as
pseudo chromosomes to the genome FASTA file, similar to tRNAs.

`getDB()`, Figure \ref{fig:3f6}, function retrieve the databases, account for the
post-translational modifications of tRNAs, alter mitochondrial tRNA
database, and create miRNA clusters and assign miRNAs and miRNA
precursors to miRNA clusters. The output from `getDB()` is then parsed to
the `prepareAnnotation()` function, which prepares features annotation as
`GRanges()`, save the genome and pseudo-genomes and index that for
alignment. Features annotation are then converted to `FactorList` using
`featuresAnnoToFL()`. The main three outputs from the annotation
preparation are features as `GRanges()` and `FactorList()`, and `indexed customGenome`.

(ref:3cf6) Schematic of obtaining database, preparing annotations and custom genome indexes for alignment

```{r 3f6, echo=FALSE, fig.align='center', fig.cap="(ref:3cf6)", fig.width = 6.41, fig.height=4}
img <- readPNG("figure/shortRNA/in/6.png")
grid.raster(img)
```



#### Alignment

Alignment is necessary to know where in the genome reads belong. The
alignment of short RNA remains a persistent under-recognized problem. We
align the FASTA file of sequences with the custom genome prepared in the
annotation step. For alignment, we use the `Rsubread` `R` package
[@liao2019] with the index of custom genome generated on the genome and
pseudo-chromosomes (`index = "customGenome"`), with the GTF from Ensembl
database obtained from `getDB()` of `shortRNA` (`GTF = exonsBy(db$ensdb)`),
and allowing to report the maximal number of equally-best mapping
locations (`nBestLocations = 16`). Please refer to the schematic in Figure
\ref{fig:3f7}.

(ref:3cf7) Schematic of alignment, overlapping features and aligned sequences, and assigning reads to features using assignment rules

```{r 3f7, echo=FALSE, fig.align='center', fig.cap="(ref:3cf7)", fig.width = 4, fig.height=4.5}
img <- readPNG("figure/shortRNA/in/7.png")
grid.raster(img)
```


#### Reads annotation and assignment

After alignment, the aligned file (BAM file) is overlapped with the
features annotation (`featuresGR`) using the `overlapWithTx2()` function. It
is possible that a read overlaps with multiple features. Next, the
overlaps are parsed with the `assignReads()` function with the assignment
rules, `defaultAssignRules()`, for validating the overlap and finding
assignment of multi mapping reads. Please refer to the schematic in
Figure \ref{fig:3f7}.

#### Assignment rules

A sequence read can be mapped to multiple locations in the genome. It is
critical to ensure that the reads are properly mapped to the feature and
that they are assigned to one or more features. In the `shortRNA` package,
we defined a set of customizable rules for this purpose. The output of
the function `defaultAssignRules()` is described below:

We consider reads with at least 50% overlap to the features to be valid
read assignments.

```r
$overlapBy
[1] 0.5
```
By default, we do not prioritize the assignment based on the size of the
overlap between reads and features (when a read overlaps multiple
features); however this option is available to users.

```r
$prioritizeByOverlapSize
[1] FALSE
```

We prioritize overlap in the same strand, but enable overlaps from the
opposite strand if there is no known feature on the same strand.

```r
$sameStrand
[1] "prioritize"
```

We give priority to known features in our assignments.

```r
$prioritizeKnown
[1] TRUE
```

Next come validation rules specific to RNA types. For example, in order
to be assigned to primary piRNAs, the read should be 26-32 nucleotides
long, with the first nucleotide being a T.

```r
$typeValidation$primary_piRNA
$typeValidation$primary_piRNA$fun
function(src, allowRevComp=FALSE, length=26:32){
  length <- as.integer(length)
  valid <- src$length >= min(length) & src$length <= max(length)
  if(length(w <- which(valid))==0) return(valid)
  seqs <- as.character(src$seq[w])
  valid[w] <- sapply(strsplit(seqs,""),FUN=function(x){ x[[1]]=="T" })
  if(allowRevComp){
    revcomp <- as.character(reverseComplement(DNAStringSet(seqs)))
    valid[w] <- valid[w] |
      sapply(strsplit(seqs,""),FUN=function(x){ x[[1]]=="T" })
  }
  valid
}
```

If the read cannot be identified as a primary piRNA, it is referred to
as a piRNA precursor.
```r
$typeValidation$primary_piRNA$fallback
[1] "piRNA_precursor"
```

Secondary piRNA sequences should be 26 to 32 nucleotides long, with A in
the 10th position [@brennecke2007].
```r
$typeValidation$secondary_piRNA
$typeValidation$secondary_piRNA$fun
function(src, allowRevComp=FALSE, length=26:32){
  length <- as.integer(length)
  valid <- src$length >= min(length) & src$length <= max(length)
  if(length(w <- which(valid))==0) return(valid)
  seqs <- as.character(src$seq[w])
  valid[w] <- sapply(strsplit(seqs,""),FUN=function(x){ x[[10]]=="A" })
  if(allowRevComp){
    revcomp <- as.character(reverseComplement(DNAStringSet(seqs)))
    valid[w] <- valid[w] |
      sapply(strsplit(seqs,""),FUN=function(x){ x[[10]]=="A" })
  }
  valid
}
```

If a read cannot be identified as secondary piRNA, it is referred to as
a piRNA precursor.
```r
$typeValidation$secondary_piRNA$fallback
[1] "piRNA_precursor"
```

For miRNAs, the length should be between 19 and 24 nucleotides, and the
read should overlap the feature by at least 16 bp. The maximum number of
non-overlapping nucleotides allowed is three.
```r
$typeValidation$miRNA
$typeValidation$miRNA$fun
function (src, length = 19:24, minOverlap = 16L, maxNonOverlap = 3L)
{
    src$length %in% length & src$overlap >= minOverlap & (src$length -
        src$overlap) <= maxNonOverlap
}

$typeValidation$miRNA$length
[1] 19 20 21 22 23 24
```

If a read cannot be assigned to a mature miRNA, it is assigned to a
miRNA precursor.
```r
$typeValidation$miRNA$fallback
[1] "miRNA_precursor"
```

Reads that are assigned to tRNAs (or pseudo tRNAs) undergo additional
classification. tRNA 5p fragments are less than 30 bp long and begin in
the feature's 5bp. tRNA 3p fragments are less than 50 bp long, with a
distance to the feature end of 5bp. Starts at +/= 1bp of the feature and
has a length of 30 to 34 nucleotides for 5p half. The 3p half begins at
+/= 1bp of the feature, has a length of 34 to 50 nucleotides, and ends
with CCA.
```r
$reclassify
$reclassify$tRNA
function(srcs, rules=list(
  "tRNA_internal_fragment"=function(x){ rep(TRUE, nrow(x)) },
  "tRNA_5p_fragment"=function(x){ x$startInFeature < 5L & x$length < 30L },
  "tRNA_3p_fragment"=function(x){ x$distanceToFeatureEnd < 5L & x$length < 50L },
  "tRNA_5p_half"=function(x){ x$startInFeature %in% -1:1 & x$length %in% 30:34 },
  "tRNA_3p_half"=function(x){ x$distanceToFeatureEnd %in% -1:1 & x$length >= 34L &
      x$length <= 50L & grepl("CCA$",x$seq) }
)){
  valids <- vapply(rules, FUN.VALUE=logical(nrow(srcs)), FUN=function(fn){
    fn(srcs)
  })
  if(nrow(srcs) == 1) valids <- t(valids)
  factor(apply(valids, 1, FUN=function(x) max(which(x))),
         seq_len(ncol(valids)), colnames(valids))
}
```

We also assign priorities to different types of RNA. Priorities can be
changed by increasing or decreasing the numbers; for example, in the
case of only miRNA sequencing, users can set higher priorities for
miRNAs by changing the number from 1 to 2 or higher.
```r
$priorities
miRNA           tRNA           tRNAp         Mt_tRNA          snRNA
  1               1              1             1                1
snoRNA       antisense      primary_piRNA      secondary_piRNA        
  1               1              1                   1              
precursor     long_RNA         longRNA
  -1             -1              -1
```

#### Features tree {#ft}

We organized the features and reads in the form of a rooted phylogenetic
tree in a hierarchical fashion. This is done by the function
`assignReadsToTree()` in `shortRNA`, Figure \ref{fig:3f12}. From the root, we have two
main branches: `shortRNA` and long RNA, Figure \ref{fig:3f8}C. We keep the features
that are longer than 200 bp in long RNA and the features that are
shorter than 200 bp in short RNA. Each branch is then further divided
into RNA types. The miRNAs and tRNAs are further divided into
sub-branches to account for the multi-mapping problems, Figure \ref{fig:3f9} and
Figure \ref{fig:3f10}. Sequence quality and a follow up trimming, may result in one
nucleotide shorter sequence. Because of the hierarchical organization,
unique reads mapping to a feature are kept in the hierarchy, Figure \ref{fig:3f8}.
If the user would like to include an additional database for short RNAs,
an additional branch can be added to the tree while preparing
annotation, `prepareAnnotation()`.


(ref:3cf8) General framework of feature tree and assignment of sequences to the tree. **A:** Example of two sequences. **B:** Alignment of both the sequences to miRNA. **C:** General outline of the feature tree and assignment of both the sequences to the tree

```{r 3f8, echo=FALSE, fig.align='center', fig.cap="(ref:3cf8)", fig.width = 6.41, fig.height=4.43}
img <- readPNG("figure/shortRNA/in/8.png")
grid.raster(img)
```



(ref:3cf9) Hierarchical organization of **A:** tRNA features and **B:** mitochondrial tRNA features.

```{r 3f9, echo=FALSE, fig.align='center', fig.cap="(ref:3cf9)", fig.width = 6.43, fig.height=2.82}
img <- readPNG("figure/shortRNA/in/9.png")
grid.raster(img)
```

(ref:3cf10) Hierarchical organization of miRNA features

```{r 3f10, echo=FALSE, fig.align='center', fig.cap="(ref:3cf10)", fig.width = 6.43, fig.height=43.67}
img <- readPNG("figure/shortRNA/in/10.png")
grid.raster(img)
```


#### Reads assignment ambiguity

Reads can align to multiple locations in the genome, and when this is
the case, typical workflows will either randomly align to one location
or not report any alignment. We allow for multi-mapping during
alignment, and address ambiguities when we assign reads to features.
Specifically, we assign it to the parent of all the features to which
the read maps. Figure \ref{fig:3f11}A depicts a read that can be mapped to multiple
tRNAs, Figure \ref{fig:3f11}B. While assigning reads to the features tree, we assign
this read as an ambiguous read to the parent of all the features it is
mapping to; in this case, tRNA-Leu-CAG, Figure \ref{fig:3f11}C.


(ref:3cf11) Multiple assignments of a sequence mapping to a tRNA. **A:** Exemplary sequence. **B:** Multiple sequence alignment plot of the sequence and the features to which it is mapping. **C:** Assignment of the reads to the tRNA feature tree

```{r 3f11, echo=FALSE, fig.align='center', fig.cap="(ref:3cf11)", fig.width = 6.43, fig.height=6}
img <- readPNG("figure/shortRNA/in/11.png")
grid.raster(img)
```


#### Construction of `TreeSummarizedExperiment` object

The feature tree and the assigned reads `DataFrame` is then used for the
construction of the `TreeSummarizedExperiment` object, which is then used
for all the downstream analysis, including differential analysis. This
is depicted in Figure \ref{fig:3f12}.


(ref:3cf12) Schematic of assigning reads to the features tree of features, creation of `TreeSummarizedExperiment` object, and downstream analysis and plotting

```{r 3f12, echo=FALSE, fig.align='center', fig.cap="(ref:3cf12)", fig.width = 4.32, fig.height=6.12}
img <- readPNG("figure/shortRNA/in/12.png")
grid.raster(img)
```


#### Differential analysis

The hierarchical structure of the features implies that differential
expression analysis could be performed at different levels of the
hierarchy, which would however increase the multiple testing problem.
We, therefore, rely on the method for dynamic testing of hierarchical
hypotheses implemented in `treeclimbR` [@huang2021] for differential
analysis of features.

Following normalization and differential analysis, users can use the
`TreeHeatmap`, `ggtree`, and `castro` `R` packages for exploratory data analysis
and figure creation. The `shortRNA` `makeTracks()` function can be used to
create genomic tracks, as shown in Figures \ref{fig:3f15}, \ref{fig:3f18a}, and \ref{fig:3f18b}. We intend to
expand ou`R` package by writing wrappers for these packages in order to
generate plots from within `shortRNA`.

#### Table of important functions

Table \ref{tab:sht1} shows some of the most important functions of the `shortRNA`
package. In the table, the functions are organized in the chronological
order of data analysis.

```{r sht1, message=FALSE, warning=FALSE, echo=FALSE}
library(readxl)
df1 <- data.frame(read_xlsx("figure/shortRNA/in/t1.xlsx", sheet = 1), check.names = F)

library(kableExtra)

kbl(df1, booktabs = T, caption = "Important functions from shortRNA") %>%
kable_styling(latex_options = c("striped", "scale_down")) %>%
  kable_styling(latex_options = c("HOLD_position")) 
```

## Results

### Datasets used for testing the pipeline

We used a previously published sperm sRNA-seq dataset from [@gapp2021]
(GEO accession: GSE162112), in which we additionally spiked simulated
reads. We also used an unpublished PBMC human dataset from the Schratt
Lab (<https://schrattlab.ethz.ch/>) at the ETH Zurich, on which qPCR
were additionally performed for 5 miRNAs (miR-30b-5p, miR-30e-5p,
miR-30d-5p, miR-499-5p, and miR-1248). We used these human data to
correlate quantifications from `shortRNA`, `seqpac`, `Sports1`, and `Oasis2`
with delta Ct values from qPCR. We analyzed both of these datasets with
`shortRNA`, `seqpac`, `Sports1`, and `Oasis2` tools to compare the
quantification of miRNAs. Further, we used the simulated reads from
miRNAs to test the reads assignment and to assess differential analysis.

For both datasets, we used miRBase (miRNA), GtRNADB (tRNA), MttRNADb
(mitochondrial tRNA), SiLVA (rRNA), and Ensemble. For the sperm dataset
from the mouse, we also used piRNA precursors.

### Comparison of quantification and identification of miRNAs

#### `shortRNA` quantification is positively correlated with `Oasis2` and `Sports1`

We used `Sports1`, `seqpac`, `Oasis2`, and `shortRNA` with the default
parameters to analyze the sperm dataset. Following the data analysis, we
limited our assessment to miRNAs for comparison across tools. We found
1970 miRNAs using `Oasis2`, 1137 using `Sports1`, 1024 using `seqpac`, and 503
using `shortRNA`. We began by examining the quantification with all four
tools. Figure \ref{fig:3f13} shows a comparison of the quantification of miRNAs from
`Sports1`, `Oasis2`, and `seqpac` with `shortRNA`. We discovered that the
quantifications of `shortRNA` and `Oasis2`, as well as `shortRNA` and `Sports1`,
are highly correlated. In contrast, the correlation of quantification
between `shortRNA` and `seqpac` was negative. We investigated this
discrepancy and discovered a large number of short reads (length: less
than 15 nucleotides) mapping to miRNAs and assigned by `seqpac`, but which
were considered invalid overlaps in light of our assignment rules.

(ref:3cf13) Comparison of Log2 TMM normalized quantification of miRNAs from `shortRNA` with `Oasis2`, `Sports1`, and `seqpac`

```{r 3f13, echo=FALSE, fig.align='center', fig.cap="(ref:3cf13)", fig.width = 6.36, fig.height=2.5}
img <- readPNG("figure/shortRNA/in/13.png")
grid.raster(img)
```



#### `shortRNA` accurately identify miRNAs

Following that, we examined the overlaps of identified miRNAs between
four tools. Despite the fact that all of the tools use miRBase, we
discovered that only 44 miRNAs were commonly detected by all of the
tools, Figure \ref{fig:3f14}. One possible explanation is that the miRBase for
mouse genome contains approximately 2000 mature miRNAs and approximately
1300 miRNA precursors, and `seqpac` and `Sports1` only use miRNA precursors
in the data analysis pipeline, whereas `Oasis2` and `shortRNA` use both
mature and miRNA precursors. Figure \ref{fig:3f14} shows that there were 231 miRNAs
that were commonly detected by `Oasis2`, `Sports1`, and `seqpac`, but not by
`shortRNA`. Next, we looked into these miRNAs and discovered that three
were removed from the most recent version of miRBase; for the other 228,
we did not find alignment in the genome using `shortRNA`, despite
accepting mismatches and soft-clipping. Based on our findings, we
believe that these miRNAs are the result of misalignment or sequential
alignment by the other three tools. When we examined the sequences
mapping to these miRNAs according to `seqpac`, we discovered that all of
the reads mapping to these miRNAs were shorter than 15 bp, and as a
result, are very unlikely to represent functional miRNAs. In conclusion,
we discovered that `shortRNA` can accurately identify miRNAs.


(ref:3cf14) Upset plot of overlaps of detected features between four methods: `shortRNA`, `Oasis2`, `Sports1`, and `seqpac`

```{r 3f14, echo=FALSE, fig.align='center', fig.cap="(ref:3cf14)", fig.width = 6.3, fig.height=4.7}
img <- readPNG("figure/shortRNA/in/14.png")
grid.raster(img)
```



### Data simulation to check the alignment and reads assignment by `shortRNA`

We validated the alignment and reads assignment by spiking the reads in
the sperm dataset ([Appendix C](#ac)). In brief, we simulated reads for a miRNA
cluster containing mmu-miR-125a-5p, mmu-miR-99b-5p, mmu-let-7e-5p,
mmu-let-7e-3p, mmu-miR-99b-3p, and mmu-miR-125a-3p, as well as reads
from a miRNA precursor, mmu-miR-144. To test for differential analysis,
the sequences from these miRNAs were spiked while also creating
differences between groups. Three invalid sequences were also simulated
for the precursor to test the accuracy of read assignment and were
properly aligned to miRNA precursors, as shown in Figure \ref{fig:3f15}.


(ref:3cf15) Genomic plots of simulated reads from miRNA precursor.

```{r 3f15, echo=FALSE, fig.align='center', fig.cap="(ref:3cf15)", fig.width = 6.41, fig.height=2.7}
img <- readPNG("figure/shortRNA/in/15.png")
grid.raster(img)
```



Further, the reads were assigned accurately to miRNAs, Table \ref{tab:sht2}.

```{r sht2, message=FALSE, warning=FALSE, echo=FALSE}
library(readxl)
df1 <- data.frame(read_xlsx("figure/shortRNA/in/t1.xlsx", sheet = 2), check.names = F)

library(kableExtra)

kbl(df1, booktabs = T, caption = "Assignment of simulated reads by shortRNA") %>%
kable_styling(latex_options = c("striped", "scale_down")) %>%
  kable_styling(latex_options = c("HOLD_position")) 
```



### Differential expression analysis

We performed differential analysis using the `treeClimbR` package. Through
differential analysis, candidate proposal, multiple testing correction,
and candidate evaluation, `treeclimbR` integrates the observations with a
tree that reflects the hierarchical relationship between entities and
finds an appropriate resolution on the tree to interpret the
association. During simulations, we spiked the mature miRNAs miR-99b-5p
and miR-99b-3p to be different. After differential analysis, because
both miR-99b-5p and miR-99b-3p were simulated to be different, we found
that miR-99b was called to be differentially expressed, Figure \ref{fig:3f16}.
Looking at the cluster, miRNAcluster_17:17830188-17830879, the full
cluster would have been differentially expressed if one read mapping to
miR-125a-5p, in grey, would also have been significantly different.


(ref:3cf16) Hierarchical representation of features and assigned reads to them. Differential expression analysis of features from two miRNA clusters. Simulated read nodes are highlighted in orange colour

```{r 3f16, echo=FALSE, fig.align='center', fig.cap="(ref:3cf16)", fig.width = 6.31, fig.height=4.92}
img <- readPNG("figure/shortRNA/in/16.png")
grid.raster(img)
```



### Validation using human data and quantitative real-time PCR

To validate the `Oasis2`, `Sports1`, `seqpac`, and `shortRNA` quantifications,
we used an unpublished PBMC human dataset and corresponding quantitative
real-time PCR (qPCR) quantifications. The PBMC dataset was analyzed
using the default parameters from `Oasis2`, `Sports1`, `seqpac`, and `shortRNA`.
Then, we correlated the qPCR quantification of five miRNAs with the
quantification using all four tools, Figure \ref{fig:3f17}. Because `Sports1` and
`seqpac` only report miRNA quantification at the level of precursors, the
precursor of mature miRNAs quantification was used for the correlation
of these tools with the qPCR. When we looked at the overall correlation
of miRNAs with qPCR quantifications, we discovered that except `seqpac`, all tools have a good correlation, Figure \ref{fig:3f17}A. Overall, we found a low correlation of
quantifications between four tools and qPCR. We found that `Oasis2`
quantification was negatively correlated with qPCR Ct values for miRNAs
miR-30d-5p and miR-30e-5p, as shown in Figure \ref{fig:3f18}B. The quantification of
miRNAs using short RNAs was found to be positively correlated with the
quantification of miRNAs using qPCR. In addition, we performed a visual
inspection of the reads mapping to these miRNAs by creating genomic
tracks, as shown in Figures \ref{fig:3f18a} and \ref{fig:3f18b}. As a result, while `shortRNA`
quantifications were visually observed to correctly count reads, they
were not substantially correlated with qPCR, but were generally better
correlated with qPCR quantifications than the other methods.



(ref:3cf17) Comparison of quantifications of five miRNAs from four different tools with -Delta Ct values from qPCR. **A:** Overall correlation. **B:** Correlation at indivisual miRNA

```{r 3f17, echo=FALSE, fig.align='center', fig.cap="(ref:3cf17)", fig.width = 6.41, fig.height=8}
img <- readPNG("figure/shortRNA/in/17.png")
grid.raster(img)
```

\begin{subfigures}
(ref:3cf18a) Genomic plots of validated miRNAs

```{r 3f18a, echo=FALSE, fig.align='center', fig.cap="(ref:3cf18a)", fig.width = 6.25, fig.height=8.5}
img <- readPNG("figure/shortRNA/in/18.png")
grid.raster(img)
```

(ref:3cf18b) Genomic plots of validated miRNAs

```{r 3f18b, echo=FALSE, fig.align='center', fig.cap="(ref:3cf18b)", fig.width = 6.25, fig.height=5.7}
img <- readPNG("figure/shortRNA/in/19.png")
grid.raster(img)
```
\end{subfigures}



### Qualitative comparison with other tools

There are several tools published for analyzing the sRNA-seq data. In
Table: [Qualitative comparison](figures/shortRNA/in/t3.xlsx), we made a qualitative comparison of 20 other tools. Table is too big to be displayed here.


## Discussion & Outlook

Due to the non-unique genomic origin, short length, and numerous
post-transcriptional modifications of short RNA species, data processing
of sRNA-Seq has proven to be difficult. Moreover, from a bioinformatics
point of view, there are several challenges in analyzing data generated
from small RNA sequencing techniques, including alignment, reads
assignment, normalization, and differential analysis. Importantly,
quantifying small RNAs correctly is a computational challenge. To a
large extent, different tools provide varying quantification. This is
startling, and it highlights the need for further research and better
tools in the field.

We present `shortRNA`, an `R` package, a novel tool, for analyzing sRNA-seq
data, which can reduce the likelihood of false discoveries from the
sRNA-seq data. `shortRNA` is available from github.com/mansuylab/`shortRNA`
and is planned to be submitted to the Bioconductor. The `shortRNA` R
package is developed around the existing frameworks of Bioconductor:
phylo, `FactorList`, `DataFrame`, and `TreeSummarizedExperiment`. Hence, it is
expected to be able to interact seamlessly with many other tools, such
as `ggtree`, castor, and `TreeHeatmap`. `shortRNA` performs preprocessing,
alignment, and downstream analysis of the sRNA-seq data. Moreover,
`shortRNA` is able to deal with the sequencing data with UMIs using `UMIc`.
Further, if an adapter sequence is unknown, it is possible to detect the
adapter sequence using `shortRNA`.

We tested the package on various human and mouse datasets and
demonstrated it using the mouse sperm dataset, simulated mouse sperm
dataset, and human dataset from PBMCs. We compared the quantifications
of miRNAs from `shortRNA` to those of three other tools: `Sports1`, `Oasis2`,
and `seqpac` and found them to be well correlated with `Sports1` and `Oasis2`.
We demonstrated `shortRNA`'s ability to assign reads to features in a
meaningful way by customisable assignment rules. `shortRNA` uses a
considerable proportion of reads, multi-mapping reads, that are either
discarded by other methods or aligned randomly. We demonstrated that
`shortRNA` correctly annotates reads to features using simulated reads
from the sperm dataset. In addition, we demonstrated that `shortRNA`
correctly identifies differentially expressed reads. Finally, using the
human PBMC data and corresponding qPCR quantifications from five mature
miRNAs, we demonstrated that `shortRNA` quantification is better
correlated with qPCR data than the other three tools.

In general, however, we observed a very low correlation of all methods
against qPCR in terms of the relative miRNA expression across samples.
There are a number of potential explanations for this discrepancy.
First, the lack of normalization of miRNA quantifications with U6
expression. In several investigations, including in this dataset, RNU6B
(U6) is used to normalize circulating miRNA data [@donati2019]. Although
qPCR is used to measure mature miRNA levels, it's great sensitivity
necessitates proper normalization to account for non-biological
variance. This is naturally different from the RNA sequencing analysis,
where the normalization is based on the whole distribution of RNAs. This
discrepancy could therefore be one reason for the low correlations. On
the contrary, it has been advised that U6 should not be utilized for
data normalization of circulating miRNAs [@donati2019; @benz2013]. Next,
the lack of UMIs in the data. UMIs make it possible to eliminate PCR
amplification biases[@fu2018]. The lack of UMIs in human PBMC data might
have resulted in technical duplication, and hence affecting the
correlation with the qPCR data.

Most tools for the sRNA-seq data analysis, including `Sports1` and `Oasis2`,
use feature-based counting, whereas `seqpac` uses sequence-based counting
for sRNA-seq data analysis. The benefit of sequence-based counting is
that it preserves sequence integrity and allows users to BLAST
[@altschul1990] candidate sequences to confirm classification and obtain
additional information. In addition, sequence-based counting allows for
the examination of sequence-specific variations or their boundaries.
Further, when using sequence-based counting, users can choose to remove
sequences that have low evidence and do not replicate across samples in
their data. `shortRNA`, on the other hand, has the advantage of both
sequence-based and feature-based counting due to the hierarchical
organization of features and mapping sequences. Because of the short
lengths of the reads and their proclivity to originate from higher-copy
number regions of the genome, multi-mapping reads are much more common
in sRNA-seq data. Commonly used sRNA-seq alignment methods for
multi-mapping reads are either very low precision (choosing an alignment
at random and make it difficult to determine their true origins) or
sensibility (ignoring multi-mapping reads, which result in losing a high
proportion of reads). `shortRNA` allows these reads to be aligned multiple
times and then assign them, using a set of customisable rules, either to
a specific feature or to a position in the feature tree which makes it
usable while preserving knowledge of eventual ambiguities.

`shortRNA` makes use of miRBase, GtRNADb, MttRNADb, SILVA and Ensembl
databases, but it is possible to manually add databases and extend the
list. We used adaptable assignment rules to identify the correct
assignment of a read. For example, if the data comes only from miRNAs,
the user can set higher priority for mapping the reads with miRNAs by
changing the number from 1 to 2 or higher. Further, we used a
phylogenetic tree structure to organize the features and to assign the
reads to features. If a sequence is overlapped with multiple features,
we assign the read to the parent of both features. This helps in solving
the problem of multi-mapping.

## Limitations

`shortRNA` has not been thoroughly tested across multiple platforms.
Although we anticipate that the `shortRNA` will be platform independent,
it has not yet been tested on other operating systems. We intend to do
so at a later date. In addition, we also intend to extensively benchmark
`shortRNA` with existing tools and strategies for reads assignment.

Simulations for other types of RNA are necessary to benchmark the read
assignment. We have only simulated reads from miRNAs and miRNA
precursors and demonstrated the proper assignment of reads using
`shortRNA`. Simulations of reads mapping to tRNAs, reads mapping to piRNAs
and reads mapping to rRNAs will be performed to better assess the reads
assignment.

The ambiguous assignment of multi-mapping reads to the parent of all
mappable features is not always biologically meaningful. If a read is
assigned to multiple tRNAs, Figure \ref{fig:3f11}, it is still meaningful to assign
it to the parent of all mapping tRNAs. In contrast, if a read maps to
both a tRNA and a protein-coding gene, assigning it to the parent of
both protein coding and tRNA provides less biological insight. Hence, we
plan to compare `shortRNA` to other read assignment strategies for
multi-mapping reads in depth. Other strategies for assigning
multi-mapping reads exist [@johnson2016; @handzlik2020], and a benchmark
with simulations would be required to assess the `shortRNA` assignment of
multi-mapping reads. Instead of assigning reads to the parent of all
mappable features, this would aid in the proper assignment and provide
more biological insight for the reads mapping to multiple biotypes.

The current implementation of `UMIc` for dealing with sRNA-seq data based
on UMIs is extremely slow. There are several for-loops for running
analysis, as well as a non-parallel implementation of the code by the
authors of `UMIc`. We intend to optimize the code for speed by
parallelizing it and replacing for-loops with parallel apply functions.

`shortRNA` has limited user-friendly plotting capabilities. Users can
currently use data visualization packages developed for
`TreeSummarizedExperiment`, such as `TreeHeatmap`, directly. However, we
intend to create wrappers for othe`R` packages in order to facilitate the
visualization of data and results. Also, we will be creating a Shiny app
for exploratory analysis of the data and results.

IsomiRs analysis could be implemented in `shortRNA`. IsomiRs are miRNAs
that vary slightly in sequence, which result from variations in the
cleavage site during miRNA biogenesis (5'-trimming and 3'-trimming
variants), nucleotide additions to the 3'-end of the mature miRNA
(3'-addition variants) and nucleotide modifications (substitution
variants). While in principle our hierarchical and sequence-based
approach is ideal for their investigation, tailored functions would make
this more straightforward for the users. We plan to extend ou`R` package
to include isoMiRs analysis by adapting functionalities of the `isomiR` `R`
package
[@lorenapantanoautcregeorgiaescaramisautchristosargyropoulosaut2017].

`shortRNA` is developed to work with single-end sRNA-seq data. We plan to
extend our package to work with paired-end sRNA-seq data.

## Conclusion

In conclusion, we developed an `R` package, platform-independent, for
analysis of the sRNA-seq data.This package provides a great overview of
the quality, performs QC, aligns the data to custom made genome,
performs reads annotation and assignment with adaptable assignment
rules, organizes reads and features as a phylogenetic tree, performs
differential analysis, and provides interactive plots for exploratory
data analysis. `shortRNA` currently works with single-end sequencing data.

## Data and code availibility

`shortRNA` is available from: github.com/mansuylab/`shortRNA`

No sequencing data were generated during this study. Code for simulating
reads is presented in [Appendix D](#ad).

Data analysis steps for sRNA-seq data with `shortRNA` is available in
[Appendix C](#ac).




<!--chapter:end:rmd/08-shortRNA.Rmd-->

# Conclusion and outlook {#conclusion .unnumbered}

## Overall conclusion{.unnumbered}

In this Doctoral thesis, first, the work is presented on the development of pipelines for multi-omics high-throughput sequencing data, [Methods](#methods), and data analysis pipelines have been used to analyze muli-omics data from the lab and literature, [Appendix A](#aa). Next as summarized in [Chapter 1](#chapter1), we compare the open chromatin landscape of developing and adult spermatogonial cells to explore the development of spermatogonial cells in postnatal development. We discovered and characterized genomic regions with differential genomic activity in spermatogonial cells by combining the data generated by the host lab with the data available from the literature. After that, as summarized in [Chapter 2](#chapter2), we studied the impact of early postnatal stress on the transcriptome of epididymal extracellular vesicles. Our findings show that chronic stress in early postnatal life affects miRNAs in adult male reproductive tract extracellular vesicles, with consequences for mature sperm and zygotes. Finally, in [Chapter 3](#chapter3) we created an `R` package, `shortRNA`, for analyzing short RNA sequencing data that is platform-independent, provides a good overview of the quality, performs quality control, aligns the data to a custom-made genome, performs read annotation and assignment with adaptable assignment rules, organizes reads and features as a phylogenetic tree, performs differential analysis, and provides interactive plots for exploratory data analysis. However, there are a number of changes or further investigations that might be needed, which we will explore in this section.


## MSUS-associated differences {.unnumbered}

The MSUS model is appropriate for examining epigenetic inheritance owing to environmental perturbations, but low effect sizes make it difficult to discover candidates that passed multiple testing corrections. The costs associated with sample creation and sequencing, makes it challenging to have a higher number of replicates in the experimental designs. After multiple testing correction, obtaining significant p-values requires either appropriate data distribution assumptions or a high number of replicates. Clipper, a new statistical framework for reducing false discovery rates without relying on p-values or specific data distributions, was published recently (Ge et al. 2021). `Clipper` has been shown to outperform conventional methods for limiting false discoveries, thus pipelines can be updated to implement it.


## RNA sequencing data analysis pipeline {.unnumbered}

Data analysis pipelines for RNA sequencing could be adapted. `Salmon`, a tool for obtaining quantifications from RNA-seq data, is used for pseudo-alignment while accounting for GC biases at the fragment level, sequence-specific biases, and position-specific fragment starts distribution. `Salmon Anomaly Detection` (`SAD`) tool can be incorporated into the RNA sequencing data analysis pipeline in addition to `Salmon` quantifications. `SAD` identifies probable misquantifications in `Salmon`'s RNA-seq transcript expression estimation. However, due to several external dependencies, implementing this tool may be difficult (https://github.com/Kingsford-Group/sad/issues), and this will almost certainly necessitate containerization from the authors, either via `conda` or `Docker`. Also, while evaluating and quantifying lowly-expressed genes and small RNAs with alignment-free pipelines, such as `Salmon`, potential pitfalls have been recognized, particularly when the small RNAs contain biological variability. As a result, when the analysis isn't limited to simply protein-coding genes, an alignment-based technique like `Rsubread` or `STAR` can be used (Wu et al. 2018). Further, for differential transcript usage and isoform switching, we employed `IsoformSwitchAnalyzeR`. Under the hood, `IsoformSwitchAnalyzeR` employs `DEXseq` for differential transcript usage. However, `DEXseq` has the drawback of not being able to account for blocking variables. `BANDITS` could also be used instead of `DEXseq` in the `IsoformSwitchAnalyzeR` pipeline because it outperforms `DEXseq` (Tiberi and Robinson 2020).


## ATAC sequencing data analysis pipeline {.unnumbered}

In recent years, ATAC-seq has grown in popularity as a tool for investigating chromatin accessibility. Despite improvements in protocols, bioinformatics analysis tools are still in their development, with no comprehensive analytical pipeline defined. The interpretation of ATAC-seq results is currently hampered by this. The majority of the tools used to analyze ATAC-seq data are from the ChIP-seq or DNA-seq fields. For example, `MACS2`, the peak caller we used for ATAC-seq, was designed originally for ChIP-seq data. `HMMRATAC` is a peak calling method for ATAC-seq data that combines the nucleosome-free and nucleosome-enriched signals from a single ATAC-seq dataset to find open chromatin regions (Tarbell and Liu 2019). Our strategy was instead to use the nucleosome-free fragments (which are indicative of TF binding) for peak calling. Because we only looked into nucleosome-free fragments, `HMMRATAC` was not used. Also, `HMMRATAC`'s authors advise against using it on ATAC-seq datasets that have undergone physical or computational size selection. But, `HMMRATC` could be employed for investigating outside the nucleosome-free fragments.

The ATAC-seq pipeline can also be extended to include tools for nucleosome positioning and transcription factor footprinting analysis. However, nucleosome detection is problematic due to an inherent shortcoming of ATAC-seq data, which typically has inadequate read coverage beyond peaks (Yan et al. 2020). Based on empirical and computational estimates, the recommended minimum number of mapped reads for open chromatin detection and differential analysis in mammalian species is 50 million, and 200 million for TF footprinting (Yan et al. 2020). Due to the reduced read coverage outside the open chromatin regions, nucleosome detection in ATAC-seq data was shown to be more difficult than in MNase-seq data (Yan et al. 2020). Because we did not sequence this deep, we did not include TF footprinting in the pipeline, which would necessitate a thorough investigation of tools. These drawbacks necessitate a detailed study of the tools employed in ATAC-seq for normalization, peak calling on NFFs, and TF footprinting.


## Updating manuscript from [Chapter 1](#chapter1) in light of latest RNA-seq data {.unnumbered}

The datasets used for the study of spermatogonial cell development were not initially designed for that purpose, but to investigate the changes at different developmental stages because of the MSUS paradigm. As a result, the datasets were not ideal for this new purpose: the study originally employed polyA RNA sequencing data from the PNW21 stage, while RNA sequencing data from the stages PND8 and PND15 came from Total RNA sequencing. We, therefore, used data from the literature but determined to re-generate the data from the PNW21 stage in the lab in order to have more impactful research and accurate findings. Because of these changes, the entire downstream analysis, including functional analysis, motif analysis, data integration, and re-creation of the figures, will have to be repeated. Some of these analyses have been completed and are included in [Chapter 1](#chapter1) as an update. In general, however, given the high consistency of the results across the two analyses, we are confident that the main results will remain unchanged.


## Low correlations of quantification between `shortRNA` and qPCR {.unnumbered}

In terms of relative miRNA expression across samples, we found a low correlation between sequencing-based quantifications (with `shortRNA` as well as with other tools) and qPCR. We covered a variety of possible explanations for this mismatch in [Chapter 3](#chapter3), including normalizing the data with U6 RNA expression and the lack of UMIs. We don't have an exact explanation for the poor correlations, but we want to look into them further using a variety of techniques, including testing the shortRNA tool with UMI-based data with corresponding qPCR quantifications and creating a more extended simulation dataset.

<!--chapter:end:rmd/09-outlook.Rmd-->

# Appendix A {- #aa}

\markboth{Datasets analyzed during the thesis}{Appendix A}

\noindent

\setlength\parindent{0pt}

## Datasets analyzed from lab's MSUS model{-}

```{r, message=FALSE, warning=FALSE, echo=FALSE}
library(readxl)
df1 <- data.frame(read_xlsx("figure/appendix/datasets_dkt.xlsx", sheet = 1), check.names = F)
df1 <- df1[,c(3,4,5,6,7,8,9,10,11,12)]
df2 <- data.frame(read_xlsx("figure/appendix/datasets_dkt.xlsx", sheet = 2), check.names = F)
df2 <- df2[,c(3:13)]
df3 <- read_xlsx("figure/appendix/datasets_dkt.xlsx", sheet = 3)
df3 <- df3[,-c(1:2)]

df1$GitHub <- sapply(df1$GitHub, function(x) paste(x, collapse = " ", sep = "\n"))

library(kableExtra)

kbl(df1, booktabs = T) %>%
kable_styling(latex_options = c("striped", "scale_down")) %>%
  kable_styling(latex_options = c("HOLD_position")) 
# %>%
  # landscape()
```

## Human dataset from the Schratt lab{-}

```{r, message=FALSE, warning=FALSE, echo=FALSE}
kbl(df3, booktabs = T) %>%
kable_styling(latex_options = c("striped", "scale_down")) %>%
  kable_styling(latex_options = c("HOLD_position")) 
```

## Mouse datasets analyzed from literature{-}

```{r, message=FALSE, warning=FALSE, echo=FALSE}
kbl(df2, booktabs = T) %>%
kable_styling(latex_options = c("striped", "scale_down")) %>%
  kable_styling(latex_options = c("HOLD_position")) 
```


<!--chapter:end:rmd/11-appendix1.Rmd-->

# Appendix B {- #ab}

\markboth{Other publications or manuscripts}{Appendix B}

\noindent

\setlength\parindent{0pt}


## Other publications and manuscripts during doctoral thesis work {-}


### Involvement of circulating factors in the transmission of paternal experiences through the germline{-}
Gretchen van Steenwyk\textsuperscript{$\dagger$}, Katharina Gapp\textsuperscript{$\dagger$}, Ali Jawaid, Pierre-Luc Germain, Francesca Manuella, **Deepak K Tanwar**, Nicola Zamboni, Niharika Gaur, Anastasiia Efimova, Kristina M Thumfart, Eric A Miska,  Isabelle Mansuy\textsuperscript{\#} \newline 

\textsuperscript{$\dagger$}Equal contributions \newline 
\textsuperscript{\#}Corresponding author \newline 

**Journal:** *The EMBO Journal* \newline 
\textbf{DOI:} [10.15252/embj.2020104579](https://doi.org/10.15252/embj.2020104579)\newline 

**Contributions:** *Analysis of RNA-seq data.*

### Symposium summary: Epigenetic inheritance-impact for biology and society 26–28 August 2019, Zurich, Switzerland{-}

Irina Lazar-Contes, Martin Roszkowski, **Deepak K Tanwar** and Isabelle M Mansuy\textsuperscript{\#}

\textsuperscript{\#}Corresponding author \newline 

**Journal:** *Environmental Epigenetics* \newline 
\textbf{DOI:} [10.15252/embj.2020104579](https://doi.org/10.15252/embj.2020104579)\newline 

**Contributions:** *Writing manuscript.*

### OmniSperm: Multiomic analyses of sperm and offspring production from a single male{-}

Martin Roszkowski, Irina Lazar-Contes, Pierre-Luc Germain, **Deepak K Tanwar**, Anara Alshanbayeva, Niharika Obrist, Ali Jawaid, Gretchen van Steenwyk, Eloise Kremer, Dalila Korkmaz, Mark Ormiston, Francesca Manuella, Johannes vom Berg, Jorg Tost, Johannes Bohacek, Isabelle M Mansuy\textsuperscript{\#}

\textsuperscript{\#}Corresponding author \newline 

\textbf{DOI:} [10.3929/ethz-b-000489446](https://doi.org/10.3929/ethz-b-000489446) (Doctoral thesis, Chapter 2)\newline 

**Contributions:** *Analysis of RNA-seq from sperm and embryos, and WGBS data from sperm.*

### Early life stress alters chromatin accessibility landscape and transcript usage in spermatogonial cells during postnatal testis maturation{-}

Irina Lazar-Contes, Gretchen van Steenwyk, **Deepak K Tanwar**, Pierre-Luc Germain, Francesca Manuella, Martin Roszkowski, Niharika Gaur, Isabelle M Mansuyy\textsuperscript{\#}

\textsuperscript{\#}Corresponding author \newline 

\textbf{DOI:} [10.1101/2020.08.20.259374](https://doi.org/10.1101/2020.08.20.259374) (Doctoral thesis, Chapter 3)\newline 

**Contributions:** *Analysis of RNA-seq and ATAC-seq data, and figure generation.*

<!--chapter:end:rmd/12-appendix2.Rmd-->

# Appendix C {- #ac}

\markboth{shortRNA analysis steps}{Appendix C}

\noindent

\setlength\parindent{0pt}


## shortRNA analysis steps in R (a short vignette){-}

### Package installation{-}
```{r, eval = FALSE}
BiocManager::install("mansuylab/shortRNA")
library(shortRNA)
```

### Quality check and trimming{-}
```{r, eval = FALSE}
qc_SE(file = fq_files, outdir = "output/", ad1 = adapter_sequence)
```

### FastQ files to sequence by counts matrix{-}
```{r, eval = FALSE}
m <- fastq2SeqCountMatrix(files = trimmed_fastq_files)
```

### Unique sequences for alignment{-}
```{r, eval = FALSE}
fa <- DNAStringSet(row.names(m))
names(fa) <- paste0("S", 1:length(fa))
writeXStringSet(fa, fasta_file)
```


### Obtaining databases for analysis{-}
```{r, eval = FALSE}
db <- getDB()
```


### Annotation preparation and genome index generation for alignment{-}
```{r, eval = FALSE}
a <- prepareAnnotation(
  ensdb = db$ensdb,
  output_dir = genomeDir,
  extra.gr = list(piRNA = db$piRNA_GR, miRNA = db$miRNA_GR),
  extra.seqs = list(rRNA = db$rRNA_fa, tRNA = db$tRNA_fa),
  resolveSplicing = NULL,
  rules = defaultAssignRules(),
  tRNAEnsembleRemove = FALSE,
  clusterMiRNA = TRUE
)
```


### Alignment{-}
```{r, eval = FALSE}
alignShortRNA(
  fastq = "unique.fasta",
  index = "genomeDir/customGenome",
  outDir = "align", GTF = exonsBy(db$ensdb),
  GTF.featureType = "exon", GTF.attrType = "gene_id"
)
```


### Overlapping aligned data with annotations{-}
```{r, eval = FALSE}
o <- overlapWithTx2(
  bamFile = align_file, annotation = a,
  ignoreStrand = TRUE, nbthreads = 16
)
```


### Assignment of overlapping reads to features with assignment rules{-}
```{r, eval = FALSE}
ar <- assignReads(sources = o, rules = defaultAssignRules())
```


### Converting features to FactorList{-}
```{r, eval = FALSE}
fl <- featuresAnnoToFL(a)
names(fl) <- lapply(fl, function(x) as.character(x[length(x)]))
```

### Assignment of reads to the features tree{-}
```{r, eval=FALSE}
mappedFeaturesDF <- ar
mappedFeaturesDF$seq <- rownames(mappedFeaturesDF)
ar_tree <- addReadsToTree(
  fL = fl,
  mappedFeaturesDF = mappedFeaturesDF,
  unassigned = FALSE,
  extraTreeBranch = NULL
)
```


### Creation of TreeSummarizedExperiment object{-}
```{r, eval = FALSE}
library(TreeSummarizedExperiment)

rt <- ar_tree
as <- list(counts = m)
cd <- DataFrame(samples = c(s1, s2), group - c("group1", "group2"))

tse <- TreeSummarizedExperiment(
  assays = as,
  rowTree = rt,
  colData = cd,
  rowData = ar[row.names(m), ],
  metadata = list(
    assignedReads = ar,
    counts = m,
    notAligned = getReadsFromBam(
      bam = align_file
    )
  )
)
```


### Differential analysis{-}
```{r, eval = FALSE}
library(treeClimbR)
dea <- runDA(TSE = tse, filter_min_count = 20)
out <- nodeResult(object = dea, n = Inf)

cand <- getCand(
  tree = dea$tree,
  score_data = out, node_column = "node",
  p_column = "PValue", sign_column = "logFC",
  message = FALSE, threshold = 0.05
)

candB <- evalCand(
  tree = dea$tree,
  levels = cand$candidate_list,
  score_data = out, node_column = "node",
  p_column = "PValue", sign_column = "logFC",
  method = "BH", limit_rej = 0.05,
  use_pseudo_leaf = FALSE,
  message = FALSE
)

result <- topNodes(object = candB, n = Inf, p_value = 0.05)
```

<!--chapter:end:rmd/13-appendix3.Rmd-->

# Appendix D {- #ad}

\markboth{Simulated reads for testing shortRNA}{Appendix D}

\noindent

\setlength\parindent{0pt}


## Simulated reads for testing shortRNA{-}

### Libraries{-}
```{r, eval=FALSE}
library(Biostrings)
library(rtracklayer)
```

### miRNAs{-}
```{r, eval=FALSE}
fa <- readRNAStringSet("https://www.mirbase.org/ftp/CURRENT/mature.fa.gz")
fa <- DNAStringSet(fa)
names(fa) <- gsub(" .+", "", names(fa))
fa <- fa[grep(pattern = "mmu", x = names(fa))]
```

### cluster chr17:17830188-17830879{-}
```{r, eval=FALSE}
clusterMirs <- c(
  "mmu-miR-125a-5p", "mmu-miR-99b-5p", "mmu-let-7e-5p",
  "mmu-let-7e-3p", "mmu-miR-99b-3p", "mmu-miR-125a-3p"
)
clusterMirs <- fa[clusterMirs]
```

### Imputing differences for differential analysis{-}
```{r, eval=FALSE}
sizefacts <- c(
  SRR13129036 = 1.2, SRR13129037 = 0.99, SRR13129038 = 1.01,
  SRR13129039 = 1.01, SRR13129040 = 1.07, SRR13129041 = 0.56,
  SRR13129042 = 0.72, SRR13129043 = 0.51
)
```
### Read counts{-}
```{r, eval=FALSE}
set.seed(123)
clusterMirsCounts <- t(sapply(rgamma(length(clusterMirs),
  shape = 3
), function(x) {
  sizefacts *
    exp(1 + x) *
    rnorm(8, mean = rep(c(1, rnorm(1, 2, sd = 0.1)), each = 4), sd = 0.4)
}))
row.names(clusterMirsCounts) <- names(clusterMirs)
clusterMirsCounts <- matrix(rpois(length(clusterMirsCounts),
  lambda = clusterMirsCounts
),
nrow = nrow(clusterMirsCounts), dimnames = dimnames(clusterMirsCounts)
)
clusterMirsCounts
row.names(clusterMirsCounts) <- as.character(clusterMirs)
```
### miRNA precursor{-}
```{r, eval=FALSE}
mir144prec <- DNAStringSet(c(mir144precursor = substr(
  "GGCTGGGATATCATCATATACTGTAAGTTTGTGATGAGACACTACAGTATAGATGATGTACTAGTC",
  1, 50
)))
mir144 <- c(
  mir144prec, fa["mmu-miR-144-3p"], fa["mmu-miR-144-5p"],
  DNAStringSet(c(
    mir144_invalid1 = "AAGTTTGTGATGAGACACTAC",
    mir144_invalid2 = "CTGTAAGTTTGTGATGAGAC",
    mir144_invalid3 = "ATGTACTAGTCTGGGTA"
  ))
)

set.seed(123)
mu <- rbind(
  abs(rnorm(8, 1)),
  abs(rnorm(8, rep(2:1, each = 4), 0.3)),
  abs(rnorm(8, rep(2:1, each = 4), 0.3)),
  abs(rnorm(8, 1)), abs(rnorm(8, 1)), abs(rnorm(8, 1))
)
mu <- 300 * t(t(mu) * sizefacts)
mir144Counts <- matrix(rpois(length(mu), lambda = mu),
  nrow = nrow(mu),
  dimnames = list(names(mir144), names(sizefacts))
)
row.names(mir144Counts) <- as.character(mir144)
```
### Data to be simulated{-}
```{r, eval=FALSE}
e <- rbind(clusterMirsCounts, mir144Counts)
e
```

<!--chapter:end:rmd/14-appendix4.Rmd-->

# Appendix E {- #ae}

Curriculum Vitae of the doctoral student.

<!-- \backmatter -->
\setcounter{secnumdepth}{-1}
\includepdf[pages=-,addtotoc={1, section, 1, CV, CV}]{data/Deepak_Tanwar.pdf}

<!--chapter:end:rmd/15-cv.Rmd-->

<!--
The bib chunk below must go last in this document according to how R Markdown renders.  More info is at http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
-->

\backmatter

<!-- 
If you'd like to change the name of the bibliography to something else,
delete "References" and replace it.
-->

# References {-}
<!--
This manually sets the header for this unnumbered chapter.
-->
\markboth{References}{References}
<!--
To remove the indentation of the first entry.
-->
\noindent

<!--
To create a hanging indent and spacing between entries. This line may need 
to be removed for styles that don't require the hanging indent.
-->

\setlength{\parindent}{-0.20in}


<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->

---
nocite: | 
  @angel2000, @angel2001, @angel2002a
...


<!--chapter:end:rmd/99-references.Rmd-->

