---
output: html_document
editor_options: 
  chunk_output_type: console
bibliography: rmd/references.bib
# header-includes:
#   \usepackage{float} \floatplacement{figure}{H}
#   \newcommand{\beginsupplement}{\setcounter{table}{0}  \renewcommand{\thetable}{S\arabic{table}} \setcounter{figure}{0} \renewcommand{\thefigure}{S\arabic{figure}}}
---

<!-- \newpage -->

## Supplementary Figures

<!-- \beginsupplement -->

\begin{subfigures}

(ref:dsc1a) \textbf{FACS leads to enrichment of PLZF+ cells in testicular preparations from mouse.} \newline \textbf{A:} Representative dot plots of the sorting strategy for spermatogonial cell enrichment. Gating based on side scatter/forward scatter (SSC-A/FSC-A) and forward scatter – height/ forward scatter – area (FSC-H/FSC-A) was performed to exclude cell debris and cell clumps. \newline \textbf{B:} PLZF+ cells are enriched following FACS, illustrated by immunocytochemistry on unsorted and sorted cell preparations. Immunocytochemistry of PND15 unsorted and the sorted testis cell suspension was performed by fixating cells on poly-L-lysine coated slides. Cells were stained with anti-PLZF antibody (S19 clone, Active Motif) and VECTASHIELD (with DAPI) antifade mounting medium was used for mounting. Cells were visualized with a fluorescence microscope and counted in 10 different fields of view/slide. The number of PLZF+ and PLZF- cells from 10 different fields of view were averaged.

```{r ds1a, echo=FALSE, fig.align='center', fig.cap="(ref:dsc1a)", fig.scap="FACS leads to enrichment of PLZF+ cells in testicular preparations from mouse"}
img <- readPNG("figure/dev/s1a.png")
grid.raster(img)
```

(ref:dsc1b) \textbf{FACS leads to enrichment of PLZF+ cells in testicular preparations from mouse.} \newline \textbf{C} Heatmap of the expression profile of selected markers for spermatogonial and different testicular somatic cells from our PND8 and PND15  RNA-seq data, and PND14 and PNW8 datasets from the literature. Key genes for stem cell potential, stem and progenitor spermatogonia, and Leydig and Sertoli cells were chosen to evaluate spermatogonial cells enrichment in the populations. Gene expression is represented in Log2CPM (counts per million).

```{r ds1b, echo=FALSE, fig.align='center', fig.cap="(ref:dsc1b)", fig.scap="FACS leads to enrichment of PLZF+ cells in testicular preparations from mouse"}
img <- readPNG("figure/dev/s1b.png")
grid.raster(img)
```

\end{subfigures}


\begin{subfigures}

(ref:dsc2a) \textbf{Omni-ATAC profiles of PND15 and adult spermatogonial cells and their genomic distribution.} \newline \textbf{A:} Genomic distribution of the 158, 978 identified Omni-ATAC regions in between exonic, intronic, intergenic and located +/- 1kb of a transcriptional start site (TSS).

```{r ds2a, echo=FALSE, fig.align='center', fig.cap="(ref:dsc2a)", fig.scap="Omni-ATAC profiles of PND15 and adult spermatogonial cells and their genomic distribution"}
img <- readPNG("figure/dev/s2a.png")
grid.raster(img)
```

(ref:dsc2b) \textbf{Omni-ATAC profiles of PND15 and adult spermatogonial cells and their genomic distribution.} \newline \textbf{B:} Dot plots of top enriched GO biological processes for regions with increased chromatin accessibility in adult spermatogonia, within gene bodies and around TSS of nearby genes (TSS +/- 1kb). The size of dots indicates the number of genes in the term and the colour of each dot corresponds to the adjusted P-value of the term’s enrichment.

```{r ds2b, echo=FALSE, fig.align='center', fig.cap="(ref:dsc2b)", fig.scap="Omni-ATAC profiles of PND15 and adult spermatogonial cells and their genomic distribution"}
img <- readPNG("figure/dev/s2b.png")
grid.raster(img)
```

\end{subfigures}


(ref:dsc3) \textbf{Representative examples from Categories 1-3 resulted from the overlap of chromatin accessibility, gene expression and histone profiling datasets.} Genomic snapshots from the Integrative Genomics Viewer (IGV, Broad Institute) of exemplary genes from \textbf{A:} Category 1 (Pdpk1 and Gata2), \textbf{B:} Category 2 (Hmx1 and Fgf8), and \textbf{C:} Category 3 (Dap2ip) showing the relative abundance of transcripts from RNA-seq, chromatin accessibility from ATAC-seq and enrichment of H3K27ac, H3K4me3 and H3K27me3 from ChIP-seq. RNA-seq data correspond to literature PND14 and PNW8 spermatogonial cells and  ATAC-seq data to PND15 and adult spermatogonial cells.

```{r ds3, echo=FALSE, fig.align='center', fig.cap="(ref:dsc3)", fig.scap="Representative examples from Categories 1-3 resulted from the overlap of chromatin accessibility, gene expression and histone profiling datasets"}
img <- readPNG("figure/dev/s3.png")
grid.raster(img)
```

(ref:dsc4) \textbf{DNAme profiles of spermatogonial cells do not show major changes over the period of testis postnatal maturation.} Enriched heatmaps showing the overlap between Category 1-4 regions, literature ChIP-seq data in PNW8 spermatogonia for H3K4me3, H3K27ac and H3K27me3, and DNAme data from BS in PND7, PND14 and PNW8 spermatogonia. For each of the Category 1-4 the following sub-categorization was applied: regions that are enriched for H3K4me3 (with or w/o H3K27ac and/or H3K27me3), regions that are enriched for H3K27ac (and lack both H3K4me3 and H3K27me3) and regions that are enriched for H3K27me3 (and lack both H3K4me3 and H3K27ac).  Each line represents a peak region and the regions are ordered by the ATAC-seq signal. Mid-x-axis corresponds to the middle of a peak region and is extended to +/- 1 kbp. The colour-key of the ATAC-seq, ChIP-seq and BS heatmaps represent ATAC-seq, ChIP-seq and BS signal, respectively. For RNA-seq, log2 FC is shown from PND8 vs PND15 and PND14 vs PNW8 comparisons. For BS, the level of DNAme is between 0 and 1, with 0 representing completely unmethylated loci and 1 fully methylated loci, respectively.

```{r ds4, echo=FALSE, fig.align='center', fig.cap="(ref:dsc4)", fig.scap="DNAme profiles of spermatogonial cells do not show major changes over the period of testis postnatal maturation"}
img <- readPNG("figure/dev/s4.png")
grid.raster(img)
```



(ref:dsc5) \textbf{Distinct chromatin profiles between PND15 and adult spermatogonia are present at distal regions across the genome.} \textbf{(A-B)} Enriched heatmaps showing the overlap between more accessible regions \textbf{(A)} and less accessible regions \textbf{(B)} situated in distal regions in spermatogonial cells, and literature ChIP-seq and DNAme data in PNW8 spermatogonia. The following sub-categorization was applied: regions that are enriched for H3K4me3 (with or w/o H3K27ac and/or H3K27me3), regions that are enriched for H3K27ac (and lack both H3K4me3 and H3K27me3) and regions that are enriched for H3K27me3 (and lack both H3K4me3 and H3K27ac).  Each line represents a peak region and the regions are ordered by the ATAC-seq signal. Mid-x-axis corresponds to the middle of a peak region and is extended to +/- 1 kbp. The colour-key of the ATAC-seq, ChIP-seq and BS heatmaps represent ATAC-seq, ChIP-seq and BS signal, respectively. For BS, the level of DNAme is between 0 and 1, with 0 representing completely unmethylated loci and 1 fully methylated loci, respectively.


```{r ds5, echo=FALSE, fig.align='center', fig.cap="(ref:dsc5)", fig.scap="Distinct chromatin profiles between PND15 and adult spermatogonia are present at distal regions across the genome"}
img <- readPNG("figure/dev/s5.png")
grid.raster(img)
```


(ref:dsc6) \textbf{Differentially accessible TEs exhibit enriched TF motifs and correspond to regions nearby non-random gene families.} \textbf{A:} HOMER extracted consensus sequences for TF motifs enriched in less accessible ERVK subtypes. Representative examples from the most enriched transcription factor families are depicted. \textbf{B:} Genomic snapshots from the Integrative Genomics Viewer (IGV, Broad Institute) of a cluster of Olfr genes on Chr2 overlapping an increased density of LINE elements relative to the neighbouring regions. The relative abundance of transcripts from RNA-seq and chromatin accessibility from ATAC-seq are shown. RNA-seq data corresponds to literature PND14 and adult (PNW8) spermatogonial cells and ATAC-seq data to PND15 and adult (PNW20) spermatogonial cells, respectively.


```{r ds6, echo=FALSE, fig.align='center', fig.cap="(ref:dsc6)", fig.scap="Differentially accessible TEs exhibit enriched TF motifs and correspond to regions nearby non-random gene families"}
img <- readPNG("figure/dev/s6.png")
grid.raster(img)
```