<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse | Understanding the molecular mechanisms of germline-dependent epigenetic inheritance: Computational analysis of multi-omics data</title>
  <meta name="description" content="Chapter 2 Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse | Understanding the molecular mechanisms of germline-dependent epigenetic inheritance: Computational analysis of multi-omics data" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse | Understanding the molecular mechanisms of germline-dependent epigenetic inheritance: Computational analysis of multi-omics data" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse | Understanding the molecular mechanisms of germline-dependent epigenetic inheritance: Computational analysis of multi-omics data" />
  
  
  

<meta name="author" content="Deepak Kumar Tanwar" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="1-developmental.html"/>
<link rel="next" href="3-shortRNA.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="abstract-de.html"><a href="abstract-de.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="0.1" data-path="intro.html"><a href="intro.html#epigenetics"><i class="fa fa-check"></i><b>0.1</b> Epigenetics</a></li>
<li class="chapter" data-level="0.2" data-path="intro.html"><a href="intro.html#germline-epigenetics"><i class="fa fa-check"></i><b>0.2</b> Germline epigenetics</a>
<ul>
<li class="chapter" data-level="0.2.1" data-path="intro.html"><a href="intro.html#spermatogonial-stem-cells"><i class="fa fa-check"></i><b>0.2.1</b> Spermatogonial Stem Cells</a></li>
<li class="chapter" data-level="0.2.2" data-path="intro.html"><a href="intro.html#reprogramming"><i class="fa fa-check"></i><b>0.2.2</b> Reprogramming</a></li>
</ul></li>
<li class="chapter" data-level="0.3" data-path="intro.html"><a href="intro.html#epigenetic-inheritance"><i class="fa fa-check"></i><b>0.3</b> Epigenetic Inheritance</a>
<ul>
<li class="chapter" data-level="0.3.1" data-path="intro.html"><a href="intro.html#transgenerational-epigenetic-inheritance"><i class="fa fa-check"></i><b>0.3.1</b> Transgenerational Epigenetic Inheritance</a></li>
</ul></li>
<li class="chapter" data-level="0.4" data-path="intro.html"><a href="intro.html#models-of-epigenetic-inheritance"><i class="fa fa-check"></i><b>0.4</b> Models of Epigenetic Inheritance</a></li>
<li class="chapter" data-level="0.5" data-path="intro.html"><a href="intro.html#msus-mouse-model"><i class="fa fa-check"></i><b>0.5</b> MSUS mouse model</a></li>
<li class="chapter" data-level="0.6" data-path="intro.html"><a href="intro.html#vectors-of-tei"><i class="fa fa-check"></i><b>0.6</b> Vectors of TEI</a></li>
<li class="chapter" data-level="0.7" data-path="intro.html"><a href="intro.html#extracellular-vesicles"><i class="fa fa-check"></i><b>0.7</b> Extracellular vesicles</a></li>
<li class="chapter" data-level="0.8" data-path="intro.html"><a href="intro.html#small-rnas"><i class="fa fa-check"></i><b>0.8</b> Small RNAs</a>
<ul>
<li class="chapter" data-level="0.8.1" data-path="intro.html"><a href="intro.html#challenges-for-analyzing-srna-seq-dataset"><i class="fa fa-check"></i><b>0.8.1</b> Challenges for analyzing sRNA-seq dataset</a></li>
</ul></li>
<li class="chapter" data-level="0.9" data-path="intro.html"><a href="intro.html#aims"><i class="fa fa-check"></i><b>0.9</b> Aims</a></li>
<li class="chapter" data-level="0.10" data-path="intro.html"><a href="intro.html#thesis-overview"><i class="fa fa-check"></i><b>0.10</b> Thesis overview</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i>Methods</a>
<ul>
<li class="chapter" data-level="0.11" data-path="methods.html"><a href="methods.html#version-controlled-data-analysis-using-git"><i class="fa fa-check"></i><b>0.11</b> Version controlled data analysis using git</a></li>
<li class="chapter" data-level="0.12" data-path="methods.html"><a href="methods.html#data-analysis-directory-organization"><i class="fa fa-check"></i><b>0.12</b> Data analysis directory organization</a></li>
<li class="chapter" data-level="0.13" data-path="methods.html"><a href="methods.html#pipelines-for-data-analysis"><i class="fa fa-check"></i><b>0.13</b> Pipelines for data analysis</a>
<ul>
<li class="chapter" data-level="0.13.1" data-path="methods.html"><a href="methods.html#bulk-rna-seq"><i class="fa fa-check"></i><b>0.13.1</b> Bulk RNA-seq</a></li>
<li class="chapter" data-level="0.13.2" data-path="methods.html"><a href="methods.html#atac-seq"><i class="fa fa-check"></i><b>0.13.2</b> ATAC-seq</a></li>
<li class="chapter" data-level="0.13.3" data-path="methods.html"><a href="methods.html#chip-seq"><i class="fa fa-check"></i><b>0.13.3</b> ChIP-seq</a></li>
<li class="chapter" data-level="0.13.4" data-path="methods.html"><a href="methods.html#wgbs-rrbs"><i class="fa fa-check"></i><b>0.13.4</b> WGBS &amp; RRBS</a></li>
<li class="chapter" data-level="0.13.5" data-path="methods.html"><a href="methods.html#small-rna-seq"><i class="fa fa-check"></i><b>0.13.5</b> small RNA-seq</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-developmental.html"><a href="1-developmental.html"><i class="fa fa-check"></i><b>1</b> Dynamic chromatin accessibility in spermatogonial cells for transcriptional programmings from early postnatal to adult stages</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-developmental.html"><a href="1-developmental.html#graphical-abstract"><i class="fa fa-check"></i><b>1.1</b> Graphical Abstract</a></li>
<li class="chapter" data-level="1.2" data-path="1-developmental.html"><a href="1-developmental.html#abstract-1"><i class="fa fa-check"></i><b>1.2</b> Abstract</a></li>
<li class="chapter" data-level="1.3" data-path="1-developmental.html"><a href="1-developmental.html#introduction"><i class="fa fa-check"></i><b>1.3</b> Introduction</a></li>
<li class="chapter" data-level="1.4" data-path="1-developmental.html"><a href="1-developmental.html#results"><i class="fa fa-check"></i><b>1.4</b> Results</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-developmental.html"><a href="1-developmental.html#enrichment-of-spermatogonial-cells-from-postnatal-and-adult-mouse-testis"><i class="fa fa-check"></i><b>1.4.1</b> Enrichment of spermatogonial cells from postnatal and adult mouse testis</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-developmental.html"><a href="1-developmental.html#chromatin-is-remodeled-in-spermatogonial-cells-during-development"><i class="fa fa-check"></i><b>1.4.2</b> Chromatin is remodeled in spermatogonial cells during development</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-developmental.html"><a href="1-developmental.html#differentially-accessible-chromatin-regions-associate-with-distinct-gene-expression-dynamics"><i class="fa fa-check"></i><b>1.4.3</b> Differentially accessible chromatin regions associate with distinct gene expression dynamics</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-developmental.html"><a href="1-developmental.html#differentially-accessible-chromatin-regions-associate-with-distinct-epigenetic-profiles"><i class="fa fa-check"></i><b>1.4.4</b> Differentially accessible chromatin regions associate with distinct epigenetic profiles</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-developmental.html"><a href="1-developmental.html#accessibility-changes-at-open-chromatin-regions-are-markedcarry-by-binding-sites-for-distinct-families-of-transcription-factors"><i class="fa fa-check"></i><b>1.4.5</b> Accessibility changes at open chromatin regions are markedcarry by binding sites for distinct families of transcription factors</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-developmental.html"><a href="1-developmental.html#chromatin-accessibility-at-transposable-elements-undergoes-significant-remodeling-in-the-transition-from-postnatal-to-adult-spermatogonia"><i class="fa fa-check"></i><b>1.4.6</b> Chromatin accessibility at transposable elements undergoes significant remodeling in the transition from postnatal to adult spermatogonia</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-developmental.html"><a href="1-developmental.html#discussion"><i class="fa fa-check"></i><b>1.5</b> Discussion</a></li>
<li class="chapter" data-level="1.6" data-path="1-developmental.html"><a href="1-developmental.html#methods-1"><i class="fa fa-check"></i><b>1.6</b> Methods</a>
<ul>
<li class="chapter" data-level="1.6.1" data-path="1-developmental.html"><a href="1-developmental.html#mouse-husbandry"><i class="fa fa-check"></i><b>1.6.1</b> Mouse husbandry</a></li>
<li class="chapter" data-level="1.6.2" data-path="1-developmental.html"><a href="1-developmental.html#germ-cells-isolation"><i class="fa fa-check"></i><b>1.6.2</b> Germ cells isolation</a></li>
<li class="chapter" data-level="1.6.3" data-path="1-developmental.html"><a href="1-developmental.html#spermatogonial-cells-enrichment-by-facs"><i class="fa fa-check"></i><b>1.6.3</b> Spermatogonial cells enrichment by FACS</a></li>
<li class="chapter" data-level="1.6.4" data-path="1-developmental.html"><a href="1-developmental.html#immunocytochemistry"><i class="fa fa-check"></i><b>1.6.4</b> Immunocytochemistry</a></li>
<li class="chapter" data-level="1.6.5" data-path="1-developmental.html"><a href="1-developmental.html#rna-extraction-and-rna-seq-library-preparation-for-rna-seq"><i class="fa fa-check"></i><b>1.6.5</b> RNA extraction and RNA-seq library preparation for RNA-seq</a></li>
<li class="chapter" data-level="1.6.6" data-path="1-developmental.html"><a href="1-developmental.html#omni-atac-llibrary-preparation-for-omni-atac-and-sequencing"><i class="fa fa-check"></i><b>1.6.6</b> Omni-ATAC Llibrary preparation for Omni-ATAC and sequencing</a></li>
<li class="chapter" data-level="1.6.7" data-path="1-developmental.html"><a href="1-developmental.html#rna-sequencing-rna-seq"><i class="fa fa-check"></i><b>1.6.7</b> RNA sequencing (RNA-seq)</a></li>
<li class="chapter" data-level="1.6.8" data-path="1-developmental.html"><a href="1-developmental.html#assay-for-transposase-accessible-chromatin-using-sequening-omni-atac"><i class="fa fa-check"></i><b>1.6.8</b> Assay for Transposase-Accessible Chromatin using sequening (Omni-ATAC)</a></li>
<li class="chapter" data-level="1.6.9" data-path="1-developmental.html"><a href="1-developmental.html#chromatin-iimmunnoprecipitation-sequencing-chip-seq"><i class="fa fa-check"></i><b>1.6.9</b> Chromatin iImmunnoprecipitation sequencing (ChIP-seq)</a></li>
<li class="chapter" data-level="1.6.10" data-path="1-developmental.html"><a href="1-developmental.html#bisulfite-sequencing-bs"><i class="fa fa-check"></i><b>1.6.10</b> Bisulfite sequencing (BS)</a></li>
<li class="chapter" data-level="1.6.11" data-path="1-developmental.html"><a href="1-developmental.html#high-throughput-sequencing-data-analysis"><i class="fa fa-check"></i><b>1.6.11</b> High-throughput sequencing data analysis</a></li>
<li class="chapter" data-level="1.6.12" data-path="1-developmental.html"><a href="1-developmental.html#figures"><i class="fa fa-check"></i><b>1.6.12</b> Figures</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="1-developmental.html"><a href="1-developmental.html#data-and-materials-availability"><i class="fa fa-check"></i><b>1.7</b> Data and materials availability</a></li>
<li class="chapter" data-level="1.8" data-path="1-developmental.html"><a href="1-developmental.html#authors-contribution"><i class="fa fa-check"></i><b>1.8</b> Authors Contribution</a></li>
<li class="chapter" data-level="1.9" data-path="1-developmental.html"><a href="1-developmental.html#competing-interest"><i class="fa fa-check"></i><b>1.9</b> Competing interest</a></li>
<li class="chapter" data-level="1.10" data-path="1-developmental.html"><a href="1-developmental.html#acknowledgments"><i class="fa fa-check"></i><b>1.10</b> Acknowledgments</a></li>
<li class="chapter" data-level="1.11" data-path="1-developmental.html"><a href="1-developmental.html#funding"><i class="fa fa-check"></i><b>1.11</b> Funding</a></li>
<li class="chapter" data-level="1.12" data-path="1-developmental.html"><a href="1-developmental.html#supplementary-figures"><i class="fa fa-check"></i><b>1.12</b> Supplementary Figures</a>
<ul>
<li class="chapter" data-level="1.12.1" data-path="1-developmental.html"><a href="1-developmental.html#figure-1"><i class="fa fa-check"></i><b>1.12.1</b> Figure 1</a></li>
<li class="chapter" data-level="1.12.2" data-path="1-developmental.html"><a href="1-developmental.html#figure-2"><i class="fa fa-check"></i><b>1.12.2</b> Figure 2</a></li>
<li class="chapter" data-level="1.12.3" data-path="1-developmental.html"><a href="1-developmental.html#figure-3"><i class="fa fa-check"></i><b>1.12.3</b> Figure 3</a></li>
<li class="chapter" data-level="1.12.4" data-path="1-developmental.html"><a href="1-developmental.html#figure-4"><i class="fa fa-check"></i><b>1.12.4</b> Figure 4</a></li>
<li class="chapter" data-level="1.12.5" data-path="1-developmental.html"><a href="1-developmental.html#figure-5"><i class="fa fa-check"></i><b>1.12.5</b> Figure 5</a></li>
<li class="chapter" data-level="1.12.6" data-path="1-developmental.html"><a href="1-developmental.html#figure-6"><i class="fa fa-check"></i><b>1.12.6</b> Figure 6</a></li>
</ul></li>
<li class="chapter" data-level="1.13" data-path="1-developmental.html"><a href="1-developmental.html#references"><i class="fa fa-check"></i><b>1.13</b> References</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-ev.html"><a href="2-ev.html"><i class="fa fa-check"></i><b>2</b> Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-ev.html"><a href="2-ev.html#abstract-2"><i class="fa fa-check"></i><b>2.1</b> Abstract</a></li>
<li class="chapter" data-level="2.2" data-path="2-ev.html"><a href="2-ev.html#summary-sentence"><i class="fa fa-check"></i><b>2.2</b> Summary sentence</a></li>
<li class="chapter" data-level="2.3" data-path="2-ev.html"><a href="2-ev.html#graphical-abstract-1"><i class="fa fa-check"></i><b>2.3</b> Graphical Abstract</a></li>
<li class="chapter" data-level="2.4" data-path="2-ev.html"><a href="2-ev.html#key-words"><i class="fa fa-check"></i><b>2.4</b> Key words</a></li>
<li class="chapter" data-level="2.5" data-path="2-ev.html"><a href="2-ev.html#introduction-1"><i class="fa fa-check"></i><b>2.5</b> Introduction</a></li>
<li class="chapter" data-level="2.6" data-path="2-ev.html"><a href="2-ev.html#results-1"><i class="fa fa-check"></i><b>2.6</b> Results</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="2-ev.html"><a href="2-ev.html#isolation-of-cauda-epididymosomes-confirmed-by-several-methods"><i class="fa fa-check"></i><b>2.6.1</b> Isolation of cauda epididymosomes confirmed by several methods</a></li>
<li class="chapter" data-level="2.6.2" data-path="2-ev.html"><a href="2-ev.html#the-number-and-size-of-epididymosomes-in-adult-males-are-not-altered-by-postnatal-stress"><i class="fa fa-check"></i><b>2.6.2</b> The number and size of epididymosomes in adult males are not altered by postnatal stress</a></li>
<li class="chapter" data-level="2.6.3" data-path="2-ev.html"><a href="2-ev.html#mirnas-are-persistently-altered-by-postnatal-stress-in-cauda-epididymosomes"><i class="fa fa-check"></i><b>2.6.3</b> miRNAs are persistently altered by postnatal stress in cauda epididymosomes</a></li>
<li class="chapter" data-level="2.6.4" data-path="2-ev.html"><a href="2-ev.html#mrna-targets-of-mirnas-from-cauda-epididymosomes-are-altered-by-postnatal-stress-in-sperm-and-in-zygotes"><i class="fa fa-check"></i><b>2.6.4</b> mRNA targets of miRNAs from cauda epididymosomes are altered by postnatal stress in sperm and in zygotes</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="2-ev.html"><a href="2-ev.html#discussion-1"><i class="fa fa-check"></i><b>2.7</b> Discussion</a></li>
<li class="chapter" data-level="2.8" data-path="2-ev.html"><a href="2-ev.html#materials-and-methods"><i class="fa fa-check"></i><b>2.8</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="2.8.1" data-path="2-ev.html"><a href="2-ev.html#animals"><i class="fa fa-check"></i><b>2.8.1</b> Animals</a></li>
<li class="chapter" data-level="2.8.2" data-path="2-ev.html"><a href="2-ev.html#msus"><i class="fa fa-check"></i><b>2.8.2</b> MSUS</a></li>
<li class="chapter" data-level="2.8.3" data-path="2-ev.html"><a href="2-ev.html#tissue-collection"><i class="fa fa-check"></i><b>2.8.3</b> Tissue collection</a></li>
<li class="chapter" data-level="2.8.4" data-path="2-ev.html"><a href="2-ev.html#electron-microscopy-images"><i class="fa fa-check"></i><b>2.8.4</b> Electron microscopy images</a></li>
<li class="chapter" data-level="2.8.5" data-path="2-ev.html"><a href="2-ev.html#epididymosomes-isolation-by-ultracentrifugation"><i class="fa fa-check"></i><b>2.8.5</b> Epididymosomes isolation by ultracentrifugation</a></li>
<li class="chapter" data-level="2.8.6" data-path="2-ev.html"><a href="2-ev.html#immunoblotting"><i class="fa fa-check"></i><b>2.8.6</b> Immunoblotting</a></li>
<li class="chapter" data-level="2.8.7" data-path="2-ev.html"><a href="2-ev.html#nanoparticle-tracking-analysis"><i class="fa fa-check"></i><b>2.8.7</b> Nanoparticle tracking analysis</a></li>
<li class="chapter" data-level="2.8.8" data-path="2-ev.html"><a href="2-ev.html#rna-isolation-and-epididymosomes-profiling"><i class="fa fa-check"></i><b>2.8.8</b> RNA isolation and epididymosomes profiling</a></li>
<li class="chapter" data-level="2.8.9" data-path="2-ev.html"><a href="2-ev.html#preparation-and-sequencing-of-srna-seq-libraries-from-epididymosomes"><i class="fa fa-check"></i><b>2.8.9</b> Preparation and sequencing of sRNA-seq libraries from epididymosomes</a></li>
<li class="chapter" data-level="2.8.10" data-path="2-ev.html"><a href="2-ev.html#rt-qpcr"><i class="fa fa-check"></i><b>2.8.10</b> RT-qPCR</a></li>
<li class="chapter" data-level="2.8.11" data-path="2-ev.html"><a href="2-ev.html#cholesterol-measurements"><i class="fa fa-check"></i><b>2.8.11</b> Cholesterol measurements</a></li>
<li class="chapter" data-level="2.8.12" data-path="2-ev.html"><a href="2-ev.html#bioinformatics-data-analysis"><i class="fa fa-check"></i><b>2.8.12</b> Bioinformatics data analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.9" data-path="2-ev.html"><a href="2-ev.html#data-availability"><i class="fa fa-check"></i><b>2.9</b> Data availability</a></li>
<li class="chapter" data-level="2.10" data-path="2-ev.html"><a href="2-ev.html#authors-contributions"><i class="fa fa-check"></i><b>2.10</b> Authors’ contributions</a></li>
<li class="chapter" data-level="2.11" data-path="2-ev.html"><a href="2-ev.html#grant-support"><i class="fa fa-check"></i><b>2.11</b> Grant Support</a></li>
<li class="chapter" data-level="2.12" data-path="2-ev.html"><a href="2-ev.html#acknowledgements"><i class="fa fa-check"></i><b>2.12</b> Acknowledgements</a></li>
<li class="chapter" data-level="2.13" data-path="2-ev.html"><a href="2-ev.html#supplementary-figures-1"><i class="fa fa-check"></i><b>2.13</b> Supplementary Figures</a>
<ul>
<li class="chapter" data-level="2.13.1" data-path="2-ev.html"><a href="2-ev.html#figure-1-1"><i class="fa fa-check"></i><b>2.13.1</b> Figure 1</a></li>
<li class="chapter" data-level="2.13.2" data-path="2-ev.html"><a href="2-ev.html#figure-2-1"><i class="fa fa-check"></i><b>2.13.2</b> Figure 2</a></li>
<li class="chapter" data-level="2.13.3" data-path="2-ev.html"><a href="2-ev.html#figure-3-1"><i class="fa fa-check"></i><b>2.13.3</b> Figure 3</a></li>
<li class="chapter" data-level="2.13.4" data-path="2-ev.html"><a href="2-ev.html#figure-4-1"><i class="fa fa-check"></i><b>2.13.4</b> Figure 4</a></li>
<li class="chapter" data-level="2.13.5" data-path="2-ev.html"><a href="2-ev.html#figure-5-1"><i class="fa fa-check"></i><b>2.13.5</b> Figure 5</a></li>
<li class="chapter" data-level="2.13.6" data-path="2-ev.html"><a href="2-ev.html#figure-6-1"><i class="fa fa-check"></i><b>2.13.6</b> Figure 6</a></li>
</ul></li>
<li class="chapter" data-level="2.14" data-path="2-ev.html"><a href="2-ev.html#supplementary-tables"><i class="fa fa-check"></i><b>2.14</b> Supplementary Tables</a>
<ul>
<li class="chapter" data-level="2.14.1" data-path="2-ev.html"><a href="2-ev.html#table-1"><i class="fa fa-check"></i><b>2.14.1</b> Table 1</a></li>
<li class="chapter" data-level="2.14.2" data-path="2-ev.html"><a href="2-ev.html#table-2"><i class="fa fa-check"></i><b>2.14.2</b> Table 2</a></li>
<li class="chapter" data-level="2.14.3" data-path="2-ev.html"><a href="2-ev.html#table-3"><i class="fa fa-check"></i><b>2.14.3</b> Table 3</a></li>
<li class="chapter" data-level="2.14.4" data-path="2-ev.html"><a href="2-ev.html#table-4"><i class="fa fa-check"></i><b>2.14.4</b> Table 4</a></li>
</ul></li>
<li class="chapter" data-level="2.15" data-path="2-ev.html"><a href="2-ev.html#references-1"><i class="fa fa-check"></i><b>2.15</b> References</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-shortRNA.html"><a href="3-shortRNA.html"><i class="fa fa-check"></i><b>3</b> shortRNA</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-shortRNA.html"><a href="3-shortRNA.html#abstract-3"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="3-shortRNA.html"><a href="3-shortRNA.html#introduction-2"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="3-shortRNA.html"><a href="3-shortRNA.html#methods-2"><i class="fa fa-check"></i><b>3.3</b> Methods</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-shortRNA.html"><a href="3-shortRNA.html#pipeline"><i class="fa fa-check"></i><b>3.3.1</b> Pipeline</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-shortRNA.html"><a href="3-shortRNA.html#qc"><i class="fa fa-check"></i><b>3.3.2</b> QC</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-shortRNA.html"><a href="3-shortRNA.html#annotation-preparation"><i class="fa fa-check"></i><b>3.3.3</b> Annotation preparation</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-shortRNA.html"><a href="3-shortRNA.html#alignment"><i class="fa fa-check"></i><b>3.3.4</b> Alignment</a></li>
<li class="chapter" data-level="3.3.5" data-path="3-shortRNA.html"><a href="3-shortRNA.html#reads-assignment"><i class="fa fa-check"></i><b>3.3.5</b> Reads assignment</a></li>
<li class="chapter" data-level="3.3.6" data-path="3-shortRNA.html"><a href="3-shortRNA.html#assignment-rules"><i class="fa fa-check"></i><b>3.3.6</b> Assignment rules</a></li>
<li class="chapter" data-level="3.3.7" data-path="3-shortRNA.html"><a href="3-shortRNA.html#treesummarizedexperiment-object"><i class="fa fa-check"></i><b>3.3.7</b> TreeSummarizedExperiment object</a></li>
<li class="chapter" data-level="3.3.8" data-path="3-shortRNA.html"><a href="3-shortRNA.html#differential-analysis"><i class="fa fa-check"></i><b>3.3.8</b> Differential analysis</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-shortRNA.html"><a href="3-shortRNA.html#results-2"><i class="fa fa-check"></i><b>3.4</b> Results</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-shortRNA.html"><a href="3-shortRNA.html#datasets-used-for-testing-the-pipeline"><i class="fa fa-check"></i><b>3.4.1</b> Datasets used for testing the pipeline</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-shortRNA.html"><a href="3-shortRNA.html#databases-included-for-analyzing-these-data"><i class="fa fa-check"></i><b>3.4.2</b> Databases included for analyzing these data</a></li>
<li class="chapter" data-level="3.4.3" data-path="3-shortRNA.html"><a href="3-shortRNA.html#result-1"><i class="fa fa-check"></i><b>3.4.3</b> result 1</a></li>
<li class="chapter" data-level="3.4.4" data-path="3-shortRNA.html"><a href="3-shortRNA.html#result-2"><i class="fa fa-check"></i><b>3.4.4</b> result 2</a></li>
<li class="chapter" data-level="3.4.5" data-path="3-shortRNA.html"><a href="3-shortRNA.html#result-3"><i class="fa fa-check"></i><b>3.4.5</b> result 3</a></li>
<li class="chapter" data-level="3.4.6" data-path="3-shortRNA.html"><a href="3-shortRNA.html#comparison-with-other-tools"><i class="fa fa-check"></i><b>3.4.6</b> Comparison with other tools</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-shortRNA.html"><a href="3-shortRNA.html#discussion-outlook"><i class="fa fa-check"></i><b>3.5</b> Discussion &amp; Outlook</a></li>
<li class="chapter" data-level="3.6" data-path="3-shortRNA.html"><a href="3-shortRNA.html#data-and-code-availibility"><i class="fa fa-check"></i><b>3.6</b> Data and code availibility</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="1-developmental.html"><a href="1-developmental.html#discussion"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="chapter" data-level="" data-path="aa.html"><a href="aa.html"><i class="fa fa-check"></i>Appendix A</a>
<ul>
<li class="chapter" data-level="3.7" data-path="aa.html"><a href="aa.html#datasets-analyzed"><i class="fa fa-check"></i><b>3.7</b> Datasets analyzed</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="ab.html"><a href="ab.html"><i class="fa fa-check"></i>Appendix B</a>
<ul>
<li class="chapter" data-level="3.8" data-path="ab.html"><a href="ab.html#other-manuscripts-during-phd"><i class="fa fa-check"></i><b>3.8</b> Other manuscripts during PhD</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="ac.html"><a href="ac.html"><i class="fa fa-check"></i>Appendix C</a></li>
<li class="chapter" data-level="" data-path="references-2.html"><a href="references-2.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Understanding the molecular mechanisms of germline-dependent epigenetic inheritance: Computational analysis of multi-omics data</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ev" class="section level1" number="2">
<h1><span class="header-section-number">Chapter 2</span> Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse</h1>
<p>Anar Alshanbayeva, <strong>Deepak K Tanwar</strong>, Martin Roszkowski, Francesca Manuella, Isabelle M Mansuy


Laboratory of Neuroepigenetics, Brain Research Institute at the Medical Faculty of the University of Zurich.

Institute for Neuroscience of the Department of Health Sciences and Technology, ETH Zurich, Zurich, Switzerland.

Zurich Neuroscience Center, ETH and University of Zurich, Zurich, Switzerland.


Corresponding author


<strong>Journal:</strong> <em>Biology of Reproduction</em>

DOI: <a href="https://doi.org/10.1093/biolre/ioab156">10.1093/biolre/ioab156</a>


<strong>Contributions:</strong> <em>I performed data analysis with Anar Alshanbayeva, generated figures with Anar Alshanbayeva, helped Anar Alshanbayeva in writing the manuscript, and revised manuscript with Anar Alshanbayeva.</em></p>
<div style="page-break-after: always;"></div>
<div id="abstract-2" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Abstract</h2>
<p>Sperm RNA can be modified by environmental factors and has been implicated in communicating signals about changes in a father’s environment to the offspring. The small RNA composition of sperm could be changed during its final stage of maturation in the epididymis by extracellular vesicles (EVs) released by epididymal cells. We studied the effect of exposure to stress in early postnatal life on the transcriptome of epididymal EVs using a mouse model of transgenerational transmission. We found that the small RNA signature of epididymal EVs, particularly miRNAs, is altered in adult males exposed to postnatal stress. In some cases, these miRNA changes correlate with differences in the expression of their target genes in sperm and zygotes generated from that sperm. These results suggest that stressful experiences in early life can have persistent biological effects on the male reproductive tract that may in part be responsible for the transmission of the effects of exposure to the offspring.</p>
</div>
<div id="summary-sentence" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Summary sentence</h2>
<p>miRNA cargo of extracellular vesicles in cauda epididymis is altered by paternal exposure to early life stress. This correlates with changes in the expression of target genes in sperm and in zygotes generated from that sperm.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="graphical-abstract-1" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Graphical Abstract</h2>
</div>
<div id="key-words" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Key words</h2>
<p>epigenetics, epididymis, epididymosomes, early life stress, extracellular vesicles, miRNAs, sperm.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="introduction-1" class="section level2" number="2.5">
<h2><span class="header-section-number">2.5</span> Introduction</h2>
<div style="page-break-after: always;"></div>
</div>
<div id="results-1" class="section level2" number="2.6">
<h2><span class="header-section-number">2.6</span> Results</h2>
<div id="isolation-of-cauda-epididymosomes-confirmed-by-several-methods" class="section level3" number="2.6.1">
<h3><span class="header-section-number">2.6.1</span> Isolation of cauda epididymosomes confirmed by several methods</h3>
</div>
<div id="the-number-and-size-of-epididymosomes-in-adult-males-are-not-altered-by-postnatal-stress" class="section level3" number="2.6.2">
<h3><span class="header-section-number">2.6.2</span> The number and size of epididymosomes in adult males are not altered by postnatal stress</h3>
</div>
<div id="mirnas-are-persistently-altered-by-postnatal-stress-in-cauda-epididymosomes" class="section level3" number="2.6.3">
<h3><span class="header-section-number">2.6.3</span> miRNAs are persistently altered by postnatal stress in cauda epididymosomes</h3>
</div>
<div id="mrna-targets-of-mirnas-from-cauda-epididymosomes-are-altered-by-postnatal-stress-in-sperm-and-in-zygotes" class="section level3" number="2.6.4">
<h3><span class="header-section-number">2.6.4</span> mRNA targets of miRNAs from cauda epididymosomes are altered by postnatal stress in sperm and in zygotes</h3>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="discussion-1" class="section level2" number="2.7">
<h2><span class="header-section-number">2.7</span> Discussion</h2>
<div style="page-break-after: always;"></div>
</div>
<div id="materials-and-methods" class="section level2" number="2.8">
<h2><span class="header-section-number">2.8</span> Materials and methods</h2>
<div id="animals" class="section level3" number="2.8.1">
<h3><span class="header-section-number">2.8.1</span> Animals</h3>
</div>
<div id="msus" class="section level3" number="2.8.2">
<h3><span class="header-section-number">2.8.2</span> MSUS</h3>
</div>
<div id="tissue-collection" class="section level3" number="2.8.3">
<h3><span class="header-section-number">2.8.3</span> Tissue collection</h3>
</div>
<div id="electron-microscopy-images" class="section level3" number="2.8.4">
<h3><span class="header-section-number">2.8.4</span> Electron microscopy images</h3>
</div>
<div id="epididymosomes-isolation-by-ultracentrifugation" class="section level3" number="2.8.5">
<h3><span class="header-section-number">2.8.5</span> Epididymosomes isolation by ultracentrifugation</h3>
</div>
<div id="immunoblotting" class="section level3" number="2.8.6">
<h3><span class="header-section-number">2.8.6</span> Immunoblotting</h3>
</div>
<div id="nanoparticle-tracking-analysis" class="section level3" number="2.8.7">
<h3><span class="header-section-number">2.8.7</span> Nanoparticle tracking analysis</h3>
</div>
<div id="rna-isolation-and-epididymosomes-profiling" class="section level3" number="2.8.8">
<h3><span class="header-section-number">2.8.8</span> RNA isolation and epididymosomes profiling</h3>
</div>
<div id="preparation-and-sequencing-of-srna-seq-libraries-from-epididymosomes" class="section level3" number="2.8.9">
<h3><span class="header-section-number">2.8.9</span> Preparation and sequencing of sRNA-seq libraries from epididymosomes</h3>
</div>
<div id="rt-qpcr" class="section level3" number="2.8.10">
<h3><span class="header-section-number">2.8.10</span> RT-qPCR</h3>
</div>
<div id="cholesterol-measurements" class="section level3" number="2.8.11">
<h3><span class="header-section-number">2.8.11</span> Cholesterol measurements</h3>
</div>
<div id="bioinformatics-data-analysis" class="section level3" number="2.8.12">
<h3><span class="header-section-number">2.8.12</span> Bioinformatics data analysis</h3>
</div>
</div>
<div id="data-availability" class="section level2" number="2.9">
<h2><span class="header-section-number">2.9</span> Data availability</h2>
<p>The datasets collected for this study are available as follows:
- sRNA-seq dataset of cauda epididymosomes before and after sizeselection: NCBI GEO under accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175976">GSE175976</a>.
- Codes for bioinformatics analysis of RNA-sequencing datasets and
all corresponding differential expression analyses: Github repository <a href="https://github.com/mansuylab/alshanbayeva_et_al_2021%5Bhttps://github.com/mansuylab/alshanbayeva_et_al_2021" class="uri">https://github.com/mansuylab/alshanbayeva_et_al_2021[https://github.com/mansuylab/alshanbayeva_et_al_2021</a>].
- Sperm and zygote sequencing datasets from previous publications
can be found in ArrayExpress database at EMBL-EBI (<a href="www.ebi.ac.uk/arrayexpress">www.ebi.ac.uk/arrayexpress</a>) with the accession number <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5834/">E-MTAB-5834</a> (sperm) and <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6589/">E-MTAB-6589</a> (zygotes).</p>
<div style="page-break-after: always;"></div>
</div>
<div id="authors-contributions" class="section level2" number="2.10">
<h2><span class="header-section-number">2.10</span> Authors’ contributions</h2>
<p>AA and IMM conceived and designed the study. FM and MR performed the MSUS breeding and collected tissue samples. AA and DKT performed data analysis and generated figures. AA wrote the manuscript with input from DKT and IMM. AA performed all
experiments for RNA sequencing and all molecular analyses. IMM supervised the project and raised funds.</p>
</div>
<div id="grant-support" class="section level2" number="2.11">
<h2><span class="header-section-number">2.11</span> Grant Support</h2>
<p>The work was supported by Swiss National Science Foundation (31003A-135715), ETH grants (ETH-10 15-2 and ETH-17 13-2), the Slack-Gyr Foundation, the Escher Foundation. The Mansuy lab is funded by the University Zürich, the Swiss Federal Institute of Technology, the Swiss National Science Foundation (31003A-135715), ETH grants (ETH-10 15-2 and ETH-17 13-2), the Slack-Gyr foundation, the Escher Foundation. Deepak K. Tanwar is supported by the Swiss Government Excellence Scholarship. Martin Roszkowski was funded by the ETH Zurich Fellowship (ETH-10 15-2).</p>
</div>
<div id="acknowledgements" class="section level2" number="2.12">
<h2><span class="header-section-number">2.12</span> Acknowledgements</h2>
<p>We thank Pierre-Luc Germain for advice on data analysis and for generating cumulative distribution plots, Irina Lazar-Contes for help with MSUS breeding, Silvia Schelbert for work on the animal license, Emilio Yandez at Function Genomics Center Zurich (FGCZ) for advice on the sRNA sequencing, Alekhya Mazumkhar for help with nanoparticle-tracking analysis, Yvonne Zipfel for animal care, Zurich Integrative Rodent Physiology facility for performing cholesterol measurements. We also thank Eloise Kremer, Ali Jawaid, and Mea Holmes for their initial contributions to the project.</p>
<p><strong>Conflict of interest:</strong> The authors declare no conflict of interest.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="supplementary-figures-1" class="section level2" number="2.13">
<h2><span class="header-section-number">2.13</span> Supplementary Figures</h2>
<div id="figure-1-1" class="section level3" number="2.13.1">
<h3><span class="header-section-number">2.13.1</span> Figure 1</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="figure-2-1" class="section level3" number="2.13.2">
<h3><span class="header-section-number">2.13.2</span> Figure 2</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="figure-3-1" class="section level3" number="2.13.3">
<h3><span class="header-section-number">2.13.3</span> Figure 3</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="figure-4-1" class="section level3" number="2.13.4">
<h3><span class="header-section-number">2.13.4</span> Figure 4</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="figure-5-1" class="section level3" number="2.13.5">
<h3><span class="header-section-number">2.13.5</span> Figure 5</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="figure-6-1" class="section level3" number="2.13.6">
<h3><span class="header-section-number">2.13.6</span> Figure 6</h3>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="supplementary-tables" class="section level2" number="2.14">
<h2><span class="header-section-number">2.14</span> Supplementary Tables</h2>
<div id="table-1" class="section level3" number="2.14.1">
<h3><span class="header-section-number">2.14.1</span> Table 1</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="table-2" class="section level3" number="2.14.2">
<h3><span class="header-section-number">2.14.2</span> Table 2</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="table-3" class="section level3" number="2.14.3">
<h3><span class="header-section-number">2.14.3</span> Table 3</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="table-4" class="section level3" number="2.14.4">
<h3><span class="header-section-number">2.14.4</span> Table 4</h3>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="references-1" class="section level2" number="2.15">
<h2><span class="header-section-number">2.15</span> References</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="1-developmental.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="3-shortRNA.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
