<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse | Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance</title>
  <meta name="description" content="Chapter 2 Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse | Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse | Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse | Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance" />
  
  
  

<meta name="author" content="Deepak Kumar Tanwar" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="developmental.html"/>
<link rel="next" href="shortRNA.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="summary-de.html"><a href="summary-de.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="0.1" data-path="intro.html"><a href="intro.html#epigenetics"><i class="fa fa-check"></i><b>0.1</b> Epigenetics</a></li>
<li class="chapter" data-level="0.2" data-path="intro.html"><a href="intro.html#germline-epigenetics"><i class="fa fa-check"></i><b>0.2</b> Germline epigenetics</a>
<ul>
<li class="chapter" data-level="0.2.1" data-path="intro.html"><a href="intro.html#spermatogonial-stem-cells"><i class="fa fa-check"></i><b>0.2.1</b> Spermatogonial Stem Cells</a></li>
<li class="chapter" data-level="0.2.2" data-path="intro.html"><a href="intro.html#reprogramming"><i class="fa fa-check"></i><b>0.2.2</b> Reprogramming</a></li>
</ul></li>
<li class="chapter" data-level="0.3" data-path="intro.html"><a href="intro.html#epigenetic-inheritance"><i class="fa fa-check"></i><b>0.3</b> Epigenetic Inheritance</a>
<ul>
<li class="chapter" data-level="0.3.1" data-path="intro.html"><a href="intro.html#transgenerational-epigenetic-inheritance"><i class="fa fa-check"></i><b>0.3.1</b> Transgenerational Epigenetic Inheritance</a></li>
</ul></li>
<li class="chapter" data-level="0.4" data-path="intro.html"><a href="intro.html#models-of-epigenetic-inheritance"><i class="fa fa-check"></i><b>0.4</b> Models of Epigenetic Inheritance</a>
<ul>
<li class="chapter" data-level="0.4.1" data-path="intro.html"><a href="intro.html#msus-mouse-model"><i class="fa fa-check"></i><b>0.4.1</b> MSUS mouse model</a></li>
</ul></li>
<li class="chapter" data-level="0.5" data-path="intro.html"><a href="intro.html#vectors-of-tei"><i class="fa fa-check"></i><b>0.5</b> Vectors of TEI</a>
<ul>
<li class="chapter" data-level="0.5.1" data-path="intro.html"><a href="intro.html#extracellular-vesicles"><i class="fa fa-check"></i><b>0.5.1</b> Extracellular vesicles</a></li>
<li class="chapter" data-level="0.5.2" data-path="intro.html"><a href="intro.html#small-rnas"><i class="fa fa-check"></i><b>0.5.2</b> Small RNAs</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i>Bioinformatics methods</a>
<ul>
<li class="chapter" data-level="0.6" data-path="methods.html"><a href="methods.html#version-controlled-data-analysis-using-git"><i class="fa fa-check"></i><b>0.6</b> Version controlled data analysis using git</a></li>
<li class="chapter" data-level="0.7" data-path="methods.html"><a href="methods.html#data-analysis-directory-organization"><i class="fa fa-check"></i><b>0.7</b> Data analysis directory organization</a></li>
<li class="chapter" data-level="0.8" data-path="methods.html"><a href="methods.html#pipelines-for-data-analysis"><i class="fa fa-check"></i><b>0.8</b> Pipelines for data analysis</a>
<ul>
<li class="chapter" data-level="0.8.1" data-path="methods.html"><a href="methods.html#bulk-rna-seq"><i class="fa fa-check"></i><b>0.8.1</b> Bulk RNA-seq</a></li>
<li class="chapter" data-level="0.8.2" data-path="methods.html"><a href="methods.html#atac-seq"><i class="fa fa-check"></i><b>0.8.2</b> ATAC-seq</a></li>
<li class="chapter" data-level="0.8.3" data-path="methods.html"><a href="methods.html#chip-seq"><i class="fa fa-check"></i><b>0.8.3</b> ChIP-seq</a></li>
<li class="chapter" data-level="0.8.4" data-path="methods.html"><a href="methods.html#wgbs-rrbs"><i class="fa fa-check"></i><b>0.8.4</b> WGBS &amp; RRBS</a></li>
<li class="chapter" data-level="0.8.5" data-path="methods.html"><a href="methods.html#small-rna-seq"><i class="fa fa-check"></i><b>0.8.5</b> small RNA-seq</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="aims.html"><a href="aims.html"><i class="fa fa-check"></i>Aims and thesis overview</a>
<ul>
<li class="chapter" data-level="0.9" data-path="aims.html"><a href="aims.html#aims-1"><i class="fa fa-check"></i><b>0.9</b> Aims</a></li>
<li class="chapter" data-level="0.10" data-path="aims.html"><a href="aims.html#thesis-overview"><i class="fa fa-check"></i><b>0.10</b> Thesis overview</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="developmental.html"><a href="developmental.html"><i class="fa fa-check"></i><b>1</b> Dynamic chromatin accessibility in spermatogonial cells for transcriptional programmings from early postnatal to adult stages</a>
<ul>
<li class="chapter" data-level="1.1" data-path="developmental.html"><a href="developmental.html#graphical-abstract"><i class="fa fa-check"></i><b>1.1</b> Graphical Abstract</a></li>
<li class="chapter" data-level="1.2" data-path="developmental.html"><a href="developmental.html#abstract"><i class="fa fa-check"></i><b>1.2</b> Abstract</a></li>
<li class="chapter" data-level="1.3" data-path="developmental.html"><a href="developmental.html#introduction"><i class="fa fa-check"></i><b>1.3</b> Introduction</a></li>
<li class="chapter" data-level="1.4" data-path="developmental.html"><a href="developmental.html#results"><i class="fa fa-check"></i><b>1.4</b> Results</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="developmental.html"><a href="developmental.html#enrichment-of-spermatogonial-cells-from-postnatal-and-adult-mouse-testis"><i class="fa fa-check"></i><b>1.4.1</b> Enrichment of spermatogonial cells from postnatal and adult mouse testis</a></li>
<li class="chapter" data-level="1.4.2" data-path="developmental.html"><a href="developmental.html#chromatin-is-remodeled-in-spermatogonial-cells-during-development"><i class="fa fa-check"></i><b>1.4.2</b> Chromatin is remodeled in spermatogonial cells during development</a></li>
<li class="chapter" data-level="1.4.3" data-path="developmental.html"><a href="developmental.html#differentially-accessible-chromatin-regions-associate-with-distinct-gene-expression-dynamics"><i class="fa fa-check"></i><b>1.4.3</b> Differentially accessible chromatin regions associate with distinct gene expression dynamics</a></li>
<li class="chapter" data-level="1.4.4" data-path="developmental.html"><a href="developmental.html#differentially-accessible-chromatin-regions-associate-with-distinct-epigenetic-profiles"><i class="fa fa-check"></i><b>1.4.4</b> Differentially accessible chromatin regions associate with distinct epigenetic profiles</a></li>
<li class="chapter" data-level="1.4.5" data-path="developmental.html"><a href="developmental.html#accessibility-changes-at-open-chromatin-regions-are-markedcarry-by-binding-sites-for-distinct-families-of-transcription-factors"><i class="fa fa-check"></i><b>1.4.5</b> Accessibility changes at open chromatin regions are markedcarry by binding sites for distinct families of transcription factors</a></li>
<li class="chapter" data-level="1.4.6" data-path="developmental.html"><a href="developmental.html#chromatin-accessibility-at-transposable-elements-undergoes-significant-remodeling-in-the-transition-from-postnatal-to-adult-spermatogonia"><i class="fa fa-check"></i><b>1.4.6</b> Chromatin accessibility at transposable elements undergoes significant remodeling in the transition from postnatal to adult spermatogonia</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="developmental.html"><a href="developmental.html#discussion"><i class="fa fa-check"></i><b>1.5</b> Discussion</a></li>
<li class="chapter" data-level="1.6" data-path="developmental.html"><a href="developmental.html#methods-1"><i class="fa fa-check"></i><b>1.6</b> Methods</a>
<ul>
<li class="chapter" data-level="1.6.1" data-path="developmental.html"><a href="developmental.html#mouse-husbandry"><i class="fa fa-check"></i><b>1.6.1</b> Mouse husbandry</a></li>
<li class="chapter" data-level="1.6.2" data-path="developmental.html"><a href="developmental.html#germ-cells-isolation"><i class="fa fa-check"></i><b>1.6.2</b> Germ cells isolation</a></li>
<li class="chapter" data-level="1.6.3" data-path="developmental.html"><a href="developmental.html#spermatogonial-cells-enrichment-by-facs"><i class="fa fa-check"></i><b>1.6.3</b> Spermatogonial cells enrichment by FACS</a></li>
<li class="chapter" data-level="1.6.4" data-path="developmental.html"><a href="developmental.html#immunocytochemistry"><i class="fa fa-check"></i><b>1.6.4</b> Immunocytochemistry</a></li>
<li class="chapter" data-level="1.6.5" data-path="developmental.html"><a href="developmental.html#rna-extraction-and-rna-seq-library-preparation-for-rna-seq"><i class="fa fa-check"></i><b>1.6.5</b> RNA extraction and RNA-seq library preparation for RNA-seq</a></li>
<li class="chapter" data-level="1.6.6" data-path="developmental.html"><a href="developmental.html#omni-atac-llibrary-preparation-for-omni-atac-and-sequencing"><i class="fa fa-check"></i><b>1.6.6</b> Omni-ATAC Llibrary preparation for Omni-ATAC and sequencing</a></li>
<li class="chapter" data-level="1.6.7" data-path="developmental.html"><a href="developmental.html#rna-sequencing-rna-seq"><i class="fa fa-check"></i><b>1.6.7</b> RNA sequencing (RNA-seq)</a></li>
<li class="chapter" data-level="1.6.8" data-path="developmental.html"><a href="developmental.html#assay-for-transposase-accessible-chromatin-using-sequening-omni-atac"><i class="fa fa-check"></i><b>1.6.8</b> Assay for Transposase-Accessible Chromatin using sequening (Omni-ATAC)</a></li>
<li class="chapter" data-level="1.6.9" data-path="developmental.html"><a href="developmental.html#chromatin-iimmunnoprecipitation-sequencing-chip-seq"><i class="fa fa-check"></i><b>1.6.9</b> Chromatin iImmunnoprecipitation sequencing (ChIP-seq)</a></li>
<li class="chapter" data-level="1.6.10" data-path="developmental.html"><a href="developmental.html#bisulfite-sequencing-bs"><i class="fa fa-check"></i><b>1.6.10</b> Bisulfite sequencing (BS)</a></li>
<li class="chapter" data-level="1.6.11" data-path="developmental.html"><a href="developmental.html#high-throughput-sequencing-data-analysis"><i class="fa fa-check"></i><b>1.6.11</b> High-throughput sequencing data analysis</a></li>
<li class="chapter" data-level="1.6.12" data-path="developmental.html"><a href="developmental.html#figures"><i class="fa fa-check"></i><b>1.6.12</b> Figures</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="developmental.html"><a href="developmental.html#data-and-materials-availability"><i class="fa fa-check"></i><b>1.7</b> Data and materials availability</a></li>
<li class="chapter" data-level="1.8" data-path="developmental.html"><a href="developmental.html#authors-contribution"><i class="fa fa-check"></i><b>1.8</b> Authors Contribution</a></li>
<li class="chapter" data-level="1.9" data-path="developmental.html"><a href="developmental.html#competing-interest"><i class="fa fa-check"></i><b>1.9</b> Competing interest</a></li>
<li class="chapter" data-level="1.10" data-path="developmental.html"><a href="developmental.html#acknowledgments"><i class="fa fa-check"></i><b>1.10</b> Acknowledgments</a></li>
<li class="chapter" data-level="1.11" data-path="developmental.html"><a href="developmental.html#funding"><i class="fa fa-check"></i><b>1.11</b> Funding</a></li>
<li class="chapter" data-level="1.12" data-path="developmental.html"><a href="developmental.html#supplementary-figures"><i class="fa fa-check"></i><b>1.12</b> Supplementary Figures</a>
<ul>
<li class="chapter" data-level="1.12.1" data-path="developmental.html"><a href="developmental.html#figure-1"><i class="fa fa-check"></i><b>1.12.1</b> Figure 1</a></li>
<li class="chapter" data-level="1.12.2" data-path="developmental.html"><a href="developmental.html#figure-2"><i class="fa fa-check"></i><b>1.12.2</b> Figure 2</a></li>
<li class="chapter" data-level="1.12.3" data-path="developmental.html"><a href="developmental.html#figure-3"><i class="fa fa-check"></i><b>1.12.3</b> Figure 3</a></li>
<li class="chapter" data-level="1.12.4" data-path="developmental.html"><a href="developmental.html#figure-4"><i class="fa fa-check"></i><b>1.12.4</b> Figure 4</a></li>
<li class="chapter" data-level="1.12.5" data-path="developmental.html"><a href="developmental.html#figure-5"><i class="fa fa-check"></i><b>1.12.5</b> Figure 5</a></li>
<li class="chapter" data-level="1.12.6" data-path="developmental.html"><a href="developmental.html#figure-6"><i class="fa fa-check"></i><b>1.12.6</b> Figure 6</a></li>
</ul></li>
<li class="chapter" data-level="1.13" data-path="developmental.html"><a href="developmental.html#references"><i class="fa fa-check"></i><b>1.13</b> References</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ev.html"><a href="ev.html"><i class="fa fa-check"></i><b>2</b> Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse</a>
<ul>
<li class="chapter" data-level="2.1" data-path="ev.html"><a href="ev.html#abstract-1"><i class="fa fa-check"></i><b>2.1</b> Abstract</a></li>
<li class="chapter" data-level="2.2" data-path="ev.html"><a href="ev.html#summary-sentence"><i class="fa fa-check"></i><b>2.2</b> Summary sentence</a></li>
<li class="chapter" data-level="2.3" data-path="ev.html"><a href="ev.html#graphical-abstract-1"><i class="fa fa-check"></i><b>2.3</b> Graphical Abstract</a></li>
<li class="chapter" data-level="2.4" data-path="ev.html"><a href="ev.html#key-words"><i class="fa fa-check"></i><b>2.4</b> Key words</a></li>
<li class="chapter" data-level="2.5" data-path="ev.html"><a href="ev.html#introduction-1"><i class="fa fa-check"></i><b>2.5</b> Introduction</a></li>
<li class="chapter" data-level="2.6" data-path="ev.html"><a href="ev.html#results-1"><i class="fa fa-check"></i><b>2.6</b> Results</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="ev.html"><a href="ev.html#isolation-of-cauda-epididymosomes-confirmed-by-several-methods"><i class="fa fa-check"></i><b>2.6.1</b> Isolation of cauda epididymosomes confirmed by several methods</a></li>
<li class="chapter" data-level="2.6.2" data-path="ev.html"><a href="ev.html#the-number-and-size-of-epididymosomes-in-adult-males-are-not-altered-by-postnatal-stress"><i class="fa fa-check"></i><b>2.6.2</b> The number and size of epididymosomes in adult males are not altered by postnatal stress</a></li>
<li class="chapter" data-level="2.6.3" data-path="ev.html"><a href="ev.html#mirnas-are-persistently-altered-by-postnatal-stress-in-cauda-epididymosomes"><i class="fa fa-check"></i><b>2.6.3</b> miRNAs are persistently altered by postnatal stress in cauda epididymosomes</a></li>
<li class="chapter" data-level="2.6.4" data-path="ev.html"><a href="ev.html#mrna-targets-of-mirnas-from-cauda-epididymosomes-are-altered-by-postnatal-stress-in-sperm-and-in-zygotes"><i class="fa fa-check"></i><b>2.6.4</b> mRNA targets of miRNAs from cauda epididymosomes are altered by postnatal stress in sperm and in zygotes</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="ev.html"><a href="ev.html#discussion-1"><i class="fa fa-check"></i><b>2.7</b> Discussion</a></li>
<li class="chapter" data-level="2.8" data-path="ev.html"><a href="ev.html#materials-and-methods"><i class="fa fa-check"></i><b>2.8</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="2.8.1" data-path="ev.html"><a href="ev.html#animals"><i class="fa fa-check"></i><b>2.8.1</b> Animals</a></li>
<li class="chapter" data-level="2.8.2" data-path="ev.html"><a href="ev.html#msus"><i class="fa fa-check"></i><b>2.8.2</b> MSUS</a></li>
<li class="chapter" data-level="2.8.3" data-path="ev.html"><a href="ev.html#tissue-collection"><i class="fa fa-check"></i><b>2.8.3</b> Tissue collection</a></li>
<li class="chapter" data-level="2.8.4" data-path="ev.html"><a href="ev.html#electron-microscopy-images"><i class="fa fa-check"></i><b>2.8.4</b> Electron microscopy images</a></li>
<li class="chapter" data-level="2.8.5" data-path="ev.html"><a href="ev.html#epididymosomes-isolation-by-ultracentrifugation"><i class="fa fa-check"></i><b>2.8.5</b> Epididymosomes isolation by ultracentrifugation</a></li>
<li class="chapter" data-level="2.8.6" data-path="ev.html"><a href="ev.html#immunoblotting"><i class="fa fa-check"></i><b>2.8.6</b> Immunoblotting</a></li>
<li class="chapter" data-level="2.8.7" data-path="ev.html"><a href="ev.html#nanoparticle-tracking-analysis"><i class="fa fa-check"></i><b>2.8.7</b> Nanoparticle tracking analysis</a></li>
<li class="chapter" data-level="2.8.8" data-path="ev.html"><a href="ev.html#rna-isolation-and-epididymosomes-profiling"><i class="fa fa-check"></i><b>2.8.8</b> RNA isolation and epididymosomes profiling</a></li>
<li class="chapter" data-level="2.8.9" data-path="ev.html"><a href="ev.html#preparation-and-sequencing-of-srna-seq-libraries-from-epididymosomes"><i class="fa fa-check"></i><b>2.8.9</b> Preparation and sequencing of sRNA-seq libraries from epididymosomes</a></li>
<li class="chapter" data-level="2.8.10" data-path="ev.html"><a href="ev.html#rt-qpcr"><i class="fa fa-check"></i><b>2.8.10</b> RT-qPCR</a></li>
<li class="chapter" data-level="2.8.11" data-path="ev.html"><a href="ev.html#cholesterol-measurements"><i class="fa fa-check"></i><b>2.8.11</b> Cholesterol measurements</a></li>
<li class="chapter" data-level="2.8.12" data-path="ev.html"><a href="ev.html#bioinformatics-data-analysis"><i class="fa fa-check"></i><b>2.8.12</b> Bioinformatics data analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.9" data-path="ev.html"><a href="ev.html#data-availability"><i class="fa fa-check"></i><b>2.9</b> Data availability</a></li>
<li class="chapter" data-level="2.10" data-path="ev.html"><a href="ev.html#authors-contributions"><i class="fa fa-check"></i><b>2.10</b> Authors’ contributions</a></li>
<li class="chapter" data-level="2.11" data-path="ev.html"><a href="ev.html#grant-support"><i class="fa fa-check"></i><b>2.11</b> Grant Support</a></li>
<li class="chapter" data-level="2.12" data-path="ev.html"><a href="ev.html#acknowledgements"><i class="fa fa-check"></i><b>2.12</b> Acknowledgements</a></li>
<li class="chapter" data-level="2.13" data-path="ev.html"><a href="ev.html#supplementary-figures-1"><i class="fa fa-check"></i><b>2.13</b> Supplementary Figures</a>
<ul>
<li class="chapter" data-level="2.13.1" data-path="ev.html"><a href="ev.html#figure-1-1"><i class="fa fa-check"></i><b>2.13.1</b> Figure 1</a></li>
<li class="chapter" data-level="2.13.2" data-path="ev.html"><a href="ev.html#figure-2-1"><i class="fa fa-check"></i><b>2.13.2</b> Figure 2</a></li>
<li class="chapter" data-level="2.13.3" data-path="ev.html"><a href="ev.html#figure-3-1"><i class="fa fa-check"></i><b>2.13.3</b> Figure 3</a></li>
<li class="chapter" data-level="2.13.4" data-path="ev.html"><a href="ev.html#figure-4-1"><i class="fa fa-check"></i><b>2.13.4</b> Figure 4</a></li>
<li class="chapter" data-level="2.13.5" data-path="ev.html"><a href="ev.html#figure-5-1"><i class="fa fa-check"></i><b>2.13.5</b> Figure 5</a></li>
<li class="chapter" data-level="2.13.6" data-path="ev.html"><a href="ev.html#figure-6-1"><i class="fa fa-check"></i><b>2.13.6</b> Figure 6</a></li>
</ul></li>
<li class="chapter" data-level="2.14" data-path="ev.html"><a href="ev.html#supplementary-tables"><i class="fa fa-check"></i><b>2.14</b> Supplementary Tables</a>
<ul>
<li class="chapter" data-level="2.14.1" data-path="ev.html"><a href="ev.html#table-1"><i class="fa fa-check"></i><b>2.14.1</b> Table 1</a></li>
<li class="chapter" data-level="2.14.2" data-path="ev.html"><a href="ev.html#table-2"><i class="fa fa-check"></i><b>2.14.2</b> Table 2</a></li>
<li class="chapter" data-level="2.14.3" data-path="ev.html"><a href="ev.html#table-3"><i class="fa fa-check"></i><b>2.14.3</b> Table 3</a></li>
<li class="chapter" data-level="2.14.4" data-path="ev.html"><a href="ev.html#table-4"><i class="fa fa-check"></i><b>2.14.4</b> Table 4</a></li>
</ul></li>
<li class="chapter" data-level="2.15" data-path="ev.html"><a href="ev.html#references-1"><i class="fa fa-check"></i><b>2.15</b> References</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="shortRNA.html"><a href="shortRNA.html"><i class="fa fa-check"></i><b>3</b> shortRNA</a>
<ul>
<li class="chapter" data-level="3.1" data-path="shortRNA.html"><a href="shortRNA.html#abstract-2"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="shortRNA.html"><a href="shortRNA.html#introduction-2"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="shortRNA.html"><a href="shortRNA.html#methods-2"><i class="fa fa-check"></i><b>3.3</b> Methods</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="shortRNA.html"><a href="shortRNA.html#pipeline"><i class="fa fa-check"></i><b>3.3.1</b> Pipeline</a></li>
<li class="chapter" data-level="3.3.2" data-path="shortRNA.html"><a href="shortRNA.html#qc"><i class="fa fa-check"></i><b>3.3.2</b> QC</a></li>
<li class="chapter" data-level="3.3.3" data-path="shortRNA.html"><a href="shortRNA.html#annotation-preparation"><i class="fa fa-check"></i><b>3.3.3</b> Annotation preparation</a></li>
<li class="chapter" data-level="3.3.4" data-path="shortRNA.html"><a href="shortRNA.html#alignment"><i class="fa fa-check"></i><b>3.3.4</b> Alignment</a></li>
<li class="chapter" data-level="3.3.5" data-path="shortRNA.html"><a href="shortRNA.html#reads-assignment"><i class="fa fa-check"></i><b>3.3.5</b> Reads assignment</a></li>
<li class="chapter" data-level="3.3.6" data-path="shortRNA.html"><a href="shortRNA.html#assignment-rules"><i class="fa fa-check"></i><b>3.3.6</b> Assignment rules</a></li>
<li class="chapter" data-level="3.3.7" data-path="shortRNA.html"><a href="shortRNA.html#treesummarizedexperiment-object"><i class="fa fa-check"></i><b>3.3.7</b> TreeSummarizedExperiment object</a></li>
<li class="chapter" data-level="3.3.8" data-path="shortRNA.html"><a href="shortRNA.html#differential-analysis"><i class="fa fa-check"></i><b>3.3.8</b> Differential analysis</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="shortRNA.html"><a href="shortRNA.html#results-2"><i class="fa fa-check"></i><b>3.4</b> Results</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="shortRNA.html"><a href="shortRNA.html#datasets-used-for-testing-the-pipeline"><i class="fa fa-check"></i><b>3.4.1</b> Datasets used for testing the pipeline</a></li>
<li class="chapter" data-level="3.4.2" data-path="shortRNA.html"><a href="shortRNA.html#databases-included-for-analyzing-these-data"><i class="fa fa-check"></i><b>3.4.2</b> Databases included for analyzing these data</a></li>
<li class="chapter" data-level="3.4.3" data-path="shortRNA.html"><a href="shortRNA.html#result-1"><i class="fa fa-check"></i><b>3.4.3</b> result 1</a></li>
<li class="chapter" data-level="3.4.4" data-path="shortRNA.html"><a href="shortRNA.html#result-2"><i class="fa fa-check"></i><b>3.4.4</b> result 2</a></li>
<li class="chapter" data-level="3.4.5" data-path="shortRNA.html"><a href="shortRNA.html#result-3"><i class="fa fa-check"></i><b>3.4.5</b> result 3</a></li>
<li class="chapter" data-level="3.4.6" data-path="shortRNA.html"><a href="shortRNA.html#comparison-with-other-tools"><i class="fa fa-check"></i><b>3.4.6</b> Comparison with other tools</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="shortRNA.html"><a href="shortRNA.html#discussion-outlook"><i class="fa fa-check"></i><b>3.5</b> Discussion &amp; Outlook</a></li>
<li class="chapter" data-level="3.6" data-path="shortRNA.html"><a href="shortRNA.html#data-and-code-availibility"><i class="fa fa-check"></i><b>3.6</b> Data and code availibility</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="developmental.html"><a href="developmental.html#discussion"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="chapter" data-level="" data-path="aa.html"><a href="aa.html"><i class="fa fa-check"></i>Appendix A</a>
<ul>
<li class="chapter" data-level="3.7" data-path="aa.html"><a href="aa.html#datasets-analyzed"><i class="fa fa-check"></i><b>3.7</b> Datasets analyzed</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="ab.html"><a href="ab.html"><i class="fa fa-check"></i>Appendix B</a>
<ul>
<li class="chapter" data-level="3.8" data-path="ab.html"><a href="ab.html#other-manuscripts-during-phd"><i class="fa fa-check"></i><b>3.8</b> Other manuscripts during PhD</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="ac.html"><a href="ac.html"><i class="fa fa-check"></i>Appendix C</a></li>
<li class="chapter" data-level="" data-path="references-2.html"><a href="references-2.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ev" class="section level1" number="2">
<h1><span class="header-section-number">Chapter 2</span> Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse</h1>
<p>Anar Alshanbayeva, <strong>Deepak K Tanwar</strong>, Martin Roszkowski, Francesca Manuella, Isabelle M Mansuy


Laboratory of Neuroepigenetics, Brain Research Institute at the Medical Faculty of the University of Zurich.

Institute for Neuroscience of the Department of Health Sciences and Technology, ETH Zurich, Zurich, Switzerland.

Zurich Neuroscience Center, ETH and University of Zurich, Zurich, Switzerland.


Corresponding author


<strong>Journal:</strong> <em>Biology of Reproduction</em>

DOI: <a href="https://doi.org/10.1093/biolre/ioab156">10.1093/biolre/ioab156</a>


<strong>Contributions:</strong> <em>I performed data analysis with Anar Alshanbayeva, generated figures with Anar Alshanbayeva, helped Anar Alshanbayeva in writing the manuscript, and revised manuscript with Anar Alshanbayeva.</em></p>
<div style="page-break-after: always;"></div>
<div id="abstract-1" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Abstract</h2>
<p>Sperm RNA can be modified by environmental factors and has been implicated in communicating signals about changes in a father’s environment to the offspring. The small RNA composition of sperm could be changed during its final stage of maturation in the epididymis by extracellular vesicles (EVs) released by epididymal cells. We studied the effect of exposure to stress in early postnatal life on the transcriptome of epididymal EVs using a mouse model of transgenerational transmission. We found that the small RNA signature of epididymal EVs, particularly miRNAs, is altered in adult males exposed to postnatal stress. In some cases, these miRNA changes correlate with differences in the expression of their target genes in sperm and zygotes generated from that sperm. These results suggest that stressful experiences in early life can have persistent biological effects on the male reproductive tract that may in part be responsible for the transmission of the effects of exposure to the offspring.</p>
</div>
<div id="summary-sentence" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Summary sentence</h2>
<p>miRNA cargo of extracellular vesicles in cauda epididymis is altered by paternal exposure to early life stress. This correlates with changes in the expression of target genes in sperm and in zygotes generated from that sperm.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="graphical-abstract-1" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Graphical Abstract</h2>
</div>
<div id="key-words" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Key words</h2>
<p>epigenetics, epididymis, epididymosomes, early life stress, extracellular vesicles, miRNAs, sperm.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="introduction-1" class="section level2" number="2.5">
<h2><span class="header-section-number">2.5</span> Introduction</h2>
<p>Post-testicular maturation of spermatozoa in the epididymis is an elaborate process that involves modifications of sperm RNA, protein, and lipid content <span class="citation">(Nixon et al., 2015; Rejraji et al., 2006; Sharma et al., 2016; Skerget, Rosenow, Petritis, &amp; Karr, 2015; Tamessar et al., 2021)</span>. The epididymis is segmented into different parts, including the initial segment, caput, corpus, and cauda. Each segment has a distinct gene expression profile, and different protein and lipid composition. Some modifications in epididymal spermatozoa are conserved across species <span class="citation">(Sellem et al., 2020)</span>. For example, ~50% of miRNAs, a class of small RNAs that are modified during caput to cauda epididymis transit, is identical in mouse and bovine spermatozoa <span class="citation">(Sellem et al., 2020)</span>. One mechanism by which small RNA load in spermatozoa is modified along the epididymis is by uptake of extracellular vesicles (EVs), also known as epididymosomes, which are produced by epididymal epithelial cells <span class="citation">(Reilly et al., 2016)</span>. Studies have shown that epididymosomes can be taken up by maturing sperm through proteins present on the sperm head such as dynamin in mice and tetraspanins or syntenins in humans <span class="citation">(Caballero, Frenette, Belleannée, &amp; Sullivan, 2013; Reilly et al., 2016; Thimon, Frenette, Saez, Thabet, &amp; Sullivan, 2008; Zhou et al., 2019)</span>. Co-incubation experiments provided evidence for epididymosome-mediated transfer of miRNAs to spermatozoa <span class="citation">(Reilly et al., 2016)</span>. Exogenous DNA and RNA can also be directly taken up by spermatozoa via artificial liposomes <span class="citation">(Bachiller, Schellander, Peli, &amp; Rüther, 1991)</span>.</p>
<p>However, it is still not clear if changes in small RNA composition of spermatozoa occurring during epididymal transit are required for embryonic development, and studies on the subject have been conflicting <span class="citation">(Conine, Sun, Song, Rivera-Pérez, &amp; Rando, 2018; Suganuma, Yanagimachi, &amp; Meistrich, 2005)</span>. Changes in sperm small RNA have nevertheless been suggested to play a role in the transmission of information about paternal experiences to the progeny and can influence their developmental trajectory <span class="citation">(Chan et al., 2020; Katharina Gapp et al., 2014; Sharma et al., 2016)</span>. Epididymosomal small RNA content can also be altered by exposure, for instance, to dietary insult and stress <span class="citation">(Chan et al., 2020; Sharma et al., 2016)</span>. For instance, epididymosomal miRNAs are changed by exposure to chronic stress <span class="citation">(Chan et al., 2020)</span> and low-protein diet <span class="citation">(Sharma et al., 2016)</span> in mice.</p>
<p>Transmission of information about paternal exposure to the offspring depends on the type of exposure, its duration and the developmental window at which it is applied. To date, little is known about the long-term effects of early life stress, particularly stress experienced after birth, on epididymosomal small RNA composition in adulthood, and whether any changes to this composition can influence gene expression in sperm and in zygotes generated from that sperm. Using a transgenerational mouse model of postnatal stress induced by unpredictable maternal separation combined with unpredictable maternal stress, unpredictable maternal separation combined with unpredictable maternal stress (MSUS) <span class="citation">(Franklin et al., 2010)</span>, we show that the miRNA signature of cauda epididymosomes in adult males is altered by exposure, and that this alteration, correlates with changes in the expression of their target genes in sperm and in zygotes.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="results-1" class="section level2" number="2.6">
<h2><span class="header-section-number">2.6</span> Results</h2>
<div id="isolation-of-cauda-epididymosomes-confirmed-by-several-methods" class="section level3" number="2.6.1">
<h3><span class="header-section-number">2.6.1</span> Isolation of cauda epididymosomes confirmed by several methods</h3>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig1"></span>
<img src="figure/anaraEV/1.jpeg" alt="testFig" width="500" />
<p class="caption">
Figure 2.1: testFig
</p>
</div>
<p> is a figure</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="ev.html#cb1-1" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> <span class="fu">matrix</span>(<span class="at">data =</span> <span class="fu">rnorm</span>(<span class="dv">100</span>, <span class="dv">1</span>,<span class="dv">1</span>), <span class="at">nrow =</span> <span class="dv">10</span>)</span>
<span id="cb1-2"><a href="ev.html#cb1-2" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(df,<span class="at">label =</span> <span class="st">&quot;tab1&quot;</span>,</span>
<span id="cb1-3"><a href="ev.html#cb1-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">&quot;Maximum Delays by Airline&quot;</span>,</span>
<span id="cb1-4"><a href="ev.html#cb1-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption.short =</span> <span class="st">&quot;Max Delays by Airline&quot;</span>,</span>
<span id="cb1-5"><a href="ev.html#cb1-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">longtable =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1-6"><a href="ev.html#cb1-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">booktabs =</span> <span class="cn">TRUE</span></span>
<span id="cb1-7"><a href="ev.html#cb1-7" aria-hidden="true" tabindex="-1"></a>)</span></code></pre></div>
<table style="width:100%;">
<caption><span id="tab:tab1">Table 2.1: </span>Maximum Delays by Airline</caption>
<colgroup>
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="9%" />
<col width="8%" />
<col width="10%" />
<col width="9%" />
<col width="10%" />
</colgroup>
<tbody>
<tr class="odd">
<td align="right">1.3481062</td>
<td align="right">1.4117682</td>
<td align="right">1.5122888</td>
<td align="right">0.0496106</td>
<td align="right">1.5089906</td>
<td align="right">0.8708267</td>
<td align="right">2.856591</td>
<td align="right">0.0299737</td>
<td align="right">1.1594168</td>
<td align="right">-0.0391803</td>
</tr>
<tr class="even">
<td align="right">0.9844369</td>
<td align="right">1.4793294</td>
<td align="right">0.5378289</td>
<td align="right">-0.0650591</td>
<td align="right">0.7715314</td>
<td align="right">0.7167377</td>
<td align="right">1.310372</td>
<td align="right">0.7362597</td>
<td align="right">0.6794936</td>
<td align="right">0.8200460</td>
</tr>
<tr class="odd">
<td align="right">0.5691062</td>
<td align="right">1.0485396</td>
<td align="right">-0.2830685</td>
<td align="right">-0.0045427</td>
<td align="right">1.9501109</td>
<td align="right">3.4328434</td>
<td align="right">1.674216</td>
<td align="right">2.4980207</td>
<td align="right">0.4659555</td>
<td align="right">1.5858225</td>
</tr>
<tr class="even">
<td align="right">2.2106748</td>
<td align="right">-0.6668658</td>
<td align="right">0.6128429</td>
<td align="right">1.4913462</td>
<td align="right">2.7258547</td>
<td align="right">0.9132302</td>
<td align="right">1.677395</td>
<td align="right">-1.0572186</td>
<td align="right">1.2401660</td>
<td align="right">3.3194090</td>
</tr>
<tr class="odd">
<td align="right">-0.6554510</td>
<td align="right">-0.3831714</td>
<td align="right">1.5427163</td>
<td align="right">2.7747070</td>
<td align="right">-0.6322579</td>
<td align="right">0.8911905</td>
<td align="right">0.286754</td>
<td align="right">1.0255944</td>
<td align="right">0.1751870</td>
<td align="right">1.8559057</td>
</tr>
<tr class="even">
<td align="right">1.4973533</td>
<td align="right">0.7887161</td>
<td align="right">1.4674989</td>
<td align="right">1.4218112</td>
<td align="right">1.7318716</td>
<td align="right">0.3196163</td>
<td align="right">1.965311</td>
<td align="right">-0.0254563</td>
<td align="right">0.4012858</td>
<td align="right">0.3532852</td>
</tr>
<tr class="odd">
<td align="right">0.1317781</td>
<td align="right">1.8324544</td>
<td align="right">1.7819952</td>
<td align="right">0.1086353</td>
<td align="right">0.1539139</td>
<td align="right">1.7511706</td>
<td align="right">1.012131</td>
<td align="right">-0.6514499</td>
<td align="right">0.4042961</td>
<td align="right">0.1994835</td>
</tr>
<tr class="even">
<td align="right">1.7285466</td>
<td align="right">1.6951024</td>
<td align="right">0.2575499</td>
<td align="right">0.9311336</td>
<td align="right">-0.3379541</td>
<td align="right">1.9261680</td>
<td align="right">1.428068</td>
<td align="right">0.2516179</td>
<td align="right">1.6285784</td>
<td align="right">0.0814184</td>
</tr>
<tr class="odd">
<td align="right">0.9695459</td>
<td align="right">1.3602344</td>
<td align="right">2.8641294</td>
<td align="right">-0.8062035</td>
<td align="right">2.2761041</td>
<td align="right">1.6590320</td>
<td align="right">2.134042</td>
<td align="right">0.6124056</td>
<td align="right">1.6164551</td>
<td align="right">1.0447776</td>
</tr>
<tr class="even">
<td align="right">-0.1787895</td>
<td align="right">1.5202238</td>
<td align="right">1.1298784</td>
<td align="right">1.2292171</td>
<td align="right">1.5834084</td>
<td align="right">2.0109505</td>
<td align="right">1.407150</td>
<td align="right">1.4397346</td>
<td align="right">0.7583192</td>
<td align="right">0.0856242</td>
</tr>
</tbody>
</table>
<p> is a table</p>
</div>
<div id="the-number-and-size-of-epididymosomes-in-adult-males-are-not-altered-by-postnatal-stress" class="section level3" number="2.6.2">
<h3><span class="header-section-number">2.6.2</span> The number and size of epididymosomes in adult males are not altered by postnatal stress</h3>
</div>
<div id="mirnas-are-persistently-altered-by-postnatal-stress-in-cauda-epididymosomes" class="section level3" number="2.6.3">
<h3><span class="header-section-number">2.6.3</span> miRNAs are persistently altered by postnatal stress in cauda epididymosomes</h3>
</div>
<div id="mrna-targets-of-mirnas-from-cauda-epididymosomes-are-altered-by-postnatal-stress-in-sperm-and-in-zygotes" class="section level3" number="2.6.4">
<h3><span class="header-section-number">2.6.4</span> mRNA targets of miRNAs from cauda epididymosomes are altered by postnatal stress in sperm and in zygotes</h3>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="discussion-1" class="section level2" number="2.7">
<h2><span class="header-section-number">2.7</span> Discussion</h2>
<p>The effects of environmental factors on RNA in the male reproductive tract, in particular, the epididymis have been examined in rodent models. Until now, most models have used invasive exposure such as dietary insult or injection of endocrine disruptors, applied prenatally and sometimes before conception. Few studies have examined the effects of non-invasive psychological/emotional exposure such as stress in early life and the effects on epididymal RNA in adulthood <span class="citation">(Chan et al., 2020)</span>. This study examines if postnatal stress affects RNAs in EVs released from the cauda epididymis and whether this has consequences for mature sperm and zygotes generated from that sperm.</p>
<p>Using a transgenerational mouse model of early postnatal stress, we show that several miRNAs, including miR-871-3p, miR-31-5p, miR-155-5p, miR-878-5p, and miR-34c-5p are altered in cauda epididymosomes in adult males exposed to postnatal stress, and that the targets of some of these miRNAs are affected in mature sperm and zygotes. Particularly, miR-31-5p is significantly decreased in cauda epididymosomes and its target genes are up-regulated in sperm but down-regulated in zygotes generated from that sperm, suggesting an over-compensation during early development. This may also be due to the heterogeneity of epididymosomes which have different size, biogenesis, and cellular targeting <span class="citation">(Sullivan, 2015)</span>, leading to a dissociation between the RNA content of epididymosomes and transcriptional changes in zygotes. It has been suggested that different subsets of epididymosomes have different roles. While a subset communicates with spermatozoa during sperm epididymal transit <span class="citation">(Reilly et al., 2016; Sharma et al., 2016)</span>, another subset serves in the communication within epididymal epithelial cells <span class="citation">(Belleannée, Calvo, Caballero, &amp; Sullivan, 2013)</span>, and a third one is delivered as part of seminal fluid during fertilization <span class="citation">(Belleannée, Légaré, Calvo, Thimon, &amp; Sullivan, 2013; Frenette, Légaré, Saez, &amp; Sullivan, 2005)</span>. Thus owing to their heterogeneity, not all cauda epididymosomes or their cargo is delivered to the oocyte upon fertilization, which may explain the differences in miRNAs targets that are affected in sperm and zygotes.</p>
<p>Several of the differentially expressed miRNAs in MSUS cauda epididymosomes play a role in metabolic processes and early development <span class="citation">(Reza et al., 2019)</span>. For instance, miR-31-5p is involved in glucose metabolism and fatty acid oxidation <span class="citation">(Reza et al., 2019)</span>. In humans, its target complement C1q Tumor Necrosis Factor-Related Protein 9A (CTRP9) protein is negatively correlated with the amount of visceral fat and positively associated with a beneficial glucose and metabolic phenotype <span class="citation">(Shao et al., 2017)</span>. This is consistent with the observation that glucose and insulin metabolism are also affected by MSUS <span class="citation">(Franklin et al., 2010; Katharina Gapp et al., 2014)</span>. The level of other miRNAs is significantly increased or decreased in MSUS epididymosomes, such as miR-155-5p, which facilitates differentiation of mouse embryonic stem cells, or miR-34c-5p that initiates the first embryonic cleavage in mice <span class="citation">(Reza et al., 2019)</span>.</p>
<p>The first days after birth are a sensitive period for the development and the establishment of cellular niches in tissues. Epithelial cells in the epididymis, which are the source of epididymosomes, undergo differentiation and expansion postnatally until puberty <span class="citation">(Robaire, Hinton, &amp; Orgebin-Crist, 2002)</span>. Once their expansion is completed, epididymal epithelial cells remain at a nearly constant number in adulthood. If they can be modified by prior exposure, they may therefore carry a memory of exposure into adulthood. The postnatal development and differentiation of epididymal epithelial cells primarily depend on testicular signals <span class="citation">(Bilińska et al., 2006; Robaire &amp; Hamzeh, 2011; Robaire et al., 2002; Zhu et al., 2000)</span>. Since chronic stress affects the coupling of the hypothalamus–pituitary and hypothalamus–gonadal axes, stress-related decrease in steroidogenesis can profoundly affect the differentiation and expansion of epididymal epithelial cells in early postnatal life. A number of studies have shown the importance of the abundance of androgens during postnatal life for epididymal development <span class="citation">(Robaire et al., 2002)</span>. Thus, the availability of testicular cholesterol during epididymal cells differentiation has implications for these cells. Systemic alteration in cholesterol metabolism seen in young MSUS males (decreased total cholesterol in testis and increased HDL cholesterol in liver) may contribute to metabolic changes seen in adult animals, for instance in plasma steroidogenesis and fatty acid pathways, and to alterations in glucose and insulin metabolism in adult MSUS males. Moreover, androgen-dependent miRNAs miR-878-5p and miR-871-3p are significantly increased in cauda epididymosomes in MSUS.</p>
<p>In conclusion, our results provide evidence that chronic stress in early postnatal life alters miRNAs in EVs of the male reproductive tract in adulthood, with effects in mature sperm and zygotes. These persistent and intergenerational effects in vivo point to the sensitivity of the reproductive system to stress exposure and the detrimental consequences for descendants. These consequences may differ depending on the time window and severity of paternal stress exposure. Further studies are necessary to more precisely define these effects and the source of vesicles and their cargo miRNAs.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="materials-and-methods" class="section level2" number="2.8">
<h2><span class="header-section-number">2.8</span> Materials and methods</h2>
<div id="animals" class="section level3" number="2.8.1">
<h3><span class="header-section-number">2.8.1</span> Animals</h3>
<p>Animal experiments were conducted according to the Swiss Law for Animal Protection and were approved by the cantonal veterinary office in Zürich under license number 83/2018. C57Bl/6 J mice were obtained from Janvier (France) and bred in-house to generate animals for the experiments. Animals were maintained under a reverse light–dark cycle in a temperature and humidity-controlled environment with constant access to food and water. Nine months old age-matched MSUS and control males were used for small RNA-sequencing (sRNA-seq) of cauda epididymosomes, tissue collection for RT-qPCR, nanoparticle-tracking analysis, and total cholesterol measurements. HDL cholesterol and total cholesterol measurements were performed on MSUS and control males at postnatal day 28. Datasets from previous publication <span class="citation">(K. Gapp et al., 2020)</span>: caudal sperm RNA-seq was performed on 5-months old males, and zygote RNA-seq was performed on zygotes from 3-months old males.</p>
</div>
<div id="msus" class="section level3" number="2.8.2">
<h3><span class="header-section-number">2.8.2</span> MSUS</h3>
<p>To obtain MSUS mice, 3-months old C57Bl/6 J females and their litters were subjected to daily 3 h separation unpredictably and females were exposed to an unpredictable stressor during separation as previously described <span class="citation">(Franklin et al., 2010)</span>. Control dams and pups were left undisturbed. After weaning at postnatal day 21, pups from different litters were randomly assigned to cages of 4–5 mice, in corresponding treatment groups to avoid litter effects.</p>
</div>
<div id="tissue-collection" class="section level3" number="2.8.3">
<h3><span class="header-section-number">2.8.3</span> Tissue collection</h3>
<p>After decapitation and blood collection, mice were pinned on a dissection board and cleaned with alcohol. Epididymis and testis were excised and separated from surrounding adipose tissue. The epididymis was separated into caput, corpus, and cauda. Cauda was excised with several incisions and sperm collected with a swim-up protocol. The supernatant was collected to isolate epididymosomes. The whole testis and caput epididymis were crushed with stainless steel beads in a tissue crusher in cold PBS, centrifuged at 3000 rcf for 10 min to pellet the tissue and cells and used for total cholesterol and HDL cholesterol measurements.</p>
</div>
<div id="electron-microscopy-images" class="section level3" number="2.8.4">
<h3><span class="header-section-number">2.8.4</span> Electron microscopy images</h3>
<p>Negative staining of cauda epididymosomes was performed with methylcellulose. Briefly, the carrier grid was glow-discharged in plasma for 10 min and washed with a drop of PBS, then incubated in 1% glutaraldehyde (GA) in water for 5 min and washed with water five times for 2 min each. Afterwards the grid was incubated in 1% UAc (uranyl acetate) for 5 min and then kept on ice in methylcellulose/UAc (900 ul methylcellulose 2% and 100 ul 3% UAc) solution. After incubation with methylcellulose/UAc, the excess liquid was removed by dipping onto a filter paper. The grid was air-dried on ice for 5 min. Imaging was performed with a transmission electron microscope.</p>
</div>
<div id="epididymosomes-isolation-by-ultracentrifugation" class="section level3" number="2.8.5">
<h3><span class="header-section-number">2.8.5</span> Epididymosomes isolation by ultracentrifugation</h3>
<p>After pelleting sperm following the sperm swim-up protocol in M2 medium (Sigma, M7167), the supernatant was centrifuged at 2000 rcf for 10 min, 10 000 rcf for 30 min and then ultracentrifuged at 120 000 rcf at 4 <span class="math inline">\(^{\circ}\)</span>C for 2 h (TH 64.1 rotor, Thermo Fisher Scientific). The epididymosomal pellet was then washed in PBS at 4 <span class="math inline">\(^{\circ}\)</span>C and ultracentrifuged at 120 000 rcf at 4 <span class="math inline">\(^{\circ}\)</span>C for 2 h. The resulting pellet was resuspended in 60 l of PBS for all downstream analysis.</p>
</div>
<div id="immunoblotting" class="section level3" number="2.8.6">
<h3><span class="header-section-number">2.8.6</span> Immunoblotting</h3>
<p>PBS-resuspended pellet containing epididymosomes was lysed in 10x RIPA buffer for 5 min at 4 <span class="math inline">\(^{\circ}\)</span>C. Equal amounts of protein were mixed with 4x Laemmli Sample Buffer (Bio-Rad Laboratories, USA) and loaded on 4–20% Tris-Glycine polyacrylamide gels (Bio-Rad Laboratories, USA). The membranes were blocked in 5% SureBlock (in Tris-buffer with 0.05% Tween-20) for 1 h at room temperature and incubated with overnight at 4 <span class="math inline">\(^{\circ}\)</span>C with primary anti-Cd9 ([1:3000; System Biosciences, USA] and anti-Gapdh [1:5000; Cell Signaling, USA; 14C10]) antibodies.</p>
</div>
<div id="nanoparticle-tracking-analysis" class="section level3" number="2.8.7">
<h3><span class="header-section-number">2.8.7</span> Nanoparticle tracking analysis</h3>
<p>Particle number and size of epididymosomes were measured using a Nanosight NS300 (Malvern, UK) at 20 <span class="math inline">\(^{\circ}\)</span>C, according to the manufacturer’s instructions and lots were generated using a published method <span class="citation">(Dragovic et al., 2011)</span>. The following parameters were kept constant for all samples: “Camera level” = 14 and “Detection threshold” = 7. For measurements with Nanosight, the resuspended pellet from ultracentrifugation was diluted to a 1:1000 concentration.</p>
</div>
<div id="rna-isolation-and-epididymosomes-profiling" class="section level3" number="2.8.8">
<h3><span class="header-section-number">2.8.8</span> RNA isolation and epididymosomes profiling</h3>
<p>To lyse purified epididymosomes, 33 l of lysis buffer (6.4 M guanidine HCl, 5% Tween 20, 5% Triton, 120 mM EDTA, and 120 mM Tris pH 8.0) per 60 l of PBS resuspended pellet was added to each sample, together with l Proteinase K and 3.3 l water. Samples were incubated at 60 <span class="math inline">\(^{\circ}\)</span>C for 15 min with shaking. A total of 40 l water were added and RNA was extracted using Trizol LS protocol, according to the manufacturer’s instructions. Profiling of extracted RNA was done using high-resolution automated electrophoresis on a 2100 Bioanalyzer (Agilent, G2939BA), according to instructions for the RNA 6000 Pico Kit (Agilent, 5067-1513) reagent.</p>
</div>
<div id="preparation-and-sequencing-of-srna-seq-libraries-from-epididymosomes" class="section level3" number="2.8.9">
<h3><span class="header-section-number">2.8.9</span> Preparation and sequencing of sRNA-seq libraries from epididymosomes</h3>
<p>sRNA-seq libraries were prepared using the NEB Next Small RNA-sequencing kit (NEB #E7300, New England BioLabs), according to the manufacturer’s instructions. About 80–90 ng of total RNA per sample was used to prepare the libraries. The same libraries were sequenced before and after size-selection (target peak 150 bp) with the BluePippin System. 200 million reads were obtained for 10 samples, with 125 bp single-stranded read-length on a HiSeq2500 sequencer.</p>
</div>
<div id="rt-qpcr" class="section level3" number="2.8.10">
<h3><span class="header-section-number">2.8.10</span> RT-qPCR</h3>
<p>For gene expression analysis in caput epididymis, RNA was extracted using the phenol/chloroform extraction method (TRIzol; Thermo Fisher Scientific). Reverse transcription was performed using miScript  RT reagents (Qiagen) - HiFlex buffer, and RT qPCR was performed with QuantiTect SYBR (Qiagen) on the Light-Cycler  480 (Roche). All samples were run in cycles as follows: 95 <span class="math inline">\(^{\circ}\)</span>C for 15 min, 45 cycles of 15 s at 94 <span class="math inline">\(^{\circ}\)</span>C, 30 s at 55 <span class="math inline">\(^{\circ}\)</span>C and 30 s at 70 <span class="math inline">\(^{\circ}\)</span>C, followed by gradual increase of temperature to <span class="math inline">\(^{\circ}\)</span>C. The endogenous control  was used for normalization. The expression level of genes was analyzed using two-tailed Student’s -test.</p>
</div>
<div id="cholesterol-measurements" class="section level3" number="2.8.11">
<h3><span class="header-section-number">2.8.11</span> Cholesterol measurements</h3>
<p>Testicular and epididymal total cholesterol and HDL cholesterol levels were measured using the CHOL reagent, in conjunction with SYNCHRON LX System(s), UniCel DxC 600/800 System(s) and Synchron Systems Multi Calibrator (Beckman Coulter), according to the manufacturer’s instructions at the Zurich Integrative Rodent Physiology (ZIRP) facility of the University of Zurich.</p>
</div>
<div id="bioinformatics-data-analysis" class="section level3" number="2.8.12">
<h3><span class="header-section-number">2.8.12</span> Bioinformatics data analysis</h3>
<p>sRNA-sequencing FASTQ files were processed using the <code>ExceRpt</code> pipeline, previously established for EV small RNA data analysis <span class="citation">(Rozowsky et al., 2019)</span>. Briefly, <code>ExceRpt</code> first automatically identifies and removes known 3<span class="math inline">\(&#39;\)</span> adapter sequences, then aligns against known spike-in sequences used for library construction, filters low-quality reads and aligns them to annotated sequences in the UniVec database. Reads not filtered out in pre-processing steps are then aligned to the mouse genome and transcriptome using <code>STAR</code> aligner <span class="citation">(Dobin et al., 2013)</span>. The annotations were performed in the following order: miRbase, tRNAscan, piRNA, GENCODE, and circRNA. rRNA counts were obtained using <code>Oasis 2</code> tool. Reads mapped to miRNAs were combined from sequencing obtained before and after size-selection and corrected for batch effect using <code>RUVSeq</code> <span class="citation">(Leek, 2014)</span>. Normalization factors were calculated using <code>TMM</code> <span class="citation">(Robinson &amp; Oshlack, 2010)</span> method and differential expression was performed using <code>edgeR</code> package <span class="citation">(Robinson, McCarthy, &amp; Smyth, 2010)</span> in <code>R</code>. For cumulative distribution plots, miRNA targets (all and conserved) were downloaded from TargetScan release 7.2 <span class="citation">(Agarwal, Bell, Nam, &amp; Bartel, 2015)</span>. When using context++ scores, targets were split into three same-frequency groups according to their scores. -values were calculated using a <code>Kolmogorov–Smirnov</code> test between the first and last groups (i.e. strongest and weakest targets). The miRNA pathway analysis was conducted using a web-server tool <code>DIANA-miRPath</code> <span class="citation">(Vlachos &amp; Hatzigeorgiou, 2017)</span>, where targets were predicted-derived from DIANA-TarBase v6.0, a database of experimentally validated miRNA targets. The adjusted  cutoff value of 0.05 was used for the identification of expressed pathways. The miRNAs and their corresponding target pathways information was extracted and plots were generated in <code>R</code>. <code>ggplot2</code> <span class="citation">(Wickham, 2016)</span> and <code>ComplexHeatmap</code> <span class="citation">(Gu, Eils, &amp; Schlesner, 2016)</span> <code>R</code> packages were used for generation of figures.</p>
</div>
</div>
<div id="data-availability" class="section level2" number="2.9">
<h2><span class="header-section-number">2.9</span> Data availability</h2>
<p>The datasets collected for this study are available as follows:</p>
<ul>
<li><p>sRNA-seq dataset of cauda epididymosomes before and after sizeselection: NCBI GEO under accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175976">GSE175976</a>.</p></li>
<li><p>Codes for bioinformatics analysis of RNA-sequencing datasets and
all corresponding differential expression analyses: Github repository <a href="https://github.com/mansuylab/alshanbayeva_et_al_2021">mansuylab/alshanbayeva_et_al_2021</a>.</p></li>
<li><p>Sperm and zygote sequencing datasets from previous publications
can be found in ArrayExpress database at EMBL-EBI (<a href="www.ebi.ac.uk/arrayexpress">www.ebi.ac.uk/arrayexpress</a>) with the accession number <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5834/">E-MTAB-5834</a> (sperm) and <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6589/">E-MTAB-6589</a> (zygotes).</p></li>
</ul>
<div style="page-break-after: always;"></div>
</div>
<div id="authors-contributions" class="section level2" number="2.10">
<h2><span class="header-section-number">2.10</span> Authors’ contributions</h2>
<p>AA and IMM conceived and designed the study. FM and MR performed the MSUS breeding and collected tissue samples. AA and DKT performed data analysis and generated figures. AA wrote the manuscript with input from DKT and IMM. AA performed all
experiments for RNA sequencing and all molecular analyses. IMM supervised the project and raised funds.</p>
</div>
<div id="grant-support" class="section level2" number="2.11">
<h2><span class="header-section-number">2.11</span> Grant Support</h2>
<p>The work was supported by Swiss National Science Foundation (31003A-135715), ETH grants (ETH-10 15-2 and ETH-17 13-2), the Slack-Gyr Foundation, the Escher Foundation. The Mansuy lab is funded by the University Zürich, the Swiss Federal Institute of Technology, the Swiss National Science Foundation (31003A-135715), ETH grants (ETH-10 15-2 and ETH-17 13-2), the Slack-Gyr foundation, the Escher Foundation. Deepak K. Tanwar is supported by the Swiss Government Excellence Scholarship. Martin Roszkowski was funded by the ETH Zurich Fellowship (ETH-10 15-2).</p>
</div>
<div id="acknowledgements" class="section level2" number="2.12">
<h2><span class="header-section-number">2.12</span> Acknowledgements</h2>
<p>We thank Pierre-Luc Germain for advice on data analysis and for generating cumulative distribution plots, Irina Lazar-Contes for help with MSUS breeding, Silvia Schelbert for work on the animal license, Emilio Yandez at Function Genomics Center Zurich (FGCZ) for advice on the sRNA sequencing, Alekhya Mazumkhar for help with nanoparticle-tracking analysis, Yvonne Zipfel for animal care, Zurich Integrative Rodent Physiology facility for performing cholesterol measurements. We also thank Eloise Kremer, Ali Jawaid, and Mea Holmes for their initial contributions to the project.</p>
<p><strong>Conflict of interest:</strong> The authors declare no conflict of interest.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="supplementary-figures-1" class="section level2" number="2.13">
<h2><span class="header-section-number">2.13</span> Supplementary Figures</h2>
<div id="figure-1-1" class="section level3" number="2.13.1">
<h3><span class="header-section-number">2.13.1</span> Figure 1</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="figure-2-1" class="section level3" number="2.13.2">
<h3><span class="header-section-number">2.13.2</span> Figure 2</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="figure-3-1" class="section level3" number="2.13.3">
<h3><span class="header-section-number">2.13.3</span> Figure 3</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="figure-4-1" class="section level3" number="2.13.4">
<h3><span class="header-section-number">2.13.4</span> Figure 4</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="figure-5-1" class="section level3" number="2.13.5">
<h3><span class="header-section-number">2.13.5</span> Figure 5</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="figure-6-1" class="section level3" number="2.13.6">
<h3><span class="header-section-number">2.13.6</span> Figure 6</h3>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="supplementary-tables" class="section level2" number="2.14">
<h2><span class="header-section-number">2.14</span> Supplementary Tables</h2>
<div id="table-1" class="section level3" number="2.14.1">
<h3><span class="header-section-number">2.14.1</span> Table 1</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="table-2" class="section level3" number="2.14.2">
<h3><span class="header-section-number">2.14.2</span> Table 2</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="table-3" class="section level3" number="2.14.3">
<h3><span class="header-section-number">2.14.3</span> Table 3</h3>
<div style="page-break-after: always;"></div>
</div>
<div id="table-4" class="section level3" number="2.14.4">
<h3><span class="header-section-number">2.14.4</span> Table 4</h3>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="references-1" class="section level2" number="2.15">
<h2><span class="header-section-number">2.15</span> References</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="developmental.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="shortRNA.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
