<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance</title>
  <meta name="description" content="References | Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="References | Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance" />
  
  
  

<meta name="author" content="Deepak Kumar Tanwar" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="ac.html"/>

<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="summary-de.html"><a href="summary-de.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="0.1" data-path="intro.html"><a href="intro.html#epigenetics"><i class="fa fa-check"></i><b>0.1</b> Epigenetics</a></li>
<li class="chapter" data-level="0.2" data-path="intro.html"><a href="intro.html#germline-epigenetics"><i class="fa fa-check"></i><b>0.2</b> Germline epigenetics</a>
<ul>
<li class="chapter" data-level="0.2.1" data-path="intro.html"><a href="intro.html#spermatogonial-stem-cells"><i class="fa fa-check"></i><b>0.2.1</b> Spermatogonial Stem Cells</a></li>
<li class="chapter" data-level="0.2.2" data-path="intro.html"><a href="intro.html#reprogramming"><i class="fa fa-check"></i><b>0.2.2</b> Reprogramming</a></li>
</ul></li>
<li class="chapter" data-level="0.3" data-path="intro.html"><a href="intro.html#epigenetic-inheritance"><i class="fa fa-check"></i><b>0.3</b> Epigenetic Inheritance</a>
<ul>
<li class="chapter" data-level="0.3.1" data-path="intro.html"><a href="intro.html#transgenerational-epigenetic-inheritance"><i class="fa fa-check"></i><b>0.3.1</b> Transgenerational Epigenetic Inheritance</a></li>
</ul></li>
<li class="chapter" data-level="0.4" data-path="intro.html"><a href="intro.html#models-of-epigenetic-inheritance"><i class="fa fa-check"></i><b>0.4</b> Models of Epigenetic Inheritance</a>
<ul>
<li class="chapter" data-level="0.4.1" data-path="intro.html"><a href="intro.html#msus-mouse-model"><i class="fa fa-check"></i><b>0.4.1</b> MSUS mouse model</a></li>
</ul></li>
<li class="chapter" data-level="0.5" data-path="intro.html"><a href="intro.html#vectors-of-tei"><i class="fa fa-check"></i><b>0.5</b> Vectors of TEI</a>
<ul>
<li class="chapter" data-level="0.5.1" data-path="intro.html"><a href="intro.html#extracellular-vesicles"><i class="fa fa-check"></i><b>0.5.1</b> Extracellular vesicles</a></li>
<li class="chapter" data-level="0.5.2" data-path="intro.html"><a href="intro.html#small-rnas"><i class="fa fa-check"></i><b>0.5.2</b> Small RNAs</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i>Bioinformatics methods</a>
<ul>
<li class="chapter" data-level="0.6" data-path="methods.html"><a href="methods.html#version-controlled-data-analysis-using-git"><i class="fa fa-check"></i><b>0.6</b> Version controlled data analysis using git</a></li>
<li class="chapter" data-level="0.7" data-path="methods.html"><a href="methods.html#data-analysis-directory-organization"><i class="fa fa-check"></i><b>0.7</b> Data analysis directory organization</a></li>
<li class="chapter" data-level="0.8" data-path="methods.html"><a href="methods.html#pipelines-for-data-analysis"><i class="fa fa-check"></i><b>0.8</b> Pipelines for data analysis</a>
<ul>
<li class="chapter" data-level="0.8.1" data-path="methods.html"><a href="methods.html#bulk-rna-seq"><i class="fa fa-check"></i><b>0.8.1</b> Bulk RNA-seq</a></li>
<li class="chapter" data-level="0.8.2" data-path="methods.html"><a href="methods.html#atac-seq"><i class="fa fa-check"></i><b>0.8.2</b> ATAC-seq</a></li>
<li class="chapter" data-level="0.8.3" data-path="methods.html"><a href="methods.html#chip-seq"><i class="fa fa-check"></i><b>0.8.3</b> ChIP-seq</a></li>
<li class="chapter" data-level="0.8.4" data-path="methods.html"><a href="methods.html#wgbs-rrbs"><i class="fa fa-check"></i><b>0.8.4</b> WGBS &amp; RRBS</a></li>
<li class="chapter" data-level="0.8.5" data-path="methods.html"><a href="methods.html#small-rna-seq"><i class="fa fa-check"></i><b>0.8.5</b> small RNA-seq</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="aims.html"><a href="aims.html"><i class="fa fa-check"></i>Aims and thesis overview</a>
<ul>
<li class="chapter" data-level="0.9" data-path="aims.html"><a href="aims.html#aims-1"><i class="fa fa-check"></i><b>0.9</b> Aims</a></li>
<li class="chapter" data-level="0.10" data-path="aims.html"><a href="aims.html#thesis-overview"><i class="fa fa-check"></i><b>0.10</b> Thesis overview</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="developmental.html"><a href="developmental.html"><i class="fa fa-check"></i><b>1</b> Dynamic chromatin accessibility in spermatogonial cells for transcriptional programmings from early postnatal to adult stages</a>
<ul>
<li class="chapter" data-level="1.1" data-path="developmental.html"><a href="developmental.html#graphical-abstract"><i class="fa fa-check"></i><b>1.1</b> Graphical Abstract</a></li>
<li class="chapter" data-level="1.2" data-path="developmental.html"><a href="developmental.html#abstract"><i class="fa fa-check"></i><b>1.2</b> Abstract</a></li>
<li class="chapter" data-level="1.3" data-path="developmental.html"><a href="developmental.html#introduction"><i class="fa fa-check"></i><b>1.3</b> Introduction</a></li>
<li class="chapter" data-level="1.4" data-path="developmental.html"><a href="developmental.html#results"><i class="fa fa-check"></i><b>1.4</b> Results</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="developmental.html"><a href="developmental.html#enrichment-of-spermatogonial-cells-from-postnatal-and-adult-mouse-testis"><i class="fa fa-check"></i><b>1.4.1</b> Enrichment of spermatogonial cells from postnatal and adult mouse testis</a></li>
<li class="chapter" data-level="1.4.2" data-path="developmental.html"><a href="developmental.html#chromatin-is-remodeled-in-spermatogonial-cells-during-development"><i class="fa fa-check"></i><b>1.4.2</b> Chromatin is remodeled in spermatogonial cells during development</a></li>
<li class="chapter" data-level="1.4.3" data-path="developmental.html"><a href="developmental.html#differentially-accessible-chromatin-regions-associate-with-distinct-gene-expression-dynamics"><i class="fa fa-check"></i><b>1.4.3</b> Differentially accessible chromatin regions associate with distinct gene expression dynamics</a></li>
<li class="chapter" data-level="1.4.4" data-path="developmental.html"><a href="developmental.html#differentially-accessible-chromatin-regions-associate-with-distinct-epigenetic-profiles"><i class="fa fa-check"></i><b>1.4.4</b> Differentially accessible chromatin regions associate with distinct epigenetic profiles</a></li>
<li class="chapter" data-level="1.4.5" data-path="developmental.html"><a href="developmental.html#accessibility-changes-at-open-chromatin-regions-are-markedcarry-by-binding-sites-for-distinct-families-of-transcription-factors"><i class="fa fa-check"></i><b>1.4.5</b> Accessibility changes at open chromatin regions are markedcarry by binding sites for distinct families of transcription factors</a></li>
<li class="chapter" data-level="1.4.6" data-path="developmental.html"><a href="developmental.html#chromatin-accessibility-at-transposable-elements-undergoes-significant-remodeling-in-the-transition-from-postnatal-to-adult-spermatogonia"><i class="fa fa-check"></i><b>1.4.6</b> Chromatin accessibility at transposable elements undergoes significant remodeling in the transition from postnatal to adult spermatogonia</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="developmental.html"><a href="developmental.html#discussion"><i class="fa fa-check"></i><b>1.5</b> Discussion</a></li>
<li class="chapter" data-level="1.6" data-path="developmental.html"><a href="developmental.html#methods-1"><i class="fa fa-check"></i><b>1.6</b> Methods</a>
<ul>
<li class="chapter" data-level="1.6.1" data-path="developmental.html"><a href="developmental.html#mouse-husbandry"><i class="fa fa-check"></i><b>1.6.1</b> Mouse husbandry</a></li>
<li class="chapter" data-level="1.6.2" data-path="developmental.html"><a href="developmental.html#germ-cells-isolation"><i class="fa fa-check"></i><b>1.6.2</b> Germ cells isolation</a></li>
<li class="chapter" data-level="1.6.3" data-path="developmental.html"><a href="developmental.html#spermatogonial-cells-enrichment-by-facs"><i class="fa fa-check"></i><b>1.6.3</b> Spermatogonial cells enrichment by FACS</a></li>
<li class="chapter" data-level="1.6.4" data-path="developmental.html"><a href="developmental.html#immunocytochemistry"><i class="fa fa-check"></i><b>1.6.4</b> Immunocytochemistry</a></li>
<li class="chapter" data-level="1.6.5" data-path="developmental.html"><a href="developmental.html#rna-extraction-and-rna-seq-library-preparation-for-rna-seq"><i class="fa fa-check"></i><b>1.6.5</b> RNA extraction and RNA-seq library preparation for RNA-seq</a></li>
<li class="chapter" data-level="1.6.6" data-path="developmental.html"><a href="developmental.html#omni-atac-llibrary-preparation-for-omni-atac-and-sequencing"><i class="fa fa-check"></i><b>1.6.6</b> Omni-ATAC Llibrary preparation for Omni-ATAC and sequencing</a></li>
<li class="chapter" data-level="1.6.7" data-path="developmental.html"><a href="developmental.html#rna-sequencing-rna-seq"><i class="fa fa-check"></i><b>1.6.7</b> RNA sequencing (RNA-seq)</a></li>
<li class="chapter" data-level="1.6.8" data-path="developmental.html"><a href="developmental.html#assay-for-transposase-accessible-chromatin-using-sequening-omni-atac"><i class="fa fa-check"></i><b>1.6.8</b> Assay for Transposase-Accessible Chromatin using sequening (Omni-ATAC)</a></li>
<li class="chapter" data-level="1.6.9" data-path="developmental.html"><a href="developmental.html#chromatin-iimmunnoprecipitation-sequencing-chip-seq"><i class="fa fa-check"></i><b>1.6.9</b> Chromatin iImmunnoprecipitation sequencing (ChIP-seq)</a></li>
<li class="chapter" data-level="1.6.10" data-path="developmental.html"><a href="developmental.html#bisulfite-sequencing-bs"><i class="fa fa-check"></i><b>1.6.10</b> Bisulfite sequencing (BS)</a></li>
<li class="chapter" data-level="1.6.11" data-path="developmental.html"><a href="developmental.html#high-throughput-sequencing-data-analysis"><i class="fa fa-check"></i><b>1.6.11</b> High-throughput sequencing data analysis</a></li>
<li class="chapter" data-level="1.6.12" data-path="developmental.html"><a href="developmental.html#figures"><i class="fa fa-check"></i><b>1.6.12</b> Figures</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="developmental.html"><a href="developmental.html#data-and-materials-availability"><i class="fa fa-check"></i><b>1.7</b> Data and materials availability</a></li>
<li class="chapter" data-level="1.8" data-path="developmental.html"><a href="developmental.html#authors-contribution"><i class="fa fa-check"></i><b>1.8</b> Authors Contribution</a></li>
<li class="chapter" data-level="1.9" data-path="developmental.html"><a href="developmental.html#competing-interest"><i class="fa fa-check"></i><b>1.9</b> Competing interest</a></li>
<li class="chapter" data-level="1.10" data-path="developmental.html"><a href="developmental.html#acknowledgments"><i class="fa fa-check"></i><b>1.10</b> Acknowledgments</a></li>
<li class="chapter" data-level="1.11" data-path="developmental.html"><a href="developmental.html#funding"><i class="fa fa-check"></i><b>1.11</b> Funding</a></li>
<li class="chapter" data-level="1.12" data-path="developmental.html"><a href="developmental.html#supplementary-figures"><i class="fa fa-check"></i><b>1.12</b> Supplementary Figures</a>
<ul>
<li class="chapter" data-level="1.12.1" data-path="developmental.html"><a href="developmental.html#figure-1"><i class="fa fa-check"></i><b>1.12.1</b> Figure 1</a></li>
<li class="chapter" data-level="1.12.2" data-path="developmental.html"><a href="developmental.html#figure-2"><i class="fa fa-check"></i><b>1.12.2</b> Figure 2</a></li>
<li class="chapter" data-level="1.12.3" data-path="developmental.html"><a href="developmental.html#figure-3"><i class="fa fa-check"></i><b>1.12.3</b> Figure 3</a></li>
<li class="chapter" data-level="1.12.4" data-path="developmental.html"><a href="developmental.html#figure-4"><i class="fa fa-check"></i><b>1.12.4</b> Figure 4</a></li>
<li class="chapter" data-level="1.12.5" data-path="developmental.html"><a href="developmental.html#figure-5"><i class="fa fa-check"></i><b>1.12.5</b> Figure 5</a></li>
<li class="chapter" data-level="1.12.6" data-path="developmental.html"><a href="developmental.html#figure-6"><i class="fa fa-check"></i><b>1.12.6</b> Figure 6</a></li>
</ul></li>
<li class="chapter" data-level="1.13" data-path="developmental.html"><a href="developmental.html#references"><i class="fa fa-check"></i><b>1.13</b> References</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ev.html"><a href="ev.html"><i class="fa fa-check"></i><b>2</b> Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse</a>
<ul>
<li class="chapter" data-level="2.1" data-path="ev.html"><a href="ev.html#abstract-1"><i class="fa fa-check"></i><b>2.1</b> Abstract</a></li>
<li class="chapter" data-level="2.2" data-path="ev.html"><a href="ev.html#summary-sentence"><i class="fa fa-check"></i><b>2.2</b> Summary sentence</a></li>
<li class="chapter" data-level="2.3" data-path="ev.html"><a href="ev.html#graphical-abstract-1"><i class="fa fa-check"></i><b>2.3</b> Graphical Abstract</a></li>
<li class="chapter" data-level="2.4" data-path="ev.html"><a href="ev.html#key-words"><i class="fa fa-check"></i><b>2.4</b> Key words</a></li>
<li class="chapter" data-level="2.5" data-path="ev.html"><a href="ev.html#introduction-1"><i class="fa fa-check"></i><b>2.5</b> Introduction</a></li>
<li class="chapter" data-level="2.6" data-path="ev.html"><a href="ev.html#results-1"><i class="fa fa-check"></i><b>2.6</b> Results</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="ev.html"><a href="ev.html#isolation-of-cauda-epididymosomes-confirmed-by-several-methods"><i class="fa fa-check"></i><b>2.6.1</b> Isolation of cauda epididymosomes confirmed by several methods</a></li>
<li class="chapter" data-level="2.6.2" data-path="ev.html"><a href="ev.html#the-number-and-size-of-epididymosomes-in-adult-males-are-not-altered-by-postnatal-stress"><i class="fa fa-check"></i><b>2.6.2</b> The number and size of epididymosomes in adult males are not altered by postnatal stress</a></li>
<li class="chapter" data-level="2.6.3" data-path="ev.html"><a href="ev.html#mirnas-are-persistently-altered-by-postnatal-stress-in-cauda-epididymosomes"><i class="fa fa-check"></i><b>2.6.3</b> miRNAs are persistently altered by postnatal stress in cauda epididymosomes</a></li>
<li class="chapter" data-level="2.6.4" data-path="ev.html"><a href="ev.html#mrna-targets-of-mirnas-from-cauda-epididymosomes-are-altered-by-postnatal-stress-in-sperm-and-in-zygotes"><i class="fa fa-check"></i><b>2.6.4</b> mRNA targets of miRNAs from cauda epididymosomes are altered by postnatal stress in sperm and in zygotes</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="ev.html"><a href="ev.html#discussion-1"><i class="fa fa-check"></i><b>2.7</b> Discussion</a></li>
<li class="chapter" data-level="2.8" data-path="ev.html"><a href="ev.html#materials-and-methods"><i class="fa fa-check"></i><b>2.8</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="2.8.1" data-path="ev.html"><a href="ev.html#animals"><i class="fa fa-check"></i><b>2.8.1</b> Animals</a></li>
<li class="chapter" data-level="2.8.2" data-path="ev.html"><a href="ev.html#msus"><i class="fa fa-check"></i><b>2.8.2</b> MSUS</a></li>
<li class="chapter" data-level="2.8.3" data-path="ev.html"><a href="ev.html#tissue-collection"><i class="fa fa-check"></i><b>2.8.3</b> Tissue collection</a></li>
<li class="chapter" data-level="2.8.4" data-path="ev.html"><a href="ev.html#electron-microscopy-images"><i class="fa fa-check"></i><b>2.8.4</b> Electron microscopy images</a></li>
<li class="chapter" data-level="2.8.5" data-path="ev.html"><a href="ev.html#epididymosomes-isolation-by-ultracentrifugation"><i class="fa fa-check"></i><b>2.8.5</b> Epididymosomes isolation by ultracentrifugation</a></li>
<li class="chapter" data-level="2.8.6" data-path="ev.html"><a href="ev.html#immunoblotting"><i class="fa fa-check"></i><b>2.8.6</b> Immunoblotting</a></li>
<li class="chapter" data-level="2.8.7" data-path="ev.html"><a href="ev.html#nanoparticle-tracking-analysis"><i class="fa fa-check"></i><b>2.8.7</b> Nanoparticle tracking analysis</a></li>
<li class="chapter" data-level="2.8.8" data-path="ev.html"><a href="ev.html#rna-isolation-and-epididymosomes-profiling"><i class="fa fa-check"></i><b>2.8.8</b> RNA isolation and epididymosomes profiling</a></li>
<li class="chapter" data-level="2.8.9" data-path="ev.html"><a href="ev.html#preparation-and-sequencing-of-srna-seq-libraries-from-epididymosomes"><i class="fa fa-check"></i><b>2.8.9</b> Preparation and sequencing of sRNA-seq libraries from epididymosomes</a></li>
<li class="chapter" data-level="2.8.10" data-path="ev.html"><a href="ev.html#rt-qpcr"><i class="fa fa-check"></i><b>2.8.10</b> RT-qPCR</a></li>
<li class="chapter" data-level="2.8.11" data-path="ev.html"><a href="ev.html#cholesterol-measurements"><i class="fa fa-check"></i><b>2.8.11</b> Cholesterol measurements</a></li>
<li class="chapter" data-level="2.8.12" data-path="ev.html"><a href="ev.html#bioinformatics-data-analysis"><i class="fa fa-check"></i><b>2.8.12</b> Bioinformatics data analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.9" data-path="ev.html"><a href="ev.html#data-availability"><i class="fa fa-check"></i><b>2.9</b> Data availability</a></li>
<li class="chapter" data-level="2.10" data-path="ev.html"><a href="ev.html#authors-contributions"><i class="fa fa-check"></i><b>2.10</b> Authors’ contributions</a></li>
<li class="chapter" data-level="2.11" data-path="ev.html"><a href="ev.html#grant-support"><i class="fa fa-check"></i><b>2.11</b> Grant Support</a></li>
<li class="chapter" data-level="2.12" data-path="ev.html"><a href="ev.html#acknowledgements"><i class="fa fa-check"></i><b>2.12</b> Acknowledgements</a></li>
<li class="chapter" data-level="2.13" data-path="ev.html"><a href="ev.html#supplementary-figures-1"><i class="fa fa-check"></i><b>2.13</b> Supplementary Figures</a>
<ul>
<li class="chapter" data-level="2.13.1" data-path="ev.html"><a href="ev.html#figure-1-1"><i class="fa fa-check"></i><b>2.13.1</b> Figure 1</a></li>
<li class="chapter" data-level="2.13.2" data-path="ev.html"><a href="ev.html#figure-2-1"><i class="fa fa-check"></i><b>2.13.2</b> Figure 2</a></li>
<li class="chapter" data-level="2.13.3" data-path="ev.html"><a href="ev.html#figure-3-1"><i class="fa fa-check"></i><b>2.13.3</b> Figure 3</a></li>
<li class="chapter" data-level="2.13.4" data-path="ev.html"><a href="ev.html#figure-4-1"><i class="fa fa-check"></i><b>2.13.4</b> Figure 4</a></li>
<li class="chapter" data-level="2.13.5" data-path="ev.html"><a href="ev.html#figure-5-1"><i class="fa fa-check"></i><b>2.13.5</b> Figure 5</a></li>
<li class="chapter" data-level="2.13.6" data-path="ev.html"><a href="ev.html#figure-6-1"><i class="fa fa-check"></i><b>2.13.6</b> Figure 6</a></li>
</ul></li>
<li class="chapter" data-level="2.14" data-path="ev.html"><a href="ev.html#supplementary-tables"><i class="fa fa-check"></i><b>2.14</b> Supplementary Tables</a>
<ul>
<li class="chapter" data-level="2.14.1" data-path="ev.html"><a href="ev.html#table-1"><i class="fa fa-check"></i><b>2.14.1</b> Table 1</a></li>
<li class="chapter" data-level="2.14.2" data-path="ev.html"><a href="ev.html#table-2"><i class="fa fa-check"></i><b>2.14.2</b> Table 2</a></li>
<li class="chapter" data-level="2.14.3" data-path="ev.html"><a href="ev.html#table-3"><i class="fa fa-check"></i><b>2.14.3</b> Table 3</a></li>
<li class="chapter" data-level="2.14.4" data-path="ev.html"><a href="ev.html#table-4"><i class="fa fa-check"></i><b>2.14.4</b> Table 4</a></li>
</ul></li>
<li class="chapter" data-level="2.15" data-path="ev.html"><a href="ev.html#references-1"><i class="fa fa-check"></i><b>2.15</b> References</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="shortRNA.html"><a href="shortRNA.html"><i class="fa fa-check"></i><b>3</b> shortRNA</a>
<ul>
<li class="chapter" data-level="3.1" data-path="shortRNA.html"><a href="shortRNA.html#abstract-2"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="shortRNA.html"><a href="shortRNA.html#introduction-2"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="shortRNA.html"><a href="shortRNA.html#methods-2"><i class="fa fa-check"></i><b>3.3</b> Methods</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="shortRNA.html"><a href="shortRNA.html#pipeline"><i class="fa fa-check"></i><b>3.3.1</b> Pipeline</a></li>
<li class="chapter" data-level="3.3.2" data-path="shortRNA.html"><a href="shortRNA.html#qc"><i class="fa fa-check"></i><b>3.3.2</b> QC</a></li>
<li class="chapter" data-level="3.3.3" data-path="shortRNA.html"><a href="shortRNA.html#annotation-preparation"><i class="fa fa-check"></i><b>3.3.3</b> Annotation preparation</a></li>
<li class="chapter" data-level="3.3.4" data-path="shortRNA.html"><a href="shortRNA.html#alignment"><i class="fa fa-check"></i><b>3.3.4</b> Alignment</a></li>
<li class="chapter" data-level="3.3.5" data-path="shortRNA.html"><a href="shortRNA.html#reads-assignment"><i class="fa fa-check"></i><b>3.3.5</b> Reads assignment</a></li>
<li class="chapter" data-level="3.3.6" data-path="shortRNA.html"><a href="shortRNA.html#assignment-rules"><i class="fa fa-check"></i><b>3.3.6</b> Assignment rules</a></li>
<li class="chapter" data-level="3.3.7" data-path="shortRNA.html"><a href="shortRNA.html#treesummarizedexperiment-object"><i class="fa fa-check"></i><b>3.3.7</b> TreeSummarizedExperiment object</a></li>
<li class="chapter" data-level="3.3.8" data-path="shortRNA.html"><a href="shortRNA.html#differential-analysis"><i class="fa fa-check"></i><b>3.3.8</b> Differential analysis</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="shortRNA.html"><a href="shortRNA.html#results-2"><i class="fa fa-check"></i><b>3.4</b> Results</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="shortRNA.html"><a href="shortRNA.html#datasets-used-for-testing-the-pipeline"><i class="fa fa-check"></i><b>3.4.1</b> Datasets used for testing the pipeline</a></li>
<li class="chapter" data-level="3.4.2" data-path="shortRNA.html"><a href="shortRNA.html#databases-included-for-analyzing-these-data"><i class="fa fa-check"></i><b>3.4.2</b> Databases included for analyzing these data</a></li>
<li class="chapter" data-level="3.4.3" data-path="shortRNA.html"><a href="shortRNA.html#result-1"><i class="fa fa-check"></i><b>3.4.3</b> result 1</a></li>
<li class="chapter" data-level="3.4.4" data-path="shortRNA.html"><a href="shortRNA.html#result-2"><i class="fa fa-check"></i><b>3.4.4</b> result 2</a></li>
<li class="chapter" data-level="3.4.5" data-path="shortRNA.html"><a href="shortRNA.html#result-3"><i class="fa fa-check"></i><b>3.4.5</b> result 3</a></li>
<li class="chapter" data-level="3.4.6" data-path="shortRNA.html"><a href="shortRNA.html#comparison-with-other-tools"><i class="fa fa-check"></i><b>3.4.6</b> Comparison with other tools</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="shortRNA.html"><a href="shortRNA.html#discussion-outlook"><i class="fa fa-check"></i><b>3.5</b> Discussion &amp; Outlook</a></li>
<li class="chapter" data-level="3.6" data-path="shortRNA.html"><a href="shortRNA.html#data-and-code-availibility"><i class="fa fa-check"></i><b>3.6</b> Data and code availibility</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="developmental.html"><a href="developmental.html#discussion"><i class="fa fa-check"></i>Discussion</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="chapter" data-level="" data-path="aa.html"><a href="aa.html"><i class="fa fa-check"></i>Appendix A</a>
<ul>
<li class="chapter" data-level="3.7" data-path="aa.html"><a href="aa.html#datasets-analyzed"><i class="fa fa-check"></i><b>3.7</b> Datasets analyzed</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="ab.html"><a href="ab.html"><i class="fa fa-check"></i>Appendix B</a>
<ul>
<li class="chapter" data-level="3.8" data-path="ab.html"><a href="ab.html#other-manuscripts-during-phd"><i class="fa fa-check"></i><b>3.8</b> Other manuscripts during PhD</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="ac.html"><a href="ac.html"><i class="fa fa-check"></i>Appendix C</a></li>
<li class="chapter" data-level="" data-path="references-2.html"><a href="references-2.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references-2" class="section level1 unnumbered">
<h1>References</h1>
<!--
This manually sets the header for this unnumbered chapter.
-->
<!--
To remove the indentation of the first entry.
-->
<p></p>
<!--
To create a hanging indent and spacing between entries. This line may need 
to be removed for styles that don't require the hanging indent.
-->
<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->

<div id="refs" class="references csl-bib-body hanging-indent" line-spacing="2">
<div class="csl-entry">
Agarwal, V., Bell, G. W., Nam, J.-W., &amp; Bartel, D. P. (2015). Predicting effective <span class="nocase">microRNA</span> target sites in mammalian <span class="nocase">mRNAs</span>. <em>eLife</em>, <em>4</em>. http://doi.org/<a href="https://doi.org/10.7554/{eLife}.05005">10.7554/{eLife}.05005</a>
</div>
<div class="csl-entry">
Bachiller, D., Schellander, K., Peli, J., &amp; Rüther, U. (1991). Liposome-mediated <span>DNA</span> uptake by sperm cells. <em>Molecular Reproduction and Development</em>, <em>30</em>(3), 194–200. http://doi.org/<a href="https://doi.org/10.1002/mrd.1080300305">10.1002/mrd.1080300305</a>
</div>
<div class="csl-entry">
Belleannée, C., Calvo, É., Caballero, J., &amp; Sullivan, R. (2013). Epididymosomes convey different repertoires of <span class="nocase">microRNAs</span> throughout the bovine epididymis. <em>Biology of Reproduction</em>, <em>89</em>(2), 30. http://doi.org/<a href="https://doi.org/10.1095/biolreprod.113.110486">10.1095/biolreprod.113.110486</a>
</div>
<div class="csl-entry">
Belleannée, C., Légaré, C., Calvo, E., Thimon, V., &amp; Sullivan, R. (2013). <span class="nocase">microRNA</span> signature is altered in both human epididymis and seminal microvesicles following vasectomy. <em>Human Reproduction</em>, <em>28</em>(6), 1455–1467. http://doi.org/<a href="https://doi.org/10.1093/humrep/det088">10.1093/humrep/det088</a>
</div>
<div class="csl-entry">
Bilińska, B., Wiszniewska, B., Kosiniak-Kamysz, K., Kotula-Balak, M., Gancarczyk, M., Hejmej, A., … Wenda-Rózewicka, L. (2006). Hormonal status of male reproductive system: Androgens and estrogens in the testis and epididymis. In vivo and in vitro approaches. <em>Reproductive Biology</em>, <em>6 Suppl 1</em>, 43–58. Retrieved from <a href="https://www.ncbi.nlm.nih.gov/pubmed/16967089">https://www.ncbi.nlm.nih.gov/pubmed/16967089</a>
</div>
<div class="csl-entry">
Caballero, J. N., Frenette, G., Belleannée, C., &amp; Sullivan, R. (2013). <span>CD9</span>-positive microvesicles mediate the transfer of molecules to bovine spermatozoa during epididymal maturation. <em>Plos One</em>, <em>8</em>(6), e65364. http://doi.org/<a href="https://doi.org/10.1371/journal.pone.0065364">10.1371/journal.pone.0065364</a>
</div>
<div class="csl-entry">
Chan, J. C., Morgan, C. P., Adrian Leu, N., Shetty, A., Cisse, Y. M., Nugent, B. M., … Bale, T. L. (2020). Reproductive tract extracellular vesicles are sufficient to transmit intergenerational stress and program neurodevelopment. <em>Nature Communications</em>, <em>11</em>(1), 1499. http://doi.org/<a href="https://doi.org/10.1038/s41467-020-15305-w">10.1038/s41467-020-15305-w</a>
</div>
<div class="csl-entry">
Conine, C. C., Sun, F., Song, L., Rivera-Pérez, J. A., &amp; Rando, O. J. (2018). Small <span>RNAs</span> gained during epididymal transit of sperm are essential for embryonic development in mice. <em>Developmental Cell</em>, <em>46</em>(4), 470–480.e3. http://doi.org/<a href="https://doi.org/10.1016/j.devcel.2018.06.024">10.1016/j.devcel.2018.06.024</a>
</div>
<div class="csl-entry">
Dobin, A., Davis, C. A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., … Gingeras, T. R. (2013). <span>STAR</span>: Ultrafast universal <span>RNA</span>-seq aligner. <em>Bioinformatics</em>, <em>29</em>(1), 15–21. http://doi.org/<a href="https://doi.org/10.1093/bioinformatics/bts635">10.1093/bioinformatics/bts635</a>
</div>
<div class="csl-entry">
Dragovic, R. A., Gardiner, C., Brooks, A. S., Tannetta, D. S., Ferguson, D. J. P., Hole, P., … Sargent, I. L. (2011). Sizing and phenotyping of cellular vesicles using nanoparticle tracking analysis. <em>Nanomedicine : Nanotechnology, Biology, and Medicine</em>, <em>7</em>(6), 780–788. http://doi.org/<a href="https://doi.org/10.1016/j.nano.2011.04.003">10.1016/j.nano.2011.04.003</a>
</div>
<div class="csl-entry">
Franklin, T. B., Russig, H., Weiss, I. C., Gräff, J., Linder, N., Michalon, A., … Mansuy, I. M. (2010). Epigenetic transmission of the impact of early stress across generations. <em>Biological Psychiatry</em>, <em>68</em>(5), 408–415. http://doi.org/<a href="https://doi.org/10.1016/j.biopsych.2010.05.036">10.1016/j.biopsych.2010.05.036</a>
</div>
<div class="csl-entry">
Frenette, G., Légaré, C., Saez, F., &amp; Sullivan, R. (2005). Macrophage migration inhibitory factor in the human epididymis and semen. <em>Molecular Human Reproduction</em>, <em>11</em>(8), 575–582. http://doi.org/<a href="https://doi.org/10.1093/molehr/gah197">10.1093/molehr/gah197</a>
</div>
<div class="csl-entry">
Gapp, Katharina, Jawaid, A., Sarkies, P., Bohacek, J., Pelczar, P., Prados, J., … Mansuy, I. M. (2014). Implication of sperm <span>RNAs</span> in transgenerational inheritance of the effects of early trauma in mice. <em>Nature Neuroscience</em>, <em>17</em>(5), 667–669. http://doi.org/<a href="https://doi.org/10.1038/nn.3695">10.1038/nn.3695</a>
</div>
<div class="csl-entry">
Gapp, K., Steenwyk, G. van, Germain, P. L., Matsushima, W., Rudolph, K. L. M., Manuella, F., … Miska, E. A. (2020). Alterations in sperm long <span>RNA</span> contribute to the epigenetic inheritance of the effects of postnatal trauma. <em>Molecular Psychiatry</em>, <em>25</em>(9), 2162–2174. http://doi.org/<a href="https://doi.org/10.1038/s41380-018-0271-6">10.1038/s41380-018-0271-6</a>
</div>
<div class="csl-entry">
Gu, Z., Eils, R., &amp; Schlesner, M. (2016). Complex heatmaps reveal patterns and correlations in multidimensional genomic data. <em>Bioinformatics</em>, <em>32</em>(18), 2847–2849. http://doi.org/<a href="https://doi.org/10.1093/bioinformatics/btw313">10.1093/bioinformatics/btw313</a>
</div>
<div class="csl-entry">
Leek, J. T. (2014). Svaseq: Removing batch effects and other unwanted noise from sequencing data. <em>Nucleic Acids Research</em>, <em>42</em>(21). http://doi.org/<a href="https://doi.org/10.1093/nar/gku864">10.1093/nar/gku864</a>
</div>
<div class="csl-entry">
Nixon, B., Stanger, S. J., Mihalas, B. P., Reilly, J. N., Anderson, A. L., Tyagi, S., … McLaughlin, E. A. (2015). The <span class="nocase">microRNA</span> signature of mouse spermatozoa is substantially modified during epididymal maturation. <em>Biology of Reproduction</em>, <em>93</em>(4), 91. http://doi.org/<a href="https://doi.org/10.1095/biolreprod.115.132209">10.1095/biolreprod.115.132209</a>
</div>
<div class="csl-entry">
Reilly, J. N., McLaughlin, E. A., Stanger, S. J., Anderson, A. L., Hutcheon, K., Church, K., … Nixon, B. (2016). Characterisation of mouse epididymosomes reveals a complex profile of <span class="nocase">microRNAs</span> and a potential mechanism for modification of the sperm epigenome. <em>Scientific Reports</em>, <em>6</em>, 31794. http://doi.org/<a href="https://doi.org/10.1038/srep31794">10.1038/srep31794</a>
</div>
<div class="csl-entry">
Rejraji, H., Sion, B., Prensier, G., Carreras, M., Motta, C., Frenoux, J.-M., … Drevet, J. R. (2006). Lipid remodeling of murine epididymosomes and spermatozoa during epididymal maturation. <em>Biology of Reproduction</em>, <em>74</em>(6), 1104–1113. http://doi.org/<a href="https://doi.org/10.1095/biolreprod.105.049304">10.1095/biolreprod.105.049304</a>
</div>
<div class="csl-entry">
Reza, A. M. M. T., Choi, Y.-J., Han, S. G., Song, H., Park, C., Hong, K., &amp; Kim, J.-H. (2019). Roles of <span class="nocase">microRNAs</span> in mammalian reproduction: From the commitment of germ cells to peri-implantation embryos. <em>Biological Reviews of the Cambridge Philosophical Society</em>, <em>94</em>(2), 415–438. http://doi.org/<a href="https://doi.org/10.1111/brv.12459">10.1111/brv.12459</a>
</div>
<div class="csl-entry">
Robaire, B., &amp; Hamzeh, M. (2011). Androgen action in the epididymis. <em>Journal of Andrology</em>, <em>32</em>(6), 592–599. http://doi.org/<a href="https://doi.org/10.2164/jandrol.111.014266">10.2164/jandrol.111.014266</a>
</div>
<div class="csl-entry">
Robaire, B., Hinton, B., &amp; Orgebin-Crist, M.-C. (Eds.). (2002). <em>The epididymis: From molecules to clinical practice: From molecules to clinical practice : A comprehensive survey of the efferent ducts, the epididymis and the vas deferens</em> (illustrated). Springer Science &amp; Business Media.
</div>
<div class="csl-entry">
Robinson, M. D., McCarthy, D. J., &amp; Smyth, G. K. (2010). <span class="nocase">edgeR</span>: A bioconductor package for differential expression analysis of digital gene expression data. <em>Bioinformatics</em>, <em>26</em>(1), 139–140. http://doi.org/<a href="https://doi.org/10.1093/bioinformatics/btp616">10.1093/bioinformatics/btp616</a>
</div>
<div class="csl-entry">
Robinson, M. D., &amp; Oshlack, A. (2010). A scaling normalization method for differential expression analysis of <span>RNA</span>-seq data. <em>Genome Biology</em>, <em>11</em>(3), R25. http://doi.org/<a href="https://doi.org/10.1186/gb-2010-11-3-r25">10.1186/gb-2010-11-3-r25</a>
</div>
<div class="csl-entry">
Rozowsky, J., Kitchen, R. R., Park, J. J., Galeev, T. R., Diao, J., Warrell, J., … Gerstein, M. (2019). <span class="nocase">exceRpt</span>: A comprehensive analytic platform for extracellular <span>RNA</span> profiling. <em>Cell Systems</em>, <em>8</em>(4), 352–357.e3. http://doi.org/<a href="https://doi.org/10.1016/j.cels.2019.03.004">10.1016/j.cels.2019.03.004</a>
</div>
<div class="csl-entry">
Sellem, E., Marthey, S., Rau, A., Jouneau, L., Bonnet, A., Perrier, J.-P., … Schibler, L. (2020). A comprehensive overview of bull sperm-borne small non-coding <span>RNAs</span> and their diversity across breeds. <em>Epigenetics &amp; Chromatin</em>, <em>13</em>(1), 19. http://doi.org/<a href="https://doi.org/10.1186/s13072-020-00340-0">10.1186/s13072-020-00340-0</a>
</div>
<div class="csl-entry">
Shao, Y., Li, C., Xu, W., Zhang, P., Zhang, W., &amp; Zhao, X. (2017). <span class="nocase">miR</span>-31 links lipid metabolism and cell apoptosis in bacteria-challenged apostichopus japonicus via targeting <span>CTRP9</span>. <em>Frontiers in Immunology</em>, <em>8</em>, 263. http://doi.org/<a href="https://doi.org/10.3389/fimmu.2017.00263">10.3389/fimmu.2017.00263</a>
</div>
<div class="csl-entry">
Sharma, U., Conine, C. C., Shea, J. M., Boskovic, A., Derr, A. G., Bing, X. Y., … Rando, O. J. (2016). Biogenesis and function of <span class="nocase">tRNA</span> fragments during sperm maturation and fertilization in mammals. <em>Science</em>, <em>351</em>(6271), 391–396. http://doi.org/<a href="https://doi.org/10.1126/science.aad6780">10.1126/science.aad6780</a>
</div>
<div class="csl-entry">
Skerget, S., Rosenow, M. A., Petritis, K., &amp; Karr, T. L. (2015). Sperm proteome maturation in the mouse epididymis. <em>Plos One</em>, <em>10</em>(11), e0140650. http://doi.org/<a href="https://doi.org/10.1371/journal.pone.0140650">10.1371/journal.pone.0140650</a>
</div>
<div class="csl-entry">
Suganuma, R., Yanagimachi, R., &amp; Meistrich, M. L. (2005). Decline in fertility of mouse sperm with abnormal chromatin during epididymal passage as revealed by <span>ICSI</span>. <em>Human Reproduction</em>, <em>20</em>(11), 3101–3108. http://doi.org/<a href="https://doi.org/10.1093/humrep/dei169">10.1093/humrep/dei169</a>
</div>
<div class="csl-entry">
Sullivan, R. (2015). Epididymosomes: A heterogeneous population of microvesicles with multiple functions in sperm maturation and storage. <em>Asian Journal of Andrology</em>, <em>17</em>(5), 726–729. http://doi.org/<a href="https://doi.org/10.4103/1008-{682X}.155255">10.4103/1008-{682X}.155255</a>
</div>
<div class="csl-entry">
Tamessar, C. T., Trigg, N. A., Nixon, B., Skerrett-Byrne, D. A., Sharkey, D. J., Robertson, S. A., … Schjenken, J. E. (2021). Roles of male reproductive tract extracellular vesicles in reproduction. <em>American Journal of Reproductive Immunology</em>, <em>85</em>(2), e13338. http://doi.org/<a href="https://doi.org/10.1111/aji.13338">10.1111/aji.13338</a>
</div>
<div class="csl-entry">
Thimon, V., Frenette, G., Saez, F., Thabet, M., &amp; Sullivan, R. (2008). Protein composition of human epididymosomes collected during surgical vasectomy reversal: A proteomic and genomic approach. <em>Human Reproduction</em>, <em>23</em>(8), 1698–1707. http://doi.org/<a href="https://doi.org/10.1093/humrep/den181">10.1093/humrep/den181</a>
</div>
<div class="csl-entry">
Vlachos, I. S., &amp; Hatzigeorgiou, A. G. (2017). Functional analysis of <span class="nocase">miRNAs</span> using the <span>DIANA</span> tools online suite. <em>Methods in Molecular Biology</em>, <em>1517</em>, 25–50. http://doi.org/<a href="https://doi.org/10.1007/978-1-4939-6563-2\_2">10.1007/978-1-4939-6563-2\_2</a>
</div>
<div class="csl-entry">
Wickham, H. (2016). <em>ggplot2: Elegant graphics for data analysis (use r!)</em> (2nd ed., p. 276). Cham: Springer.
</div>
<div class="csl-entry">
Zhou, W., Stanger, S. J., Anderson, A. L., Bernstein, I. R., De Iuliis, G. N., McCluskey, A., … Nixon, B. (2019). Mechanisms of tethering and cargo transfer during epididymosome-sperm interactions. <em><span>BMC</span> Biology</em>, <em>17</em>(1), 35. http://doi.org/<a href="https://doi.org/10.1186/s12915-019-0653-5">10.1186/s12915-019-0653-5</a>
</div>
<div class="csl-entry">
Zhu, L. J., Hardy, M. P., Inigo, I. V., Huhtaniemi, I., Bardin, C. W., &amp; Moo-Young, A. J. (2000). Effects of androgen on androgen receptor expression in rat testicular and epididymal cells: A quantitative immunohistochemical study. <em>Biology of Reproduction</em>, <em>63</em>(2), 368–376. http://doi.org/<a href="https://doi.org/10.1095/biolreprod63.2.368">10.1095/biolreprod63.2.368</a>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="ac.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
