% This is the Reed College LaTeX thesis template. Most of the work
% for the document class was done by Sam Noble (SN), as well as this
% template. Later comments etc. by Ben Salzberg (BTS). Additional
% restructuring and APA support by Jess Youngberg (JY).
% Your comments and suggestions are more than welcome; please email
% them to cus@reed.edu
%
% See https://www.reed.edu/cis/help/LaTeX/index.html for help. There are a
% great bunch of help pages there, with notes on
% getting started, bibtex, etc. Go there and read it if you're not
% already familiar with LaTeX.
%
% Any line that starts with a percent symbol is a comment.
% They won't show up in the document, and are useful for notes
% to yourself and explaining commands.
% Commenting also removes a line from the document;
% very handy for troubleshooting problems. -BTS

% As far as I know, this follows the requirements laid out in
% the 2002-2003 Senior Handbook. Ask a librarian to check the
% document before binding. -SN

%%
%% Preamble
%%
% \documentclass{<something>} must begin each LaTeX document
\documentclass[12pt,twoside]{reedthesis}
% Packages are extensions to the basic LaTeX functions. Whatever you
% want to typeset, there is probably a package out there for it.
% Chemistry (chemtex), screenplays, you name it.
% Check out CTAN to see: https://www.ctan.org/
%%
\usepackage{graphicx,latexsym}
\usepackage{amsmath}
\usepackage{amssymb,amsthm}
\usepackage{longtable,booktabs,setspace}
\usepackage{chemarr} %% Useful for one reaction arrow, useless if you're not a chem major
\usepackage[hyphens]{url}
% Added by CII
\usepackage{hyperref}
\usepackage{lmodern}
\usepackage{float}
\floatplacement{figure}{H}
% Thanks, @Xyv
\usepackage{calc}
% End of CII addition
\usepackage{rotating}

% Next line commented out by CII
%%% \usepackage{natbib}
% Comment out the natbib line above and uncomment the following two lines to use the new
% biblatex-chicago style, for Chicago A. Also make some changes at the end where the
% bibliography is included.
%\usepackage{biblatex-chicago}
%\bibliography{thesis}


% Added by CII (Thanks, Hadley!)
% Use ref for internal links
\renewcommand{\hyperref}[2][???]{\autoref{#1}}
\def\chapterautorefname{Chapter}
\def\sectionautorefname{Section}
\def\subsectionautorefname{Subsection}
% End of CII addition

% Added by CII
\usepackage{caption}
\captionsetup{width=5in}
% End of CII addition

% \usepackage{times} % other fonts are available like times, bookman, charter, palatino

% Syntax highlighting #22

% To pass between YAML and LaTeX the dollar signs are added by CII
\title{Understanding the molecular mechanisms of germline-dependent epigenetic inheritance: Computational analysis of multi-omics data}
\author{Deepak Kumar Tanwar}
% The month and year that you submit your FINAL draft TO THE LIBRARY (May or December)
\date{}
\division{}
\advisor{}
\institution{ETH Zurich}
\degree{}
%If you have two advisors for some reason, you can use the following
% Uncommented out by CII
% End of CII addition

%%% Remember to use the correct department!
\department{}
% if you're writing a thesis in an interdisciplinary major,
% uncomment the line below and change the text as appropriate.
% check the Senior Handbook if unsure.
%\thedivisionof{The Established Interdisciplinary Committee for}
% if you want the approval page to say "Approved for the Committee",
% uncomment the next line
%\approvedforthe{Committee}

% Added by CII
%%% Copied from knitr
%% maxwidth is the original width if it's less than linewidth
%% otherwise use linewidth (to make sure the graphics do not exceed the margin)
\makeatletter
\def\maxwidth{ %
  \ifdim\Gin@nat@width>\linewidth
    \linewidth
  \else
    \Gin@nat@width
  \fi
}
\makeatother

% From {rticles}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
% for Pandoc 2.8 to 2.10.1
\newenvironment{cslreferences}%
  {}%
  {\par}
% For Pandoc 2.11+
% As noted by @mirh [2] is needed instead of [3] for 2.12
\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
 }%
 {}
\usepackage{calc} % for calculating minipage widths
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}

\renewcommand{\contentsname}{Table of Contents}
% End of CII addition

\setlength{\parskip}{0pt}

% Added by CII

\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\Acknowledgements{
I want to thank a few people.
}

\Dedication{

}

\Preface{

}

\Abstract{

}

	\usepackage{setspace}\onehalfspacing
 \usepackage{acronym}
 \usepackage{glossaries}
% End of CII addition
%%
%% End Preamble
%%
%
\begin{document}

% Everything below added by CII
  \maketitle

\frontmatter % this stuff will be roman-numbered
\pagestyle{empty} % this removes page numbers from the frontmatter
  \begin{acknowledgements}
    I want to thank a few people.
  \end{acknowledgements}

  \hypersetup{linkcolor=black}
  \setcounter{secnumdepth}{2}
  \setcounter{tocdepth}{2}
  \tableofcontents

  \listoftables

  \listoffigures



\mainmatter % here the regular arabic numbering starts
\pagestyle{fancyplain} % turns page numbering back on

\hypertarget{list-of-abbreviations}{%
\chapter*{List of Abbreviations}\label{list-of-abbreviations}}
\addcontentsline{toc}{chapter}{List of Abbreviations}
\begin{acronym}[MPC]
\acro{MPC}{model predictive control}
\acro{TLA}{Three Letter Acronym}
\end{acronym}
\hypertarget{abstract}{%
\chapter*{Abstract}\label{abstract}}
\addcontentsline{toc}{chapter}{Abstract}

\hypertarget{abstract-de}{%
\chapter*{Zusammenfassung}\label{abstract-de}}
\addcontentsline{toc}{chapter}{Zusammenfassung}

\hypertarget{intro}{%
\chapter*{Introduction}\label{intro}}
\addcontentsline{toc}{chapter}{Introduction}

\hypertarget{epigenetics}{%
\section{Epigenetics}\label{epigenetics}}

\hypertarget{germline-epigenetics}{%
\section{Germline epigenetics}\label{germline-epigenetics}}

\hypertarget{spermatogonial-stem-cells}{%
\subsection{Spermatogonial Stem Cells}\label{spermatogonial-stem-cells}}

\hypertarget{reprogramming}{%
\subsection{Reprogramming}\label{reprogramming}}

\hypertarget{epigenetic-inheritance}{%
\section{Epigenetic Inheritance}\label{epigenetic-inheritance}}

\hypertarget{transgenerational-epigenetic-inheritance}{%
\subsection{Transgenerational Epigenetic Inheritance}\label{transgenerational-epigenetic-inheritance}}

\hypertarget{models-of-epigenetic-inheritance}{%
\section{Models of Epigenetic Inheritance}\label{models-of-epigenetic-inheritance}}

\hypertarget{msus-mouse-model}{%
\section{MSUS mouse model}\label{msus-mouse-model}}

\hypertarget{vectors-of-tei}{%
\section{Vectors of TEI}\label{vectors-of-tei}}

\hypertarget{extracellular-vesicles}{%
\section{Extracellular vesicles}\label{extracellular-vesicles}}

\hypertarget{small-rnas}{%
\section{Small RNAs}\label{small-rnas}}

\hypertarget{challenges-for-analyzing-srna-seq-dataset}{%
\subsection{Challenges for analyzing sRNA-seq dataset}\label{challenges-for-analyzing-srna-seq-dataset}}

\hypertarget{aims}{%
\section{Aims}\label{aims}}

\hypertarget{thesis-overview}{%
\section{Thesis overview}\label{thesis-overview}}

\hypertarget{methods}{%
\chapter*{Methods}\label{methods}}
\addcontentsline{toc}{chapter}{Methods}

\hypertarget{version-controlled-data-analysis-using-git}{%
\section{Version controlled data analysis using git}\label{version-controlled-data-analysis-using-git}}

\hypertarget{data-analysis-directory-organization}{%
\section{Data analysis directory organization}\label{data-analysis-directory-organization}}

\hypertarget{pipelines-for-data-analysis}{%
\section{Pipelines for data analysis}\label{pipelines-for-data-analysis}}

\hypertarget{bulk-rna-seq}{%
\subsection{Bulk RNA-seq}\label{bulk-rna-seq}}

\hypertarget{atac-seq}{%
\subsection{ATAC-seq}\label{atac-seq}}

\hypertarget{chip-seq}{%
\subsection{ChIP-seq}\label{chip-seq}}

\hypertarget{wgbs-rrbs}{%
\subsection{WGBS \& RRBS}\label{wgbs-rrbs}}

\hypertarget{developmental}{%
\chapter{Dynamic chromatin accessibility in spermatogonial cells for transcriptional programmings from early postnatal to adult stages}\label{developmental}}

\newpage

\hypertarget{graphical-abstract}{%
\section{Graphical Abstract}\label{graphical-abstract}}

\newpage

\hypertarget{abstract-1}{%
\section{Abstract}\label{abstract-1}}

\newpage

\hypertarget{introduction}{%
\section{Introduction}\label{introduction}}

\newpage

\hypertarget{results}{%
\section{Results}\label{results}}

\hypertarget{enrichment-of-spermatogonial-cells-from-postnatal-and-adult-mouse-testis}{%
\subsection{Enrichment of spermatogonial cells from postnatal and adult mouse testis}\label{enrichment-of-spermatogonial-cells-from-postnatal-and-adult-mouse-testis}}

\hypertarget{chromatin-is-remodeled-in-spermatogonial-cells-during-development}{%
\subsection{Chromatin is remodeled in spermatogonial cells during development}\label{chromatin-is-remodeled-in-spermatogonial-cells-during-development}}

\hypertarget{differentially-accessible-chromatin-regions-associate-with-distinct-gene-expression-dynamics}{%
\subsection{Differentially accessible chromatin regions associate with distinct gene expression dynamics}\label{differentially-accessible-chromatin-regions-associate-with-distinct-gene-expression-dynamics}}

\hypertarget{differentially-accessible-chromatin-regions-associate-with-distinct-epigenetic-profiles}{%
\subsection{Differentially accessible chromatin regions associate with distinct epigenetic profiles}\label{differentially-accessible-chromatin-regions-associate-with-distinct-epigenetic-profiles}}

\hypertarget{accessibility-changes-at-open-chromatin-regions-are-markedcarry-by-binding-sites-for-distinct-families-of-transcription-factors}{%
\subsection{Accessibility changes at open chromatin regions are markedcarry by binding sites for distinct families of transcription factors}\label{accessibility-changes-at-open-chromatin-regions-are-markedcarry-by-binding-sites-for-distinct-families-of-transcription-factors}}

\hypertarget{chromatin-accessibility-at-transposable-elements-undergoes-significant-remodeling-in-the-transition-from-postnatal-to-adult-spermatogonia}{%
\subsection{Chromatin accessibility at transposable elements undergoes significant remodeling in the transition from postnatal to adult spermatogonia}\label{chromatin-accessibility-at-transposable-elements-undergoes-significant-remodeling-in-the-transition-from-postnatal-to-adult-spermatogonia}}

\newpage

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

\newpage

\hypertarget{methods-1}{%
\section{Methods}\label{methods-1}}

\hypertarget{mouse-husbandry}{%
\subsection{Mouse husbandry}\label{mouse-husbandry}}

\hypertarget{germ-cells-isolation}{%
\subsection{Germ cells isolation}\label{germ-cells-isolation}}

\hypertarget{spermatogonial-cells-enrichment-by-facs}{%
\subsection{Spermatogonial cells enrichment by FACS}\label{spermatogonial-cells-enrichment-by-facs}}

\hypertarget{immunocytochemistry}{%
\subsection{Immunocytochemistry}\label{immunocytochemistry}}

\hypertarget{rna-extraction-and-rna-seq-library-preparation-for-rna-seq}{%
\subsection{RNA extraction and RNA-seq library preparation for RNA-seq}\label{rna-extraction-and-rna-seq-library-preparation-for-rna-seq}}

\hypertarget{omni-atac-llibrary-preparation-for-omni-atac-and-sequencing}{%
\subsection{Omni-ATAC Llibrary preparation for Omni-ATAC and sequencing}\label{omni-atac-llibrary-preparation-for-omni-atac-and-sequencing}}

\hypertarget{rna-sequencing-rna-seq}{%
\subsection{RNA sequencing (RNA-seq)}\label{rna-sequencing-rna-seq}}

\hypertarget{assay-for-transposase-accessible-chromatin-using-sequening-omni-atac}{%
\subsection{Assay for Transposase-Accessible Chromatin using sequening (Omni-ATAC)}\label{assay-for-transposase-accessible-chromatin-using-sequening-omni-atac}}

\hypertarget{chromatin-iimmunnoprecipitation-sequencing-chip-seq}{%
\subsection{Chromatin iImmunnoprecipitation sequencing (ChIP-seq)}\label{chromatin-iimmunnoprecipitation-sequencing-chip-seq}}

\hypertarget{bisulfite-sequencing-bs}{%
\subsection{Bisulfite sequencing (BS)}\label{bisulfite-sequencing-bs}}

\hypertarget{high-throughput-sequencing-data-analysis}{%
\subsection{High-throughput sequencing data analysis}\label{high-throughput-sequencing-data-analysis}}

\hypertarget{figures}{%
\subsection{Figures}\label{figures}}

\newpage

\hypertarget{data-and-materials-availability}{%
\section{Data and materials availability}\label{data-and-materials-availability}}

\hypertarget{authors-contribution}{%
\section{Authors Contribution}\label{authors-contribution}}

\hypertarget{competing-interest}{%
\section{Competing interest}\label{competing-interest}}

\newpage

\hypertarget{acknowledgments}{%
\section{Acknowledgments}\label{acknowledgments}}

\hypertarget{funding}{%
\section{Funding}\label{funding}}

\newpage

\hypertarget{supplementary-figures}{%
\section{Supplementary Figures}\label{supplementary-figures}}

\hypertarget{figure-1}{%
\subsection{Figure 1}\label{figure-1}}

\hypertarget{figure-2}{%
\subsection{Figure 2}\label{figure-2}}

\hypertarget{figure-3}{%
\subsection{Figure 3}\label{figure-3}}

\hypertarget{figure-4}{%
\subsection{Figure 4}\label{figure-4}}

\hypertarget{figure-5}{%
\subsection{Figure 5}\label{figure-5}}

\hypertarget{figure-6}{%
\subsection{Figure 6}\label{figure-6}}

\newpage

\hypertarget{references}{%
\section{References}\label{references}}

\hypertarget{ev}{%
\chapter{Early life stress affects the miRNA cargo of epididymal extracellular vesicles in mouse}\label{ev}}

Anar Alshanbayeva\textsuperscript{1,2,3}, \textbf{Deepak K Tanwar}\textsuperscript{1,2,3}, Martin Roszkowski\textsuperscript{1,2,3}, Francesca Manuella\textsuperscript{1,2,3}, Isabelle M Mansuy\textsuperscript{1,2,3,\#}
\newline
\newline
\textsuperscript{1}Laboratory of Neuroepigenetics, Brain Research Institute at the Medical Faculty of the University of Zurich.
\newline
\textsuperscript{2}Institute for Neuroscience of the Department of Health Sciences and Technology, ETH Zurich, Zurich, Switzerland.
\newline
\textsuperscript{3}Zurich Neuroscience Center, ETH and University of Zurich, Zurich, Switzerland.
\newline
\newline
\textsuperscript{\#}Corresponding author
\newline
\newline
\textbf{Journal:} \emph{Biology of Reproduction}
\newline
DOI: \href{https://doi.org/10.1093/biolre/ioab156}{10.1093/biolre/ioab156}
\newline
\newline
\textbf{Contributions:} \emph{I performed data analysis with Anar Alshanbayeva, generated figures with Anar Alshanbayeva, helped Anar Alshanbayeva in writing the manuscript, and revised manuscript with Anar Alshanbayeva.}

\newpage

\hypertarget{abstract-2}{%
\section{Abstract}\label{abstract-2}}

Sperm RNA can be modified by environmental factors and has been implicated in communicating signals about changes in a father's environment to the offspring. The small RNA composition of sperm could be changed during its final stage of maturation in the epididymis by extracellular vesicles (EVs) released by epididymal cells. We studied the effect of exposure to stress in early postnatal life on the transcriptome of epididymal EVs using a mouse model of transgenerational transmission. We found that the small RNA signature of epididymal EVs, particularly miRNAs, is altered in adult males exposed to postnatal stress. In some cases, these miRNA changes correlate with differences in the expression of their target genes in sperm and zygotes generated from that sperm. These results suggest that stressful experiences in early life can have persistent biological effects on the male reproductive tract that may in part be responsible for the transmission of the effects of exposure to the offspring.

\hypertarget{summary-sentence}{%
\section{Summary sentence}\label{summary-sentence}}

miRNA cargo of extracellular vesicles in cauda epididymis is altered by paternal exposure to early life stress. This correlates with changes in the expression of target genes in sperm and in zygotes generated from that sperm.

\newpage

\hypertarget{graphical-abstract-1}{%
\section{Graphical Abstract}\label{graphical-abstract-1}}

\hypertarget{key-words}{%
\section{Key words}\label{key-words}}

epigenetics, epididymis, epididymosomes, early life stress, extracellular vesicles, miRNAs, sperm.

\newpage

\hypertarget{introduction-1}{%
\section{Introduction}\label{introduction-1}}

\newpage

\hypertarget{results-1}{%
\section{Results}\label{results-1}}

\hypertarget{isolation-of-cauda-epididymosomes-confirmed-by-several-methods}{%
\subsection{Isolation of cauda epididymosomes confirmed by several methods}\label{isolation-of-cauda-epididymosomes-confirmed-by-several-methods}}

\hypertarget{the-number-and-size-of-epididymosomes-in-adult-males-are-not-altered-by-postnatal-stress}{%
\subsection{The number and size of epididymosomes in adult males are not altered by postnatal stress}\label{the-number-and-size-of-epididymosomes-in-adult-males-are-not-altered-by-postnatal-stress}}

\hypertarget{mirnas-are-persistently-altered-by-postnatal-stress-in-cauda-epididymosomes}{%
\subsection{miRNAs are persistently altered by postnatal stress in cauda epididymosomes}\label{mirnas-are-persistently-altered-by-postnatal-stress-in-cauda-epididymosomes}}

\hypertarget{mrna-targets-of-mirnas-from-cauda-epididymosomes-are-altered-by-postnatal-stress-in-sperm-and-in-zygotes}{%
\subsection{mRNA targets of miRNAs from cauda epididymosomes are altered by postnatal stress in sperm and in zygotes}\label{mrna-targets-of-mirnas-from-cauda-epididymosomes-are-altered-by-postnatal-stress-in-sperm-and-in-zygotes}}

\newpage

\hypertarget{discussion-1}{%
\section{Discussion}\label{discussion-1}}

\newpage

\hypertarget{materials-and-methods}{%
\section{Materials and methods}\label{materials-and-methods}}

\hypertarget{animals}{%
\subsection{Animals}\label{animals}}

\hypertarget{msus}{%
\subsection{MSUS}\label{msus}}

\hypertarget{tissue-collection}{%
\subsection{Tissue collection}\label{tissue-collection}}

\hypertarget{electron-microscopy-images}{%
\subsection{Electron microscopy images}\label{electron-microscopy-images}}

\hypertarget{epididymosomes-isolation-by-ultracentrifugation}{%
\subsection{Epididymosomes isolation by ultracentrifugation}\label{epididymosomes-isolation-by-ultracentrifugation}}

\hypertarget{immunoblotting}{%
\subsection{Immunoblotting}\label{immunoblotting}}

\hypertarget{nanoparticle-tracking-analysis}{%
\subsection{Nanoparticle tracking analysis}\label{nanoparticle-tracking-analysis}}

\hypertarget{rna-isolation-and-epididymosomes-profiling}{%
\subsection{RNA isolation and epididymosomes profiling}\label{rna-isolation-and-epididymosomes-profiling}}

\hypertarget{preparation-and-sequencing-of-srna-seq-libraries-from-epididymosomes}{%
\subsection{Preparation and sequencing of sRNA-seq libraries from epididymosomes}\label{preparation-and-sequencing-of-srna-seq-libraries-from-epididymosomes}}

\hypertarget{rt-qpcr}{%
\subsection{RT-qPCR}\label{rt-qpcr}}

\hypertarget{cholesterol-measurements}{%
\subsection{Cholesterol measurements}\label{cholesterol-measurements}}

\hypertarget{bioinformatics-data-analysis}{%
\subsection{Bioinformatics data analysis}\label{bioinformatics-data-analysis}}

\hypertarget{data-availability}{%
\section{Data availability}\label{data-availability}}

The datasets collected for this study are available as follows:
- sRNA-seq dataset of cauda epididymosomes before and after sizeselection: NCBI GEO under accession number \href{https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175976}{GSE175976}.
- Codes for bioinformatics analysis of RNA-sequencing datasets and
all corresponding differential expression analyses: Github repository \url{https://github.com/mansuylab/alshanbayeva_et_al_2021\%5Bhttps://github.com/mansuylab/alshanbayeva_et_al_2021}{]}.
- Sperm and zygote sequencing datasets from previous publications
can be found in ArrayExpress database at EMBL-EBI (\url{www.ebi.ac.uk/arrayexpress}) with the accession number \href{https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5834/}{E-MTAB-5834} (sperm) and \href{https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6589/}{E-MTAB-6589} (zygotes).

\newpage

\hypertarget{authors-contributions}{%
\section{Authors' contributions}\label{authors-contributions}}

AA and IMM conceived and designed the study. FM and MR performed the MSUS breeding and collected tissue samples. AA and DKT performed data analysis and generated figures. AA wrote the manuscript with input from DKT and IMM. AA performed all
experiments for RNA sequencing and all molecular analyses. IMM supervised the project and raised funds.

\hypertarget{grant-support}{%
\section{Grant Support}\label{grant-support}}

The work was supported by Swiss National Science Foundation (31003A-135715), ETH grants (ETH-10 15-2 and ETH-17 13-2), the Slack-Gyr Foundation, the Escher Foundation. The Mansuy lab is funded by the University ZÃ¼rich, the Swiss Federal Institute of Technology, the Swiss National Science Foundation (31003A-135715), ETH grants (ETH-10 15-2 and ETH-17 13-2), the Slack-Gyr foundation, the Escher Foundation. Deepak K. Tanwar is supported by the Swiss Government Excellence Scholarship. Martin Roszkowski was funded by the ETH Zurich Fellowship (ETH-10 15-2).

\hypertarget{acknowledgements}{%
\section{Acknowledgements}\label{acknowledgements}}

We thank Pierre-Luc Germain for advice on data analysis and for generating cumulative distribution plots, Irina Lazar-Contes for help with MSUS breeding, Silvia Schelbert for work on the animal license, Emilio Yandez at Function Genomics Center Zurich (FGCZ) for advice on the sRNA sequencing, Alekhya Mazumkhar for help with nanoparticle-tracking analysis, Yvonne Zipfel for animal care, Zurich Integrative Rodent Physiology facility for performing cholesterol measurements. We also thank Eloise Kremer, Ali Jawaid, and Mea Holmes for their initial contributions to the project.

\textbf{Conflict of interest:} The authors declare no conflict of interest.

\newpage

\hypertarget{supplementary-figures-1}{%
\section{Supplementary Figures}\label{supplementary-figures-1}}

\hypertarget{figure-1-1}{%
\subsection{Figure 1}\label{figure-1-1}}

\newpage

\hypertarget{figure-2-1}{%
\subsection{Figure 2}\label{figure-2-1}}

\newpage

\hypertarget{figure-3-1}{%
\subsection{Figure 3}\label{figure-3-1}}

\newpage

\hypertarget{figure-4-1}{%
\subsection{Figure 4}\label{figure-4-1}}

\newpage

\hypertarget{figure-5-1}{%
\subsection{Figure 5}\label{figure-5-1}}

\newpage

\hypertarget{figure-6-1}{%
\subsection{Figure 6}\label{figure-6-1}}

\newpage

\hypertarget{supplementary-tables}{%
\section{Supplementary Tables}\label{supplementary-tables}}

\hypertarget{table-1}{%
\subsection{Table 1}\label{table-1}}

\newpage

\hypertarget{table-2}{%
\subsection{Table 2}\label{table-2}}

\newpage

\hypertarget{table-3}{%
\subsection{Table 3}\label{table-3}}

\newpage

\hypertarget{table-4}{%
\subsection{Table 4}\label{table-4}}

\newpage

\hypertarget{references-1}{%
\section{References}\label{references-1}}

\hypertarget{shortRNA}{%
\chapter{shortRNA}\label{shortRNA}}

\hypertarget{abstract-3}{%
\section{Abstract}\label{abstract-3}}

\hypertarget{introduction-2}{%
\section{Introduction}\label{introduction-2}}

\hypertarget{methods-2}{%
\section{Methods}\label{methods-2}}

\hypertarget{pipeline}{%
\subsection{Pipeline}\label{pipeline}}

\hypertarget{qc}{%
\subsection{QC}\label{qc}}

\hypertarget{annotation-preparation}{%
\subsection{Annotation preparation}\label{annotation-preparation}}

\hypertarget{alignment}{%
\subsection{Alignment}\label{alignment}}

\hypertarget{reads-assignment}{%
\subsection{Reads assignment}\label{reads-assignment}}

\hypertarget{assignment-rules}{%
\subsection{Assignment rules}\label{assignment-rules}}

\hypertarget{treesummarizedexperiment-object}{%
\subsection{TreeSummarizedExperiment object}\label{treesummarizedexperiment-object}}

\hypertarget{differential-analysis}{%
\subsection{Differential analysis}\label{differential-analysis}}

\hypertarget{results-2}{%
\section{Results}\label{results-2}}

\hypertarget{datasets-used-for-testing-the-pipeline}{%
\subsection{Datasets used for testing the pipeline}\label{datasets-used-for-testing-the-pipeline}}

\hypertarget{databases-included-for-analyzing-these-data}{%
\subsection{Databases included for analyzing these data}\label{databases-included-for-analyzing-these-data}}

\hypertarget{result-1}{%
\subsection{result 1}\label{result-1}}

\hypertarget{result-2}{%
\subsection{result 2}\label{result-2}}

\hypertarget{result-3}{%
\subsection{result 3}\label{result-3}}

\hypertarget{comparison-with-other-tools}{%
\subsection{Comparison with other tools}\label{comparison-with-other-tools}}

\hypertarget{discussion-outlook}{%
\section{Discussion \& Outlook}\label{discussion-outlook}}

\hypertarget{data-and-code-availibility}{%
\section{Data and code availibility}\label{data-and-code-availibility}}

\hypertarget{discussion}{%
\chapter*{Discussion}\label{discussion}}
\addcontentsline{toc}{chapter}{Discussion}

\hypertarget{conclusion}{%
\chapter*{Conclusion}\label{conclusion}}
\addcontentsline{toc}{chapter}{Conclusion}

\hypertarget{aa}{%
\chapter*{Appendix A}\label{aa}}
\addcontentsline{toc}{chapter}{Appendix A}

\hypertarget{datasets-analyzed}{%
\section{Datasets analyzed}\label{datasets-analyzed}}

\hypertarget{ab}{%
\chapter*{Appendix B}\label{ab}}
\addcontentsline{toc}{chapter}{Appendix B}

\hypertarget{other-manuscripts-during-phd}{%
\section{Other manuscripts during PhD}\label{other-manuscripts-during-phd}}

\hypertarget{ac}{%
\chapter*{Appendix C}\label{ac}}
\addcontentsline{toc}{chapter}{Appendix C}

\backmatter

\hypertarget{references-2}{%
\chapter*{References}\label{references-2}}
\addcontentsline{toc}{chapter}{References}

\markboth{References}{References}

\noindent

\setlength{\parindent}{-0.20in}

\hypertarget{refs}{}
\begin{CSLReferences}{1}{0}
\leavevmode\hypertarget{ref-angel2000}{}%
Angel, E. (2000). \emph{Interactive computer graphics : A top-down approach with OpenGL}. Boston, MA: Addison Wesley Longman.

\leavevmode\hypertarget{ref-angel2001}{}%
Angel, E. (2001a). \emph{Batch-file computer graphics : A bottom-up approach with QuickTime}. Boston, MA: Wesley Addison Longman.

\leavevmode\hypertarget{ref-angel2002a}{}%
Angel, E. (2001b). \emph{Test second book by angel}. Boston, MA: Wesley Addison Longman.

\end{CSLReferences}

% Index?

\end{document}
