% This is the Reed College LaTeX thesis template. Most of the work
% for the document class was done by Sam Noble (SN), as well as this
% template. Later comments etc. by Ben Salzberg (BTS). Additional
% restructuring and APA support by Jess Youngberg (JY).
% Your comments and suggestions are more than welcome; please email
% them to cus@reed.edu
%
% See https://www.reed.edu/cis/help/LaTeX/index.html for help. There are a
% great bunch of help pages there, with notes on
% getting started, bibtex, etc. Go there and read it if you're not
% already familiar with LaTeX.
%
% Any line that starts with a percent symbol is a comment.
% They won't show up in the document, and are useful for notes
% to yourself and explaining commands.
% Commenting also removes a line from the document;
% very handy for troubleshooting problems. -BTS

% As far as I know, this follows the requirements laid out in
% the 2002-2003 Senior Handbook. Ask a librarian to check the
% document before binding. -SN

%%
%% Preamble
%%
% \documentclass{<something>} must begin each LaTeX document
\documentclass[12pt,twoside]{reedthesis}
% Packages are extensions to the basic LaTeX functions. Whatever you
% want to typeset, there is probably a package out there for it.
% Chemistry (chemtex), screenplays, you name it.
% Check out CTAN to see: https://www.ctan.org/
%%
\usepackage{graphicx,latexsym}
\usepackage{amsmath}
\usepackage{amssymb,amsthm}
\usepackage{longtable,booktabs,setspace}
\usepackage{chemarr} %% Useful for one reaction arrow, useless if you're not a chem major
\usepackage[hyphens]{url}
% Added by CII
\usepackage{hyperref}
\usepackage{lmodern}
\usepackage{float}
\floatplacement{figure}{H}
% Thanks, @Xyv
\usepackage{calc}
% End of CII addition
\usepackage{rotating}

% Next line commented out by CII
%%% \usepackage{natbib}
% Comment out the natbib line above and uncomment the following two lines to use the new
% biblatex-chicago style, for Chicago A. Also make some changes at the end where the
% bibliography is included.
%\usepackage{biblatex-chicago}
%\bibliography{thesis}


% Added by CII (Thanks, Hadley!)
% Use ref for internal links
\renewcommand{\hyperref}[2][???]{\autoref{#1}}
\def\chapterautorefname{Chapter}
\def\sectionautorefname{Section}
\def\subsectionautorefname{Subsection}
% End of CII addition

% Added by CII
\usepackage{caption}
\captionsetup{width=5in}
% End of CII addition

% \usepackage{times} % other fonts are available like times, bookman, charter, palatino

% Syntax highlighting #22
  \usepackage{color}
  \usepackage{fancyvrb}
  \newcommand{\VerbBar}{|}
  \newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
  \DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
  % Add ',fontsize=\small' for more characters per line
  \usepackage{framed}
  \definecolor{shadecolor}{RGB}{248,248,248}
  \newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
  \newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
  \newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
  \newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
  \newcommand{\BuiltInTok}[1]{#1}
  \newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
  \newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
  \newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
  \newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
  \newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
  \newcommand{\ExtensionTok}[1]{#1}
  \newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
  \newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\ImportTok}[1]{#1}
  \newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
  \newcommand{\NormalTok}[1]{#1}
  \newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
  \newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
  \newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
  \newcommand{\RegionMarkerTok}[1]{#1}
  \newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}

% To pass between YAML and LaTeX the dollar signs are added by CII
\title{Computational analysis of multi-omics data to understand the molecular mechanisms of germline-dependent epigenetic inheritance}
\author{Deepak Kumar Tanwar}
% The month and year that you submit your FINAL draft TO THE LIBRARY (May or December)
\date{}
\division{}
\advisor{}
\institution{ETH Zurich}
\degree{}
%If you have two advisors for some reason, you can use the following
% Uncommented out by CII
% End of CII addition

%%% Remember to use the correct department!
\department{}
% if you're writing a thesis in an interdisciplinary major,
% uncomment the line below and change the text as appropriate.
% check the Senior Handbook if unsure.
%\thedivisionof{The Established Interdisciplinary Committee for}
% if you want the approval page to say "Approved for the Committee",
% uncomment the next line
%\approvedforthe{Committee}

% Added by CII
%%% Copied from knitr
%% maxwidth is the original width if it's less than linewidth
%% otherwise use linewidth (to make sure the graphics do not exceed the margin)
\makeatletter
\def\maxwidth{ %
  \ifdim\Gin@nat@width>\linewidth
    \linewidth
  \else
    \Gin@nat@width
  \fi
}
\makeatother

% From {rticles}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
% for Pandoc 2.8 to 2.10.1
\newenvironment{cslreferences}%
  {}%
  {\par}
% For Pandoc 2.11+
% As noted by @mirh [2] is needed instead of [3] for 2.12
\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
 }%
 {}
\usepackage{calc} % for calculating minipage widths
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}

\renewcommand{\contentsname}{Table of Contents}
% End of CII addition

\setlength{\parskip}{0pt}

% Added by CII

\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\Acknowledgements{
This doctoral thesis has been an accomplishment that would not have been possible, or anywhere near as enjoyable, without the help and support of a number of incredible individuals. Thank you for all that you have contributed to this thesis. \newline\newline I am eternally grateful to Prof.~Dr.~Isabelle Mansuy for allowing me to do Doctoral research in her lab. It is a pleasure to be a member of the Neuroepigenetics lab. Thank you to all the members of the Neuroepigenetics lab, past and present. The Neuroepigenetics group was always fun to work with and I learned a lot about experiments. Thank you so much! A special thanks to the present and past members of Y38 for motivating, and various scientific and non-scientific discussions. \newline\newline My sincere thanks to Prof.~Dr.~Mark Robinson and Prof.~Dr.~Tuncay Baubec for accepting to be my co-examiners and supporting me throughout my doctoral research. Thank you so much Mark for always being available and providing all the help in writing proposals, helping in supervising students, and discussing and providing solutions to the problems I was facing. \newline\newline I am indebted to Dr.~Pierre-Luc Germain for his expert supervision and all the support he has provided. I am grateful for all discussions and suggestions on both scientific and non-scientific issues. Thank you for being so nice and for all the support which was extremely helpful to reach this point. I have learnt a lot from you. \newline\newline I would also like to thank the Directors and members of the Brain Research Institute for creating such a stimulating environment for research. \newline\newline I am extremely grateful for my doctoral funding, the Swiss Government Excellence Scholarship. Thank you so much Sandra Zweifel for always being there. \newline\newline Looking back into the past, this thesis would not have happened without my bachelor's thesis supervisors Rainer König and Sandro Lindig, who got me interested in the field of bioinformatics data science. \newline\newline Finally, I am very grateful to my family for their continuous support and belief in me. Visiting you always helped me to relax and to not worry about work so much. Last of all, I want to thank my wife, Palni Kundra, for always cheering me up when I was down and for helping me with stupid issues.
}

\Dedication{

}

\Preface{

}

\Abstract{

}

	\usepackage{setspace}\onehalfspacing
 \usepackage{acronym}
 \usepackage{glossaries}
 \usepackage{biblatex}
 \usepackage{fancyhdr}
 \usepackage{graphics}
 \usepackage{float}
 \usepackage{lmodern}
 \usepackage{pdflscape}
 \usepackage{caption}
 \captionsetup{font=footnotesize}
 \usepackage[labelfont=bf]{caption}
 \usepackage{wrapfig}
 \usepackage{pdfpages}
 \usepackage{afterpage}
 \usepackage[section]{placeins}
	\usepackage{flafter}
 \usepackage{float}
 \usepackage{subfig}
 \usepackage{subfloat}
 \usepackage{booktabs}
 \usepackage{longtable}
 \usepackage{array}
 \usepackage{multirow}
 \usepackage{wrapfig}
 \usepackage{colortbl}
 \usepackage{pdflscape}
 \usepackage{tabu}
 \usepackage{threeparttable}
 \usepackage{threeparttablex}
 \usepackage[normalem]{ulem}
 \usepackage{makecell}
 \usepackage{xcolor}
% End of CII addition
%%
%% End Preamble
%%
%
\begin{document}

% Everything below added by CII
  \maketitle

\frontmatter % this stuff will be roman-numbered
\pagestyle{empty} % this removes page numbers from the frontmatter
  \begin{acknowledgements}
    This doctoral thesis has been an accomplishment that would not have been possible, or anywhere near as enjoyable, without the help and support of a number of incredible individuals. Thank you for all that you have contributed to this thesis. \newline\newline I am eternally grateful to Prof.~Dr.~Isabelle Mansuy for allowing me to do Doctoral research in her lab. It is a pleasure to be a member of the Neuroepigenetics lab. Thank you to all the members of the Neuroepigenetics lab, past and present. The Neuroepigenetics group was always fun to work with and I learned a lot about experiments. Thank you so much! A special thanks to the present and past members of Y38 for motivating, and various scientific and non-scientific discussions. \newline\newline My sincere thanks to Prof.~Dr.~Mark Robinson and Prof.~Dr.~Tuncay Baubec for accepting to be my co-examiners and supporting me throughout my doctoral research. Thank you so much Mark for always being available and providing all the help in writing proposals, helping in supervising students, and discussing and providing solutions to the problems I was facing. \newline\newline I am indebted to Dr.~Pierre-Luc Germain for his expert supervision and all the support he has provided. I am grateful for all discussions and suggestions on both scientific and non-scientific issues. Thank you for being so nice and for all the support which was extremely helpful to reach this point. I have learnt a lot from you. \newline\newline I would also like to thank the Directors and members of the Brain Research Institute for creating such a stimulating environment for research. \newline\newline I am extremely grateful for my doctoral funding, the Swiss Government Excellence Scholarship. Thank you so much Sandra Zweifel for always being there. \newline\newline Looking back into the past, this thesis would not have happened without my bachelor's thesis supervisors Rainer König and Sandro Lindig, who got me interested in the field of bioinformatics data science. \newline\newline Finally, I am very grateful to my family for their continuous support and belief in me. Visiting you always helped me to relax and to not worry about work so much. Last of all, I want to thank my wife, Palni Kundra, for always cheering me up when I was down and for helping me with stupid issues.
  \end{acknowledgements}

  \hypersetup{linkcolor=black}
  \setcounter{secnumdepth}{4}
  \setcounter{tocdepth}{4}
  \tableofcontents

  \listoftables

  \listoffigures



\mainmatter % here the regular arabic numbering starts
\pagestyle{fancyplain} % turns page numbering back on

\hypertarget{list-of-abbreviations}{%
\chapter*{List of Abbreviations}\label{list-of-abbreviations}}
\addcontentsline{toc}{chapter}{List of Abbreviations}
\begin{tabular}{ll}
\toprule
Abbreviation & Term\\
\midrule
5mc & 5-Methylcytosine\\
ATAC-seq & Assay for Transposase-Accessible Chromatin using sequencing\\
BS-seq & Bisulfite sequencing\\
ChIP-seq & Chromatin immunoprecipitation with sequencing\\
DARs & Differentially accessible regions\\
\addlinespace
DGEs & Differentially expressed genes\\
DNAme & DNA methylation\\
EV & Extracellular vescicles\\
GO & Gene ontology\\
GSEA & Gene set enrichment analysis\\
\addlinespace
HTS & High-throughput sequencing\\
miRNA & micro RNA\\
ncRNA & non-coding RNA\\
piRNA & Piwi-interacting RNA\\
PND & Postnatal day\\
\addlinespace
PNW & Postnatal week\\
PGCs & Primordial germ cells\\
qPCR & quantitative polymerase chain reaction\\
RRBS & Reduced representation bisulfite sequencing\\
RNA-seq & RNA sequencing\\
\addlinespace
sRNA-seq & Short RNA sequening\\
siRNA & Small interfering RNA\\
SSCs & Spermatogonial stem cells\\
TF & Transcription factor\\
TFBS & Transcription factor binding site\\
\addlinespace
MSUS & Unpredictable maternal separation combined with unpredictable maternal stress\\
VCS & Version control system\\
\bottomrule
\end{tabular}
\hypertarget{summary}{%
\chapter*{Summary}\label{summary}}
\addcontentsline{toc}{chapter}{Summary}

The environment has a strong influence on an organism's development, especially in the early stages. Environmental influences and life experiences can alter the phenotypic features of exposed individuals and their offspring in many animals. In mammals, traumatic stress is a type of environmental event that can change their behaviour, cognition, and physiological functions. Some of the impacts of severe stress can be passed down through generations, even if those generations have never been exposed to a traumatic stressor. Environmentally generated effects that involve non-genetic germline alterations can be an important route of trait transmission over generations. Even if the subsequent generations have never been exposed to similar stressful exposures, they may be impaired in similar functions. The host lab's unpredictable maternal separation combined with unpredictable maternal stress (MSUS) mouse model is perfect for learning more about the mechanisms behind epigenetic inheritance induced by environmental perturbations.

In this thesis, to study the mechanisms of epigenetic inheritance, multi-omics data analysis pipelines were built, \protect\hyperlink{methods}{Methods}, and a number of high-throughput sequencing datasets generated using the MSUS model were analyzed using these pipelines, \protect\hyperlink{aa}{Appendix A} and \protect\hyperlink{ab}{Appendix B}. Also, the datasets generated from spermatogonial cells were used to study the developmental trajectory across postnatal and adult stages. Further, the effect of early life trauma on epididymal extracellular vesicles transcriptome was investigated using short RNA sequencing data. At last, a tool was built to analyze short RNA sequencing data to address issues in the current tools and pipelines.

Spermatogonial cells are the postnatal initiators of spermatogenesis, and they undertake critical proliferative and differentiation tasks throughout one's life. They are the only type of stem cell capable of transmitting genetic information to an embryo. Open chromatin in spermatogonial cells undergoes significant remodelling throughout testes development. Throughout postnatal testes development, they have significantly different transcriptional pathways. We investigated the dynamics of chromatin accessibility and gene expression during the developmental stages of postnatal day 8 (PND8), PND15, and postnatal week 21 (PNW21) using multi-omics data collected from spermatogonial cells. By comparing chromatin accessibility changes between early postnatal and steady-state adult cells in the mouse testes, the age-dependent molecular phenotype of spermatogonial cells was identified. The differentially accessible regions were discovered to be relevant to developmental and metabolic processes and enriched for important transcription factors. Also, the accessibility of chromatin at transposable elements in spermatogonia has been examined. Specific transposable elements subtypes become less accessible as the adult testis matures, whereas specific LINE L1 subtypes become more accessible. The genome's olfactory receptor gene regions were discovered to be highly connected to several of the more accessible subtypes. In conclusion, these findings reveal that chromatin accessibility in spermatogonial cells changes from postnatal to adult stage, as well as interesting biological patterns when combined with expression and histone modifications data. Furthermore, we show how bioinformatics approaches may be used to integrate multi-omics datasets. These results are available in \protect\hyperlink{chapter1}{Chapter 1}.

If environmentally-triggered marks are not inherited from SSCs, another possible way in which sperm may carry them is through the uptake of extracellular vesicles, in particular those produced by epididymal cells. By using the MSUS model, we studied the impact of early postnatal stress on the transcriptome of epididymal extracellular vesicles. The short RNA profile of epididymal EVs, particularly miRNAs, were found to be altered in adult males exposed to postnatal stress. In certain cases, these miRNA changes have been related to differences in the expression of their target genes in sperm and zygotes produced from those sperm. To summarize, chronic stress in early postnatal life changes miRNAs in mature extracellular vesicles of the male reproductive tract, with implications for mature sperm and zygotes. These results are summarized in \protect\hyperlink{chapter2}{Chapter 2}.

While working on \protect\hyperlink{chapter1}{Chapter 1} and \protect\hyperlink{chapter2}{Chapter 2}, we discovered a lot of issues with tools available for analyzing short RNA sequencing data. Short RNAs are significant molecules that play an important role in the control of the genome. Short RNAs have been classified into several categories, including miRNA, tRNA and tRNA fragments, and piRNAs, all of which have a complex biosynthesis. As a result, an analysis framework is necessary that is both sufficiently specialized to reflect the specificities of different classes of short RNA and their biogenesis, as well as sufficiently generic and thorough to conduct global analyses. We created a user-friendly, highly customizable, and comprehensive \texttt{R} package, \texttt{shortRNA} to do a full end-to-end analysis of short RNA sequencing data. Our package has been specifically designed to account for changes in short RNA types and their biogenesis, and it may readily be modified to include further annotations. We also use a customised genome annotation with pseudo chromosomes to account for post-transcriptional modifications, as well as a flexible rule-based approach to allocate reads along a tree of hierarchically ordered attributes. This enables systematic querying, exploration, and differential expression analysis of short RNAs at several levels of granularity, ranging from single sequences to entire RNA classes. The shortRNA R package, which is platform-agnostic, was designed to carry out all of the analysis steps from within R. These results are available in \protect\hyperlink{chapter3}{Chapter 3}.

In summary, the work presented in this thesis investigated various mechanisms of epigenetic inheritance and developed several pipelines for data analysis from the MSUS model, investigated changes in spermatogonial cells across development, studied the impact of early life stress on the transcriptome of epididymal extracellular vesicles, and developed an \texttt{R} package for short RNA sequencing data analysis.

\hypertarget{chapter3}{%
\chapter{shortRNA}\label{chapter3}}

\begingroup\LARGE

\texttt{shortRNA}: a flexible framework for the analysis of short RNA sequencing data
\begingroup\Large
\normalsize

\textbf{Contributions:} \emph{I worked towards the tool development, data analysis, results interpretation. This has been done together with Pierre-Luc Germain.}

\hypertarget{abstract}{%
\section{Abstract}\label{abstract}}

Short RNA are important molecules that play key functional roles in the
regulation of the genome. Several classes of short RNAs, such as miRNA,
tRNA and tRNA fragments, and piRNAs have been characterized and have
complex biogenesis. Short RNA sequencing is becoming increasingly
relevant in the research of regulatory mechanisms in a wide range of
biological functions. From an analysis point of view, each type of RNA
has its own features, and hence, specialized methods have been developed
focused on particular types, which not only multiply the work needed for
a comprehensive analysis but potentially create misassignment problems.
In addition, methods are typically divided into genome-based methods
that do not deal adequately with post-transcriptional modifications, and
transcript-based methods that are blind to unannotated features.
Finally, there are several outstanding issues in the analysis of short
RNAs, which are critical in the analysis of some samples. There is,
therefore, a need for an analysis framework that is: sufficiently
tailored to consider the specificities of different classes of short RNA
and their biogenesis, and sufficiently general and exhaustive to perform
global analyses. We developed a user-friendly, highly flexible and
comprehensive \texttt{R} package for a thorough end-to-end analysis of short RNA
sequencing data (planned submission to Bioconductor). Our package is
appropriately adapted to take into account the differences between
different types of short RNA and their biogenesis and is seamlessly
expandable to include additional annotation. We use a customized genome
annotation with artificial chromosomes to account for
post-transcriptional modifications, and a flexible rule-based approach
to assign reads along a tree of hierarchically organized features. This
enables the systematic querying, exploration, and differential
expression analysis of short RNAs at various degrees of granularity,
from specific sequences to RNA classes. We also include various
normalization and experimental-bias correction methods.The \texttt{shortRNA} \texttt{R} package is developed to perform all the steps from within \texttt{R} and is
platform-independent.

\hypertarget{introduction-and-background}{%
\section{Introduction and background}\label{introduction-and-background}}

Previous decades have revealed a number of RNAs that are distinct from
the messenger RNAs. Short-RNAs are non-coding RNA molecules that are
fewer than 200 nucleotides long and play an important role in genome
regulation. Several types of short RNAs, of different lengths, have been
discovered including microRNA (miRNA; 18-24 nt); small interfering RNA
(siRNA; 21-27 nt); small nucleolar RNA (snoRNA; 60-170 nt); small
nuclear RNA (snRNA/ U-RNA; 10-300 nt); Small temporal RNA (stRNA; 18-24
nt); tRNA-derived small RNA (tsRNA), including tRNA halves (tsRNA; 28-36
nt) and tRNA fragments (tRFs; 14-22 nt); small rDNA-derived RNA (srRNA;
18-30 nt); repeat-associated small interfering RNA (rasiRNA; 24-27 nt);
and Piwi-interacting RNA {[}piRNA; 26-31 nt, present in animals in the
germline cells and also in the brain (\protect\hyperlink{ref-zuo2016}{Zuo, Wang, Tan, Chen, \& Luo, 2016}). Some short RNAs, for
instance, miRNA and siRNA, are known to suppress gene expression by
sequence-specific interactions with regulatory regions during
transcription, RNA processing and translation (\protect\hyperlink{ref-zhu2016}{Zhu et al., 2016}), and by forming
the core with RNA-induced silencing complex (RISC) (\protect\hyperlink{ref-pratt2009}{Pratt \& MacRae, 2009}). In
addition, short-RNAs have been related to cancer, Parkinson's disease,
Alzheimer's disease, and prion disease (\protect\hyperlink{ref-gong2005}{Gong, Liu, Liu, \& Liang, 2005}), and hence, are used
as biomarkers.

Several methods have been developed for the analyses of short RNA
sequencing data. The development of high-throughput sequencing (HTS)
techniques allows researchers to study short RNAs in diverse tissues or
cells. HTS not only allows for the quantification of known short RNAs
but also for the identification and quantification of novel short RNAs.
The small RNA sequencing (sRNA-Seq) bears challenges and biases that
researchers need to be informed of in order to properly analyze the
data. First, most of the methods concentrate on a specific type of short
RNA, such as \texttt{MINTmap} (\protect\hyperlink{ref-loher2017}{Loher, Telonis, \& Rigoutsos, 2017}) focus on analyzing tRFs, \texttt{TAM\ 2.0}
(\protect\hyperlink{ref-li2018}{J. Li et al., 2018}), \texttt{Prost!} (\protect\hyperlink{ref-desvignes2019}{Desvignes, Batzel, Sydes, Eames, \& Postlethwait, 2019}), \texttt{Chimira} (\protect\hyperlink{ref-vitsios2015}{Vitsios \& Enright, 2015}), and \texttt{mirTools}
2.0 (\protect\hyperlink{ref-wu2013}{J. Wu et al., 2013}) focus on miRNAs. Next, a few tools concentrate on most
types of short RNAs (not all), such as \texttt{ncPRO-seq} (\protect\hyperlink{ref-chen2012}{Chen et al., 2012}),
\texttt{sRNAtoolbox} (\protect\hyperlink{ref-rueda2015}{Rueda et al., 2015}), \texttt{SPAR} (\protect\hyperlink{ref-kuksa2018}{Kuksa et al., 2018}), \texttt{Threshold-seq} (\protect\hyperlink{ref-magee2017}{Magee, Loher, Londin, \& Rigoutsos, 2017}),
\texttt{sRNAnalyzer} (\protect\hyperlink{ref-wu2017}{X. Wu et al., 2017}), \texttt{UEA\ sRNA\ Workbench} (\protect\hyperlink{ref-stocks2012}{Stocks et al., 2012}), \texttt{Sports1}
(\protect\hyperlink{ref-shi2018}{Shi, Ko, Sanders, Chen, \& Zhou, 2018}), and \texttt{Oasis2} (\protect\hyperlink{ref-rahman2018}{Rahman et al., 2018}). Tools that concentrate on many
types of short RNAs, such as \texttt{Oasis2} and \texttt{Sports1}, perform sequential
mapping, for example, they perform sequential mapping by first assigning
the reads to miRNAs, then assigning the leftover to tRNAs, and then the
remaining ones to mRNAs, which can create a misassignment problem as
short reads can map to multiple locations. But this information is lost
because only unmapped reads are aligned against the next annotation. In
addition, althoughthe libraries for sequencing are tailored to short
RNAs, other long RNAs are also often detected in the sequencing data,
and also because short RNAs often overlap with other features,
prioritization between the biotypes is crucial. Further, there are quite
a high number of sequenced reads left that map to the reference genome
but are not assigned to the known features. Moreover, although most of
the tools do not consider assignment rules for reads assignment, few
tools such as Prost! have a set of defined rules for reads assignment.
But, the rules are only for the miRNAs. Furthermore, the multi-mapping
reads are either excluded or randomly assigned to the multi-mapping
positions in the genome. As well as, not all the tools deal with the
post-transcriptional modifications, such as the addition of ``CCA''
towards the 3' end of tRNAs (\protect\hyperlink{ref-barraud2019}{Barraud \& Tisné, 2019}; \protect\hyperlink{ref-hou2010}{Hou, 2010}; \protect\hyperlink{ref-ibba2000}{Ibba \& Soll, 2000}), which
is important for the recognition of tRNA by enzymes and translation
(\protect\hyperlink{ref-green1997}{Green \& Noller, 1997}; \protect\hyperlink{ref-sprinzl1979}{Sprinzl \& Cramer, 1979}); and the addition of ``G'' towards the 5' end
in histidine tRNAs (\protect\hyperlink{ref-cooley1982}{Cooley, Appel, \& Soll, 1982}; \protect\hyperlink{ref-cozen2015}{Cozen et al., 2015}), this is critical for
histidyl-tRNA synthetase (HisRS) recognition (\protect\hyperlink{ref-fromant2000}{Fromant, Plateau, \& Blanquet, 2000}), which is
responsible for the integration of histidine into proteins
(\protect\hyperlink{ref-freist1999}{Freist, Verhey, Rühlmann, Gauss, \& Arnez, 1999}). On top of these limitations, most of the tools are
written in different programming languages, are platform-dependent and
depend on external tools, which can create a barrier for installation
for the user; and the tools are either web-based, which restrict the
user for additional downstream analysis or command-line based, which
creates a restriction for the non-computational researchers. Besides,
the available tools use feature-based counting (number of reads, counts,
associated with each feature), except for \texttt{seqpac} (\protect\hyperlink{ref-skog2021}{Skog et al., 2021}), a recently
published tool, which uses sequence-based counting (number of reads,
counts, associated with each unique sequence). Sequence-based counting
would prevent the same sequence from being annotated multiple times
within and across samples, hence increasing the efficiency for alignment
and reads assignment. Also, it directly enables looking at specific
variations in sequences or their boundaries. However, due to this
complexity, it can be challenging to explore data at the level of
individual sequences, which calls for methods that can afford different
levels of granularity. Hence, specialized tools are required with
specialized pipelines to analyze sRNA-seq data.

We developed \texttt{shortRNA}, an R tool that addresses all of the issues listed
above for processing sRNA-seq data. Our tool is cross-platform (it may
run on any operating system) and built on the Bioconductor framework, in
particular making use of efficient data structures (e.g., \texttt{DataFrame},
\texttt{FactorList}, and \texttt{TreeSummarizedExperiment}) enabling interoperability with
other Bioconducto\texttt{R} packages. Users can use \texttt{shortRNA} to conduct a
thorough analysis of their data, from quality control (trimming, adapter
removal, UMI compressing) and alignment to quantification and downstream
analysis. \texttt{shortRNA} also has a set of customizable criteria for assigning
reads to various types of short RNAs and seamlessly enables querying
features at the level of individual fragments. We have tested our tool
using a published mouse dataset from sperm, simulated data from sperm,
and human data from peripheral blood mononuclear cells (PBMC;
unpublished).

\hypertarget{methods}{%
\section{Methods}\label{methods}}

\hypertarget{development-and-testing-environment}{%
\subsection{Development and testing environment}\label{development-and-testing-environment}}

The \texttt{shortRNA} package is developed and tested on the Linux (GNU/Linux
4.4.0-210-generic x86\_64) operating system, Ubuntu (version 16.04.7 LTS)
with 16 processors and 124 GB of RAM, using R 3.6.3 and R 4.0.5. As the
\texttt{shortRNA} tool is developed as an \texttt{R} package, it is platform-independent
(can easily be installed and run on other operating systems).

\hypertarget{backbone-data-structures-of-shortrna}{%
\subsection{\texorpdfstring{Backbone data structures of \texttt{shortRNA}}{Backbone data structures of shortRNA}}\label{backbone-data-structures-of-shortrna}}

The \texttt{shortRNA} package is developed around four main data structures from
Bioconductor: phylo, \texttt{FactorList}, \texttt{DataFrame}, and
\texttt{TreeSummarizedExperiment}. These four data structures store the data and
results in the most efficient manner and enable fast computation.

\hypertarget{phylo}{%
\subsubsection{phylo}\label{phylo}}

The phylogenetic tree is a branching diagram used to depict evolutionary
relationships. In R, the phylo class stores the phylogenetic
relationship. In \texttt{shortRNA}, we save the relationship between RNA
biotypes, features, and reads as a phylo object, section \protect\hyperlink{ft}{Features Tree}.

\hypertarget{factorlist}{%
\subsubsection{\texorpdfstring{\texttt{FactorList}}{FactorList}}\label{factorlist}}

In the case of a long vector of repeated characters, R takes more memory
to store them and also the computation could be quite slow. These
repeated character vectors could be stored as factors, where levels are
provided to each unique character in the vector
(\url{https://datascience.stackexchange.com/questions/12018/when-to-choose-character-instead-of-factor-in-r}).
However, when factors are stored in a list, the levels are traditionally
defined for each element of the list, which can be very inefficient
memory-wise. The \texttt{FactorList}, a class from the IRanges Bioconductor
package, instead stores lists of factors as a single factor vector with
added list membership information. In addition, like other AtomicList
objects, it enables list operations without iteration. This is helpful
for efficiently saving the object in R memory and for fast computations.

\hypertarget{dataframe}{%
\subsubsection{\texorpdfstring{\texttt{DataFrame}}{DataFrame}}\label{dataframe}}

Rectangular data can be stored as \texttt{data.frame} class object in R.
\texttt{DataFrame} functions from S4Vectors Bioconducto\texttt{R} package and behaves
similar to \texttt{data.frame} in terms of construction and subsetting. An
advantage of using \texttt{DataFrame} is that it can store any type of object in
a column, for example, a \texttt{FactorList}, or even another \texttt{DataFrame}, while
retaining all the methods and functionalities of traditional
\texttt{data.frame}s.

\hypertarget{treesummarizedexperiment}{%
\subsubsection{\texorpdfstring{\texttt{TreeSummarizedExperiment}}{TreeSummarizedExperiment}}\label{treesummarizedexperiment}}

\texttt{TreeSummarizedExperiment} is a Bioconducto\texttt{R} package extending the
classical \texttt{SummarizedExperiment} (\protect\hyperlink{ref-morgan2020}{Morgan, Obenchain, Hester, \& Pagès, 2020}) and with additional
hierarchy data and operations (\protect\hyperlink{ref-huang2020}{Huang et al., 2020}). The \texttt{SummarizedExperiment}
class holds rectangular matrices of experiment data, accompanied by row
and column annotation data. There are two classes in the
SummarizedExperiment package: \texttt{SummarizedExperiment} and
RangedSummarizedExperiment. Instead of a \texttt{DataFrame} of features,
RangedSummarizedExperiment objects represent genomic ranges of interest.
Figure \ref{fig:3f1}A depicts the structure of the \texttt{SummarizedExperiment}. The
rectangular data matrices are stored as assays, rowData stores
annotation for corresponding rows in the assays, colData stores
annotation for corresponding columns in the assays, ranges are described
by a GRanges or a GRangesList object, which are stored as rowRanges, and
metadata can be used to store additional data-related information.

In extension to the \texttt{SummarizedExperiment}, \texttt{TreeSummarizedExperiment} has
rowTree, which stores the hierarchical structure of rows of assays;
colTree, which stores hierarchical structure of columns of the assays;
rowLinks, which stores the link information between the rows of the
assays and rowTree; colLinks, which stores the link information between
the columns of the assays; and referenceSeq, which stores the reference
sequence for the features, depicted in Figure \ref{fig:3f1}B.



\hypertarget{pipeline}{%
\subsection{Pipeline}\label{pipeline}}

The \texttt{shortRNA} pipeline includes three major data analysis steps:
preprocessing of data, alignment and reads assignment with the
customisable assignment rules, and statistical analysis and
visualization. Figure \ref{fig:3f2} shows the outline of the pipeline. First, the
data goes through the preprocessing steps that consist of filtering and
alignment of unique sequenced reads. The raw data undergo quality
assessment and then the quality control steps are performed. The data is
then aligned to a custom genome containing, in addition to the reference
genome, extra pseudo-chromosomes for handling post-transcriptional
modifications. Aligned reads are then overlapped with the features
annotation and reads are assigned to the features using customisable
assignment rules, which is an important part of this pipeline, after
that, a features tree is formed. After the reads assignment, the data
goes through the statistical analysis steps that consist of
normalization and differential analysis. The quality controlled data are
normalized and then the test for differential expression of short RNAs
is performed in the user-defined biological groups.



\hypertarget{quality-control-and-trimming}{%
\subsubsection{Quality control and trimming}\label{quality-control-and-trimming}}

Quality control (QC) and trimming is a vital step in HTS data analysis.
To assess and control the quality of sRNA-Seq data, we adapted the
functionalities from two \texttt{R} packages: \texttt{Rfastp} (\protect\hyperlink{ref-wang2020}{Wang \& Carroll, 2020}) and seqTools
(\protect\hyperlink{ref-kaisers2020}{Kaisers, 2020}). \texttt{Rfastp} is used for trimming the low-quality reads and
adapter trimming. Further, with the summary files from \texttt{Rfastp}, the
functions in our tool could be used for making a table of comparison for
before and after QC, plots for duplicated reads, reads quality plot, and
base-content plots. We benchmarked Biostrings (\protect\hyperlink{ref-paguxe8s2020}{Pagès, Aboyoun, Gentleman, \& DebRoy, 2020}), qrqc
(\protect\hyperlink{ref-buffalo2020}{Buffalo, 2020}), ShortRead (\protect\hyperlink{ref-morgan2009}{Morgan et al., 2009}), and seqTools for reading and
storing FASTQ files quality data in \texttt{R}, using an 863 MB FASTQ file.
seqTools outperformed the other three tools in terms of reading and
storing the FASTQ file quality data, as shown in Figure \ref{fig:3f3}. Hence, we
used it for reading the FASTQ files quality data, before and after QC,
to make reads length distribution plots. To summarize, the QC functions
provide information about sequencing depth, reads quality, possible
adapter sequences, duplicated reads, and sequence length distribution,
both before and after QC. The interactive plots provide users with
greater insight into their data. Exemplary plots and tables from the QC
report of a sample are shown in Figure \ref{fig:3f4}.





Unique Molecular Identifiers (UMIs) are complex indices of 8-16
nucleotide lengths that are introduced to sequencing libraries before
PCR amplification steps. Researchers can use UMI to evaluate the
efficiency with which they collect input molecules, identify sampling
bias, and, most importantly, identify and compensate for the effects of
PCR amplification bias (\protect\hyperlink{ref-fu2018}{Fu, Wu, Beane, Zamore, \& Weng, 2018}). Preprocessing of data sequenced with
UMIs is required for reads deduplication and correction, as well as the
creation of consensus sequences from each UMI. In \texttt{shortRNA}, we have
adapted the \texttt{UMIc} tool for UMIs collapsing (\protect\hyperlink{ref-tsagiopoulou2021}{Tsagiopoulou et al., 2021}), which
takes into account the frequency and the Phred quality of nucleotides
and the distances between the UMIs and the actual sequences for
collapsing the sequences.

Schematic of quality checks and quality control is shown in Figure \ref{fig:3f5}.



\hypertarget{sequence-count-matrix}{%
\subsubsection{Sequence count matrix}\label{sequence-count-matrix}}

From the trimmed FASTQ files, we extract the unique reads sequences and
make a sample by sequence count matrix. This matrix represents the
counts of sequences in each sample, keeping only sequences occurring
more than once. This is done by the \texttt{fastq2SeqCountMatrix()} function from
\texttt{shortRNA}. Further, we export these sequences as a fasta file. An
overview of this step is shown in Figure \ref{fig:3f5}. The two main outputs from
this step are sequenceFasta, which would be used for alignment, and
countsMatrix.

\hypertarget{annotation-preparation}{%
\subsubsection{Annotation preparation}\label{annotation-preparation}}

Sequencing generates nucleotide sequences with unknown functions, while
sequence annotation provides descriptive information about sequenced DNA
sequences. Several databases are available for the annotation of the
sequences. In \texttt{shortRNA}, we used miRBase (\protect\hyperlink{ref-griffiths-jones2008}{Griffiths-Jones, Saini, van Dongen, \& Enright, 2008}) for
miRNAs; GtRNADb (\protect\hyperlink{ref-chan2016}{P. P. Chan \& Lowe, 2016}) for tRNAs; mitoRNADb (\protect\hyperlink{ref-juxfchling2009}{Jühling et al., 2009}) for
mitochondrial tRNAs; for mouse, we use piRNA precursors from (\protect\hyperlink{ref-li2013}{X. Z. Li et al., 2013}),
rRNAs from SILVA (\protect\hyperlink{ref-quast2013}{Quast et al., 2013}), and mRNAs and other biotypes from Ensembl
(\protect\hyperlink{ref-howe2021}{Howe et al., 2021}). It is possible for users to have additional databases of
their choice.

The miRBase database is a collection of miRNA sequences and annotations.
In miRBase, miRNAs are named as, for example, org-mir-20a. The first 3
letters signify the organism, ``20'' tell us that it was an
early-discovered family (20th family that was named), ``20a'' tell us that
there is possibly another related miRNA, for example, org-mir-20a. Here,
org-mir-20a is a precursor. The mature miRNA species may be derived from
both the 5' and 3' arms of the precursor duplex, and are called the
miRNA-5p and -3p species, respectively. In this example, it would be
org-mir20a-5p and org-mir20a-3p. All the miRNAs overlapping between
Ensembl and miRBase were removed from Ensembl and the remaining one from
Ensembl, not overlapping, were renamed to match the nomenclature of
miRBase. For example, Mir7679 is renamed to org-miR-7679. Further, if
the length of Ensembl miRNAs were more than 25bp, we labelled them as
precursors.

A miRNA cluster is a group of two or more miRNA hairpins that are
transcribed from miRNA genes that are physically adjacent, transcribed
in the same direction, and are not separated by a transcription unit or
a miRNA in the opposite orientation. There are mostly 2-3 mature miRNAs
in a miRNA cluster, but there is an existence of a bigger miRNA cluster,
which is found in humans on chromosome 13: miR-17 to miR-92, involved in
tumour formation, and development of heart lungs and immune systems
(\protect\hyperlink{ref-lai2013}{Lai \& Vera, 2013}). Researchers group the miRNAs to form a miRNA cluster by the
distance between them. For example, (\protect\hyperlink{ref-baumgart2017}{Baumgart et al., 2017}; \protect\hyperlink{ref-griffiths-jones2008}{Griffiths-Jones et al., 2008}) call a group of miRNA as clusters if miRNAs are within
10kb and (\protect\hyperlink{ref-yuan2009}{Yuan et al., 2009}) used a distance of 50kb. In \texttt{shortRNA}, we use a
distance of 10kb for clustering miRNAs as more than 40\% of
experimentally validated human miRNA cluster genes have been identified
within 10kb (\protect\hyperlink{ref-griffiths-jones2008}{Griffiths-Jones et al., 2008}; \protect\hyperlink{ref-lai2013}{Lai \& Vera, 2013}).

The genomic tRNA database (GtRNADb) is a database of tRNA gene
predictions created by tRNAscan-SE (\protect\hyperlink{ref-chan2021}{P. Chan, Lin, Mak, \& Lowe, 2021}) on whole or almost
complete genomes. In GtRNADb, tRNAs symbols consist of 5 parts,
separated by a ``-'', for example, tRNA-Ala-AGC-9-2. In this example, tRNA
(prefix) represents tRNA genes that are high scoring and not predicted
as pseudogenes. If it was a pseudogene, it would have been represented
as ``tRX''. At the second position, we have three-letter amino acids
(isotype), which stand for tRNA isotype. At the third position, we have
anticodon detected in the gene sequence. At the fourth position, there
is a unique ID (transcript ID) of a tRNA transcript or ``isodecoder'' with
a particular isotype and anticodon. The fifth position represents the
gene locus ID and for tRNA genes with multiple identical copies, this
gene locus ID represents the particular gene copy in the genome. For the
mitochondrial tRNAs, we obtained the sequences from mitotRNAdb
(\protect\hyperlink{ref-juxfchling2009}{Jühling et al., 2009}) using the tRNAdbImport (\protect\hyperlink{ref-gm2018}{GM, 2018}) \texttt{R} package. Further, we
removed all the duplicated sequences and renamed the mitochondrial tRNAs
as per the naming convention of GtRNADb, adding ``mt'' as a prefix. Mature
tRNAs receive a post-transcriptional addition of ``CCA'' (\protect\hyperlink{ref-barraud2019}{Barraud \& Tisné, 2019}),
hence, we added ``CCA'' to all the tRNA/ mt-tRNA sequences at the 3' end.
All histidine tRNAs of known sequence are one nucleotide longer at the
5' end than are other tRNA species(\protect\hyperlink{ref-cozen2015}{Cozen et al., 2015}),hence we added ``G'' at the 5' end of the sequences of Histidine tRNAs/ mt-tRNAs to account for this post-translational modification, as also
done in (\protect\hyperlink{ref-cozen2015}{Cozen et al., 2015}; \protect\hyperlink{ref-shi2018}{Shi et al., 2018}). Because of these post-transcriptional
modifications, we add the sequences of tRNA into the genome FASTA file
as pseudo chromosomes. For example:
\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{\textgreater{}pseudoChr\_tRNA{-}Thr{-}CGT{-}1{-}1}
\NormalTok{GGCGCGGTGGCCAAGTGGTAAGGCGTCGGTCTCGTAAACCGAAGATCACGGGTTCGAACCCCGTCCGTGCCTCCA}

\NormalTok{\textgreater{}pseudoChr\_mt\_tRNA{-}His{-}GTG{-}1{-}1}
\NormalTok{GGTGAATATAGTTTACAAAAAACATTAGACTGTGAATCTGACAACAGGAAATAAACCTCCTTGTTCACCCCA}
\end{Highlighting}
\end{Shaded}
All the tRNAs overlapping between Ensembl and GtRNADb were removed from
Ensembl and the remaining tRNA from Ensembl, not overlapping, were
labelled as pseudo tRNAs.

SILVA is an rRNA database, which provides extensive, quality-checked,
and regularly updated datasets of aligned short (16S/18S, SSU) and large
subunit (23S/28S, LSU) ribosomal RNA sequences. rRNAs are repeated
sequences and are masked in the genomes in the Ensembl database, and are
not listed in the features annotation. Hence, we add them as well as
pseudo chromosomes to the genome FASTA file, similar to tRNAs.

\texttt{getDB()}, Figure \ref{fig:3f6}, function retrieve the databases, account for the
post-translational modifications of tRNAs, alter mitochondrial tRNA
database, and create miRNA clusters and assign miRNAs and miRNA
precursors to miRNA clusters. The output from \texttt{getDB()} is then parsed to
the \texttt{prepareAnnotation()} function, which prepares features annotation as
\texttt{GRanges()}, save the genome and pseudo-genomes and index that for
alignment. Features annotation are then converted to \texttt{FactorList} using
\texttt{featuresAnnoToFL()}. The main three outputs from the annotation
preparation are features as \texttt{GRanges()} and \texttt{FactorList()}, and \texttt{indexed\ customGenome}.



\hypertarget{alignment}{%
\subsubsection{Alignment}\label{alignment}}

Alignment is necessary to know where in the genome reads belong. The
alignment of short RNA remains a persistent under-recognized problem. We
align the FASTA file of sequences with the custom genome prepared in the
annotation step. For alignment, we use the \texttt{Rsubread} \texttt{R} package
(\protect\hyperlink{ref-liao2019}{Liao, Smyth, \& Shi, 2019}) with the index of custom genome generated on the genome and
pseudo-chromosomes (\texttt{index\ =\ "customGenome"}), with the GTF from Ensembl
database obtained from \texttt{getDB()} of \texttt{shortRNA} (\texttt{GTF\ =\ exonsBy(db\$ensdb)}),
and allowing to report the maximal number of equally-best mapping
locations (\texttt{nBestLocations\ =\ 16}). Please refer to the schematic in Figure
\ref{fig:3f7}.



\hypertarget{reads-annotation-and-assignment}{%
\subsubsection{Reads annotation and assignment}\label{reads-annotation-and-assignment}}

After alignment, the aligned file (BAM file) is overlapped with the
features annotation (\texttt{featuresGR}) using the \texttt{overlapWithTx2()} function. It
is possible that a read overlaps with multiple features. Next, the
overlaps are parsed with the \texttt{assignReads()} function with the assignment
rules, \texttt{defaultAssignRules()}, for validating the overlap and finding
assignment of multi mapping reads. Please refer to the schematic in
Figure \ref{fig:3f7}.

\hypertarget{assignment-rules}{%
\subsubsection{Assignment rules}\label{assignment-rules}}

A sequence read can be mapped to multiple locations in the genome. It is
critical to ensure that the reads are properly mapped to the feature and
that they are assigned to one or more features. In the \texttt{shortRNA} package,
we defined a set of customizable rules for this purpose. The output of
the function \texttt{defaultAssignRules()} is described below:

We consider reads with at least 50\% overlap to the features to be valid
read assignments.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{overlapBy}
\NormalTok{[}\DecValTok{1}\NormalTok{] }\FloatTok{0.5}
\end{Highlighting}
\end{Shaded}
By default, we do not prioritize the assignment based on the size of the
overlap between reads and features (when a read overlaps multiple
features); however this option is available to users.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{prioritizeByOverlapSize}
\NormalTok{[}\DecValTok{1}\NormalTok{] }\ConstantTok{FALSE}
\end{Highlighting}
\end{Shaded}
We prioritize overlap in the same strand, but enable overlaps from the
opposite strand if there is no known feature on the same strand.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{sameStrand}
\NormalTok{[}\DecValTok{1}\NormalTok{] }\StringTok{"prioritize"}
\end{Highlighting}
\end{Shaded}
We give priority to known features in our assignments.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{prioritizeKnown}
\NormalTok{[}\DecValTok{1}\NormalTok{] }\ConstantTok{TRUE}
\end{Highlighting}
\end{Shaded}
Next come validation rules specific to RNA types. For example, in order
to be assigned to primary piRNAs, the read should be 26-32 nucleotides
long, with the first nucleotide being a T.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{typeValidation}\SpecialCharTok{$}\NormalTok{primary\_piRNA}
\SpecialCharTok{$}\NormalTok{typeValidation}\SpecialCharTok{$}\NormalTok{primary\_piRNA}\SpecialCharTok{$}\NormalTok{fun}
\ControlFlowTok{function}\NormalTok{(src, }\AttributeTok{allowRevComp=}\ConstantTok{FALSE}\NormalTok{, }\AttributeTok{length=}\DecValTok{26}\SpecialCharTok{:}\DecValTok{32}\NormalTok{)\{}
\NormalTok{  length }\OtherTok{\textless{}{-}} \FunctionTok{as.integer}\NormalTok{(length)}
\NormalTok{  valid }\OtherTok{\textless{}{-}}\NormalTok{ src}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{\textgreater{}=} \FunctionTok{min}\NormalTok{(length) }\SpecialCharTok{\&}\NormalTok{ src}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{\textless{}=} \FunctionTok{max}\NormalTok{(length)}
  \ControlFlowTok{if}\NormalTok{(}\FunctionTok{length}\NormalTok{(w }\OtherTok{\textless{}{-}} \FunctionTok{which}\NormalTok{(valid))}\SpecialCharTok{==}\DecValTok{0}\NormalTok{) }\FunctionTok{return}\NormalTok{(valid)}
\NormalTok{  seqs }\OtherTok{\textless{}{-}} \FunctionTok{as.character}\NormalTok{(src}\SpecialCharTok{$}\NormalTok{seq[w])}
\NormalTok{  valid[w] }\OtherTok{\textless{}{-}} \FunctionTok{sapply}\NormalTok{(}\FunctionTok{strsplit}\NormalTok{(seqs,}\StringTok{""}\NormalTok{),}\AttributeTok{FUN=}\ControlFlowTok{function}\NormalTok{(x)\{ x[[}\DecValTok{1}\NormalTok{]]}\SpecialCharTok{==}\StringTok{"T"}\NormalTok{ \})}
  \ControlFlowTok{if}\NormalTok{(allowRevComp)\{}
\NormalTok{    revcomp }\OtherTok{\textless{}{-}} \FunctionTok{as.character}\NormalTok{(}\FunctionTok{reverseComplement}\NormalTok{(}\FunctionTok{DNAStringSet}\NormalTok{(seqs)))}
\NormalTok{    valid[w] }\OtherTok{\textless{}{-}}\NormalTok{ valid[w] }\SpecialCharTok{|}
      \FunctionTok{sapply}\NormalTok{(}\FunctionTok{strsplit}\NormalTok{(seqs,}\StringTok{""}\NormalTok{),}\AttributeTok{FUN=}\ControlFlowTok{function}\NormalTok{(x)\{ x[[}\DecValTok{1}\NormalTok{]]}\SpecialCharTok{==}\StringTok{"T"}\NormalTok{ \})}
\NormalTok{  \}}
\NormalTok{  valid}
\NormalTok{\}}
\end{Highlighting}
\end{Shaded}
If the read cannot be identified as a primary piRNA, it is referred to
as a piRNA precursor.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{typeValidation}\SpecialCharTok{$}\NormalTok{primary\_piRNA}\SpecialCharTok{$}\NormalTok{fallback}
\NormalTok{[}\DecValTok{1}\NormalTok{] }\StringTok{"piRNA\_precursor"}
\end{Highlighting}
\end{Shaded}
Secondary piRNA sequences should be 26 to 32 nucleotides long, with A in
the 10th position (\protect\hyperlink{ref-brennecke2007}{Brennecke et al., 2007}).
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{typeValidation}\SpecialCharTok{$}\NormalTok{secondary\_piRNA}
\SpecialCharTok{$}\NormalTok{typeValidation}\SpecialCharTok{$}\NormalTok{secondary\_piRNA}\SpecialCharTok{$}\NormalTok{fun}
\ControlFlowTok{function}\NormalTok{(src, }\AttributeTok{allowRevComp=}\ConstantTok{FALSE}\NormalTok{, }\AttributeTok{length=}\DecValTok{26}\SpecialCharTok{:}\DecValTok{32}\NormalTok{)\{}
\NormalTok{  length }\OtherTok{\textless{}{-}} \FunctionTok{as.integer}\NormalTok{(length)}
\NormalTok{  valid }\OtherTok{\textless{}{-}}\NormalTok{ src}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{\textgreater{}=} \FunctionTok{min}\NormalTok{(length) }\SpecialCharTok{\&}\NormalTok{ src}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{\textless{}=} \FunctionTok{max}\NormalTok{(length)}
  \ControlFlowTok{if}\NormalTok{(}\FunctionTok{length}\NormalTok{(w }\OtherTok{\textless{}{-}} \FunctionTok{which}\NormalTok{(valid))}\SpecialCharTok{==}\DecValTok{0}\NormalTok{) }\FunctionTok{return}\NormalTok{(valid)}
\NormalTok{  seqs }\OtherTok{\textless{}{-}} \FunctionTok{as.character}\NormalTok{(src}\SpecialCharTok{$}\NormalTok{seq[w])}
\NormalTok{  valid[w] }\OtherTok{\textless{}{-}} \FunctionTok{sapply}\NormalTok{(}\FunctionTok{strsplit}\NormalTok{(seqs,}\StringTok{""}\NormalTok{),}\AttributeTok{FUN=}\ControlFlowTok{function}\NormalTok{(x)\{ x[[}\DecValTok{10}\NormalTok{]]}\SpecialCharTok{==}\StringTok{"A"}\NormalTok{ \})}
  \ControlFlowTok{if}\NormalTok{(allowRevComp)\{}
\NormalTok{    revcomp }\OtherTok{\textless{}{-}} \FunctionTok{as.character}\NormalTok{(}\FunctionTok{reverseComplement}\NormalTok{(}\FunctionTok{DNAStringSet}\NormalTok{(seqs)))}
\NormalTok{    valid[w] }\OtherTok{\textless{}{-}}\NormalTok{ valid[w] }\SpecialCharTok{|}
      \FunctionTok{sapply}\NormalTok{(}\FunctionTok{strsplit}\NormalTok{(seqs,}\StringTok{""}\NormalTok{),}\AttributeTok{FUN=}\ControlFlowTok{function}\NormalTok{(x)\{ x[[}\DecValTok{10}\NormalTok{]]}\SpecialCharTok{==}\StringTok{"A"}\NormalTok{ \})}
\NormalTok{  \}}
\NormalTok{  valid}
\NormalTok{\}}
\end{Highlighting}
\end{Shaded}
If a read cannot be identified as secondary piRNA, it is referred to as
a piRNA precursor.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{typeValidation}\SpecialCharTok{$}\NormalTok{secondary\_piRNA}\SpecialCharTok{$}\NormalTok{fallback}
\NormalTok{[}\DecValTok{1}\NormalTok{] }\StringTok{"piRNA\_precursor"}
\end{Highlighting}
\end{Shaded}
For miRNAs, the length should be between 19 and 24 nucleotides, and the
read should overlap the feature by at least 16 bp. The maximum number of
non-overlapping nucleotides allowed is three.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{typeValidation}\SpecialCharTok{$}\NormalTok{miRNA}
\SpecialCharTok{$}\NormalTok{typeValidation}\SpecialCharTok{$}\NormalTok{miRNA}\SpecialCharTok{$}\NormalTok{fun}
\ControlFlowTok{function}\NormalTok{ (src, }\AttributeTok{length =} \DecValTok{19}\SpecialCharTok{:}\DecValTok{24}\NormalTok{, }\AttributeTok{minOverlap =}\NormalTok{ 16L, }\AttributeTok{maxNonOverlap =}\NormalTok{ 3L)}
\NormalTok{\{}
\NormalTok{    src}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{\%in\%}\NormalTok{ length }\SpecialCharTok{\&}\NormalTok{ src}\SpecialCharTok{$}\NormalTok{overlap }\SpecialCharTok{\textgreater{}=}\NormalTok{ minOverlap }\SpecialCharTok{\&}\NormalTok{ (src}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{{-}}
\NormalTok{        src}\SpecialCharTok{$}\NormalTok{overlap) }\SpecialCharTok{\textless{}=}\NormalTok{ maxNonOverlap}
\NormalTok{\}}

\SpecialCharTok{$}\NormalTok{typeValidation}\SpecialCharTok{$}\NormalTok{miRNA}\SpecialCharTok{$}\NormalTok{length}
\NormalTok{[}\DecValTok{1}\NormalTok{] }\DecValTok{19} \DecValTok{20} \DecValTok{21} \DecValTok{22} \DecValTok{23} \DecValTok{24}
\end{Highlighting}
\end{Shaded}
If a read cannot be assigned to a mature miRNA, it is assigned to a
miRNA precursor.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{typeValidation}\SpecialCharTok{$}\NormalTok{miRNA}\SpecialCharTok{$}\NormalTok{fallback}
\NormalTok{[}\DecValTok{1}\NormalTok{] }\StringTok{"miRNA\_precursor"}
\end{Highlighting}
\end{Shaded}
Reads that are assigned to tRNAs (or pseudo tRNAs) undergo additional
classification. tRNA 5p fragments are less than 30 bp long and begin in
the feature's 5bp. tRNA 3p fragments are less than 50 bp long, with a
distance to the feature end of 5bp. Starts at +/= 1bp of the feature and
has a length of 30 to 34 nucleotides for 5p half. The 3p half begins at
+/= 1bp of the feature, has a length of 34 to 50 nucleotides, and ends
with CCA.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{reclassify}
\SpecialCharTok{$}\NormalTok{reclassify}\SpecialCharTok{$}\NormalTok{tRNA}
\ControlFlowTok{function}\NormalTok{(srcs, }\AttributeTok{rules=}\FunctionTok{list}\NormalTok{(}
  \StringTok{"tRNA\_internal\_fragment"}\OtherTok{=}\ControlFlowTok{function}\NormalTok{(x)\{ }\FunctionTok{rep}\NormalTok{(}\ConstantTok{TRUE}\NormalTok{, }\FunctionTok{nrow}\NormalTok{(x)) \},}
  \StringTok{"tRNA\_5p\_fragment"}\OtherTok{=}\ControlFlowTok{function}\NormalTok{(x)\{ x}\SpecialCharTok{$}\NormalTok{startInFeature }\SpecialCharTok{\textless{}}\NormalTok{ 5L }\SpecialCharTok{\&}\NormalTok{ x}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{\textless{}}\NormalTok{ 30L \},}
  \StringTok{"tRNA\_3p\_fragment"}\OtherTok{=}\ControlFlowTok{function}\NormalTok{(x)\{ x}\SpecialCharTok{$}\NormalTok{distanceToFeatureEnd }\SpecialCharTok{\textless{}}\NormalTok{ 5L }\SpecialCharTok{\&}\NormalTok{ x}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{\textless{}}\NormalTok{ 50L \},}
  \StringTok{"tRNA\_5p\_half"}\OtherTok{=}\ControlFlowTok{function}\NormalTok{(x)\{ x}\SpecialCharTok{$}\NormalTok{startInFeature }\SpecialCharTok{\%in\%} \SpecialCharTok{{-}}\DecValTok{1}\SpecialCharTok{:}\DecValTok{1} \SpecialCharTok{\&}\NormalTok{ x}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{\%in\%} \DecValTok{30}\SpecialCharTok{:}\DecValTok{34}\NormalTok{ \},}
  \StringTok{"tRNA\_3p\_half"}\OtherTok{=}\ControlFlowTok{function}\NormalTok{(x)\{ x}\SpecialCharTok{$}\NormalTok{distanceToFeatureEnd }\SpecialCharTok{\%in\%} \SpecialCharTok{{-}}\DecValTok{1}\SpecialCharTok{:}\DecValTok{1} \SpecialCharTok{\&}\NormalTok{ x}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{\textgreater{}=}\NormalTok{ 34L }\SpecialCharTok{\&}
\NormalTok{      x}\SpecialCharTok{$}\NormalTok{length }\SpecialCharTok{\textless{}=}\NormalTok{ 50L }\SpecialCharTok{\&} \FunctionTok{grepl}\NormalTok{(}\StringTok{"CCA$"}\NormalTok{,x}\SpecialCharTok{$}\NormalTok{seq) \}}
\NormalTok{))\{}
\NormalTok{  valids }\OtherTok{\textless{}{-}} \FunctionTok{vapply}\NormalTok{(rules, }\AttributeTok{FUN.VALUE=}\FunctionTok{logical}\NormalTok{(}\FunctionTok{nrow}\NormalTok{(srcs)), }\AttributeTok{FUN=}\ControlFlowTok{function}\NormalTok{(fn)\{}
    \FunctionTok{fn}\NormalTok{(srcs)}
\NormalTok{  \})}
  \ControlFlowTok{if}\NormalTok{(}\FunctionTok{nrow}\NormalTok{(srcs) }\SpecialCharTok{==} \DecValTok{1}\NormalTok{) valids }\OtherTok{\textless{}{-}} \FunctionTok{t}\NormalTok{(valids)}
  \FunctionTok{factor}\NormalTok{(}\FunctionTok{apply}\NormalTok{(valids, }\DecValTok{1}\NormalTok{, }\AttributeTok{FUN=}\ControlFlowTok{function}\NormalTok{(x) }\FunctionTok{max}\NormalTok{(}\FunctionTok{which}\NormalTok{(x))),}
         \FunctionTok{seq\_len}\NormalTok{(}\FunctionTok{ncol}\NormalTok{(valids)), }\FunctionTok{colnames}\NormalTok{(valids))}
\NormalTok{\}}
\end{Highlighting}
\end{Shaded}
We also assign priorities to different types of RNA. Priorities can be
changed by increasing or decreasing the numbers; for example, in the
case of only miRNA sequencing, users can set higher priorities for
miRNAs by changing the number from 1 to 2 or higher.
\begin{Shaded}
\begin{Highlighting}[]
\SpecialCharTok{$}\NormalTok{priorities}
\NormalTok{miRNA           tRNA           tRNAp         Mt\_tRNA          snRNA}
  \DecValTok{1}               \DecValTok{1}              \DecValTok{1}             \DecValTok{1}                \DecValTok{1}
\NormalTok{snoRNA       antisense      primary\_piRNA      secondary\_piRNA        }
  \DecValTok{1}               \DecValTok{1}              \DecValTok{1}                   \DecValTok{1}              
\NormalTok{precursor     long\_RNA         longRNA}
  \SpecialCharTok{{-}}\DecValTok{1}             \SpecialCharTok{{-}}\DecValTok{1}              \SpecialCharTok{{-}}\DecValTok{1}
\end{Highlighting}
\end{Shaded}
\hypertarget{ft}{%
\subsubsection{Features tree}\label{ft}}

We organized the features and reads in the form of a rooted phylogenetic
tree in a hierarchical fashion. This is done by the function
\texttt{assignReadsToTree()} in \texttt{shortRNA}, Figure \ref{fig:3f12}. From the root, we have two
main branches: \texttt{shortRNA} and long RNA, Figure \ref{fig:3f8}C. We keep the features
that are longer than 200 bp in long RNA and the features that are
shorter than 200 bp in short RNA. Each branch is then further divided
into RNA types. The miRNAs and tRNAs are further divided into
sub-branches to account for the multi-mapping problems, Figure \ref{fig:3f9} and
Figure \ref{fig:3f10}. Sequence quality and a follow up trimming, may result in one
nucleotide shorter sequence. Because of the hierarchical organization,
unique reads mapping to a feature are kept in the hierarchy, Figure \ref{fig:3f8}.
If the user would like to include an additional database for short RNAs,
an additional branch can be added to the tree while preparing
annotation, \texttt{prepareAnnotation()}.







\hypertarget{reads-assignment-ambiguity}{%
\subsubsection{Reads assignment ambiguity}\label{reads-assignment-ambiguity}}

Reads can align to multiple locations in the genome, and when this is
the case, typical workflows will either randomly align to one location
or not report any alignment. We allow for multi-mapping during
alignment, and address ambiguities when we assign reads to features.
Specifically, we assign it to the parent of all the features to which
the read maps. Figure \ref{fig:3f11}A depicts a read that can be mapped to multiple
tRNAs, Figure \ref{fig:3f11}B. While assigning reads to the features tree, we assign
this read as an ambiguous read to the parent of all the features it is
mapping to; in this case, tRNA-Leu-CAG, Figure \ref{fig:3f11}C.



\hypertarget{construction-of-treesummarizedexperiment-object}{%
\subsubsection{\texorpdfstring{Construction of \texttt{TreeSummarizedExperiment} object}{Construction of TreeSummarizedExperiment object}}\label{construction-of-treesummarizedexperiment-object}}

The feature tree and the assigned reads \texttt{DataFrame} is then used for the
construction of the \texttt{TreeSummarizedExperiment} object, which is then used
for all the downstream analysis, including differential analysis. This
is depicted in Figure \ref{fig:3f12}.



\hypertarget{differential-analysis}{%
\subsubsection{Differential analysis}\label{differential-analysis}}

The hierarchical structure of the features implies that differential
expression analysis could be performed at different levels of the
hierarchy, which would however increase the multiple testing problem.
We, therefore, rely on the method for dynamic testing of hierarchical
hypotheses implemented in \texttt{treeclimbR} (\protect\hyperlink{ref-huang2021}{Huang et al., 2021}) for differential
analysis of features.

Following normalization and differential analysis, users can use the
\texttt{TreeHeatmap}, \texttt{ggtree}, and \texttt{castro} \texttt{R} packages for exploratory data analysis
and figure creation. The \texttt{shortRNA} \texttt{makeTracks()} function can be used to
create genomic tracks, as shown in Figures \ref{fig:3f15}, \ref{fig:3f18a}, and \ref{fig:3f18b}. We intend to
expand ou\texttt{R} package by writing wrappers for these packages in order to
generate plots from within \texttt{shortRNA}.

\hypertarget{table-of-important-functions}{%
\subsubsection{Table of important functions}\label{table-of-important-functions}}

Table \ref{tab:sht1} shows some of the most important functions of the \texttt{shortRNA}
package. In the table, the functions are organized in the chronological
order of data analysis.

\hypertarget{results}{%
\section{Results}\label{results}}

\hypertarget{datasets-used-for-testing-the-pipeline}{%
\subsection{Datasets used for testing the pipeline}\label{datasets-used-for-testing-the-pipeline}}

We used a previously published sperm sRNA-seq dataset from (\protect\hyperlink{ref-gapp2021}{Gapp et al., 2021})
(GEO accession: GSE162112), in which we additionally spiked simulated
reads. We also used an unpublished PBMC human dataset from the Schratt
Lab (\url{https://schrattlab.ethz.ch/}) at the ETH Zurich, on which qPCR
were additionally performed for 5 miRNAs (miR-30b-5p, miR-30e-5p,
miR-30d-5p, miR-499-5p, and miR-1248). We used these human data to
correlate quantifications from \texttt{shortRNA}, \texttt{seqpac}, \texttt{Sports1}, and \texttt{Oasis2}
with delta Ct values from qPCR. We analyzed both of these datasets with
\texttt{shortRNA}, \texttt{seqpac}, \texttt{Sports1}, and \texttt{Oasis2} tools to compare the
quantification of miRNAs. Further, we used the simulated reads from
miRNAs to test the reads assignment and to assess differential analysis.

For both datasets, we used miRBase (miRNA), GtRNADB (tRNA), MttRNADb
(mitochondrial tRNA), SiLVA (rRNA), and Ensemble. For the sperm dataset
from the mouse, we also used piRNA precursors.

\hypertarget{comparison-of-quantification-and-identification-of-mirnas}{%
\subsection{Comparison of quantification and identification of miRNAs}\label{comparison-of-quantification-and-identification-of-mirnas}}

\hypertarget{shortrna-quantification-is-positively-correlated-with-oasis2-and-sports1}{%
\subsubsection{\texorpdfstring{\texttt{shortRNA} quantification is positively correlated with \texttt{Oasis2} and \texttt{Sports1}}{shortRNA quantification is positively correlated with Oasis2 and Sports1}}\label{shortrna-quantification-is-positively-correlated-with-oasis2-and-sports1}}

We used \texttt{Sports1}, \texttt{seqpac}, \texttt{Oasis2}, and \texttt{shortRNA} with the default
parameters to analyze the sperm dataset. Following the data analysis, we
limited our assessment to miRNAs for comparison across tools. We found
1970 miRNAs using \texttt{Oasis2}, 1137 using \texttt{Sports1}, 1024 using \texttt{seqpac}, and 503
using \texttt{shortRNA}. We began by examining the quantification with all four
tools. Figure \ref{fig:3f13} shows a comparison of the quantification of miRNAs from
\texttt{Sports1}, \texttt{Oasis2}, and \texttt{seqpac} with \texttt{shortRNA}. We discovered that the
quantifications of \texttt{shortRNA} and \texttt{Oasis2}, as well as \texttt{shortRNA} and \texttt{Sports1},
are highly correlated. In contrast, the correlation of quantification
between \texttt{shortRNA} and \texttt{seqpac} was negative. We investigated this
discrepancy and discovered a large number of short reads (length: less
than 15 nucleotides) mapping to miRNAs and assigned by \texttt{seqpac}, but which
were considered invalid overlaps in light of our assignment rules.



\hypertarget{shortrna-accurately-identify-mirnas}{%
\subsubsection{\texorpdfstring{\texttt{shortRNA} accurately identify miRNAs}{shortRNA accurately identify miRNAs}}\label{shortrna-accurately-identify-mirnas}}

Following that, we examined the overlaps of identified miRNAs between
four tools. Despite the fact that all of the tools use miRBase, we
discovered that only 44 miRNAs were commonly detected by all of the
tools, Figure \ref{fig:3f14}. One possible explanation is that the miRBase for
mouse genome contains approximately 2000 mature miRNAs and approximately
1300 miRNA precursors, and \texttt{seqpac} and \texttt{Sports1} only use miRNA precursors
in the data analysis pipeline, whereas \texttt{Oasis2} and \texttt{shortRNA} use both
mature and miRNA precursors. Figure \ref{fig:3f14} shows that there were 231 miRNAs
that were commonly detected by \texttt{Oasis2}, \texttt{Sports1}, and \texttt{seqpac}, but not by
\texttt{shortRNA}. Next, we looked into these miRNAs and discovered that three
were removed from the most recent version of miRBase; for the other 228,
we did not find alignment in the genome using \texttt{shortRNA}, despite
accepting mismatches and soft-clipping. Based on our findings, we
believe that these miRNAs are the result of misalignment or sequential
alignment by the other three tools. When we examined the sequences
mapping to these miRNAs according to \texttt{seqpac}, we discovered that all of
the reads mapping to these miRNAs were shorter than 15 bp, and as a
result, are very unlikely to represent functional miRNAs. In conclusion,
we discovered that \texttt{shortRNA} can accurately identify miRNAs.



\hypertarget{data-simulation-to-check-the-alignment-and-reads-assignment-by-shortrna}{%
\subsection{\texorpdfstring{Data simulation to check the alignment and reads assignment by \texttt{shortRNA}}{Data simulation to check the alignment and reads assignment by shortRNA}}\label{data-simulation-to-check-the-alignment-and-reads-assignment-by-shortrna}}

We validated the alignment and reads assignment by spiking the reads in
the sperm dataset (\protect\hyperlink{ac}{Appendix C}). In brief, we simulated reads for a miRNA
cluster containing mmu-miR-125a-5p, mmu-miR-99b-5p, mmu-let-7e-5p,
mmu-let-7e-3p, mmu-miR-99b-3p, and mmu-miR-125a-3p, as well as reads
from a miRNA precursor, mmu-miR-144. To test for differential analysis,
the sequences from these miRNAs were spiked while also creating
differences between groups. Three invalid sequences were also simulated
for the precursor to test the accuracy of read assignment and were
properly aligned to miRNA precursors, as shown in Figure \ref{fig:3f15}.



Further, the reads were assigned accurately to miRNAs, Table \ref{tab:sht2}.

\hypertarget{differential-expression-analysis}{%
\subsection{Differential expression analysis}\label{differential-expression-analysis}}

We performed differential analysis using the \texttt{treeClimbR} package. Through
differential analysis, candidate proposal, multiple testing correction,
and candidate evaluation, \texttt{treeclimbR} integrates the observations with a
tree that reflects the hierarchical relationship between entities and
finds an appropriate resolution on the tree to interpret the
association. During simulations, we spiked the mature miRNAs miR-99b-5p
and miR-99b-3p to be different. After differential analysis, because
both miR-99b-5p and miR-99b-3p were simulated to be different, we found
that miR-99b was called to be differentially expressed, Figure \ref{fig:3f16}.
Looking at the cluster, miRNAcluster\_17:17830188-17830879, the full
cluster would have been differentially expressed if one read mapping to
miR-125a-5p, in grey, would also have been significantly different.



\hypertarget{validation-using-human-data-and-quantitative-real-time-pcr}{%
\subsection{Validation using human data and quantitative real-time PCR}\label{validation-using-human-data-and-quantitative-real-time-pcr}}

To validate the \texttt{Oasis2}, \texttt{Sports1}, \texttt{seqpac}, and \texttt{shortRNA} quantifications,
we used an unpublished PBMC human dataset and corresponding quantitative
real-time PCR (qPCR) quantifications. The PBMC dataset was analyzed
using the default parameters from \texttt{Oasis2}, \texttt{Sports1}, \texttt{seqpac}, and \texttt{shortRNA}.
Then, we correlated the qPCR quantification of five miRNAs with the
quantification using all four tools, Figure \ref{fig:3f17}. Because \texttt{Sports1} and
\texttt{seqpac} only report miRNA quantification at the level of precursors, the
precursor of mature miRNAs quantification was used for the correlation
of these tools with the qPCR. When we looked at the overall correlation
of miRNAs with qPCR quantifications, we discovered that except \texttt{seqpac}, all tools have a good correlation, Figure \ref{fig:3f17}A. Overall, we found a low correlation of
quantifications between four tools and qPCR. We found that \texttt{Oasis2}
quantification was negatively correlated with qPCR Ct values for miRNAs
miR-30d-5p and miR-30e-5p, as shown in Figure \ref{fig:3f18}B. The quantification of
miRNAs using short RNAs was found to be positively correlated with the
quantification of miRNAs using qPCR. In addition, we performed a visual
inspection of the reads mapping to these miRNAs by creating genomic
tracks, as shown in Figures \ref{fig:3f18a} and \ref{fig:3f18b}. As a result, while \texttt{shortRNA}
quantifications were visually observed to correctly count reads, they
were not substantially correlated with qPCR, but were generally better
correlated with qPCR quantifications than the other methods.


\begin{subfigures}







\end{subfigures}
\hypertarget{qualitative-comparison-with-other-tools}{%
\subsection{Qualitative comparison with other tools}\label{qualitative-comparison-with-other-tools}}

There are several tools published for analyzing the sRNA-seq data. In
Table: \href{figures/shortRNA/in/t3.xlsx}{Qualitative comparison}, we made a qualitative comparison of 20 other tools. Table is too big to be displayed here.

\hypertarget{discussion-outlook}{%
\section{Discussion \& Outlook}\label{discussion-outlook}}

Due to the non-unique genomic origin, short length, and numerous
post-transcriptional modifications of short RNA species, data processing
of sRNA-Seq has proven to be difficult. Moreover, from a bioinformatics
point of view, there are several challenges in analyzing data generated
from small RNA sequencing techniques, including alignment, reads
assignment, normalization, and differential analysis. Importantly,
quantifying small RNAs correctly is a computational challenge. To a
large extent, different tools provide varying quantification. This is
startling, and it highlights the need for further research and better
tools in the field.

We present \texttt{shortRNA}, an \texttt{R} package, a novel tool, for analyzing sRNA-seq
data, which can reduce the likelihood of false discoveries from the
sRNA-seq data. \texttt{shortRNA} is available from github.com/mansuylab/\texttt{shortRNA}
and is planned to be submitted to the Bioconductor. The \texttt{shortRNA} R
package is developed around the existing frameworks of Bioconductor:
phylo, \texttt{FactorList}, \texttt{DataFrame}, and \texttt{TreeSummarizedExperiment}. Hence, it is
expected to be able to interact seamlessly with many other tools, such
as \texttt{ggtree}, castor, and \texttt{TreeHeatmap}. \texttt{shortRNA} performs preprocessing,
alignment, and downstream analysis of the sRNA-seq data. Moreover,
\texttt{shortRNA} is able to deal with the sequencing data with UMIs using \texttt{UMIc}.
Further, if an adapter sequence is unknown, it is possible to detect the
adapter sequence using \texttt{shortRNA}.

We tested the package on various human and mouse datasets and
demonstrated it using the mouse sperm dataset, simulated mouse sperm
dataset, and human dataset from PBMCs. We compared the quantifications
of miRNAs from \texttt{shortRNA} to those of three other tools: \texttt{Sports1}, \texttt{Oasis2},
and \texttt{seqpac} and found them to be well correlated with \texttt{Sports1} and \texttt{Oasis2}.
We demonstrated \texttt{shortRNA}'s ability to assign reads to features in a
meaningful way by customisable assignment rules. \texttt{shortRNA} uses a
considerable proportion of reads, multi-mapping reads, that are either
discarded by other methods or aligned randomly. We demonstrated that
\texttt{shortRNA} correctly annotates reads to features using simulated reads
from the sperm dataset. In addition, we demonstrated that \texttt{shortRNA}
correctly identifies differentially expressed reads. Finally, using the
human PBMC data and corresponding qPCR quantifications from five mature
miRNAs, we demonstrated that \texttt{shortRNA} quantification is better
correlated with qPCR data than the other three tools.

In general, however, we observed a very low correlation of all methods
against qPCR in terms of the relative miRNA expression across samples.
There are a number of potential explanations for this discrepancy.
First, the lack of normalization of miRNA quantifications with U6
expression. In several investigations, including in this dataset, RNU6B
(U6) is used to normalize circulating miRNA data (\protect\hyperlink{ref-donati2019}{Donati, Ciuffi, \& Brandi, 2019}). Although
qPCR is used to measure mature miRNA levels, it's great sensitivity
necessitates proper normalization to account for non-biological
variance. This is naturally different from the RNA sequencing analysis,
where the normalization is based on the whole distribution of RNAs. This
discrepancy could therefore be one reason for the low correlations. On
the contrary, it has been advised that U6 should not be utilized for
data normalization of circulating miRNAs (\protect\hyperlink{ref-benz2013}{Benz et al., 2013}; \protect\hyperlink{ref-donati2019}{Donati et al., 2019}). Next,
the lack of UMIs in the data. UMIs make it possible to eliminate PCR
amplification biases(\protect\hyperlink{ref-fu2018}{Fu et al., 2018}). The lack of UMIs in human PBMC data might
have resulted in technical duplication, and hence affecting the
correlation with the qPCR data.

Most tools for the sRNA-seq data analysis, including \texttt{Sports1} and \texttt{Oasis2},
use feature-based counting, whereas \texttt{seqpac} uses sequence-based counting
for sRNA-seq data analysis. The benefit of sequence-based counting is
that it preserves sequence integrity and allows users to BLAST
(\protect\hyperlink{ref-altschul1990}{Altschul, Gish, Miller, Myers, \& Lipman, 1990}) candidate sequences to confirm classification and obtain
additional information. In addition, sequence-based counting allows for
the examination of sequence-specific variations or their boundaries.
Further, when using sequence-based counting, users can choose to remove
sequences that have low evidence and do not replicate across samples in
their data. \texttt{shortRNA}, on the other hand, has the advantage of both
sequence-based and feature-based counting due to the hierarchical
organization of features and mapping sequences. Because of the short
lengths of the reads and their proclivity to originate from higher-copy
number regions of the genome, multi-mapping reads are much more common
in sRNA-seq data. Commonly used sRNA-seq alignment methods for
multi-mapping reads are either very low precision (choosing an alignment
at random and make it difficult to determine their true origins) or
sensibility (ignoring multi-mapping reads, which result in losing a high
proportion of reads). \texttt{shortRNA} allows these reads to be aligned multiple
times and then assign them, using a set of customisable rules, either to
a specific feature or to a position in the feature tree which makes it
usable while preserving knowledge of eventual ambiguities.

\texttt{shortRNA} makes use of miRBase, GtRNADb, MttRNADb, SILVA and Ensembl
databases, but it is possible to manually add databases and extend the
list. We used adaptable assignment rules to identify the correct
assignment of a read. For example, if the data comes only from miRNAs,
the user can set higher priority for mapping the reads with miRNAs by
changing the number from 1 to 2 or higher. Further, we used a
phylogenetic tree structure to organize the features and to assign the
reads to features. If a sequence is overlapped with multiple features,
we assign the read to the parent of both features. This helps in solving
the problem of multi-mapping.

\hypertarget{limitations}{%
\section{Limitations}\label{limitations}}

\texttt{shortRNA} has not been thoroughly tested across multiple platforms.
Although we anticipate that the \texttt{shortRNA} will be platform independent,
it has not yet been tested on other operating systems. We intend to do
so at a later date. In addition, we also intend to extensively benchmark
\texttt{shortRNA} with existing tools and strategies for reads assignment.

Simulations for other types of RNA are necessary to benchmark the read
assignment. We have only simulated reads from miRNAs and miRNA
precursors and demonstrated the proper assignment of reads using
\texttt{shortRNA}. Simulations of reads mapping to tRNAs, reads mapping to piRNAs
and reads mapping to rRNAs will be performed to better assess the reads
assignment.

The ambiguous assignment of multi-mapping reads to the parent of all
mappable features is not always biologically meaningful. If a read is
assigned to multiple tRNAs, Figure \ref{fig:3f11}, it is still meaningful to assign
it to the parent of all mapping tRNAs. In contrast, if a read maps to
both a tRNA and a protein-coding gene, assigning it to the parent of
both protein coding and tRNA provides less biological insight. Hence, we
plan to compare \texttt{shortRNA} to other read assignment strategies for
multi-mapping reads in depth. Other strategies for assigning
multi-mapping reads exist (\protect\hyperlink{ref-handzlik2020}{Handzlik, Tastsoglou, Vlachos, \& Hatzigeorgiou, 2020}; \protect\hyperlink{ref-johnson2016}{Johnson, Yeoh, Coruh, \& Axtell, 2016}), and a benchmark
with simulations would be required to assess the \texttt{shortRNA} assignment of
multi-mapping reads. Instead of assigning reads to the parent of all
mappable features, this would aid in the proper assignment and provide
more biological insight for the reads mapping to multiple biotypes.

The current implementation of \texttt{UMIc} for dealing with sRNA-seq data based
on UMIs is extremely slow. There are several for-loops for running
analysis, as well as a non-parallel implementation of the code by the
authors of \texttt{UMIc}. We intend to optimize the code for speed by
parallelizing it and replacing for-loops with parallel apply functions.

\texttt{shortRNA} has limited user-friendly plotting capabilities. Users can
currently use data visualization packages developed for
\texttt{TreeSummarizedExperiment}, such as \texttt{TreeHeatmap}, directly. However, we
intend to create wrappers for othe\texttt{R} packages in order to facilitate the
visualization of data and results. Also, we will be creating a Shiny app
for exploratory analysis of the data and results.

IsomiRs analysis could be implemented in \texttt{shortRNA}. IsomiRs are miRNAs
that vary slightly in sequence, which result from variations in the
cleavage site during miRNA biogenesis (5'-trimming and 3'-trimming
variants), nucleotide additions to the 3'-end of the mature miRNA
(3'-addition variants) and nucleotide modifications (substitution
variants). While in principle our hierarchical and sequence-based
approach is ideal for their investigation, tailored functions would make
this more straightforward for the users. We plan to extend ou\texttt{R} package
to include isoMiRs analysis by adapting functionalities of the \texttt{isomiR} \texttt{R}
package
(\protect\hyperlink{ref-lorenapantanoautcregeorgiaescaramisautchristosargyropoulosaut2017}{Lorena Pantano {[}Aut, Cre{]}, Georgia Escaramis {[}Aut{]}, ChristosArgyropoulos {[}Aut{]}, 2017}).

\texttt{shortRNA} is developed to work with single-end sRNA-seq data. We plan to
extend our package to work with paired-end sRNA-seq data.

\hypertarget{conclusion}{%
\section{Conclusion}\label{conclusion}}

In conclusion, we developed an \texttt{R} package, platform-independent, for
analysis of the sRNA-seq data.This package provides a great overview of
the quality, performs QC, aligns the data to custom made genome,
performs reads annotation and assignment with adaptable assignment
rules, organizes reads and features as a phylogenetic tree, performs
differential analysis, and provides interactive plots for exploratory
data analysis. \texttt{shortRNA} currently works with single-end sequencing data.

\hypertarget{data-and-code-availibility}{%
\section{Data and code availibility}\label{data-and-code-availibility}}

\texttt{shortRNA} is available from: github.com/mansuylab/\texttt{shortRNA}

No sequencing data were generated during this study. Code for simulating
reads is presented in \protect\hyperlink{ad}{Appendix D}.

Data analysis steps for sRNA-seq data with \texttt{shortRNA} is available in
\protect\hyperlink{ac}{Appendix C}.

\backmatter

\hypertarget{references}{%
\chapter*{References}\label{references}}
\addcontentsline{toc}{chapter}{References}

\markboth{References}{References}

\noindent

\setlength{\parindent}{-0.20in}

\hypertarget{refs}{}
\begin{CSLReferences}{1}{0}
\leavevmode\vadjust pre{\hypertarget{ref-altschul1990}{}}%
Altschul, S. F., Gish, W., Miller, W., Myers, E. W., \& Lipman, D. J. (1990). Basic local alignment search tool. \emph{Journal of Molecular Biology}, \emph{215}(3), 403--410. http://doi.org/\href{https://doi.org/10.1016/s0022-2836(05)80360-2}{10.1016/s0022-2836(05)80360-2}

\leavevmode\vadjust pre{\hypertarget{ref-barraud2019}{}}%
Barraud, P., \& Tisné, C. (2019). To be or not to be modified: Miscellaneous aspects influencing nucleotide modifications in tRNAs. \emph{IUBMB Life}, \emph{71}(8), 1126--1140. http://doi.org/\href{https://doi.org/10.1002/iub.2041}{10.1002/iub.2041}

\leavevmode\vadjust pre{\hypertarget{ref-baumgart2017}{}}%
Baumgart, M., Barth, E., Savino, A., Groth, M., Koch, P., Petzold, A., \ldots{} Cellerino, A. (2017). A miRNA catalogue and ncRNA annotation of the short-living fish Nothobranchius furzeri. \emph{BMC Genomics}, \emph{18}(1). http://doi.org/\href{https://doi.org/10.1186/s12864-017-3951-8}{10.1186/s12864-017-3951-8}

\leavevmode\vadjust pre{\hypertarget{ref-benz2013}{}}%
Benz, F., Roderburg, C., Vargas Cardenas, D., Vucur, M., Gautheron, J., Koch, A., \ldots{} Luedde, T. (2013). U6 is unsuitable for normalization of serum miRNA levels in patients with sepsis or liver fibrosis. \emph{Experimental \& Molecular Medicine}, \emph{45}, e42. http://doi.org/\href{https://doi.org/10.1038/emm.2013.81}{10.1038/emm.2013.81}

\leavevmode\vadjust pre{\hypertarget{ref-brennecke2007}{}}%
Brennecke, J., Aravin, A. A., Stark, A., Dus, M., Kellis, M., Sachidanandam, R., \& Hannon, G. J. (2007). Discrete small RNA-generating loci as master regulators of transposon activity in drosophila. \emph{Cell}, \emph{128}(6), 1089--1103. http://doi.org/\href{https://doi.org/10.1016/j.cell.2007.01.043}{10.1016/j.cell.2007.01.043}

\leavevmode\vadjust pre{\hypertarget{ref-buffalo2020}{}}%
Buffalo, V. (2020). Qrqc: Quick read quality control. Retrieved from \url{http://github.com/vsbuffalo/qrqc}

\leavevmode\vadjust pre{\hypertarget{ref-chan2016}{}}%
Chan, P. P., \& Lowe, T. M. (2016). GtRNAdb 2.0: An expanded database of transfer RNA genes identified in complete and draft genomes. \emph{Nucleic Acids Research}, \emph{44}(D1), D184--9. http://doi.org/\href{https://doi.org/10.1093/nar/gkv1309}{10.1093/nar/gkv1309}

\leavevmode\vadjust pre{\hypertarget{ref-chan2021}{}}%
Chan, P., Lin, B., Mak, A., \& Lowe, T. (2021). tRNAscan-SE 2.0: improved detection and functional classification of transfer RNA genes. \emph{Nucleic Acids Research}, \emph{49}(16), 9077--9096. http://doi.org/\href{https://doi.org/10.1093/nar/gkab688}{10.1093/nar/gkab688}

\leavevmode\vadjust pre{\hypertarget{ref-chen2012}{}}%
Chen, C.-J., Servant, N., Toedling, J., Sarazin, A., Marchais, A., Duvernois-Berthet, E., \ldots{} Barillot, E. (2012). ncPRO-seq: A tool for annotation and profiling of ncRNAs in sRNA-seq data. \emph{Bioinformatics}, \emph{28}(23), 3147--3149. http://doi.org/\href{https://doi.org/10.1093/bioinformatics/bts587}{10.1093/bioinformatics/bts587}

\leavevmode\vadjust pre{\hypertarget{ref-cooley1982}{}}%
Cooley, L., Appel, B., \& Soll, D. (1982). Post-transcriptional nucleotide addition is responsible for the formation of the 5' terminus of histidine tRNA. \emph{Proceedings of the National Academy of Sciences}, \emph{79}(21), 6475--6479. http://doi.org/\href{https://doi.org/10.1073/pnas.79.21.6475}{10.1073/pnas.79.21.6475}

\leavevmode\vadjust pre{\hypertarget{ref-cozen2015}{}}%
Cozen, A. E., Quartley, E., Holmes, A. D., Hrabeta-Robinson, E., Phizicky, E. M., \& Lowe, T. M. (2015). ARM-seq: AlkB-facilitated RNA methylation sequencing reveals a complex landscape of modified tRNA fragments. \emph{Nature Methods}, \emph{12}(9), 879--884. http://doi.org/\href{https://doi.org/10.1038/nmeth.3508}{10.1038/nmeth.3508}

\leavevmode\vadjust pre{\hypertarget{ref-desvignes2019}{}}%
Desvignes, T., Batzel, P., Sydes, J., Eames, B. F., \& Postlethwait, J. H. (2019). miRNA analysis with prost! Reveals evolutionary conservation of organ-enriched expression and post-transcriptional modifications in three-spined stickleback and zebrafish. \emph{Scientific Reports}, \emph{9}(1), 3913. http://doi.org/\href{https://doi.org/10.1038/s41598-019-40361-8}{10.1038/s41598-019-40361-8}

\leavevmode\vadjust pre{\hypertarget{ref-donati2019}{}}%
Donati, S., Ciuffi, S., \& Brandi, M. L. (2019). Human circulating miRNAs real-time qRT-PCR-based analysis: An overview of endogenous reference genes used for data normalization. \emph{International Journal of Molecular Sciences}, \emph{20}(18). http://doi.org/\href{https://doi.org/10.3390/ijms20184353}{10.3390/ijms20184353}

\leavevmode\vadjust pre{\hypertarget{ref-freist1999}{}}%
Freist, W., Verhey, J. F., Rühlmann, A., Gauss, D. H., \& Arnez, J. G. (1999). Histidyl-tRNA synthetase. \emph{Biological Chemistry}, \emph{380}(6), 623--646. http://doi.org/\href{https://doi.org/10.1515/BC.1999.079}{10.1515/BC.1999.079}

\leavevmode\vadjust pre{\hypertarget{ref-fromant2000}{}}%
Fromant, M., Plateau, P., \& Blanquet, S. (2000). Function of the Extra 5{`}-Phosphate Carried by Histidine tRNA. \emph{Biochemistry}, \emph{39}(14), 4062--4067. http://doi.org/\href{https://doi.org/10.1021/bi9923297}{10.1021/bi9923297}

\leavevmode\vadjust pre{\hypertarget{ref-fu2018}{}}%
Fu, Y., Wu, P.-H., Beane, T., Zamore, P. D., \& Weng, Z. (2018). Elimination of PCR duplicates in RNA-seq and small RNA-seq using unique molecular identifiers. \emph{BMC Genomics}, \emph{19}(1), 531. http://doi.org/\href{https://doi.org/10.1186/s12864-018-4933-1}{10.1186/s12864-018-4933-1}

\leavevmode\vadjust pre{\hypertarget{ref-gapp2021}{}}%
Gapp, K., Parada, G. E., Gross, F., Corcoba, A., Kaur, J., Grau, E., \ldots{} Miska, E. A. (2021). Single paternal dexamethasone challenge programs offspring metabolism and reveals multiple candidates in RNA-mediated inheritance. \emph{iScience}, \emph{24}(8), 102870. http://doi.org/\href{https://doi.org/10.1016/j.isci.2021.102870}{10.1016/j.isci.2021.102870}

\leavevmode\vadjust pre{\hypertarget{ref-gm2018}{}}%
GM, F. (2018). tRNAdbImport. http://doi.org/\href{https://doi.org/10.18129/B9.BIOC.TRNADBIMPORT}{10.18129/B9.BIOC.TRNADBIMPORT}

\leavevmode\vadjust pre{\hypertarget{ref-gong2005}{}}%
Gong, H., Liu, C.-M., Liu, D.-P., \& Liang, C.-C. (2005). The role of small RNAs in human diseases: Potential troublemaker and therapeutic tools. \emph{Medicinal Research Reviews}, \emph{25}(3), 361--381. http://doi.org/\href{https://doi.org/10.1002/med.20023}{10.1002/med.20023}

\leavevmode\vadjust pre{\hypertarget{ref-green1997}{}}%
Green, R., \& Noller, H. F. (1997). Ribosomes and translation. \emph{Annual Review of Biochemistry}, \emph{66}, 679--716. http://doi.org/\href{https://doi.org/10.1146/annurev.biochem.66.1.679}{10.1146/annurev.biochem.66.1.679}

\leavevmode\vadjust pre{\hypertarget{ref-griffiths-jones2008}{}}%
Griffiths-Jones, S., Saini, H. K., van Dongen, S., \& Enright, A. J. (2008). miRBase: Tools for microRNA genomics. \emph{Nucleic Acids Research}, \emph{36}(Database issue), D154--8. http://doi.org/\href{https://doi.org/10.1093/nar/gkm952}{10.1093/nar/gkm952}

\leavevmode\vadjust pre{\hypertarget{ref-handzlik2020}{}}%
Handzlik, J. E., Tastsoglou, S., Vlachos, I. S., \& Hatzigeorgiou, A. G. (2020). Manatee: Detection and quantification of small non-coding RNAs from next-generation sequencing data. \emph{Scientific Reports}, \emph{10}(1), 705. http://doi.org/\href{https://doi.org/10.1038/s41598-020-57495-9}{10.1038/s41598-020-57495-9}

\leavevmode\vadjust pre{\hypertarget{ref-hou2010}{}}%
Hou, Y.-M. (2010). CCA addition to tRNA: Implications for tRNA quality control. \emph{IUBMB Life}, \emph{62}(4), 251--260. http://doi.org/\href{https://doi.org/10.1002/iub.301}{10.1002/iub.301}

\leavevmode\vadjust pre{\hypertarget{ref-howe2021}{}}%
Howe, K. L., Achuthan, P., Allen, J., Allen, J., Alvarez-Jarreta, J., Amode, M. R., \ldots{} Flicek, P. (2021). Ensembl 2021. \emph{Nucleic Acids Research}, \emph{49}(D1), D884--D891. http://doi.org/\href{https://doi.org/10.1093/nar/gkaa942}{10.1093/nar/gkaa942}

\leavevmode\vadjust pre{\hypertarget{ref-huang2020}{}}%
Huang, R., Soneson, C., Ernst, F. G. M., Rue-Albrecht, K. C., Yu, G., Hicks, S. C., \& Robinson, M. D. (2020). TreeSummarizedExperiment: A S4 class for data with hierarchical structure. \emph{F1000Research}, \emph{9}, 1246. http://doi.org/\href{https://doi.org/10.12688/f1000research.26669.1}{10.12688/f1000research.26669.1}

\leavevmode\vadjust pre{\hypertarget{ref-huang2021}{}}%
Huang, R., Soneson, C., Germain, P.-L., Schmidt, T. S. B., Mering, C. V., \& Robinson, M. D. (2021). treeclimbR pinpoints the data-dependent resolution of hierarchical hypotheses. \emph{Genome Biology}, \emph{22}(1). http://doi.org/\href{https://doi.org/10.1186/s13059-021-02368-1}{10.1186/s13059-021-02368-1}

\leavevmode\vadjust pre{\hypertarget{ref-ibba2000}{}}%
Ibba, M., \& Soll, D. (2000). Aminoacyl-tRNA synthesis. \emph{Annual Review of Biochemistry}, \emph{69}, 617--650. http://doi.org/\href{https://doi.org/10.1146/annurev.biochem.69.1.617}{10.1146/annurev.biochem.69.1.617}

\leavevmode\vadjust pre{\hypertarget{ref-johnson2016}{}}%
Johnson, N. R., Yeoh, J. M., Coruh, C., \& Axtell, M. J. (2016). Improved placement of multi-mapping small RNAs. \emph{G3 (Bethesda, Md.)}, \emph{6}(7), 2103--2111. http://doi.org/\href{https://doi.org/10.1534/g3.116.030452}{10.1534/g3.116.030452}

\leavevmode\vadjust pre{\hypertarget{ref-juxfchling2009}{}}%
Jühling, F., Mörl, M., Hartmann, R. K., Sprinzl, M., Stadler, P. F., \& Pütz, J. (2009). tRNAdb 2009: Compilation of tRNA sequences and tRNA genes. \emph{Nucleic Acids Research}, \emph{37}(Database issue), D159--62. http://doi.org/\href{https://doi.org/10.1093/nar/gkn772}{10.1093/nar/gkn772}

\leavevmode\vadjust pre{\hypertarget{ref-kaisers2020}{}}%
Kaisers, W. (2020). seqTools: Analysis of nucleotide, sequence and quality content on fastqfiles. Retrieved from \url{https://bioconductor.org/packages/release/bioc/html/seqTools.html}

\leavevmode\vadjust pre{\hypertarget{ref-kuksa2018}{}}%
Kuksa, P. P., Amlie-Wolf, A., Katanic, Ž., Valladares, O., Wang, L.-S., \& Leung, Y. Y. (2018). SPAR: Small RNA-seq portal for analysis of sequencing experiments. \emph{Nucleic Acids Research}, \emph{46}(W1), W36--W42. http://doi.org/\href{https://doi.org/10.1093/nar/gky330}{10.1093/nar/gky330}

\leavevmode\vadjust pre{\hypertarget{ref-lai2013}{}}%
Lai, X., \& Vera, J. (2013). MicroRNA clusters (pp. 1310--1314). Springer New York. http://doi.org/\href{https://doi.org/10.1007/978-1-4419-9863-7_1121}{10.1007/978-1-4419-9863-7\_1121}

\leavevmode\vadjust pre{\hypertarget{ref-li2018}{}}%
Li, J., Han, X., Wan, Y., Zhang, S., Zhao, Y., Fan, R., \ldots{} Zhou, Y. (2018). TAM 2.0: Tool for MicroRNA set analysis. \emph{Nucleic Acids Research}, \emph{46}(W1), W180--W185. http://doi.org/\href{https://doi.org/10.1093/nar/gky509}{10.1093/nar/gky509}

\leavevmode\vadjust pre{\hypertarget{ref-li2013}{}}%
Li, X. Z., Roy, C. K., Dong, X., Bolcun-Filas, E., Wang, J., Han, B. W., \ldots{} Zamore, P. D. (2013). An ancient transcription factor initiates the burst of piRNA production during early meiosis in mouse testes. \emph{Molecular Cell}, \emph{50}(1), 67--81. http://doi.org/\href{https://doi.org/10.1016/j.molcel.2013.02.016}{10.1016/j.molcel.2013.02.016}

\leavevmode\vadjust pre{\hypertarget{ref-liao2019}{}}%
Liao, Y., Smyth, G. K., \& Shi, W. (2019). The r package rsubread is easier, faster, cheaper and better for alignment and quantification of RNA sequencing reads. \emph{Nucleic Acids Research}, \emph{47}(8), e47e47. http://doi.org/\href{https://doi.org/10.1093/nar/gkz114}{10.1093/nar/gkz114}

\leavevmode\vadjust pre{\hypertarget{ref-loher2017}{}}%
Loher, P., Telonis, A. G., \& Rigoutsos, I. (2017). MINTmap: Fast and exhaustive profiling of nuclear and mitochondrial tRNA fragments from short RNA-seq data. \emph{Scientific Reports}, \emph{7}, 41184. http://doi.org/\href{https://doi.org/10.1038/srep41184}{10.1038/srep41184}

\leavevmode\vadjust pre{\hypertarget{ref-lorenapantanoautcregeorgiaescaramisautchristosargyropoulosaut2017}{}}%
Lorena Pantano {[}Aut, Cre{]}, Georgia Escaramis {[}Aut{]}, ChristosArgyropoulos {[}Aut{]}. (2017). isomiRs. http://doi.org/\href{https://doi.org/10.18129/B9.BIOC.ISOMIRS}{10.18129/B9.BIOC.ISOMIRS}

\leavevmode\vadjust pre{\hypertarget{ref-magee2017}{}}%
Magee, R., Loher, P., Londin, E., \& Rigoutsos, I. (2017). Threshold-seq: A tool for determining the threshold in short RNA-seq datasets. \emph{Bioinformatics}, \emph{33}(13), 2034--2036. http://doi.org/\href{https://doi.org/10.1093/bioinformatics/btx073}{10.1093/bioinformatics/btx073}

\leavevmode\vadjust pre{\hypertarget{ref-morgan2009}{}}%
Morgan, M., Anders, S., Lawrence, M., Aboyoun, P., Pagès, H., \& Gentleman, R. (2009). ShortRead: A bioconductor package for input, quality assessment and exploration of high-throughput sequence data. \emph{Bioinformatics}, \emph{25}(19), 2607--2608. http://doi.org/\href{https://doi.org/10.1093/bioinformatics/btp450}{10.1093/bioinformatics/btp450}

\leavevmode\vadjust pre{\hypertarget{ref-morgan2020}{}}%
Morgan, M., Obenchain, V., Hester, J., \& Pagès, H. (2020). SummarizedExperiment: SummarizedExperiment container. Retrieved from \url{https://bioconductor.org/packages/SummarizedExperiment}

\leavevmode\vadjust pre{\hypertarget{ref-paguxe8s2020}{}}%
Pagès, H., Aboyoun, P., Gentleman, R., \& DebRoy, S. (2020). Biostrings: Efficient manipulation of biological strings. Retrieved from \url{https://bioconductor.org/packages/Biostrings}

\leavevmode\vadjust pre{\hypertarget{ref-pratt2009}{}}%
Pratt, A. J., \& MacRae, I. J. (2009). The RNA-induced silencing complex: A versatile gene-silencing machine. \emph{The Journal of Biological Chemistry}, \emph{284}(27), 17897--17901. http://doi.org/\href{https://doi.org/10.1074/jbc.R900012200}{10.1074/jbc.R900012200}

\leavevmode\vadjust pre{\hypertarget{ref-quast2013}{}}%
Quast, C., Pruesse, E., Yilmaz, P., Gerken, J., Schweer, T., Yarza, P., \ldots{} Glöckner, F. O. (2013). The SILVA ribosomal RNA gene database project: Improved data processing and web-based tools. \emph{Nucleic Acids Research}, \emph{41}(Database issue), D590--6. http://doi.org/\href{https://doi.org/10.1093/nar/gks1219}{10.1093/nar/gks1219}

\leavevmode\vadjust pre{\hypertarget{ref-rahman2018}{}}%
Rahman, R.-U., Gautam, A., Bethune, J., Sattar, A., Fiosins, M., Magruder, D. S., \ldots{} Bonn, S. (2018). Oasis 2: Improved online analysis of small RNA-seq data. \emph{BMC Bioinformatics}, \emph{19}(1), 54. http://doi.org/\href{https://doi.org/10.1186/s12859-018-2047-z}{10.1186/s12859-018-2047-z}

\leavevmode\vadjust pre{\hypertarget{ref-rueda2015}{}}%
Rueda, A., Barturen, G., Lebrón, R., Gómez-Martín, C., Alganza, Á., Oliver, J. L., \& Hackenberg, M. (2015). sRNAtoolbox: An integrated collection of small RNA research tools. \emph{Nucleic Acids Research}, \emph{43}(W1), W467--73. http://doi.org/\href{https://doi.org/10.1093/nar/gkv555}{10.1093/nar/gkv555}

\leavevmode\vadjust pre{\hypertarget{ref-shi2018}{}}%
Shi, J., Ko, E.-A., Sanders, K. M., Chen, Q., \& Zhou, T. (2018). SPORTS1.0: A tool for annotating and profiling non-coding RNAs optimized for rRNA- and tRNA-derived small RNAs. \emph{Genomics, Proteomics \& Bioinformatics / Beijing Genomics Institute}, \emph{16}(2), 144--151. http://doi.org/\href{https://doi.org/10.1016/j.gpb.2018.04.004}{10.1016/j.gpb.2018.04.004}

\leavevmode\vadjust pre{\hypertarget{ref-skog2021}{}}%
Skog, S., Örkenby, L., Kugelberg, U., Tariq, K., Farrants, A.-K. Ö., Öst, A., \& Nätt, D. (2021). Seqpac: A new framework for small RNA analysis in r using sequence-based counts. \emph{BioRxiv}. http://doi.org/\href{https://doi.org/10.1101/2021.03.19.436151}{10.1101/2021.03.19.436151}

\leavevmode\vadjust pre{\hypertarget{ref-sprinzl1979}{}}%
Sprinzl, M., \& Cramer, F. (1979). The -c-c-a end of tRNA and its role in protein biosynthesis (pp. 1--69). Elsevier. http://doi.org/\href{https://doi.org/10.1016/s0079-6603(08)60798-9}{10.1016/s0079-6603(08)60798-9}

\leavevmode\vadjust pre{\hypertarget{ref-stocks2012}{}}%
Stocks, M. B., Moxon, S., Mapleson, D., Woolfenden, H. C., Mohorianu, I., Folkes, L., \ldots{} Moulton, V. (2012). The UEA sRNA workbench: A suite of tools for analysing and visualizing next generation sequencing microRNA and small RNA datasets. \emph{Bioinformatics}, \emph{28}(15), 2059--2061. http://doi.org/\href{https://doi.org/10.1093/bioinformatics/bts311}{10.1093/bioinformatics/bts311}

\leavevmode\vadjust pre{\hypertarget{ref-tsagiopoulou2021}{}}%
Tsagiopoulou, M., Maniou, M. C., Pechlivanis, N., Togkousidis, A., Kotrová, M., Hutzenlaub, T., \ldots{} Psomopoulos, F. (2021). UMIc: A preprocessing method for UMI deduplication and reads correction. \emph{Frontiers in Genetics}, \emph{12}. http://doi.org/\href{https://doi.org/10.3389/fgene.2021.660366}{10.3389/fgene.2021.660366}

\leavevmode\vadjust pre{\hypertarget{ref-vitsios2015}{}}%
Vitsios, D. M., \& Enright, A. J. (2015). Chimira: Analysis of small RNA sequencing data and microRNA modifications. \emph{Bioinformatics}, \emph{31}(20), 3365--3367. http://doi.org/\href{https://doi.org/10.1093/bioinformatics/btv380}{10.1093/bioinformatics/btv380}

\leavevmode\vadjust pre{\hypertarget{ref-wang2020}{}}%
Wang, W., \& Carroll, T. (2020). Rfastp: An ultra-fast and all-in-one fastq preprocessor (QualityControl, adapter, low quality and polyX trimming) and UMISequence parsing). Retrieved from \url{http://www.bioconductor.org/packages/release/bioc/html/Rfastp.html}

\leavevmode\vadjust pre{\hypertarget{ref-wu2013}{}}%
Wu, J., Liu, Q., Wang, X., Zheng, J., Wang, T., You, M., \ldots{} Shi, Q. (2013). mirTools 2.0 for non-coding RNA discovery, profiling, and functional annotation based on high-throughput sequencing. \emph{RNA Biology}, \emph{10}(7), 1087--1092. http://doi.org/\href{https://doi.org/10.4161/rna.25193}{10.4161/rna.25193}

\leavevmode\vadjust pre{\hypertarget{ref-wu2017}{}}%
Wu, X., Kim, T.-K., Baxter, D., Scherler, K., Gordon, A., Fong, O., \ldots{} Wang, K. (2017). sRNAnalyzer-a flexible and customizable small RNA sequencing data analysis pipeline. \emph{Nucleic Acids Research}, \emph{45}(21), 12140--12151. http://doi.org/\href{https://doi.org/10.1093/nar/gkx999}{10.1093/nar/gkx999}

\leavevmode\vadjust pre{\hypertarget{ref-yuan2009}{}}%
Yuan, X., Liu, C., Yang, P., He, S., Liao, Q., Kang, S., \& Zhao, Y. (2009). Clustered microRNAs' coordination in regulating protein-protein interaction network. \emph{BMC Systems Biology}, \emph{3}, 65. http://doi.org/\href{https://doi.org/10.1186/1752-0509-3-65}{10.1186/1752-0509-3-65}

\leavevmode\vadjust pre{\hypertarget{ref-zhu2016}{}}%
Zhu, J., Chen, G., Zhu, S., Li, S., Wen, Z., Bin Li, \ldots{} Shi, L. (2016). Identification of tissue-specific protein-coding and noncoding transcripts across 14 human tissues using RNA-seq. \emph{Scientific Reports}, \emph{6}, 28400. http://doi.org/\href{https://doi.org/10.1038/srep28400}{10.1038/srep28400}

\leavevmode\vadjust pre{\hypertarget{ref-zuo2016}{}}%
Zuo, L., Wang, Z., Tan, Y., Chen, X., \& Luo, X. (2016). piRNAs and Their Functions in the Brain. \emph{International Journal of Human Genetics}, \emph{16}(1-2), 53--60. http://doi.org/\href{https://doi.org/10.1080/09723757.2016.11886278}{10.1080/09723757.2016.11886278}

\end{CSLReferences}

% Index?

\end{document}
\newpage

Dedicated to
